--- VASTRDI_revision_71_June_20_9:22_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_72_June_20_9:23_AM	2019-06-27 13:08:01.000000000 -0400
@@ -9,7 +9,9 @@
 -Edvard Vann detained, article mentions he has same name as  POK member.  
 -Works in Security 
  Kronos Government 
--Cesare Nespola. Minister of Health 
+-Cesare Nespola. Minister of Health. Died 2001 
+-Succeeded by Vincent Kapelou, 
+-Unsympathetic to POK  
  Either 10 or 14 Kidnapped, Ransom of 20M, (protectors of kronos claimed responsibility 
  President and CEO Sten Sanjorge Jr, - assumed kidnapped, but returned safely 
 CFO Ingrid Barranco,  
