--- VASTRDI_revision_56_June_19_2:07_PM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_57_June_19_2:08_PM	2019-06-27 13:08:01.000000000 -0400
@@ -37,6 +37,11 @@
 1/20   UNK  Miriam Avila confirms that leadership missing- denied abduction - Central 
 Bulletin 
   1/20    1230  Departure of first plane with 7-8 People 
+1/201245 UPDATE - Abila Police have cordoned off the GAStech headquarters with 
+additional officers arriving to secure the perimeter and to investigate what can only be 
+described as a crime scene though the crime has not be revealed. The police response 
+seems out of proportion for a false fire alarm. Police officials have declined to comment 
+at this time. 
 1/201345 UPDATE - Confirmed report of missing GAStech employees from GAStech 
 administrative assistant in Tethys headquarters 
 1/20    1430 Departure of second plane to Rome with 7-8 people 
