--- VASTRDI_revision_43_June_19_1:31_PM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_44_June_19_1:32_PM	2019-06-27 13:08:01.000000000 -0400
@@ -41,6 +41,6 @@
   E-mail “Subject” Files  
 -Sent on W/F in the afternoon over previous 2 weeks 
  Sanjorge/MiesCEO - Facilities +2 
-Arpa/Onda 
+Arpa/OndaFacilities +2 - Site Manager 
 Osvaldo/Pantanal 
 Dedos/Orilla 
\ No newline at end of file
