--- VASTRDI_revision_77_June_20_9:28_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_78_June_20_9:28_AM	2019-06-27 13:08:01.000000000 -0400
@@ -6,8 +6,8 @@
 Elian Karel- Current Leader? 
 -Jeroen Karel, Elian’s Father - Organized recruitment and personnel 
 Silvia Marek 
-Mandor Vann 
--TiesUncle to Isia Vann & Juliana Vann 
+Mandor Vann (enlisted lingusit)  
+-Uncle to Isia Vann & Juliana Vann 
 - 
 Carmine Osvaldo. - Military 
  Isia Vann -  
