--- VASTRDI_revision_10_June_19_11:16_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_11_June_19_11:18_AM	2019-06-27 13:08:01.000000000 -0400
@@ -8,4 +8,6 @@
 COO Orhan Strum, and environment civil servant  
 environment civil servant Willem Vasco-Pais  
  Kidnapping Expert - American John Rathburn  
-Lives in Tethys 
\ No newline at end of file
+Lives in Tethys 
+ Security Guard Edvard Vann 
+Interrogated and released after 6 hrs, indicated POK is i 
\ No newline at end of file
