--- VASTRDI_revision_8_June_19_11:15_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_9_June_19_11:16_AM	2019-06-27 13:08:01.000000000 -0400
@@ -6,4 +6,5 @@
 CFO Ingrid Barranco,  
 CIO Ada campo-Corrente,  
 COO Orhan Strum, and environment civil servant  
-environment civil servant Willem Vasco-Pais  
\ No newline at end of file
+environment civil servant Willem Vasco-Pais  
+ Kidnapping Expert - American  
\ No newline at end of file
