--- VASTRDI_revision_86_June_20_10:01_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_87_June_20_10:06_AM	2019-06-27 13:08:01.000000000 -0400
@@ -20,7 +20,8 @@
 -Edvard Vann detained, article mentions he has same name as  POK member.  
 -Works in Security 
  ARISE Inspiration for defenders of Kronos 
-Inga.Ferro@gastech.com.kronos, Loreto.Bodrogi@gastech.com.kronos, 
+From: Ruscella.Mies.Haber@gastech.com.kronos 
+To: Inga.Ferro@gastech.com.kronos, Loreto.Bodrogi@gastech.com.kronos, 
 Isia.Vann@gastech.com.kronos, Hennie.Osvaldo@gastech.com.kronos, 
 Minke.Mies@gastech.com.kronos 
    Kronos Government 
