--- VASTRDI_revision_21_June_19_11:34_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_22_June_19_11:34_AM	2019-06-27 13:08:01.000000000 -0400
@@ -11,8 +11,9 @@
 Lives in Tethys 
  Security Guard Edvard Vann 
 Interrogated and released after 6 hrs, indicated POK  is suspected. 
-Name is similar to POK Member 
-   Timeline: 
+Name is similar to POK Member - GASTECH EMPLOYEE INTERROGATED, RELEASED 
+Central Bulletin | 01/20/2014 
+    Timeline: 
  1/20    UNKKidnapping takes place  
 1/20    1000Fire trucks dispached to GASTech HQ on Kronos in response to fire alarm - 
 KIDNAPPING AT GASTECH KRONOS Who What News | 01/20/2014 
