--- VASTRDI_revision_80_June_20_9:30_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_81_June_20_9:31_AM	2019-06-27 13:08:01.000000000 -0400
@@ -11,6 +11,8 @@
 -Political Strategist 
 -Juliana Died at age 10 from drinking contaminated water 
 Carmine Osvaldo. - Military 
+Lucio Jakab 
+Lorenzo Di Stefano 
  Isia Vann -  
 -Edvard Vann detained, article mentions he has same name as  POK member.  
 -Works in Security 
