--- VASTRDI_revision_40_June_19_1:25_PM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_41_June_19_1:27_PM	2019-06-27 13:08:01.000000000 -0400
@@ -38,7 +38,8 @@
 Ngohebo 
 1/21     1000News conference to take place  -ON THE SCENE BLOG By Haneson 
 Ngohebo 
-  E-mail “Subject” Files 
+  E-mail “Subject” Files  
+-Sent on W/F in the afternoon over previous 2 weeks 
  Sanjorge/Mies 
 Arpa/Onda 
 Osvaldo/Pantanal 
