--- VASTRDI_revision_65_June_20_8:29_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_66_June_20_8:31_AM	2019-06-27 13:08:01.000000000 -0400
@@ -1,7 +1,8 @@
 ID 3 individuals associated with the Protectors of Kronos 
  Describe their involvement, and provide evidence.  
   Eventually construct POK org chart, figure out why people disappeared 
- Elian Karel- Leadership 
+ Elian Karel- Current Leadership? 
+Henk Bodrogi- initial Grassroots leader? 
  Isia Vann -  
 -Edvard Vann detained, article mentions he has same name as  POK member.  
 -Works in Security 
