--- VASTRDI_revision_36_June_19_1:03_PM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_37_June_19_1:06_PM	2019-06-27 13:08:01.000000000 -0400
@@ -11,7 +11,8 @@
 Lives in Tethys 
  Security Guard Edvard Vann 
 Interrogated and released after 6 hrs, indicated POK  is suspected. 
-Name is similar to POK Member - GASTECH EMPLOYEE INTERROGATED, RELEASED 
+Name is similar to POK Member - GASTECH EMPLOYEE INTERROGATED, 
+RELEASED 
 Central Bulletin | 01/20/2014 
 Armed Forces of Kronos 
  Ravi Danisliau- fueler at airport 
@@ -20,9 +21,11 @@
 -2nd departed 1430 w/7-8 business passengers  
 -“Slackend” more than first group 
 -Destination Rome 
-    Timeline: 
-  1/20    1000Fire trucks dispatched to GASTech HQ on Kronos in response to fire alarm - 
-KIDNAPPING AT GASTECH KRONOS Who What News | 01/20/2014 
+ Henk Mies 
+- 
+ Timeline: 
+  1/20    1000Fire trucks dispatched to GASTech HQ on Kronos in response to fire alarm 
+- KIDNAPPING AT GASTECH KRONOS Who What News | 01/20/2014 
 1/20    1100 Edvard Vann taken into custody? 
 1/201155Fire alarmed cleared. False alarm 
 1/20    1157  noted as not having arrived at Government Reception - VOICES - a blog 
@@ -31,6 +34,8 @@
   1/20    1230  Departure of first plane with 7-8 People 
 1/20    1430 Departure of second plane to Rome with 7-8 people 
 1/20    1700  Edvard Vann Released - News Online Today | 01/20/2014 
-   1/21  0934 estimate revised from 14 to 10. - ON THE SCENE BLOG By Haneson Ngohebo 
-1/21     1000News conference to take place  -ON THE SCENE BLOG By Haneson Ngohebo 
+   1/21  0934 estimate revised from 14 to 10. - ON THE SCENE BLOG By Haneson 
+Ngohebo 
+1/21     1000News conference to take place  -ON THE SCENE BLOG By Haneson 
+Ngohebo 
  
\ No newline at end of file
