--- VASTRDI_revision_85_June_20_10:00_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_86_June_20_10:01_AM	2019-06-27 13:08:01.000000000 -0400
@@ -19,7 +19,11 @@
  Isia Vann -  
 -Edvard Vann detained, article mentions he has same name as  POK member.  
 -Works in Security 
- Kronos Government 
+ ARISE Inspiration for defenders of Kronos 
+Inga.Ferro@gastech.com.kronos, Loreto.Bodrogi@gastech.com.kronos, 
+Isia.Vann@gastech.com.kronos, Hennie.Osvaldo@gastech.com.kronos, 
+Minke.Mies@gastech.com.kronos 
+   Kronos Government 
 -Cesare Nespola. Minister of Health. Died 2001 
 -Succeeded by Vincent Kapelou, 
 -Unsympathetic to POK  
