--- VAST3FF_revision_52_July_18_11:12_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_53_July_18_11:13_AM	2019-07-19 14:01:52.000000000 -0400
@@ -1,6 +1,7 @@
 POK Organization: 
  Current figurehead Silvia Marek. Mandor Vann is the top political strategist.  
  Leaders: Carmine Osvaldo, Henk Bodrogi, Silvia Marek, Mandor Vann. 
+, (poss Carmine Osvaldo) 
  -It is possible that Carmine Osvaldo is still in the leadership as he was 
 one of the founders, does not mention if he died/retired. 
  Extended network: Isia Vann, Lorenzo Di Stefano, Lucio Jakab, 
@@ -20,8 +21,8 @@
 leadership/key people in the POK. Several of the Kronos born members of 
 the security team served in the Kronos military and received less than 
 honorable discharged, multiple having started to work for GAStech just a 
-few months prior to the kidnapping. I believe that Isia Vann (older brother to 
-Juliana Vann) has been planning this event for several years, using his connections within the GAStech organization to build a security team to 
+few months prior to the kidnapping. I believe that Isia Vann (older brother to Juliana Vann) has been planning this event for several years, using his 
+connections within the GAStech organization to build a security team to 
 enable this “kidnapping.”  
  Alternate Theory: 
 All of the following were known to have used their credit cards at Guy’s 
@@ -56,7 +57,8 @@
 each other and cross reference their answers. Furthermore, we need the 
 actual content of the emails to and from all the security personnel prior to 
 the events in question.  
-    March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison  fourteen employees, probably including five executive leaders, 
+    March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison 
+ fourteen employees, probably including five executive leaders, 
     Lais, Cornelia 
 Position: Assistant to Security Group Manager (Administration) 
 Date of Hire: 12/22/2011 
@@ -75,8 +77,8 @@
   Morluniau, Cecilia 
 -Joined GasTech right after leader imprisoned/protests began, former Kronos military 
  Position: Truck Driver (Facilities) 
-Date of Hire: 04/30/2009 
-Email: Cecilia.Morluniau@gastech.com.kronos  Sex: Female 
+Date of Hire: 04/30/2009 Email: Cecilia.Morluniau@gastech.com.kronos 
+ Sex: Female 
 DOB: 09/02/1988 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
