--- VAST3FF_revision_34_July_18_10:01_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_35_July_18_10:02_AM	2019-07-19 14:01:52.000000000 -0400
@@ -5,7 +5,8 @@
  Leaders: Carmine Osvaldo, Henk Bodrogi, Silvia Marek, Mandor Vann. 
  Extended network: Isia Vann, Lorenzo Di Stefano, Lucio Jakab, 
  GAStech-POK Affiliates: Isia Vann,  
- -Need to look further into: Bodrogi, Loreto, Osvaldo, Hennie, Vann, Edvard,  
+ -Need to look further into: Bodrogi, Loreto, Osvaldo, Hennie, Vann, Edvard, Mies, 
+Henk, Lais, Cornelia,  
   What happened on 20-21 2014? 
 Timeline:  
  Reported that 14 were kidnapped, revised to 10, including CEO, CFO, CIO, 
@@ -17,67 +18,4 @@
  President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. 
- Rabadams - brexit, anti corporations    March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison 
- fourteen employees, probably including five executive leaders, 
-    Lais, Cornelia 
-Position: Assistant to Security Group Manager (Administration) 
-Date of Hire: 12/22/2011 
-Email: Cornelia.Lais@gastech.com.kronos 
- Sex: Female 
-DOB: 07/07/1991 
- Citizenship: Kronos 
-Passport Issued: 01/01/2000 
-Passport Expires: 01/01/2000 
- Military Branch: Armed Forces of Kronos 
-Discharge Date: 10/01/2011 
-Discharge Type: General Discharge 
- Administrative and business support to the Security Group Manager. 
-Coordinate schedules among multiple personnel that are constantly on the 
-move.  Arrange travel and training for all staff in the group. 
-  Morluniau, Cecilia -Joined GasTech right after leader imprisoned/protests began, former Kronos military 
- Position: Truck Driver (Facilities) 
-Date of Hire: 04/30/2009 
-Email: Cecilia.Morluniau@gastech.com.kronos 
- Sex: Female 
-DOB: 09/02/1988 
- Citizenship: Kronos 
-Passport Issued: 01/01/2000 
-Passport Expires: 01/01/2000 
- Military Branch: Armed Forces of Kronos 
-Discharge Date: 10/01/2008 
-Discharge Type: General Discharge 
-     Mies, Henk 
-Position: Truck Driver (Facilities) 
-Date of Hire: 07/12/2013 
-Email: Henk.Mies@gastech.com.kronos 
- Sex: Male 
-DOB: 09/23/1984 
- Citizenship: Kronos 
-Passport Issued: 01/01/2000 
-Passport Expires: 01/01/2000 
- Military Branch: Armed Forces of Kronos 
-Discharge Date: 10/01/2004 
-Discharge Type: Honorable Discharge    Vann, Isia 
--Same name as girl killed, general discharge from kronos military, 
- Position: Perimeter Control (Security) 
-Date of Hire: 12/14/2007 
-Email: Isia.Vann@gastech.com.kronos 
- Sex: Male 
-DOB: 12/13/1986 
- Citizenship: Kronos 
-Passport Issued: 01/01/2000 
-Passport Expires: 01/01/2000 
- Military Branch: Armed Forces of Kronos 
-Discharge Date: 10/01/2007 
-Discharge Type: General Discharge 
- GASTech - Kronos Security Officer 2007- Present  Secures premises and personnel by 
-patrolling property; monitoring surveillance equipment; inspecting buildings, equipment, and 
-access points; permitting entry.  Obtains help by sounding alarms.  Prevents losses and 
-damage by reporting irregularities; informing violators of policy and procedures; restraining 
-trespassers.  Controls traffic by directing drivers.  Completes reports by recording 
-observations, information, occ 
-  Vann, Edvard 
--Same name as girl who died, got job soon before everything happened 
- Position: Perimeter Control (Security) 
-Date of Hire: 08/15/2013 
-Email: Edvard.Vann@gastech.com.kronos 
\ No newline at end of file
+ 
\ No newline at end of file
