--- VAST3FF_revision_51_July_18_11:10_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_52_July_18_11:12_AM	2019-07-19 14:01:52.000000000 -0400
@@ -48,15 +48,15 @@
 Lagos 
 Sten 
 Sanjorge Jr. 
- Note that most of these are mid-low level security with the  
+ Note that most of these are mid-low level security with the exception of the 
+CEO, just a day prior to the kidnapping.  
  Next Steps: First, we need to find out who actually was kidnapped/is still 
 missing. Next, we need to interview the entire security team to find out how 
 this happened. We need to keep the names listed above separate from 
 each other and cross reference their answers. Furthermore, we need the 
 actual content of the emails to and from all the security personnel prior to 
 the events in question.  
-    March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison 
- fourteen employees, probably including five executive leaders, 
+    March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison  fourteen employees, probably including five executive leaders, 
     Lais, Cornelia 
 Position: Assistant to Security Group Manager (Administration) 
 Date of Hire: 12/22/2011 
@@ -76,8 +76,7 @@
 -Joined GasTech right after leader imprisoned/protests began, former Kronos military 
  Position: Truck Driver (Facilities) 
 Date of Hire: 04/30/2009 
-Email: Cecilia.Morluniau@gastech.com.kronos 
- Sex: Female 
+Email: Cecilia.Morluniau@gastech.com.kronos  Sex: Female 
 DOB: 09/02/1988 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
