--- VAST3FF_revision_5_July_18_8:43_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_6_July_18_8:46_AM	2019-07-19 14:01:52.000000000 -0400
@@ -2,7 +2,10 @@
  What happened on 20-21 2014? 
 Timeline:  
   POK $20 million ransom 
-  President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
+  President and CEO Sten Sanjorge Jr, CFO Ingrid Barranco, CIO Ada 
+campo-Corrente, COO Orhan Strum, and environment civil servant Willem 
+Vasco-Pais van GAStech. 
+ President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. 
  Rabadams - brexit, anti corporations 
