--- VAST3FF_revision_56_July_18_11:16_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_57_July_18_11:17_AM	2019-07-19 14:01:52.000000000 -0400
@@ -7,6 +7,7 @@
  GAStech-POK Affiliates: Isia Vann,  
  -Need to look further into: Loreto Bodrogi, Hennie Osvaldo, Edvard Vann, Henk 
 Mies, Cornelia Lais, Cecilia Morluniau, Claudio Hawelon, Inga Ferro,  
+-^ these are all suspicious individuals with possible ties to the POK as well as  
   20-21 2014 Timeline:  
 -Fire alarm was set at 10:00am though no fire was found to have 
 occurred 
@@ -20,8 +21,8 @@
 leadership/key people in the POK. Several of the Kronos born members of 
 the security team served in the Kronos military and received less than 
 honorable discharged, multiple having started to work for GAStech just a 
-few months prior to the kidnapping. I believe that Isia Vann (older brother to 
-Juliana Vann) has been planning this event for several years, using his connections within the GAStech organization to build a security team to 
+few months prior to the kidnapping. I believe that Isia Vann (older brother to Juliana Vann) has been planning this event for several years, using his 
+connections within the GAStech organization to build a security team to 
 enable this “kidnapping.”  
  Alternate Theory: 
 All of the following were known to have used their credit cards at Guy’s 
