--- VAST3FF_revision_28_July_18_9:56_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_29_July_18_9:57_AM	2019-07-19 14:01:52.000000000 -0400
@@ -1,8 +1,9 @@
 POK Organization: 
  The likely leaders of the POK include the founders who established Karel as 
 the first leader, Carmine Osvaldo and Henk Bodrogi as well as current 
-figurehead Silvia Marek.  
- Leaders: Carmine Osvaldo, Henk Bodrogi, Silvia Marek. 
+figurehead Silvia Marek. Mandor Vann is the top political strategist.  
+ Leaders: Carmine Osvaldo, Henk Bodrogi, Silvia Marek., Mandor Vann. 
+ Extended network: Isia Vann 
   What happened on 20-21 2014? 
 Timeline:  
  Reported that 14 were kidnapped, revised to 10, including CEO, CFO, CIO, 
@@ -15,8 +16,8 @@
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. 
  Rabadams - brexit, anti corporations 
-   March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison 
- fourteen employees, probably including five executive leaders,     Lais, Cornelia 
+   March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison  fourteen employees, probably including five executive leaders, 
+    Lais, Cornelia 
 Position: Assistant to Security Group Manager (Administration) 
 Date of Hire: 12/22/2011 
 Email: Cornelia.Lais@gastech.com.kronos 
@@ -35,8 +36,8 @@
 -Joined GasTech right after leader imprisoned/protests began, former Kronos military 
  Position: Truck Driver (Facilities) 
 Date of Hire: 04/30/2009 
-Email: Cecilia.Morluniau@gastech.com.kronos 
- Sex: Female DOB: 09/02/1988 
+Email: Cecilia.Morluniau@gastech.com.kronos  Sex: Female 
+DOB: 09/02/1988 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
 Passport Expires: 01/01/2000 
@@ -56,7 +57,8 @@
 Discharge Date: 10/01/2004 
 Discharge Type: Honorable Discharge 
    Vann, Isia 
--Same name as girl killed, general discharge from kronos military,  Position: Perimeter Control (Security) 
+-Same name as girl killed, general discharge from kronos military, 
+ Position: Perimeter Control (Security) 
 Date of Hire: 12/14/2007 
 Email: Isia.Vann@gastech.com.kronos 
  Sex: Male 
@@ -79,10 +81,10 @@
 Date of Hire: 08/15/2013 
 Email: Edvard.Vann@gastech.com.kronos 
  Sex: Male 
-DOB: 03/18/1991 
- Citizenship: Kronos 
+DOB: 03/18/1991  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
-Passport Expires: 01/01/2000  Military Branch: Armed Forces of Kronos 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2011 
 Discharge Type: Honorable Discharge 
   Mies, Minke 
@@ -102,10 +104,10 @@
 Date of Hire: 06/07/2011 
 Email: Hennie.Osvaldo@gastech.com.kronos 
  Sex: Male 
-DOB: 05/31/1988 
- Citizenship: Kronos 
+DOB: 05/31/1988  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
-Passport Expires: 01/01/2000  Military Branch: Armed Forces of Kronos 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2010 
 Discharge Type: General Discharge 
  Bodrogi, Loreto 
@@ -121,7 +123,8 @@
 Discharge Date: 10/01/2008 
 Discharge Type: Honorable Discharge 
       GASTECH EMPLOYEE INTERROGATED, 
-RELEASED News Online Today | 01/20/2014 
+RELEASED 
+News Online Today | 01/20/2014 
 ABILA, Kronos - For roughly 6 hours GAStech - Kronos employee Edvard 
 Vann was interrogated by Kronos Police and Government officials before 
 being released at 17:00 this evening. With some 14 GAStech employees 
