--- VAST3FF_revision_4_July_18_8:43_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_5_July_18_8:43_AM	2019-07-19 14:01:52.000000000 -0400
@@ -1,7 +1,8 @@
 POK Organization: 
  What happened on 20-21 2014? 
 Timeline:  
-    President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
+  POK $20 million ransom 
+  President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. 
  Rabadams - brexit, anti corporations 
