--- VAST3FF_revision_40_July_18_10:07_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_41_July_18_10:22_AM	2019-07-19 14:01:52.000000000 -0400
@@ -1,13 +1,13 @@
 POK Organization: 
  The likely leaders of the POK include the founders who established Karel as 
-the first leader, Carmine Osvaldo and Henk Bodrogi as well as current 
-figurehead Silvia Marek. Mandor Vann is the top political strategist.  
+the first leader, Carmine Osvaldo and Henk Bodrogi as well as  
+cCurrent figurehead Silvia Marek. Mandor Vann is the top political 
+strategist.  
  Leaders: Carmine Osvaldo, Henk Bodrogi, Silvia Marek, Mandor Vann. 
  Extended network: Isia Vann, Lorenzo Di Stefano, Lucio Jakab, 
  GAStech-POK Affiliates: Isia Vann,  
- -Need to look further into: Loreto Bodrogi, Loreto, Hennie Osvaldo, Hennie, Vann, 
-Edvard Vann, Henk Mies, Henk, Lais, Cornelia Lais, Cecilia Morluniau, Cecilia 
-- 
+ -Need to look further into: Loreto Bodrogi, Hennie Osvaldo, Edvard Vann, Henk 
+Mies, Cornelia Lais, Cecilia Morluniau,  
  What happened on 20-21 2014? 
 Timeline:  
 -Was reported that 14 GAStech employees were missing, later revised 
