--- VAST3FF_revision_9_July_18_8:54_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_10_July_18_8:57_AM	2019-07-19 14:01:52.000000000 -0400
@@ -11,5 +11,5 @@
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. 
  Rabadams - brexit, anti corporations 
-   March 2009 Karel arrested, June 2009 leader Elian Karel dies 
+   March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison 
  fourteen employees, probably including five executive leaders, 
\ No newline at end of file
