--- VAST3FF_revision_16_July_18_9:21_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_17_July_18_9:22_AM	2019-07-19 14:01:52.000000000 -0400
@@ -64,4 +64,16 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2007 
 Discharge Type: General Discharge 
+  Vann, Edvard 
+Position: Perimeter Control (Security) 
+Date of Hire: 08/15/2013 
+Email: Edvard.Vann@gastech.com.kronos 
+ Sex: Male 
+DOB: 03/18/1991 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2011 
+Discharge Type: Honorable Discharge 
  
\ No newline at end of file
