--- VAST3FF_revision_20_July_18_9:28_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_21_July_18_9:31_AM	2019-07-19 14:01:52.000000000 -0400
@@ -95,4 +95,16 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2011 
 Discharge Type: General Discharge 
+  Osvaldo, Hennie 
+Position: Perimeter Control (Security) 
+Date of Hire: 06/07/2011 
+Email: Hennie.Osvaldo@gastech.com.kronos 
+ Sex: Male 
+DOB: 05/31/1988 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2010 
+Discharge Type: General Discharge 
  
\ No newline at end of file
