--- VAST3FF_revision_14_July_18_9:12_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_15_July_18_9:16_AM	2019-07-19 14:01:52.000000000 -0400
@@ -39,4 +39,16 @@
 Passport Expires: 01/01/2000 
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2008 Discharge Type: General Discharge 
+     Mies, Henk 
+Position: Truck Driver (Facilities) 
+Date of Hire: 07/12/2013 
+Email: Henk.Mies@gastech.com.kronos 
+ Sex: Male 
+DOB: 09/23/1984 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2004 
+Discharge Type: Honorable Discharge 
  
\ No newline at end of file
