--- VAST3FF_revision_59_July_18_11:52_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_60_July_18_11:52_AM	2019-07-19 14:01:52.000000000 -0400
@@ -22,7 +22,8 @@
 leadership/key people in the POK. Several of the Kronos born members of 
 the security team served in the Kronos military and received less than 
 honorable discharged, multiple having started to work for GAStech just a 
-few months prior to the kidnapping. I believe that Isia Vann (older brother to Juliana Vann) has been planning this event for several years, using his 
+few months prior to the kidnapping. I believe that Isia Vann (older brother to 
+Juliana Vann) has been planning this event for several years, using his 
 connections within the GAStech organization to build a security team to 
 enable this “kidnapping.”  
  Alternate Theory: 
@@ -58,4 +59,8 @@
 each other and cross reference their answers. Furthermore, we need the 
 actual content of the emails to and from all the security personnel prior to 
 the events in question.  
-      
\ No newline at end of file
+ 1
+1
+2
+3
+4
\ No newline at end of file
