--- VAST3FF_revision_26_July_18_9:54_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_27_July_18_9:55_AM	2019-07-19 14:01:52.000000000 -0400
@@ -1,6 +1,7 @@
 POK Organization: 
  The likely leaders of the POK include the founders who established Karel as 
-the first leader, Carmine Osvaldo and Henk Bodrogi. 
+the first leader, Carmine Osvaldo and Henk Bodrogi. as well as current 
+figurehead Silvia Marek.  
   What happened on 20-21 2014? 
 Timeline:  
  Reported that 14 were kidnapped, revised to 10, including CEO, CFO, CIO, 
@@ -37,8 +38,8 @@
 Email: Cecilia.Morluniau@gastech.com.kronos 
  Sex: Female 
 DOB: 09/02/1988 
- Citizenship: Kronos 
-Passport Issued: 01/01/2000 Passport Expires: 01/01/2000 
+ Citizenship: Kronos Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2008 
 Discharge Type: General Discharge 
@@ -57,8 +58,8 @@
    Vann, Isia 
 -Same name as girl killed, general discharge from kronos military, 
  Position: Perimeter Control (Security) 
-Date of Hire: 12/14/2007 
-Email: Isia.Vann@gastech.com.kronos  Sex: Male 
+Date of Hire: 12/14/2007 Email: Isia.Vann@gastech.com.kronos 
+ Sex: Male 
 DOB: 12/13/1986 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
@@ -83,8 +84,8 @@
 Passport Issued: 01/01/2000 
 Passport Expires: 01/01/2000 
  Military Branch: Armed Forces of Kronos 
-Discharge Date: 10/01/2011 
-Discharge Type: Honorable Discharge   Mies, Minke 
+Discharge Date: 10/01/2011 Discharge Type: Honorable Discharge 
+  Mies, Minke 
 Position: Perimeter Control (Security) 
 Date of Hire: 05/22/2013 
 Email: Minke.Mies@gastech.com.kronos 
@@ -106,8 +107,8 @@
 Passport Issued: 01/01/2000 
 Passport Expires: 01/01/2000 
  Military Branch: Armed Forces of Kronos 
-Discharge Date: 10/01/2010 
-Discharge Type: General Discharge  Bodrogi, Loreto 
+Discharge Date: 10/01/2010 Discharge Type: General Discharge 
+ Bodrogi, Loreto 
 Position: Site Control (Security) 
 Date of Hire: 08/17/2013 
 Email: Loreto.Bodrogi@gastech.com.kronos 
@@ -123,7 +124,7 @@
 RELEASED 
 News Online Today | 01/20/2014 
 ABILA, Kronos - For roughly 6 hours GAStech - Kronos employee Edvard 
-Vann was interrogated by Kronos Police and Government officials before 
-being released at 17:00 this evening. With some 14 GAStech employees still unaccounted for, one cannot blame the police or the government for 
+Vann was interrogated by Kronos Police and Government officials before being released at 17:00 this evening. With some 14 GAStech employees 
+still unaccounted for, one cannot blame the police or the government for 
 not leaving any stone unturned. 
  
\ No newline at end of file
