--- VAST3FF_revision_31_July_18_10:00_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_32_July_18_10:01_AM	2019-07-19 14:01:52.000000000 -0400
@@ -5,6 +5,7 @@
  Leaders: Carmine Osvaldo, Henk Bodrogi, Silvia Marek, Mandor Vann. 
  Extended network: Isia Vann, Lorenzo Di Stefano, Lucio Jakab, 
  GAStech-POK Affiliates: Isia Vann,  
+ -Need to look further into: Bodrogi, Loreto,  
   What happened on 20-21 2014? 
 Timeline:  
  Reported that 14 were kidnapped, revised to 10, including CEO, CFO, CIO, 
@@ -16,8 +17,7 @@
  President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. 
- Rabadams - brexit, anti corporations 
-   March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison 
+ Rabadams - brexit, anti corporations    March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison 
  fourteen employees, probably including five executive leaders, 
     Lais, Cornelia 
 Position: Assistant to Security Group Manager (Administration) 
@@ -34,9 +34,9 @@
  Administrative and business support to the Security Group Manager. 
 Coordinate schedules among multiple personnel that are constantly on the 
 move.  Arrange travel and training for all staff in the group. 
-  Morluniau, Cecilia 
--Joined GasTech right after leader imprisoned/protests began, former Kronos military 
- Position: Truck Driver (Facilities) Date of Hire: 04/30/2009 
+  Morluniau, Cecilia -Joined GasTech right after leader imprisoned/protests began, former Kronos military 
+ Position: Truck Driver (Facilities) 
+Date of Hire: 04/30/2009 
 Email: Cecilia.Morluniau@gastech.com.kronos 
  Sex: Female 
 DOB: 09/02/1988 
@@ -57,8 +57,7 @@
 Passport Expires: 01/01/2000 
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2004 
-Discharge Type: Honorable Discharge 
-   Vann, Isia 
+Discharge Type: Honorable Discharge    Vann, Isia 
 -Same name as girl killed, general discharge from kronos military, 
  Position: Perimeter Control (Security) 
 Date of Hire: 12/14/2007 
@@ -81,8 +80,8 @@
 -Same name as girl who died, got job soon before everything happened 
  Position: Perimeter Control (Security) 
 Date of Hire: 08/15/2013 
-Email: Edvard.Vann@gastech.com.kronos 
- Sex: Male DOB: 03/18/1991 
+Email: Edvard.Vann@gastech.com.kronos  Sex: Male 
+DOB: 03/18/1991 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
 Passport Expires: 01/01/2000 
@@ -104,8 +103,8 @@
   Osvaldo, Hennie 
 Position: Perimeter Control (Security) 
 Date of Hire: 06/07/2011 
-Email: Hennie.Osvaldo@gastech.com.kronos 
- Sex: Male DOB: 05/31/1988 
+Email: Hennie.Osvaldo@gastech.com.kronos  Sex: Male 
+DOB: 05/31/1988 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
 Passport Expires: 01/01/2000 
