--- VAST3FF_revision_17_July_18_9:22_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_18_July_18_9:23_AM	2019-07-19 14:01:52.000000000 -0400
@@ -65,7 +65,8 @@
 Discharge Date: 10/01/2007 
 Discharge Type: General Discharge 
   Vann, Edvard 
-Position: Perimeter Control (Security) 
+-Same name as girl who died, got job soon before everything happened 
+ Position: Perimeter Control (Security) 
 Date of Hire: 08/15/2013 
 Email: Edvard.Vann@gastech.com.kronos 
  Sex: Male 
