--- VAST3FF_revision_3_July_18_8:42_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_4_July_18_8:43_AM	2019-07-19 14:01:52.000000000 -0400
@@ -1,5 +1,8 @@
 POK Organization: 
  What happened on 20-21 2014? 
 Timeline:  
-      Rabadams - brexit, anti corporations 
+    President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
+Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
+pianotent, and environmental leader Willem Vasco-Feed. 
+ Rabadams - brexit, anti corporations 
      fourteen employees, probably including five executive leaders, 
\ No newline at end of file
