--- VAST3FF_revision_38_July_18_10:05_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_39_July_18_10:05_AM	2019-07-19 14:01:52.000000000 -0400
@@ -12,12 +12,27 @@
 -Was reported that 14 GAStech employees were missing, later revised 
 to 10 when the company leaders were found, safe.  
 -POK claimed responsibility and demanded $20 mil in ransom 
- Reported that 14 were kidnapped, revised to 10, including CEO, CFO, CIO, 
-COO, among others. 
- POK $20 million ransom 
-  President and CEO Sten Sanjorge Jr, CFO Ingrid Barranco, CIO Ada 
+    President and CEO Sten Sanjorge Jr, CFO Ingrid Barranco, CIO Ada 
 campo-Corrente, COO Orhan Strum, and environment civil servant Willem 
 Vasco-Pais van GAStech. 
+ President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
+Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
+pianotent, and environmental leader Willem Vasco-Feed. 
+ Rabadams - brexit, anti corporations 
+   March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison 
+ fourteen employees, probably including five executive leaders, 
+    Lais, Cornelia 
+Position: Assistant to Security Group Manager (Administration) 
+Date of Hire: 12/22/2011 
+Email: Cornelia.Lais@gastech.com.kronos 
+ Sex: Female 
+DOB: 07/07/1991 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2011 
+Discharge Type: General Discharge 
  Administrative and business support to the Security Group Manager. 
 Coordinate schedules among multiple personnel that are constantly on the 
 move.  Arrange travel and training for all staff in the group. 
@@ -38,7 +53,8 @@
 Position: Truck Driver (Facilities) 
 Date of Hire: 07/12/2013 
 Email: Henk.Mies@gastech.com.kronos 
- Sex: Male DOB: 09/23/1984 
+ Sex: Male 
+DOB: 09/23/1984 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
 Passport Expires: 01/01/2000 
@@ -88,8 +104,8 @@
 Passport Expires: 01/01/2000 
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2011 
-Discharge Type: General Discharge   Osvaldo, Hennie 
-Position: Perimeter Control (Security) 
+Discharge Type: General Discharge 
+  Osvaldo, Hennie Position: Perimeter Control (Security) 
 Date of Hire: 06/07/2011 
 Email: Hennie.Osvaldo@gastech.com.kronos 
  Sex: Male 
