--- VAST3FF_revision_2_July_18_8:39_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_3_July_18_8:42_AM	2019-07-19 14:01:52.000000000 -0400
@@ -1,2 +1,5 @@
-Rabadams - brexit, anti corporations 
+POK Organization: 
+ What happened on 20-21 2014? 
+Timeline:  
+      Rabadams - brexit, anti corporations 
      fourteen employees, probably including five executive leaders, 
\ No newline at end of file
