--- VAST3FF_revision_15_July_18_9:16_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_16_July_18_9:21_AM	2019-07-19 14:01:52.000000000 -0400
@@ -51,4 +51,17 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2004 
 Discharge Type: Honorable Discharge 
+   Vann, Isia 
+-Same name as girl killed, general discharge from kronos military, 
+ Position: Perimeter Control (Security) 
+Date of Hire: 12/14/2007 
+Email: Isia.Vann@gastech.com.kronos 
+ Sex: Male 
+DOB: 12/13/1986 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2007 
+Discharge Type: General Discharge 
  
\ No newline at end of file
