--- VAST3FF_revision_35_July_18_10:02_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_36_July_18_10:02_AM	2019-07-19 14:01:52.000000000 -0400
@@ -6,7 +6,7 @@
  Extended network: Isia Vann, Lorenzo Di Stefano, Lucio Jakab, 
  GAStech-POK Affiliates: Isia Vann,  
  -Need to look further into: Bodrogi, Loreto, Osvaldo, Hennie, Vann, Edvard, Mies, 
-Henk, Lais, Cornelia,  
+Henk, Lais, Cornelia, Morluniau, Cecilia 
   What happened on 20-21 2014? 
 Timeline:  
  Reported that 14 were kidnapped, revised to 10, including CEO, CFO, CIO, 
