--- VAST3FF_revision_36_July_18_10:02_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_37_July_18_10:04_AM	2019-07-19 14:01:52.000000000 -0400
@@ -9,13 +9,13 @@
 Henk, Lais, Cornelia, Morluniau, Cecilia 
   What happened on 20-21 2014? 
 Timeline:  
+-Was reported that 14 GAStech employees were missing, later revised 
+to 10 when the company leaders were found, safe.  
+-POK claimed responsibility and demanded  
  Reported that 14 were kidnapped, revised to 10, including CEO, CFO, CIO, 
 COO, among others. 
  POK $20 million ransom 
   President and CEO Sten Sanjorge Jr, CFO Ingrid Barranco, CIO Ada 
 campo-Corrente, COO Orhan Strum, and environment civil servant Willem 
 Vasco-Pais van GAStech. 
- President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
-Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
-pianotent, and environmental leader Willem Vasco-Feed. 
  
\ No newline at end of file
