--- VAST3FF_revision_21_July_18_9:31_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_22_July_18_9:34_AM	2019-07-19 14:01:52.000000000 -0400
@@ -1,5 +1,5 @@
 POK Organization: 
- What happened on 20-21 2014? 
+    What happened on 20-21 2014? 
 Timeline:  
  Reported that 14 were kidnapped, revised to 10, including CEO, CFO, CIO, 
 COO, among others. 
@@ -13,7 +13,8 @@
  Rabadams - brexit, anti corporations 
    March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison 
  fourteen employees, probably including five executive leaders, 
-    Lais, Cornelia Position: Assistant to Security Group Manager (Administration) 
+    Lais, Cornelia 
+Position: Assistant to Security Group Manager (Administration) 
 Date of Hire: 12/22/2011 
 Email: Cornelia.Lais@gastech.com.kronos 
  Sex: Female 
@@ -36,9 +37,9 @@
 DOB: 09/02/1988 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
-Passport Expires: 01/01/2000 
- Military Branch: Armed Forces of Kronos 
-Discharge Date: 10/01/2008 Discharge Type: General Discharge 
+Passport Expires: 01/01/2000  Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2008 
+Discharge Type: General Discharge 
      Mies, Henk 
 Position: Truck Driver (Facilities) 
 Date of Hire: 07/12/2013 
