--- VAST3FF_revision_10_July_18_8:57_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_11_July_18_9:01_AM	2019-07-19 14:01:52.000000000 -0400
@@ -12,4 +12,16 @@
 pianotent, and environmental leader Willem Vasco-Feed. 
  Rabadams - brexit, anti corporations 
    March 2009 Karel arrested, June 2009 leader Elian Karel dies in prison 
- fourteen employees, probably including five executive leaders, 
\ No newline at end of file
+ fourteen employees, probably including five executive leaders, 
+    Lais, Cornelia Position: Assistant to Security Group Manager (Administration) 
+Date of Hire: 12/22/2011 
+Email: Cornelia.Lais@gastech.com.kronos 
+ Sex: Female 
+DOB: 07/07/1991 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2011 
+Discharge Type: General Discharge 
+ 
\ No newline at end of file
