--- VAST3FF_revision_50_July_18_11:08_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_51_July_18_11:10_AM	2019-07-19 14:01:52.000000000 -0400
@@ -48,7 +48,7 @@
 Lagos 
 Sten 
 Sanjorge Jr. 
- Note that most of these  
+ Note that most of these are mid-low level security with the  
  Next Steps: First, we need to find out who actually was kidnapped/is still 
 missing. Next, we need to interview the entire security team to find out how 
 this happened. We need to keep the names listed above separate from 
