--- VAST3FF_revision_19_July_18_9:24_AM	2019-07-19 14:01:52.000000000 -0400
+++ VAST3FF_revision_20_July_18_9:28_AM	2019-07-19 14:01:52.000000000 -0400
@@ -83,4 +83,16 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2011 
 Discharge Type: Honorable Discharge 
+  Mies, Minke 
+Position: Perimeter Control (Security) 
+Date of Hire: 05/22/2013 
+Email: Minke.Mies@gastech.com.kronos 
+ Sex: Male 
+DOB: 11/19/1992 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2011 
+Discharge Type: General Discharge 
  
\ No newline at end of file
