--- VASTSYZ_revision_27_June_19_11:41_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_28_June_19_11:41_AM	2019-06-27 13:12:26.000000000 -0400
@@ -67,8 +67,9 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2011 
 Discharge Type: General Discharge 
-                        Ruscella.Mies.HaberRuscella MIes Haber 
-  Position: Assistant to Engineering Group Manager (Administration) 
+                        Ruscella MIes Haber 
+Possibly related to Minke Mies? 
+ Position: Assistant to Engineering Group Manager (Administration) 
 Date of Hire: 04/02/2003 
 Email: Ruscella.Mies.Haber@gastech.com.kronos 
  Sex: Female 
