--- VASTSYZ_revision_37_June_20_8:34_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_38_June_20_8:52_AM	2019-06-27 13:12:26.000000000 -0400
@@ -19,7 +19,8 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2008 
 Discharge Type: Honorable Discharge 
-                       Isia Vann 
+     ========================== 
+                 Isia Vann 
 Possibly related to Juliana Vann, girl who died in Elodis due to water poisoning 
 sends/receives emails related to POK on corp email 
  Position: Perimeter Control (Security) 
@@ -33,7 +34,7 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2007 
 Discharge Type: General Discharge 
-                         Inga Ferro 
+                          Inga Ferro 
 Possibly related to the Ferros of Elodis (Ferro Hardware) 
 sends/receives emails related to POK on corp email 
  Position: Site Control (Security) 
@@ -47,7 +48,8 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2012 
 Discharge Type: General Discharge 
-                       Hennie Osvaldo 
+                       ============================= 
+ Hennie Osvaldo 
 There’s a Osvaldo warehouse related to Ferro Hardware in Elodis 
 Carmin Osvaldo (sp?) listed as co-leader of POK in 1999 
  Position: Perimeter Control (Security) 
@@ -61,7 +63,7 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2010 
 Discharge Type: General Discharge 
-                       Minke Mies 
+                         Minke Mies 
 sends/receives emails related to POK on corp email 
  Position: Perimeter Control (Security) 
 Date of Hire: 05/22/2013 
@@ -74,7 +76,8 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2011 
 Discharge Type: General Discharge 
-                        Ruscella MIes Haber 
+                        ============================= 
+ Ruscella MIes Haber 
 Possibly related to Minke Mies? 
 Part of email forwarding ARISE materials 
  Position: Assistant to Engineering Group Manager (Administration) 
