--- VASTSYZ_revision_43_June_20_9:12_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_44_June_20_9:14_AM	2019-06-27 13:12:26.000000000 -0400
@@ -104,9 +104,9 @@
 Discharge Date: 10/01/2004 
 Discharge Type: Honorable Discharge 
   Notes: 
-Email frombetween Ruscella Mies Haber toand Ada Campo Corrento re: change to “the 
-deadline” 
- Email frombetween Sten Sanjorge and Henk Mies re: files  
+Email between Ruscella Mies Haber and Ada Campo Corrento re: change to “the deadline” 
+ Email between Sten Sanjorge and Henk Mies re: files 
+ Check on Vera Frentes (emailing  Minke Mies) 
                  New info from day 2 supplemental materials 
  Mandor Vann 
 Mandor Vann, uncle to Isia and Juliana Vann, has been with the POK since retiring from the 
