--- VASTSYZ_revision_38_June_20_8:52_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_39_June_20_8:52_AM	2019-06-27 13:12:26.000000000 -0400
@@ -19,8 +19,8 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2008 
 Discharge Type: Honorable Discharge 
-     ========================== 
-                 Isia Vann 
+  ========================== 
+ Isia Vann 
 Possibly related to Juliana Vann, girl who died in Elodis due to water poisoning 
 sends/receives emails related to POK on corp email 
  Position: Perimeter Control (Security) 
@@ -34,7 +34,7 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2007 
 Discharge Type: General Discharge 
-                          Inga Ferro 
+ Inga Ferro 
 Possibly related to the Ferros of Elodis (Ferro Hardware) 
 sends/receives emails related to POK on corp email 
  Position: Site Control (Security) 
@@ -48,7 +48,7 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2012 
 Discharge Type: General Discharge 
-                       ============================= 
+ ============================= 
  Hennie Osvaldo 
 There’s a Osvaldo warehouse related to Ferro Hardware in Elodis 
 Carmin Osvaldo (sp?) listed as co-leader of POK in 1999 
@@ -63,7 +63,7 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2010 
 Discharge Type: General Discharge 
-                         Minke Mies 
+  Minke Mies 
 sends/receives emails related to POK on corp email 
  Position: Perimeter Control (Security) 
 Date of Hire: 05/22/2013 
@@ -76,7 +76,7 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2011 
 Discharge Type: General Discharge 
-                        ============================= 
+ ============================= 
  Ruscella MIes Haber 
 Possibly related to Minke Mies? 
 Part of email forwarding ARISE materials 
@@ -91,7 +91,19 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/1984 
 Discharge Type: Honorable Discharge 
-                       New info from day 2 supplemental materials 
+   Henk Mies 
+Position: Truck Driver (Facilities) 
+Date of Hire: 07/12/2013 
+Email: Henk.Mies@gastech.com.kronos 
+ Sex: Male 
+DOB: 09/23/1984 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2004 
+Discharge Type: Honorable Discharge 
+                    New info from day 2 supplemental materials 
  Mandor Vann 
 Mandor Vann, uncle to Isia and Juliana Vann, has been with the POK since retiring from the 
 Kronos military in 2004. Vann, who was an enlisted linguist, spent much of his military 
