--- VASTSYZ_revision_11_June_19_11:27_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_12_June_19_11:27_AM	2019-06-27 13:12:26.000000000 -0400
@@ -13,8 +13,8 @@
 Discharge Date: 10/01/2008 
 Discharge Type: Honorable Discharge 
                        Isia Vann 
-Possibly related to Juliana Vann, girl who died in  
-                  Inga.Ferro@gastech.com.kronos 
+Possibly related to Juliana Vann, girl who died in Elodis due to water poisoning 
+                    Inga.Ferro@gastech.com.kronos 
 Loreto.Bodrogi@gastech.com.kronos 
 Isia.Vann@gastech.com.kronos 
 Hennie.Osvaldo@gastech.com.kronos 
