--- VASTSYZ_revision_9_June_19_11:26_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_10_June_19_11:26_AM	2019-06-27 13:12:26.000000000 -0400
@@ -1,6 +1,5 @@
 Loreto Bodrogi 
-Security - Employee since 2013 - Former military (honorable discharge) 
-  Possibly related to Henk Bodrogi, founder of POK 
+ Possibly related to Henk Bodrogi, founder of POK 
 sends/receives emails related to POK on corp email 
  Position: Site Control (Security) 
 Date of Hire: 08/17/2013 
@@ -13,7 +12,9 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2008 
 Discharge Type: Honorable Discharge 
-                                          Inga.Ferro@gastech.com.kronos 
+                       Isia Vann 
+Possibly related to  
+                  Inga.Ferro@gastech.com.kronos 
 Loreto.Bodrogi@gastech.com.kronos 
 Isia.Vann@gastech.com.kronos 
 Hennie.Osvaldo@gastech.com.kronos 
