--- VASTN5Z_revision_17_July_18_10:33_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_18_July_18_10:34_AM	2019-07-19 13:39:19.000000000 -0400
@@ -19,8 +19,7 @@
  POK 5 year report notes: 
  Isia Vann ‘s father was/is member of POK but also works for GasTECH as a security officer. 
 Controls physical security 
-  Edvard Vann (no known relation to Mandor) Questioned on 1/20 by police 
+Isia has a resume for Gastech but none for Edvard (what if they are the same person) 
+ Edvard Vann (no known relation to Mandor) Questioned on 1/20 by police 
 Security guard at GasTech 
-Edvard Vann, GAStech - the guard of safety of Kronos which was 
-questioned and released a 
- 
\ No newline at end of file
+  
\ No newline at end of file
