--- VASTN5Z_revision_35_July_18_11:36_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_36_July_18_11:36_AM	2019-07-19 13:39:19.000000000 -0400
@@ -25,7 +25,7 @@
 (This email is going to Hennie Osvaldo and Loreto Bodrogi, who both share names with Henk 
 Bodrogi and Carmine Osvaldo who were leaders in the earlier days of POK) 
  1018 - Reports of fire in the morning but was false alarm 
-https://vast.ncsu-las.net/?article=72 Limo with execs to a breakfast? 
+https://vast.ncsu-las.net/?article=72 
 Update, 3:40 PM: One GAStech employee, who declined to be identified, reports having seen 
 several people dressed in black in the building before the fire alarm. These people were not 
 GAStech employees are 'were acting odd and lurking about'. Another report says these were 
@@ -47,4 +47,8 @@
 Isia has a resume for Gastech but none for Edvard (what if they are the same person) 
 Edvard Vann (no known relation to Mandor) Questioned on 1/20 by police 
 Security guard at GasTech 
-  
\ No newline at end of file
+  1
+1
+2
+3
+4
\ No newline at end of file
