--- VASTN5Z_revision_5_July_18_10:00_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_6_July_18_10:01_AM	2019-07-19 13:39:19.000000000 -0400
@@ -3,6 +3,8 @@
 Connections to gastech 
 Org over time 
   POK 5 year report notes: 
+The grassroots organization coalesced under the leadership of Henk Bodrogi, a floodplain 
+farmer 
 Today the POK has grown under the charismatic leadership of Elian Karel to loosely organized 
 group with an estimated membership of 200-300 people. 
 Started in Elodis 
\ No newline at end of file
