--- VASTN5Z_revision_29_July_18_11:06_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_30_July_18_11:07_AM	2019-07-19 13:39:19.000000000 -0400
@@ -29,7 +29,8 @@
 First group was rushing and second was slacking and “celebrating” 
 Passengers looked business-y 
 Update, 6:30 PM: The GAStech executive jet arrived at the airport. The occupants were taken 
-by limousine to the GAStech building.  explanation 
+by limousine to the GAStech building. Ingrid Barranco 
+   explanation 
  POK 5 year report notes: 
  Isia Vann ‘s father was/is member of POK but also works for GasTECH as a security officer. 
 Controls physical security 
