--- VASTN5Z_revision_33_July_18_11:20_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_34_July_18_11:20_AM	2019-07-19 13:39:19.000000000 -0400
@@ -22,10 +22,11 @@
 Varja.Lagos@gastech.com.kronos, Inga.Ferro@gastech.com.kronos, 
 Kanon.Herrero@gastech.com.kronos, Hideki.Cocinaro@gastech.com.kronos, 
 Minke.Mies@gastech.com.kronos 
-(This email is going to Hennie Osvaldo and Loreto Bodrogi, who both share names with  
+(This email is going to Hennie Osvaldo and Loreto Bodrogi, who both share names with Henk 
+Bodrogi and Carmine Osvaldo who were leaders in the earlier days of POK) 
  1018 - Reports of fire in the morning but was false alarm 
-https://vast.ncsu-las.net/?article=72 
-Limo with execs to a breakfast? Update, 3:40 PM: One GAStech employee, who declined to be identified, reports having seen 
+https://vast.ncsu-las.net/?article=72 Limo with execs to a breakfast? 
+Update, 3:40 PM: One GAStech employee, who declined to be identified, reports having seen 
 several people dressed in black in the building before the fire alarm. These people were not 
 GAStech employees are 'were acting odd and lurking about'. Another report says these were 
 caterers 
