--- VASTN5Z_revision_26_July_18_11:01_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_27_July_18_11:02_AM	2019-07-19 13:39:19.000000000 -0400
@@ -19,8 +19,8 @@
 https://vast.ncsu-las.net/?article=72 
 Limo with execs to a breakfast? 
 Update, 2:00 PM (GAStech): A government car was allowed through the police perimeter. The 
-occupant, who appears to be a security official (possibly Isia/the other guy, was let into the 
-building. 
+occupant, who appears to be a security official (possibly Isia/the other guyEdvard), was let into 
+the building. 
 Limo at airport 
 2 flights out of Abila airport, one at 12:30 one at 14:30 each had 7-8 passengers. Second of 
 which was confirmed to be headed toward Rome, Italy 
