--- VASTN5Z_revision_31_July_18_11:08_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_32_July_18_11:19_AM	2019-07-19 13:39:19.000000000 -0400
@@ -15,7 +15,7 @@
  Today - POK has grown under the charismatic leadership of Elian Karel to loosely organized 
 group with an estimated membership of 200-300 people. 
   Timeline on jan 20-21 
- 1018 - Reports of fire in the morning but was false alarm 
+   1018 - Reports of fire in the morning but was false alarm 
 https://vast.ncsu-las.net/?article=72 
 Limo with execs to a breakfast? 
 Update, 3:40 PM: One GAStech employee, who declined to be identified, reports having seen 
@@ -27,8 +27,8 @@
 Limo at airport 
 2 flights out of Abila airport, one at 12:30 one at 14:30 each had 7-8 passengers. Second of 
 which was confirmed to be headed toward Rome, Italy 
-First group was rushing and second was slacking and “celebrating” 
-Passengers looked business-y Update, 6:30 PM: The GAStech executive jet arrived at the airport. The occupants were taken 
+First group was rushing and second was slacking and “celebrating” Passengers looked business-y 
+Update, 6:30 PM: The GAStech executive jet arrived at the airport. The occupants were taken 
 by limousine to the GAStech building. 
 Ingrid Barranco 
    explanation 
