--- VASTN5Z_revision_22_July_18_10:52_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_23_July_18_10:53_AM	2019-07-19 13:39:19.000000000 -0400
@@ -15,18 +15,17 @@
  Today - POK has grown under the charismatic leadership of Elian Karel to loosely organized 
 group with an estimated membership of 200-300 people. 
   Timeline on jan 20-21 
- Reports of fire in the morning 
+ Reports of fire in the morning but was false alarm 
 Limo with execs to a breakfast 
 Limo at airport 
- 2 flights out of Abila airport, one at 12:30 one at 14:30 each had 7-8 passengers. Second of 
+2 flights out of Abila airport, one at 12:30 one at 14:30 each had 7-8 passengers. Second of 
 which was confirmed to be headed toward Rome, Italy 
 First group was rushing and second was slacking and “celebrating” 
 Passengers looked business-y 
   explanation 
  POK 5 year report notes: 
  Isia Vann ‘s father was/is member of POK but also works for GasTECH as a security officer. 
-Controls physical security 
-Isia has a resume for Gastech but none for Edvard (what if they are the same person) 
+Controls physical security Isia has a resume for Gastech but none for Edvard (what if they are the same person) 
 Edvard Vann (no known relation to Mandor) Questioned on 1/20 by police 
 Security guard at GasTech 
   
\ No newline at end of file
