--- VASTWI0-1_revision_40_July_18_10:41_AM	2019-07-22 15:59:10.000000000 -0400
+++ VASTWI0-1_revision_41_July_18_10:41_AM	2019-07-22 15:59:10.000000000 -0400
@@ -25,7 +25,8 @@
  POK Organization 
 ●1997 - Grassroots movement led under Henk Bodrogi 
 ●1997 - Carmine Osvaldo assists in organizing the POK 
-●2001 - Elian Karel takes over as leader of POK ●Silvia Marek / Mandor Vann (Uncle of Isia VAnn, a GASTech employee. 
+●2001 - Elian Karel takes over as leader of POK 
+●Silvia Marek / Mandor Vann (Uncle of Isia VAnn, a GASTech employee. 
 ●Isia Vann (Older brother of Juliana Vann) 
 ●Rachel Pantanal (Executive Assistant at Gastech, native of Tethys) 
 ●Loreto Bodrogi (Gastech employee)? Relative of Henk Bodrogi? 
@@ -47,4 +48,9 @@
 communication. That’s why the company and media were unable to contact any of the 
 executives. While the CEO and executives are out of country, the POK try to capitalize on the 
 short window of opportunity and make their demands. However, Sanjorge arrives back with the 
-executives a few hours later and hurriedly make their way to the police station.  
\ No newline at end of file
+executives a few hours later and hurriedly make their way to the police station.  
+1
+1
+2
+3
+4
\ No newline at end of file
