--- VASTWI0-1_revision_27_July_18_8:52_AM	2019-07-22 15:59:10.000000000 -0400
+++ VASTWI0-1_revision_28_July_18_8:53_AM	2019-07-22 15:59:10.000000000 -0400
@@ -17,7 +17,8 @@
  Sanjorge’s jet took the executives to Tethys and Sanjorge returned back to Kronos by 
 himself. 
   Timeline 
-   POK Organization 
+ ●6:30 pm - The jet arrived back at  
+ POK Organization 
 ●1997 - Grassroots movement led under Henk Bodrogi 
 ●1997 - Carmine Osvaldo assists in organizing the POK 
 ●2001 - Elian Karel takes over as leader of POK 
