--- VASTUBX-2_revision_25_July_17_10:08_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_26_July_17_10:09_AM	2019-07-30 15:49:45.000000000 -0400
@@ -5,7 +5,7 @@
  Orhan Strum/ CIO Y y
  Sten Sanjorge, Jr. / CEO Y Y
  William Vasco-Pais/ Env. Safety Off.
- Adan Morlun / Truck Driver Y Prior-Military, only other employement is with GasTech
+ Adan Morlun / Truck Driver Y Prior- Kronos Military, only other employement is with GasTech
  Albina Hafon / Truck Driver Y Prior- Kronos Military, only other employement is with GasTech, Employment Gap of 2 years
  Anda Ribera / Executive Assistant Y Y
  Axel Calzas / Snr. Drill Tech y y
