--- VASTUBX-2_revision_154_July_18_9:17_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_155_July_18_10:22_AM	2019-07-30 15:49:45.000000000 -0400
@@ -2,7 +2,7 @@
  Name/Position Tethys Citizen Kronos Citizen Unknown Time Employed w/ GasTech Ex-Kronos Mil. Ex-Tethys Mil. Notes
  Ada Campo/ CIO Y 10
  Ingrid Barranco/ CFO Y 11
- Orhan Strum/ CIO Y ?
+ Orhan Strum/ COO Y ?
  Sten Sanjorge, Jr. / CEO Y NA
  William Vasco-Pais/ Env. Safety Off. Y 4
  Adan Morlun / Truck Driver Y 8 y only other employement is with GasTech
@@ -11,7 +11,7 @@
  Axel Calzas / Snr. Drill Tech y 17
  Bertrand Ovan / Facilities Manager Y 16 y GasTech only other employment
  Carla Forluniau / Admin. Asst. Y 9 y prior - Onda Inc employment overlap with Kronos employment?
- Claudio Hawelon /Truck Driver y 7 y prior- employment on-time delivery
+ Claudio Hawelon /Truck Driver y 7 y general discharge, prior- employment on-time delivery
  Cornelia Lais / Exec. Asst. y 3 y
  Dante Coginian / Cleaning TL y 19 y
  Elsa Orilla / Drill Tech Y 10
@@ -25,7 +25,7 @@
  Isia Vann / Sec. Officer 7 y brother of Juliana Vann, nephew of PoK strategist Mando Vann, multiple arrests in connection w/ PoK
  Lars Azada / Research Engineer 19 y
  Lidelse Dedos / Engineering Group Manager y 11 prior w/ G&A and w/ Sanli
- Linda Lagos / Exec. Asst. y 4 prior w/ Bank of Kronos, avoided military conscription?, not captured
+ Linda Lagos / Exec. Asst. y 4 prior w/ Bank of Kronos, not captured
  Linnea Bergen / IT Group Manager y 10 prior w/ IT4U
  Loreto Bodrogi / Sec. Officer y 1 y prior w/ RQS security
  Minke Mies / Sec. Officer y 1 y prior w/ Bank of Kronos
