--- VASTUBX-2_revision_39_July_17_10:17_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_40_July_17_10:17_AM	2019-07-30 15:49:45.000000000 -0400
@@ -10,4 +10,4 @@
  Anda Ribera / Exec. Asst. Y Y 5
  Axel Calzas / Snr. Drill Tech y y 17
  Bertrand Ovan / Facilities Manager Y 16 Prior- Tethys Military Officer, GasTech only other employment
- Carla Forluniau / Admin. Asst. Y Prior- Kronos Military, prior - Onda Inc employment
+ Carla Forluniau / Admin. Asst. Y 9 Prior- Kronos Military, prior - Onda Inc employment overlap with Kronos employment?
