--- VASTUBX-2_revision_119_July_17_11:00_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_120_July_17_11:04_AM	2019-07-30 15:49:45.000000000 -0400
@@ -22,7 +22,7 @@
  Irene Nant / Truck Driver y 6 y (early discharge?), prior w/ Hernandez Retail
  Isak Baza / IT tech y 16 prior w/ Pharma Inc
  Isanda Borassca / Snr. Drill Tech. y 10 prior w/ TO&G
- Isia Vann / Sec. Officer 7 y
+ Isia Vann / Sec. Officer 7 y father killed by cancer of toxins?
  Lars Azada / Research Engineer 19 y
  Lidelse Dedos / Engineering Group Manager y 11 prior w/ G&A and w/ Sanli
  Linda Lagos / Exec. Asst. y 4 prior w/ Bank of Kronos, avoided military conscription?
