--- VASTUBX-2_revision_21_July_17_10:04_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_22_July_17_10:05_AM	2019-07-30 15:49:45.000000000 -0400
@@ -7,4 +7,4 @@
  William Vasco-Pais/ Env. Safety Off.
  Adan Morlun / Truck Driver Y Prior-Military, only other employement is with GasTech
  Albina Hafon / Truck Driver Y Prior-Military, only other employement is with GasTech, Employment Gap of 2 years
- Anda Ribera / Executive Assistant
+ Anda Ribera / Executive Assistant Y Y
