--- VASTUBX-2_revision_72_July_17_10:33_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_73_July_17_10:34_AM	2019-07-30 15:49:45.000000000 -0400
@@ -20,4 +20,4 @@
  Felix Resumir / Security Grp. Manager y 11 y prior employment Tarryn Security
  Hideki Cocinaro / Sec. Officer y 4 y
  Irene Nant / Truck Driver y 6 y (early discharge?), prior w/ Hernandez Retail
- Isak Baza / IT tech
+ Isak Baza / IT tech 16
