--- VASTUBX-2_revision_26_July_17_10:09_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_27_July_17_10:10_AM	2019-07-30 15:49:45.000000000 -0400
@@ -6,7 +6,7 @@
  Sten Sanjorge, Jr. / CEO Y Y
  William Vasco-Pais/ Env. Safety Off.
  Adan Morlun / Truck Driver Y Prior- Kronos Military, only other employement is with GasTech
- Albina Hafon / Truck Driver Y Prior- Kronos Military, only other employement is with GasTech, Employment Gap of 2 years
+ Albina Hafon / Truck Driver Y Prior- Kronos Military (possible early discharge?), only other employement is with GasTech, Employment Gap of 2 years
  Anda Ribera / Executive Assistant Y Y
  Axel Calzas / Snr. Drill Tech y y
  Bertrand Ovan / Facilities Manager Y Prior- Tethys Military
