--- VASTFOB_revision_49_July_18_8:54_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_50_July_18_8:54_AM	2019-07-22 16:07:08.000000000 -0400
@@ -25,7 +25,7 @@
 the POK)  
 ●Mandor Vann- Uncle to Isia and Juliana Vann (servedat the Kronosian 
 embassies in Tethys and Asteria) 
-●Isia Vann- servedat the Kronosian embassies in Tethys and Asteria. 
+●Isia Vann- older brother of Juliana Vann 
 ●Lucio Jakab-  
 ●Lorenzo Di Stefano 
 ● 
