--- VASTFOB_revision_21_July_17_10:36_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_22_July_17_10:37_AM	2019-07-22 16:07:08.000000000 -0400
@@ -14,4 +14,5 @@
 ●Albina Hafon- truck driver, Armed Forces of Kronos  
 ●Bertrand Ovan-Facilities Group Manager, retired major Armed Forces of Kronos 
 ●Carla Forluniau-Armed Forces of Kronos  
+●Claudio Hawelon - 
    
\ No newline at end of file
