--- VASTFOB_revision_18_July_17_10:34_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_19_July_17_10:35_AM	2019-07-22 16:07:08.000000000 -0400
@@ -12,6 +12,6 @@
   People of interest 
 ●Adan Morlun- truck driver, Armed Forces of Kronos  
 ●Albina Hafon- truck driver, Armed Forces of Kronos  
-● 
-●Bertrand Ovan-Facilities Group Manager, retired major of the  
+●Bertrand Ovan-Facilities Group Manager, retired major of the Armed Forces of 
+Kronos 
    
\ No newline at end of file
