--- VASTFOB_revision_60_July_18_9:04_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_61_July_18_9:05_AM	2019-07-22 16:07:08.000000000 -0400
@@ -21,7 +21,7 @@
 ●Isia Vann- Security Officer, Armed Forces of Kronos ** 
 ●Edvard Vann- interrogated employee** 
       POK members 
-●Elian Karel- most likely the leader of POK 
+●Elian Karel- most likely thecurrent leader of POK 
 -Henk Bodrogi- previous leader (wife died from contaminated water) 
 ●Silvia Marek-founder of Save Our Wildlands (which later joined with 
 the POK)  
