--- VASTFOB_revision_16_July_17_10:32_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_17_July_17_10:33_AM	2019-07-22 16:07:08.000000000 -0400
@@ -11,5 +11,6 @@
  -7Sep2012 ABILA, Kronos. Protest by the POK 
   People of interest 
 ●Adan Morlun- truck driver, Armed Forces of Kronos  
-●Albina Hafon- truck driver, Armed Forces of Kronos 
+●Albina Hafon- truck driver, Armed Forces of Kronos  
+● 
     
\ No newline at end of file
