--- VASTFOB_revision_2_July_17_10:01_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_3_July_17_10:01_AM	2019-07-22 16:07:08.000000000 -0400
@@ -1,6 +1,6 @@
 Kidnapped personel of GAStech 
-President and CEO  Sten Sanjorge Jr,  
-CFO Ingrid Barranco,  
-CIO Ada campo-Corrente,  
-COO Orhan Strum, and  
-environment civil servant Willem Vasco-Pais van GAStech.an 
\ No newline at end of file
+●President Sten Sanjorge Jr 
+●CFO Ingrid Barranco  
+●CIO Ada campo-Corrente  
+●COO Orhan Strum  
+●environment civil servant Willem Vasco-Pais van 
\ No newline at end of file
