--- VASTFOB_revision_10_July_17_10:15_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_11_July_17_10:16_AM	2019-07-22 16:07:08.000000000 -0400
@@ -6,6 +6,7 @@
 ●environment civil servant Willem Vasco-Pais van 
  - Kidnapped after a council meeting at hoofdkwartier GASTech. Taken by 
 POK 
+-ransom note $20 million 
  - 03/26/2014 attack on the presidential boat during boat of the world race 
  -7Sep2012 ABILA, Kronos. Protest by the POK 
     
\ No newline at end of file
