--- VASTFOB_revision_25_July_17_10:39_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_26_July_17_10:56_AM	2019-07-22 16:07:08.000000000 -0400
@@ -6,7 +6,7 @@
 ●environment civil servant Willem Vasco-Pais van 
  - Kidnapped after a council meeting at hoofdkwartier GASTech. Taken by 
 POK 
- -ransom note $20 million 
+ -r-Ransom note $20 million 
  - 03/26/2014 attack on the presidential boat during boat of the world race 
  -7Sep2012 ABILA, Kronos. Protest by the POK 
   People of interest 
