--- VASTFOB_revision_38_July_18_8:47_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_39_July_18_8:48_AM	2019-07-22 16:07:08.000000000 -0400
@@ -20,7 +20,7 @@
 ●Isia Vann- Security Officer, Armed Forces of Kronos ** 
 ●Edvard Vann- interrogated employee** 
   POK members 
-Mestache Vardshvna- spoke out during protest 
 ●Elian Karel- most likely the leader of POK 
+●Silvia Marek 
 ● ●Mestache Vardshvna- spoke out during protest 
  
\ No newline at end of file
