--- VASTKW0_revision_17_June_20_10:38_AM	2019-06-27 13:09:31.000000000 -0400
+++ VASTKW0_revision_18_June_20_10:49_AM	2019-06-27 13:09:31.000000000 -0400
@@ -8,8 +8,9 @@
 Loreto Bodrogi (Security Officer) 
   The company GasTech was began to benefit the community. Instead it has been causing 
 pollution and environmental damage that has had an immediate and detrimental impact on the 
-community. The GasTech Executives have ignored the outcry from the community and 
-continues to make billions of dollars off the natural gas industry. In order to get the companies 
-attention I believe theyPOK kidnapped the high ranking executives of GasTech in order to get 
-their attention. Possibly use the ransom to support medical research endeavors or to aid in 
-medical assistance for those mad sick but the contamination caused by GasTech. 
\ No newline at end of file
+community. Their only interest has been to gain investors and to become an IPO. 
+The GasTech Executives have ignored the outcry from the community and continues to make 
+billions of dollars off the natural gas industry. In order to get the companies attention I believe 
+POK kidnapped the high ranking executives of GasTech in order to get their attention. Possibly 
+use the ransom to support medical research endeavors or to aid in medical assistance for those 
+mad sick but the contamination caused by GasTech. 
\ No newline at end of file
