--- VASTBR1_revision_50_July_18_9:43_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_51_July_18_9:43_AM	2019-07-30 15:52:54.000000000 -0400
@@ -26,8 +26,8 @@
 some hours ago with 6 other employees. 
 1840 experts from GASTech arrive in Abilia thought security experts 
 2021:  Government of Kronos confirms kidnapping. 
-   12/18/13 POK attacks in broad day light. 
- Question to answer:  3 insiders of POK and supporting document/hypothesis. 
+   12/18/13 POK attacks in broad day light. 2 civil servants. 
+  Question to answer:  3 insiders of POK and supporting document/hypothesis. 
   Disappeared executives on 1/24/14: 
 Disappeared paid include:  
 President and PRESIDENT Sten Sanjorge Jr,  
@@ -45,8 +45,7 @@
 with Onda, Inc. -2010.  But also works at GASTech starting 2005. 
  Minke Mies Security Officer | GASTech – Kronos | Abila, Kronos, recent employment at 
 GASTEC 2013. 
- Loreto Bodrogi Security Officer, Abila, Kronos, 2013 started at GASTech. 
- Vann, Edvard 
+ Loreto Bodrogi Security Officer, Abila, Kronos, 2013 started at GASTech.  Vann, Edvard 
 Position: Perimeter Control (Security) 
 Date of Hire: 08/15/2013 
 Email: Edvard.Vann@gastech.com.kronos 
@@ -66,8 +65,8 @@
 Kanon.Herrero@gastech.com.kronos, Hideki.Cocinaro@gastech.com.kronos, 
 Minke.Mies@gastech.com.kronos 
    Recent hire after 2010 and citizen of Kronos: 
-Lais, Cornelia 
-Position: Assistant to Security Group Manager (Administration) Date of Hire: 12/22/2011 
+Lais, Cornelia Position: Assistant to Security Group Manager (Administration) 
+Date of Hire: 12/22/2011 
 Email: Cornelia.Lais@gastech.com.kronos 
 Arpa, Emile 
 Position: Janitor (Facilities) 
