--- VASTBR1_revision_64_July_18_10:43_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_65_July_18_10:46_AM	2019-07-30 15:52:54.000000000 -0400
@@ -27,6 +27,7 @@
 1840 experts from GASTech arrive in Abilia thought security experts 
 2021:  Government of Kronos confirms kidnapping. 
   2000:  Minister of Health Nespola concerned about cancer near Eldoris. 
+2001: Protectors of Kronos leader Henk Bodrogi 
 2001:  Nespola wants tax increase to help health issues.  Defeated. 
 1998:  Nespola investigates health issues near Eldoris. 
 1998:  Nespola meets with POK to discuss issues. 
