--- VASTBR1_revision_38_July_18_9:04_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_39_July_18_9:04_AM	2019-07-30 15:52:54.000000000 -0400
@@ -15,10 +15,10 @@
  1/20/14 at 1035am fire trucks arrive at GAStech building. 
 1145 all clear by fire department. 
 12:45 police cordon off GASTech building 
+1345 confirm missing GASTech employees. 
 1745 2 private jets leave airport.  First to Rome.  Second unknown. 
 2021:  Government of Kronos confirms kidnapping. 
-     Question to answer:  3 insiders of POK and supporting document/hypothesis. 
-  Disappeared executives on 1/24/14: 
+     Question to answer:  3 insiders of POK and supporting document/hypothesis.   Disappeared executives on 1/24/14: 
 Disappeared paid include:  
 President and PRESIDENT Sten Sanjorge Jr,  
 CFO Ingrid Barranco,  
@@ -43,8 +43,7 @@
  Bodrogi, Loreto 
 Position: Site Control (Security) 
 Date of Hire: 08/17/2013 
-Email: Loreto.Bodrogi@gastech.com.kronos 
- Ferro, Inga 
+Email: Loreto.Bodrogi@gastech.com.kronos  Ferro, Inga 
 Position: Site Control (Security) 
 Date of Hire: 01/11/2013 
 Email: Inga.Ferro@gastech.com.kronos 
@@ -62,8 +61,8 @@
 Email: Cornelia.Lais@gastech.com.kronos 
 Arpa, Emile 
 Position: Janitor (Facilities) 
-Date of Hire: 03/18/2013 
-Email: Emile.Arpa@gastech.com.kronos  Awelon, Varro 
+Date of Hire: 03/18/2013 Email: Emile.Arpa@gastech.com.kronos 
+ Awelon, Varro 
 Position: Janitor (Facilities) 
 Date of Hire: 10/22/2012 
 Email: Varro.Awelon@gastech.com.kronos 
