--- VASTBR1_revision_45_July_18_9:25_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_46_July_18_9:28_AM	2019-07-30 15:52:54.000000000 -0400
@@ -12,16 +12,18 @@
  Edvard Vann, a GAStech was interrogated. 
  2 private jets with 14 people take off to Rome, Italy.  Second group 
 celebrating something. 
- 1/20/14 at 1035am fire trucks arrive at GAStech building. 
+ 1/20/14 at  
+1018 fire alarm goes off at GAStech building 
+1035am fire trucks arrive at GAStech building. 
 1145 all clear by fire department. 
 12:25 sees Sanjorge but not these others: have not seen Ingrid Barranca, 
 live-Current of the Ada, the Strum di Orhan, or Willem Vasco-Pais. 
  12:45 police cordon off GASTech building 
 1345 confirm missing GASTech employees. 
 1430 private jet leaves Abila 
-1745 2 private jets leave airport.  First to Rome.  Second unknown. 
-18:10 jet arrives in arrived in Tethys Jr. of Sten Sanjorge of the bearing of 
-some hours ago with 6 other employees. 1840 experts from GASTech arrive in Abilia thought security experts 
+1745 2 private jets leave airport.  First to Rome.  Second unknown. 18:10 jet arrives in arrived in Tethys Jr. of Sten Sanjorge of the bearing of 
+some hours ago with 6 other employees. 
+1840 experts from GASTech arrive in Abilia thought security experts 
 2021:  Government of Kronos confirms kidnapping. 
      Question to answer:  3 insiders of POK and supporting document/hypothesis. 
   Disappeared executives on 1/24/14: 
@@ -43,7 +45,8 @@
 GASTEC 2013. 
  Loreto Bodrogi Security Officer, Abila, Kronos, 2013 started at GASTech. 
  Vann, Edvard 
-Position: Perimeter Control (Security) Date of Hire: 08/15/2013 
+Position: Perimeter Control (Security) 
+Date of Hire: 08/15/2013 
 Email: Edvard.Vann@gastech.com.kronos 
  Bodrogi, Loreto 
 Position: Site Control (Security) 
