--- VASTBR1_revision_62_July_18_10:41_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_63_July_18_10:42_AM	2019-07-30 15:52:54.000000000 -0400
@@ -28,6 +28,7 @@
 2021:  Government of Kronos confirms kidnapping. 
   2000:  Minister of Health Nespola concerned about cancer near Eldoris. 
 2001:  Nespola wants tax increase to help health issues.  Defeated. 
+1998:  Nespola investigates health issues near Eldoris. 
   12/16/13 POK attacks in broad day light 2 police officers 
 12/17/13 GASTech IPO. 
 6/22/13 POK demonstration in front of Abila capitol 
