--- VASTWI0-1_revision_36_July_18_10:34_AM	2019-07-22 15:59:10.000000000 -0400
+++ VASTWI0-1_revision_37_July_18_10:36_AM	2019-07-22 15:59:10.000000000 -0400
@@ -34,4 +34,7 @@
 ●Hinke Mies 
 ●Minke Mies 
 Theory 
-● 
\ No newline at end of file
+●The executive assistants are working with the security team to coordinate the events of the day to 
+route the executives of GasTech to a vulnerable position to be kidnapped. They are then escorted 
+from GasTech to the Airport and then departed Abila Airport for Tethys. One of the jets flew back 
+to Kronos shortly after with the CEO Sanjorge and a few of the executives  
\ No newline at end of file
