--- VASTWI0-1_revision_23_July_18_8:43_AM	2019-07-22 15:59:10.000000000 -0400
+++ VASTWI0-1_revision_24_July_18_8:43_AM	2019-07-22 15:59:10.000000000 -0400
@@ -17,12 +17,11 @@
  Sanjorge’s jet took the executives to Tethys and Sanjorge returned back to Kronos by 
 himself. 
   Timeline 
- POK LeadershipOrganization 
+ POK Organization 
 ●1997 - Grassroots movement led under Henk Bodrogi 
 ●1997 - Carmine Osvaldo assists in organizing the POK 
 ●2001 - Elian Karel takes over as leader of POK 
 ●Silvia Marek / Mandor Vann (Uncle of Isia VAnn, a GASTech employee. 
- ● 
 ●Isia Vann 
 ●Loreto Bodrogi (Gastech employee)? Relative of Henk Bodrogi? 
-● 
\ No newline at end of file
+●Hennie Osvaldo (Gastech employee), relative of Carmine Osvaldo? 
\ No newline at end of file
