--- VASTWI0-1_revision_9_July_17_10:36_AM	2019-07-22 15:59:10.000000000 -0400
+++ VASTWI0-1_revision_10_July_17_10:54_AM	2019-07-22 15:59:10.000000000 -0400
@@ -4,13 +4,14 @@
 Possibly kidnapped by the POK 
  POK established in 1998 in Elodis 
 POK Members 
-1.Petra Hohimae 
-2.Carol/KarelHenk Bodrogi 
-3. 
+1.Henk Bodrogi 
+2. 
     Action: Virus detected on your system 
 From: Linnea.Bergen@gastech.com.kronos 
 01/14/2014, 14:33:00 
 To: Inga.Ferro@gastech.com.kronos, Loreto.Bodrogi@gastech.com.kronos, 
 Isia.Vann@gastech.com.kronos, Hennie.Osvaldo@gastech.com.kronos, 
 Minke.Mies@gastech.com.kronos, Ruscella.Mies.Haber@gastech.com.kronos 
+  Two planes were reported to have left the Abila airport around 12:30 and 2:30 the day of the 
+kidnapping, each carrying about 7-8 passengers. Destination Rome, Italy. 
  
\ No newline at end of file
