--- VASTWI0-1_revision_28_July_18_8:53_AM	2019-07-22 15:59:10.000000000 -0400
+++ VASTWI0-1_revision_29_July_18_8:53_AM	2019-07-22 15:59:10.000000000 -0400
@@ -17,14 +17,15 @@
  Sanjorge’s jet took the executives to Tethys and Sanjorge returned back to Kronos by 
 himself. 
   Timeline 
- ●6:30 pm - The jet arrived back at  
+ ●6:30 pm - The jet arrived back at The GAStech executive jet arrived at the airport. The 
+occupants were taken by limousine to the GAStech building. 
  POK Organization 
 ●1997 - Grassroots movement led under Henk Bodrogi 
 ●1997 - Carmine Osvaldo assists in organizing the POK 
 ●2001 - Elian Karel takes over as leader of POK 
 ●Silvia Marek / Mandor Vann (Uncle of Isia VAnn, a GASTech employee. 
 ●Isia Vann 
-●Loreto Bodrogi (Gastech employee)? Relative of Henk Bodrogi? 
-●Hennie Osvaldo (Gastech employee), relative of Carmine Osvaldo? ●Inga Ferro (Gastech employee)? 
+●Loreto Bodrogi (Gastech employee)? Relative of Henk Bodrogi? ●Hennie Osvaldo (Gastech employee), relative of Carmine Osvaldo? 
+●Inga Ferro (Gastech employee)? 
 ●Hinke Mies 
 ●Minke Mies 
\ No newline at end of file
