--- VASTWI0-1_revision_2_July_17_9:50_AM	2019-07-22 15:59:10.000000000 -0400
+++ VASTWI0-1_revision_3_July_17_9:51_AM	2019-07-22 15:59:10.000000000 -0400
@@ -1,4 +1,4 @@
-January 21, 2014 - 14 employees fear kidnapped Including President and CEO Sten Sanjorge Jr, CFO 
+January 21, 2014 - 1410 employees fear kidnapped Including President and CEO Sten Sanjorge Jr, CFO 
 Ingrid Barranco, CIO Ada campo-Corrente, COO Orhan Strum, and environment civil servant Willem 
 Vasco-Pais van GAStech. 
 Possibly kidnapped by Kronos guards 
\ No newline at end of file
