--- VASTGZ2_revision_7_June_19_10:32_AM	2019-06-27 13:14:02.000000000 -0400
+++ VASTGZ2_revision_8_June_19_10:34_AM	2019-06-27 13:14:02.000000000 -0400
@@ -6,5 +6,6 @@
 Long winded answers from kronovians 
  Mies Harber, Ruscella/Forluniau, Caela 
 -Passport single issue? 
--Says shethey isare from Kronos and in the Kronovian Armed forces 
-- 
\ No newline at end of file
+-Says they are from Kronos and in the Kronovian Armed forces 
+  Mies Harber Ruscella is not listed in resumes 
+It is shown as Minke Mies 
\ No newline at end of file
