--- VASTGZ2_revision_6_June_19_10:31_AM	2019-06-27 13:14:02.000000000 -0400
+++ VASTGZ2_revision_7_June_19_10:32_AM	2019-06-27 13:14:02.000000000 -0400
@@ -4,7 +4,7 @@
  Struggling in business toward outside companies 
 -Disheart, mistrust? 
 Long winded answers from kronovians 
- Mies Harber Ruscella 
+ Mies Harber, Ruscella/Forluniau, Caela 
 -Passport single issue? 
--Says she is from Kronos and in the Kronovian Armed forces 
+-Says shethey isare from Kronos and in the Kronovian Armed forces 
 - 
\ No newline at end of file
