--- VASTXYZ-1_revision_0_July_17_9:26_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_1_July_17_9:55_AM	2019-07-22 16:15:24.000000000 -0400
@@ -1 +1,8 @@
- 
\ No newline at end of file
+Willem Vasco-Pais Environmental Safety Advisor Willem Vasco-Pais joined GASTech in 2010 as 
+its Environmental Safety Advisor. In this role, he provides strong leadership as GASTech 
+continues to innovate and grow into new countries and technologies. Although concerns have 
+been raised in the past, Mr. Vasco-Pais has put these to rest with new oversight programs and 
+close relations with government officials. Prior to joining GASTech, Mr. Vasco-Pais worked in the 
+pharmecutical industry insuring ethical and responsible development of new drugs and medical 
+procedures. 
+   
\ No newline at end of file
