--- VASTXYZ-1_revision_10_July_17_10:05_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_11_July_17_10:06_AM	2019-07-22 16:15:24.000000000 -0400
@@ -15,4 +15,5 @@
  Carla Forluniau - Kronos Military  Required service complete 1999. Specialized in coordination 
 of large group events, such as training and deployment. (Is well-trained to organize a 
 kidnapping) 
-  
\ No newline at end of file
+ Claudio Hawelon - Quick to fix vehicle problems, makes him an asset for get-away, former 
+Armed Forces of Kronos. 
\ No newline at end of file
