--- VASTXYZ-1_revision_29_July_17_11:01_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_30_July_17_11:02_AM	2019-07-22 16:15:24.000000000 -0400
@@ -46,9 +46,18 @@
     From: Sten.Sanjorge Jr.@gastech.com.kronos 
 01/17/2014, 13:03:00 
 To: Henk.Mies@gastech.com.kronos 
-      RE: Be Careful! 
+      Be Careful! 
+From: Bertrand.Ovan@gastech.com.kronos 
+01/14/2014, 12:22:00 
+To: Emile.Arpa@gastech.com.kronos, Varro.Awelon@gastech.com.kronos, 
+Dante.Coginian@gastech.com.kronos, Albina.Hafon@gastech.com.kronos, 
+Benito.Hawelon@gastech.com.kronos, Claudio.Hawelon@gastech.com.kronos, 
+Henk.Mies@gastech.com.kronos, Valeria.Morlun@gastech.com.kronos, 
+Adan.Morlun@gastech.com.kronos, Cecilia.Morluniau@gastech.com.kronos, 
+Irene.Nant@gastech.com.kronos, Dylan.Scozzese@gastech.com.kronos 
+RE: Be Careful! 
 From: Henk.Mies@gastech.com.kronos 
 01/14/2014, 15:55:00 
 To: Dylan.Scozzese@gastech.com.kronos, Varro.Awelon@gastech.com.kronos, 
 Emile.Arpa@gastech.com.kronos 
- 
\ No newline at end of file
+   
\ No newline at end of file
