--- VASTPSZ_revision_23_July_18_9:44_AM	2019-07-22 15:24:53.000000000 -0400
+++ VASTPSZ_revision_24_July_18_9:44_AM	2019-07-22 15:24:53.000000000 -0400
@@ -14,7 +14,6 @@
 GAStech two founders - Sten Sanjorge Sr and Hank Fluss 
 Protectors of Kronos leader Henk Bodrogi 
 environmental leader Willem Vasco-Feed 
-was in transit from the GAStech building 
    Emile Arpa 
 Irene Nant 
 Mies Haber, Ruscella 
