--- VASTPSZ_revision_17_July_18_8:54_AM	2019-07-22 15:24:53.000000000 -0400
+++ VASTPSZ_revision_18_July_18_8:59_AM	2019-07-22 15:24:53.000000000 -0400
@@ -9,7 +9,8 @@
 environment civil servant Willem Vasco-Pais van GAStec 
 Disappeared after council meeting at hoofdkwartier  
 Distraction - Fire alarm 
-Jeroen Carol, were an original member 
+Jeroen Carol, were an original member of POK 
+two founders - Sten Sanjorge Sr and Hank Fluss 
     Emile Arpa 
 Irene Nant 
 Mies Haber, Ruscella 
