--- VASTPSZ_revision_19_July_18_8:59_AM	2019-07-22 15:24:53.000000000 -0400
+++ VASTPSZ_revision_20_July_18_9:08_AM	2019-07-22 15:24:53.000000000 -0400
@@ -11,7 +11,8 @@
 Distraction - Fire alarm 
 Jeroen Carol, were an original member of POK 
 GAStech two founders - Sten Sanjorge Sr and Hank Fluss 
-    Emile Arpa 
+Protectors of Kronos leader Henk Bodrogi 
+   Emile Arpa 
 Irene Nant 
 Mies Haber, Ruscella 
 Forluniau, Carla 
