--- VASTPSZ_revision_15_July_17_11:31_AM	2019-07-22 15:24:53.000000000 -0400
+++ VASTPSZ_revision_16_July_18_8:43_AM	2019-07-22 15:24:53.000000000 -0400
@@ -9,6 +9,7 @@
 environment civil servant Willem Vasco-Pais van GAStec 
 Disappeared after council meeting at hoofdkwartier  
 Distraction - Fire alarm 
+Jeroen Carol, were an original member 
     Emile Arpa 
 Irene Nant 
 Mies Haber, Ruscella 
