--- VASTPSZ_revision_21_July_18_9:33_AM	2019-07-22 15:24:53.000000000 -0400
+++ VASTPSZ_revision_22_July_18_9:43_AM	2019-07-22 15:24:53.000000000 -0400
@@ -13,6 +13,7 @@
 GAStech two founders - Sten Sanjorge Sr and Hank Fluss 
 Protectors of Kronos leader Henk Bodrogi 
 environmental leader Willem Vasco-Feed 
+was in transit from the GAStech building 
    Emile Arpa 
 Irene Nant 
 Mies Haber, Ruscella 
