--- VASTUBX-1_revision_9_July_17_11:25_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_10_July_18_8:18_AM	2019-07-19 13:57:14.000000000 -0400
@@ -6,4 +6,6 @@
  Silvia Marek - Protectors of Kronos protest leader 
 Juliana Vann - 10 yr old killed by cancer, environmental 
 Ilian Karel - killed in prison, assassination?, pre. PoK leader 
-Petrus Gerhard - reporter 
\ No newline at end of file
+Petrus Gerhard - reporter 
+ PoK 
+Leader: Elian Karel 
\ No newline at end of file
