--- VASTUBX-1_revision_42_July_18_9:24_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_43_July_18_10:22_AM	2019-07-19 13:57:14.000000000 -0400
@@ -5,7 +5,7 @@
  Kronos Military is mandatory 
  Juliana Vann - 10 yr old killed by cancer, environmental 
 Petrus Gerhard - Homeland Illumination 
-Michale Kraft: ELian Karel’s lawyer 
+Michale Kraft: ELlian Karel’s lawyer 
  PoK 
 Leader: Henk Bodrogi (1997 - 2001) with Bodrogi’s departure, original 7 members quit, 
 concerns about environment, and health from GasTech drilling 
