--- VASTUBX-1_revision_44_July_18_10:53_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_45_July_18_11:04_AM	2019-07-19 13:57:14.000000000 -0400
@@ -7,7 +7,8 @@
 Petrus Gerhard - Homeland Illumination 
 Michale Kraft: Elian Karel’s lawyer 
  Why was Sanjorge and a bunch of the low-ranking security and truck drivers (numerous PoK 
-poss. affiliates) all at the Gyro place at the same time the night before the kidnapping 
+poss. affiliates) all at the Gyro place at the same time the night before the kidnapping? The only 
+time he is at that restaurant.  
  PoK 
 Leader: Henk Bodrogi (1997 - 2001) with Bodrogi’s departure, original 7 members quit, 
 concerns about environment, and health from GasTech drilling 
