--- VASTUBX-1_revision_26_July_18_8:37_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_27_July_18_8:38_AM	2019-07-19 13:57:14.000000000 -0400
@@ -16,4 +16,5 @@
 Leader: Silvia Marek - peaceful protest 
  Member: Carmine Osvaldo 
 Member: Mando Vann, uncle of Isia and Juliana Vann 
-Member: Lucio Jakab 
\ No newline at end of file
+Member: Lucio Jakab, media 
+ 
\ No newline at end of file
