--- VASTUBX-1_revision_37_July_18_9:15_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_38_July_18_9:16_AM	2019-07-19 13:57:14.000000000 -0400
@@ -20,6 +20,6 @@
  Chronology: 
 -Admin Asst. claims executives are missing, denies that employees are missing 
 -14 now claimed missing, Edvard Vann arrested and released at 1700 
-- 2 planes confirmed having left Abila filled with 7-8 employees each, 2nd group celebrating en 
-route to Rome, Italy 
+- 2 planes confirmed having left Abila filled with 7-8 employees each,. 1st group leaves @ 
+12:30. 2nd group celebrating en route to Rome, Italy @ 14:30 
   
\ No newline at end of file
