--- VASTUBX-1_revision_31_July_18_9:05_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_32_July_18_9:07_AM	2019-07-19 13:57:14.000000000 -0400
@@ -14,7 +14,7 @@
 increasingly radical and violent 
 Leader: Silvia Marek - peaceful protest 
  Member: Carmine Osvaldo 
-Member: Mando Vann, uncle of Isia and Juliana Vann 
+Member: Mandor Vann, uncle of Isia and Juliana Vann, possible radical leader within PoK 
 Member: Lucio Jakab, media 
 Member: Lorenzo di Stephano, enviro scientist 
  Chronology: 
