--- VASTUBX-1_revision_30_July_18_8:46_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_31_July_18_9:05_AM	2019-07-19 13:57:14.000000000 -0400
@@ -3,9 +3,7 @@
 Info given by other: 14 kidnapped, 5 executives 
  Articles start in 1982 
  Kronos Military is mandatory 
- Silvia Marek - Protectors of Kronos protest leader 
-Juliana Vann - 10 yr old killed by cancer, environmental 
-Ilian Karel - killed in prison, assassination?, pre. PoK leader 
+ Juliana Vann - 10 yr old killed by cancer, environmental 
 Petrus Gerhard - Homeland Illumination 
 Michale Kraft: ELian Karel’s lawyer 
  PoK 
@@ -19,4 +17,5 @@
 Member: Mando Vann, uncle of Isia and Juliana Vann 
 Member: Lucio Jakab, media 
 Member: Lorenzo di Stephano, enviro scientist 
-  
\ No newline at end of file
+ Chronology: 
+ 
\ No newline at end of file
