--- VASTBTZ_revision_41_June_20_8:53_AM	2019-06-27 13:08:54.000000000 -0400
+++ VASTBTZ_revision_42_June_20_8:55_AM	2019-06-27 13:08:54.000000000 -0400
@@ -58,5 +58,7 @@
 Ruscella.Mies.Haber@gastech.com.kronos, Carla.Forluniau@gastech.com.kronos, 
 Cornelia.Lais@gastech.com.kronos  
  Lais arranges travels for all staff 
- Article states that  
-POK would most likley end up disbanding if their methods don’t chnage 
\ No newline at end of file
+ Article states that POK would most likley end up disbanding if their methods don’t 
+chnagechange and thet they would need to grow larger to become a realistic force. It would 
+seem that they want to be peaceful and that kidnapping would not make since and would 
+hurt their brand 
\ No newline at end of file
