--- VASTBTZ_revision_46_June_20_8:59_AM	2019-06-27 13:08:54.000000000 -0400
+++ VASTBTZ_revision_47_June_20_9:03_AM	2019-06-27 13:08:54.000000000 -0400
@@ -63,5 +63,6 @@
 want to be peaceful and that kidnapping would not make since and would hurt their brand. 
 The kidnappers also asked for money and the POK has never indicated that money was a 
 motive for anything 
- Silvia Marek or Mandor Vann could be leaders after Elian Karel. Vann has ties is uncle to Isia 
-and Juliana Vann. Isia Vann now works at Gastech 
\ No newline at end of file
+ Silvia Marek or Mandor Vann could be leaders after Elian Karel. Vann is uncle to Isia and 
+Juliana Vann. Isia Vann now works at Gastech 
+ Kidnapping happened in transit. EA Lais arranges travel 
\ No newline at end of file
