--- VASTBTZ_revision_38_June_20_8:28_AM	2019-06-27 13:08:54.000000000 -0400
+++ VASTBTZ_revision_39_June_20_8:29_AM	2019-06-27 13:08:54.000000000 -0400
@@ -57,4 +57,5 @@
 Rachel.Pantanal@gastech.com.kronos, Linda.Lagos@gastech.com.kronos, 
 Ruscella.Mies.Haber@gastech.com.kronos, Carla.Forluniau@gastech.com.kronos, 
 Cornelia.Lais@gastech.com.kronos  
-  
\ No newline at end of file
+ Lais arranges travels for all staff 
+ 
\ No newline at end of file
