--- VASTBTZ_revision_50_June_20_9:11_AM	2019-06-27 13:08:54.000000000 -0400
+++ VASTBTZ_revision_51_June_20_9:14_AM	2019-06-27 13:08:54.000000000 -0400
@@ -67,4 +67,5 @@
 Juliana Vann. Isia Vann now works at Gastech and is ex Kronos military 
  Kidnapping happened in transit. EA Lais arranges travel 
  Three Mies that work at GAStech all ex Kronos military. One is EA, one truck driver, last is 
-security 
\ No newline at end of file
+security 
+HASR banned in Tethys but GAStech still does in Kronos 
\ No newline at end of file
