--- VASTBTZ_revision_55_June_20_9:24_AM	2019-06-27 13:08:54.000000000 -0400
+++ VASTBTZ_revision_56_June_20_9:25_AM	2019-06-27 13:08:54.000000000 -0400
@@ -69,7 +69,8 @@
  Three Mies that work at GAStech all ex Kronos military. One is EA, one truck driver, last is 
 security 
 Bodrogi, Loreto also ex Kronos military and site controlat Gastech 
- HASR banned in Tethys but GAStech still does in Kronos 
+Ferro, Inga ex Kronos military works at Gastech 
+  HASR banned in Tethys but GAStech still does in Kronos 
  Inga.Ferro@gastech.com.kronos, Loreto.Bodrogi@gastech.com.kronos, 
 Isia.Vann@gastech.com.kronos, Hennie.Osvaldo@gastech.com.kronos, 
 Minke.Mies@gastech.com.kronos, Ruscella.Mies.Haber@gastech.com.kronos 
