--- VASTOMS_revision_135_June_20_10:44_AM	2019-06-27 13:15:28.000000000 -0400
+++ VASTOMS_revision_136_June_20_10:45_AM	2019-06-27 13:15:28.000000000 -0400
@@ -151,6 +151,9 @@
 ---Reported Timeline---
  Breaking: Emergency at GAStech Headquarters Building [Updates] Cato Rossini, Marcella Trapani
  Kronos Star | 01/20/2014
+ Update, 11:45 AM: Fire department officials allow GAStech employees to re-enter the building.
+ Update, 12:20 PM: The Abila police have been called to respond to GAStech headquarters for an unspecified emergency. (editor's note - The call was overheard on a police scanner. The Abila police know this channel is monitored by the media and often do not give the reason for the response)
+ Update, 12:40 PM: Additional police have arrived at GAStech headquarters. They are setting up crowd control barricades around the building. Police have not commented on their activities.
  Update 1:00 PM: Police have come and gone from the GAStech building, but up to this point no GAStech employees have left the building. We still have no official word on why the police are present.
  Update, 1:35 PM: We called the office of Ingrid Barranco, the GSAtech Chief Financial Officer. Her administrator, who was very uneasy during the conversation, informed us that Ms. Barranco is unavailable for comment.
  Update, 1:50 PM: We have tried twenty more phone numbers at GAStech, but no one else has answered. We tried calling Mr. Strum's administrator again, but she did not answer the phone.
