--- VASTOMS_revision_134_June_20_10:44_AM	2019-06-27 13:15:28.000000000 -0400
+++ VASTOMS_revision_135_June_20_10:44_AM	2019-06-27 13:15:28.000000000 -0400
@@ -153,9 +153,11 @@
  Kronos Star | 01/20/2014
  Update 1:00 PM: Police have come and gone from the GAStech building, but up to this point no GAStech employees have left the building. We still have no official word on why the police are present.
  Update, 1:35 PM: We called the office of Ingrid Barranco, the GSAtech Chief Financial Officer. Her administrator, who was very uneasy during the conversation, informed us that Ms. Barranco is unavailable for comment.
+ Update, 1:50 PM: We have tried twenty more phone numbers at GAStech, but no one else has answered. We tried calling Mr. Strum's administrator again, but she did not answer the phone.
  Update, 2:00 PM: A government car was allowed through the police perimeter. The occupant, who appears to be a security official, was let into the building.
  Update, 3:05 PM: Some police have left the scene, but others remain to outside to secure the building while others are still in the building. There is much speculation about their activities, but police have yet to respond to our queries for information.
  Update, 3:20 PM: GAStech employees are being allowed to leave the building.
+ Update, 5:50 PM: According to a GAStech employee, the GAStech executives left the building after their meeting this morning and were headed to the reception at the capitol building. When asked about why we were unable to reach them by phone, the employee replied that it was standard corporate security procedure for the executives to be rushed to a secure location in the event of a threat like the kidnapping.
  Update, 6:40 PM: There is speculation that the GAStech personnel who have arrived from Tethys are security experts who are assisting police with the investigation.
  Update, 7:15 PM: Abila police have announced there will be a short statement made to the press at 7:30 pm.
  Update, 7:40 PM: Carman informed the press that police do not believe the missing people have left the island of Kronos.
