--- VASTXYZ-1_revision_20_July_17_10:27_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_21_July_17_10:31_AM	2019-07-22 16:15:24.000000000 -0400
@@ -9,16 +9,12 @@
 truck driver, knowing the best routes, etc. He used to work for the Kornos military and is 
 probably still loyal to that country, wanting to stand up to the industrialization that is not 
 improving the overall infrastructure or economy for Kronos itself. 
- Albina Hafon - same as before, Irene Nant - same as before 
- Axel Calzas - kidnapee - picked the site  
+  Axel Calzas - kidnapee - picked the site  
  Bertrand Ovan - managed budget & finances, staff, etc 
  Carla Forluniau - Kronos Military  Required service complete 1999. Specialized in coordination 
 of large group events, such as training and deployment. (Is well-trained to organize a 
 kidnapping) 
- Claudio Hawelon - Quick to fix vehicle problems, makes him an asset for get-away, former 
-Armed Forces of Kronos. “Works effectively in a team support large pickup and delivery 
-operations.” 
- Cornelia Lais - high up in company, has some power aka inside knowledge. Used to work for 
+  Cornelia Lais - high up in company, has some power aka inside knowledge. Used to work for 
 AF/K.  
 Administrative and business support to the Security Group Manager.  Coordinate schedules 
 among multiple personnel that are constantly on the move.  Arrange travel and training for all 
@@ -26,5 +22,6 @@
  Dante Coginian, Emile Arpa  - janitor, previous AFK, easy to fly under radar 
  Isia Vann , Loreto Bodrogi, - has a lot of access as far as security goes… almost definitely 
 involved. Former AFK.  
+ Marin Onda -  
  Linda Lagos - Planned and coordinated initial hiring screening procedures. 
   
\ No newline at end of file
