--- VASTXYZ-1_revision_19_July_17_10:26_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_20_July_17_10:27_AM	2019-07-22 16:15:24.000000000 -0400
@@ -5,10 +5,10 @@
 close relations with government officials. Prior to joining GASTech, Mr. Vasco-Pais worked in the 
 pharmecutical industry insuring ethical and responsible development of new drugs and medical 
 procedures. 
-  Adan Morlun - he is all about being an established truck driver, knowing the best routes, etc. He 
-used to work for the Kornos military and is probably still loyal to that country, wanting to stand up 
-to the industrialization that is not improving the overall infrastructure or economy for Kronos 
-itself. 
+  Adan Morlun, Claudio Hawelon, Albina Hafon, Irene Nant   - he is all about being an established 
+truck driver, knowing the best routes, etc. He used to work for the Kornos military and is 
+probably still loyal to that country, wanting to stand up to the industrialization that is not 
+improving the overall infrastructure or economy for Kronos itself. 
  Albina Hafon - same as before, Irene Nant - same as before 
  Axel Calzas - kidnapee - picked the site  
  Bertrand Ovan - managed budget & finances, staff, etc 
@@ -23,8 +23,7 @@
 Administrative and business support to the Security Group Manager.  Coordinate schedules 
 among multiple personnel that are constantly on the move.  Arrange travel and training for all 
 staff in the group 
- Dante Coginian, Emile Arpa  - lead janitor, previous AFK, easy to fly under radar 
- Emile Arpa - ^^^ 
+ Dante Coginian, Emile Arpa  - janitor, previous AFK, easy to fly under radar 
  Isia Vann , Loreto Bodrogi, - has a lot of access as far as security goes… almost definitely 
 involved. Former AFK.  
  Linda Lagos - Planned and coordinated initial hiring screening procedures. 
