--- VASTXYZ-1_revision_37_July_17_11:24_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_38_July_17_11:24_AM	2019-07-22 16:15:24.000000000 -0400
@@ -11,18 +11,7 @@
 improving the overall infrastructure or economy for Kronos itself. 
   Axel Calzas - kidnapee - picked the site  
  Bertrand Ovan - managed budget & finances, staff, etc 
- Carla Forluniau - Kronos Military  Required service complete 1999. Specialized in coordination 
-of large group events, such as training and deployment. (Is well-trained to organize a 
-kidnapping) 
-  Cornelia Lais - high up in company, has some power aka inside knowledge. Used to work for 
-AF/K.  
-Administrative and business support to the Security Group Manager.  Coordinate schedules 
-among multiple personnel that are constantly on the move.  Arrange travel and training for all 
-staff in the group 
- Dante Coginian, Emile Arpa  - janitor, previous AFK, easy to fly under radar 
- Isia Vann , - has a lot of access as far as security goes… almost definitely involved. Former 
-AFK. Loreto Bodrogi, 
- Marin Onda - he maintained current knowledge of all industry standard techniques, as well as, 
+  Marin Onda - he maintained current knowledge of all industry standard techniques, as well as, 
 innovations inside GASTech. He is technically wellversed in all substrate extraction techniques. 
  Linda Lagos - Planned and coordinated initial hiring screening procedures. 
  Minke Mies - Armed Forces of Kronos 2009-2011  Specialized in security procedures and 
@@ -83,4 +72,8 @@
 From: Emile.Arpa@gastech.com.kronos 
 01/15/2014, 13:45:00 
 To: Marin.Onda@gastech.com.kronos 
- 
\ No newline at end of file
+ 1
+1
+2
+3
+4
\ No newline at end of file
