--- VASTXYZ-1_revision_11_July_17_10:06_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_12_July_17_10:07_AM	2019-07-22 16:15:24.000000000 -0400
@@ -16,4 +16,6 @@
 of large group events, such as training and deployment. (Is well-trained to organize a 
 kidnapping) 
  Claudio Hawelon - Quick to fix vehicle problems, makes him an asset for get-away, former 
-Armed Forces of Kronos. 
\ No newline at end of file
+Armed Forces of Kronos. “Works effectively in a team support large pickup and delivery 
+operations.” 
+  
\ No newline at end of file
