--- VASTXYZ-1_revision_21_July_17_10:31_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_22_July_17_10:31_AM	2019-07-22 16:15:24.000000000 -0400
@@ -22,6 +22,7 @@
  Dante Coginian, Emile Arpa  - janitor, previous AFK, easy to fly under radar 
  Isia Vann , Loreto Bodrogi, - has a lot of access as far as security goes… almost definitely 
 involved. Former AFK.  
- Marin Onda -  
+ Marin Onda - he maintained current knowledge of all industry standard techniques, as well as, 
+innovations inside GASTech. He is technically wellversed in all substrate extraction techniques. 
  Linda Lagos - Planned and coordinated initial hiring screening procedures. 
   
\ No newline at end of file
