--- VASTXYZ-1_revision_15_July_17_10:11_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_16_July_17_10:14_AM	2019-07-22 16:15:24.000000000 -0400
@@ -9,7 +9,7 @@
 used to work for the Kornos military and is probably still loyal to that country, wanting to stand up 
 to the industrialization that is not improving the overall infrastructure or economy for Kronos 
 itself. 
- Albina Hafon - same as before 
+ Albina Hafon - same as before, Irene Nant - same as before 
  Axel Calzas - kidnapee - picked the site  
  Bertrand Ovan - managed budget & finances, staff, etc 
  Carla Forluniau - Kronos Military  Required service complete 1999. Specialized in coordination 
