--- VASTXYZ-1_revision_17_July_17_10:16_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_18_July_17_10:24_AM	2019-07-22 16:15:24.000000000 -0400
@@ -25,4 +25,6 @@
 staff in the group 
  Dante Coginian - lead janitor, previous AFK, easy to fly under radar 
  Emile Arpa - ^^^ 
- Isia Vann - has a lot of access as far as security goes… almost definitely involved. Former AFK.  
\ No newline at end of file
+ Isia Vann - has a lot of access as far as security goes… almost definitely involved. Former AFK.  
+ Linda Lagos - Planned and coordinated initial hiring screening procedures. 
+  
\ No newline at end of file
