--- VASTXYZ-1_revision_31_July_17_11:04_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_32_July_17_11:05_AM	2019-07-22 16:15:24.000000000 -0400
@@ -67,4 +67,9 @@
 Dylan.Scozzese@gastech.com.kronos, Irene.Nant@gastech.com.kronos, 
 Varro.Awelon@gastech.com.kronos, Benito.Hawelon@gastech.com.kronos, 
 Dante.Coginian@gastech.com.kronos 
- 
\ No newline at end of file
+   Union meeting 
+From: Valeria.Morlun@gastech.com.kronos 
+01/16/2014, 12:43:00 
+To: Dylan.Scozzese@gastech.com.kronos, Adan.Morlun@gastech.com.kronos, 
+Henk.Mies@gastech.com.kronos, Albina.Hafon@gastech.com.kronos 
+     
\ No newline at end of file
