--- VASTXYZ-1_revision_1_July_17_9:55_AM	2019-07-22 16:15:24.000000000 -0400
+++ VASTXYZ-1_revision_2_July_17_9:58_AM	2019-07-22 16:15:24.000000000 -0400
@@ -5,4 +5,8 @@
 close relations with government officials. Prior to joining GASTech, Mr. Vasco-Pais worked in the 
 pharmecutical industry insuring ethical and responsible development of new drugs and medical 
 procedures. 
-   
\ No newline at end of file
+  Adan Morlun - he is all about being an established truck driver, knowing the best routes, etc. He 
+used to work for the Kornos military and is probably still loyal to that country, wanting to stand up 
+to the industrialization that is not improving the overall infrastructure or economy for Kronos 
+itself. 
+  
\ No newline at end of file
