--- VASTSYZ_revision_8_June_19_11:25_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_9_June_19_11:26_AM	2019-06-27 13:12:26.000000000 -0400
@@ -1,8 +1,8 @@
 Loreto Bodrogi 
 Security - Employee since 2013 - Former military (honorable discharge) 
- Possibly related to Henk Bodrogi, founder of POK 
+  Possibly related to Henk Bodrogi, founder of POK 
 sends/receives emails related to POK on corp email 
-   Position: Site Control (Security) 
+ Position: Site Control (Security) 
 Date of Hire: 08/17/2013 
 Email: Loreto.Bodrogi@gastech.com.kronos 
  Sex: Male 
