--- VASTSYZ_revision_2_June_19_11:21_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_3_June_19_11:22_AM	2019-06-27 13:12:26.000000000 -0400
@@ -1,8 +1,6 @@
-Inga.Ferro@gastech.com.kronos,  
-Loreto.Bodrogi@gastech.com.kronos,  
-Isia.Vann@gastech.com.kronos,  
-Hennie.Osvaldo@gastech.com.kronos,  
+                                           Inga.Ferro@gastech.com.kronos 
+Loreto.Bodrogi@gastech.com.kronos 
+Isia.Vann@gastech.com.kronos 
+Hennie.Osvaldo@gastech.com.kronos 
 Minke.Mies@gastech.com.kronos 
-Ruscella.Mies.Haber 
- Send to: Inga.Ferro@gastech.com
- 
\ No newline at end of file
+Ruscella.Mies.Haber 
\ No newline at end of file
