--- VASTSYZ_revision_21_June_19_11:35_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_22_June_19_11:36_AM	2019-06-27 13:12:26.000000000 -0400
@@ -42,7 +42,7 @@
 Discharge Type: General Discharge 
                        Hennie Osvaldo 
 There’s a Osvaldo warehouse related to Ferro Hardware in Elodis 
-Carmin Osvaldo listed as co-leader of POK in 1999 
-   Hennie.Osvaldo@gastech.com.kronos 
+Carmin Osvaldo (sp?) listed as co-leader of POK in 1999 
+     Hennie.Osvaldo@gastech.com.kronos 
 Minke.Mies@gastech.com.kronos 
 Ruscella.Mies.Haber 
\ No newline at end of file
