--- VASTSYZ_revision_4_June_19_11:23_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_5_June_19_11:24_AM	2019-06-27 13:12:26.000000000 -0400
@@ -1,4 +1,7 @@
 Loreto Bodrogi 
+ Security - Employee since 2013 - Former military (honorable discharge) 
+ Possibly related to Henk Bodrogi, founder of POK 
+sends/receives emails related to POK on corp email 
                                           Inga.Ferro@gastech.com.kronos 
 Loreto.Bodrogi@gastech.com.kronos 
 Isia.Vann@gastech.com.kronos 
