--- VASTSYZ_revision_17_June_19_11:31_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_18_June_19_11:31_AM	2019-06-27 13:12:26.000000000 -0400
@@ -14,6 +14,7 @@
 Discharge Type: Honorable Discharge 
                        Isia Vann 
 Possibly related to Juliana Vann, girl who died in Elodis due to water poisoning 
+sends/receives emails related to POK on corp email 
  Position: Perimeter Control (Security) 
 Date of Hire: 12/14/2007 
 Email: Isia.Vann@gastech.com.kronos 
@@ -27,6 +28,7 @@
 Discharge Type: General Discharge 
                          Inga Ferro 
 Possibly related to the Ferros of Elodis (Ferro Hardware) 
+sends/receives emails related to POK on corp email 
  Position: Site Control (Security) 
 Date of Hire: 01/11/2013 
 Email: Inga.Ferro@gastech.com.kronos 
