--- VASTSYZ_revision_14_June_19_11:29_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_15_June_19_11:30_AM	2019-06-27 13:12:26.000000000 -0400
@@ -26,8 +26,8 @@
 Discharge Date: 10/01/2007 
 Discharge Type: General Discharge 
                          Isia Ferro 
-Possibly related to the Ferros of Elodis (Vann’s hardware) 
-                  Inga.Ferro@gastech.com.kronos 
+Possibly related to the Ferros of Elodis (Vann’sFerro hHardware) 
+                    Inga.Ferro@gastech.com.kronos 
 Loreto.Bodrogi@gastech.com.kronos 
 Isia.Vann@gastech.com.kronos 
 Hennie.Osvaldo@gastech.com.kronos 
