--- VASTSYZ_revision_34_June_20_8:32_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_35_June_20_8:32_AM	2019-06-27 13:12:26.000000000 -0400
@@ -1,10 +1,10 @@
-POK leadership 
+POK 
 founded in 1998 in Elodis, a rural township outside of the capital Abila. They have been 
 protesting for clean water, human rights, and against the corruption of the Government of 
 Kronos and GAStech International. 
-  Leadership 
+ Leadership 
 Henk Bodrogi (founder) -> Elian Karel -> Silvia Marek 
-   Loreto Bodrogi 
+                                    Loreto Bodrogi 
  Possibly related to Henk Bodrogi, founder of POK 
 sends/receives emails related to POK on corp email 
  Position: Site Control (Security) 
