--- VASTSYZ_revision_35_June_20_8:32_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_36_June_20_8:33_AM	2019-06-27 13:12:26.000000000 -0400
@@ -4,7 +4,8 @@
 Kronos and GAStech International. 
  Leadership 
 Henk Bodrogi (founder) -> Elian Karel -> Silvia Marek 
-                                    Loreto Bodrogi 
+  It’s likely that the POK has infiltrated GASTech  
+                                  Loreto Bodrogi 
  Possibly related to Henk Bodrogi, founder of POK 
 sends/receives emails related to POK on corp email 
  Position: Site Control (Security) 
