--- VASTSYZ_revision_16_June_19_11:30_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_17_June_19_11:31_AM	2019-06-27 13:12:26.000000000 -0400
@@ -25,9 +25,20 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2007 
 Discharge Type: General Discharge 
-                         Isianga Ferro 
+                         Inga Ferro 
 Possibly related to the Ferros of Elodis (Ferro Hardware) 
-                    Inga.Ferro@gastech.com.kronos 
+ Position: Site Control (Security) 
+Date of Hire: 01/11/2013 
+Email: Inga.Ferro@gastech.com.kronos 
+ Sex: Female 
+DOB: 06/17/1989 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2012 
+Discharge Type: General Discharge 
+                   Inga.Ferro@gastech.com.kronos 
 Loreto.Bodrogi@gastech.com.kronos 
 Isia.Vann@gastech.com.kronos 
 Hennie.Osvaldo@gastech.com.kronos 
