--- VASTSYZ_revision_31_June_20_8:28_AM	2019-06-27 13:12:26.000000000 -0400
+++ VASTSYZ_revision_32_June_20_8:29_AM	2019-06-27 13:12:26.000000000 -0400
@@ -90,4 +90,19 @@
 most 
 experience in government, Vann has become the group’s top political strategist. POK 
 members have great respect for Vann and describe him as intense, intelligent, organized, 
-and ambitious. 
\ No newline at end of file
+and ambitious. 
+ Lucio Jakab 
+Lucio Jakab was a member and co-founder of a small environmental activist group, Save 
+Our 
+Wildlands, which formed at the University of Abila in 1998. Elian Karel convinced Jakab and 
+the other members of Save Our Wildlands to join forces with the POK in 2005. Jakab is an 
+activist who believes that the habitat of wild life unique to Kronos is imperiled by drilling, 
+logging, and pollution. A media communications professional, Jakab is responsible for 
+many of the handbills, signs, banners, and slogans used by the POK. 
+ Lorenzo Di Stefano 
+Lorenzo Di Stefano is a professor of environmental science at the University of Abila and a 
+POK activist who believes that the government needs to enact stronger environmental laws 
+to protect the country’s agrarian lifestyle. Di Stefano has recently authored several scientific 
+articles on the environmental contamination associated with the Hyper Acidic Substrate 
+Removal (HASR) technique used at several Kronosian gas fields and has advocated for 
+regulations that would outlaw this technique. 
\ No newline at end of file
