--- VAST32P_revision_29_July_18_8:22_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_30_July_18_8:23_AM	2019-07-19 13:59:36.000000000 -0400
@@ -1,26 +1,12 @@
 Vast32P GasTech Scenario Notes 
  Who are POK? 
-What happened 1/20-21? 
- Kidnapping happened while in transit GasTech HQ: 
-“Sanjorge, that was been assistiendo to a corporative meeting, was in 
-transit of the GAStech that constructed capitol when it happened the 
-kidnapping. Sanjorge and the four executives of the headquarters of 
-GAStech of the premises were to take care of a reception that was received 
-by president Kapelou in the building of the Capitol.” 
-The Guide | 01/21/2014 
- Possible inside job: 
-“The number of the missing has been revised from fourteen to ten.” 
-The Abila Post | 01/21/2014 
- Fire Alarm, possible signal or distraction: 
-“The information without confirming of a fire in the office could have served 
-like distraction kidnapping them or at least to have contributed to the 
-confusion during the event. The cars of the body of firemen of Avila were 
-sent to the headquarters of GAStech - of Kronos around 10:00 this local 
-time of Kronos in the morning to respond to a fire alarm.” 
-The General Post | 01/20/2014 
  Suspects: 
  E. Vann, Security (Org Chart) 
 (There is also an I. Vann in Security) 
+ “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
+released after 6 hours, indicated the Protectors of Kronos were suspected 
+as being the perpetrators behind today's events.” 
+Who What News | 01/20/2014 
  Possible relative died from GasTech pollution 
 “Julian Vann died in 1998 of the cancer caused by environmental toxins in 
 its potable water.”  
@@ -28,7 +14,8 @@
  Name is actually Juliana Vann 
 “sweet-faced ten year dead old girl - Juliana Vann who died in 1998 of 
 cancer caused by the benzene in her drinking water.” 
-All News Today | 06/21/2013    “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
+All News Today | 06/21/2013 
+   “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
 released after 6 hours, indicated the Protectors of Kronos were suspected 
 as being the perpetrators behind today's events.” 
 Who What News | 01/20/2014 
@@ -38,8 +25,7 @@
 GAStech, was furious, indicating, 'I do not have anything to do nobodies 
 with no members of group POK of the terrorist. I know nobody of Elodis. I 
 was born here in Abila and support to my government and my president.” 
-The Explainer | 01/20/2014 
-    What happened on 1/20-21? 
+The Explainer | 01/20/2014     What happened on 1/20-21? 
  Kidnapping happened while in transit GasTech HQ: 
 “Sanjorge, that was been assistiendo to a corporative meeting, was in 
 transit of the GAStech that constructed capitol when it happened the 
@@ -53,7 +39,8 @@
  Fire Alarm, possible signal or distraction: 
 “The information without confirming of a fire in the office could have served 
 like distraction kidnapping them or at least to have contributed to the 
-confusion during the event. The cars of the body of firemen of Avila were sent to the headquarters of GAStech - of Kronos around 10:00 this local 
+confusion during the event. The cars of the body of firemen of Avila were 
+sent to the headquarters of GAStech - of Kronos around 10:00 this local 
 time of Kronos in the morning to respond to a fire alarm.” 
 The General Post | 01/20/2014 
   
\ No newline at end of file
