--- VAST32P_revision_34_July_18_9:00_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_35_July_18_9:01_AM	2019-07-19 13:59:36.000000000 -0400
@@ -3,8 +3,9 @@
 Organization: 
 “The guards of Kronos or POK were founded in 1997 by residents of rural 
 Elodis in accordance with the opinion of now old health for all (WFA), an 
-international group of safety of water. 
+international group of safety of water.” 
 ” 
+Athena Speaks | 08/24/2012 
   Suspects: 
  E. Vann, Security (Org Chart) 
 (There is also an I. Vann in Security) 
@@ -25,8 +26,8 @@
 from a protector of the member of Kronos. Vann, protector of security for 
 GAStech, was furious, indicating, 'I do not have anything to do nobodies 
 with no members of group POK of the terrorist. I know nobody of Elodis. I 
-was born here in Abila and support to my government and my president.” 
-The Explainer | 01/20/2014  “Edvard Vann of Abila, was questioned for hours because he shares a 
+was born here in Abila and support to my government and my president.” The Explainer | 01/20/2014 
+ “Edvard Vann of Abila, was questioned for hours because he shares a 
 family name with several known POK members.”News Online Today | 01/20/2014 
      What happened on 1/20-21? 
  Kidnapping happened while in transit GasTech HQ: 
