--- VAST32P_revision_39_July_18_9:13_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_40_July_18_9:13_AM	2019-07-19 13:59:36.000000000 -0400
@@ -9,7 +9,7 @@
 Kronos homage paid to their cases LEADER, Elian Karel, who on 19 June 
 2009 whereas resistance worker is held in a prison Kronos assassination. 
 We collect us today to remind Elian Karel, Silvia Marek told rap assembly, 
-much of which supported white candles in the honor of Karel.”..He had on 
+much of which supported white candles in the honor of Karel...He had on 
 its father, Jeroen and the founder of POK, the pace of Henk Bodrogi actual 
 gone paid attention which he would enter.” 
 The Wrap | 06/21/2012 
@@ -36,7 +36,7 @@
 The Explainer | 01/20/2014 
  “Edvard Vann of Abila, was questioned for hours because he shares a 
 family name with several known POK members.”News Online Today | 01/20/2014 
-     What happened on 1/20-21? 
+     What happened on 16/20-21? 
  Kidnapping happened while in transit GasTech HQ: 
 “Sanjorge, that was been assistiendo to a corporative meeting, was in 
 transit of the GAStech that constructed capitol when it happened the 
