--- VAST32P_revision_40_July_18_9:13_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_41_July_18_9:17_AM	2019-07-19 13:59:36.000000000 -0400
@@ -13,7 +13,10 @@
 its father, Jeroen and the founder of POK, the pace of Henk Bodrogi actual 
 gone paid attention which he would enter.” 
 The Wrap | 06/21/2012 
-  Suspects: 
+ “Elian knew his would be a difficult journey. He had watched his father, 
+Jeroen and the founder of the POK, Henk Bodrogi walk the very path he 
+would tread. 
+ Suspects: 
  E. Vann, Security (Org Chart) 
 (There is also an I. Vann in Security) 
  “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
@@ -26,7 +29,8 @@
 Daily Pegasus | 03/26/2014 
  Name is actually Juliana Vann 
 “sweet-faced ten year dead old girl - Juliana Vann who died in 1998 of 
-cancer caused by the benzene in her drinking water.” All News Today | 06/21/2013 
+cancer caused by the benzene in her drinking water.” 
+All News Today | 06/21/2013 
   Vann Possible mistaken identity: 
 “The evident confusion comes the similarity between its name and the one 
 from a protector of the member of Kronos. Vann, protector of security for 
@@ -36,7 +40,7 @@
 The Explainer | 01/20/2014 
  “Edvard Vann of Abila, was questioned for hours because he shares a 
 family name with several known POK members.”News Online Today | 01/20/2014 
-     What happened on 16/20-21? 
+   What happened on 6/20-21? 
  Kidnapping happened while in transit GasTech HQ: 
 “Sanjorge, that was been assistiendo to a corporative meeting, was in 
 transit of the GAStech that constructed capitol when it happened the 
@@ -50,7 +54,7 @@
  Fire Alarm, possible signal or distraction: 
 “The information without confirming of a fire in the office could have served 
 like distraction kidnapping them or at least to have contributed to the 
-confusion during the event. The cars of the body of firemen of Avila were 
-sent to the headquarters of GAStech - of Kronos around 10:00 this local 
-time of Kronos in the morning to respond to a fire alarm.” The General Post | 01/20/2014 
+confusion during the event. The cars of the body of firemen of Avila were sent to the headquarters of GAStech - of Kronos around 10:00 this local 
+time of Kronos in the morning to respond to a fire alarm.” 
+The General Post | 01/20/2014 
   
\ No newline at end of file
