--- VAST32P_revision_73_July_18_11:08_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_74_July_18_11:08_AM	2019-07-19 13:59:36.000000000 -0400
@@ -25,7 +25,8 @@
 government,' he said. 'This is important, so I took time extinguished from 
 the work to be a part of this important event.'’ 
 Daily Pegasus | 03/26/2014 
-   Suspects:  E. Vann, Security (Org Chart) 
+   Suspects: 
+ E. Vann, Security (Org Chart) 
 (There is also an I. Vann in Security) 
  “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
 released after 6 hours, indicated the Protectors of Kronos were suspected 
@@ -51,7 +52,8 @@
    What happened on 1/20-21? Timeline of Events 
  Meeting ends at 1000, GasTech Corporate HQ 
 “The 20 of January were annual corporative meeting of GAStech with a 
-reception of the government of Kronos to follow. Shortly after tomorrow the meeting finished in the 10:00, Abila that the cars of the body of firemen 
+reception of the government of Kronos to follow. Shortly after tomorrow the 
+meeting finished in the 10:00, Abila that the cars of the body of firemen 
 responded to the information of a fire in the office of GAStech.” 
  Fire Alarm, possible signal or distraction, executives leave building: 
 “The information without confirming of a fire in the office could have served 
@@ -68,7 +70,7 @@
  “A GAStech International senior security executive has been on site to 
 support the investigation aiding the Abila Police Department and Kronos 
 Government officials.”The Abila Post | 01/20/2014 
-  “The Protectors of Kronos (POK) are suspected of kidnapping up to 
+ “The Protectors of Kronos (POK) are suspected of kidnapping up to 
 fourteen GAStech employees from their headquarters building earlier today. 
 Abila police quickly secured the building and interviewed the remaining 
 GAStech employees.”International News | 01/20/2014 
@@ -77,7 +79,8 @@
 Homeland Illumination | 01/20/2014 
  “1837 - There are unconfirmed reports that the GAStech employees were 
 kidnapped.”Homeland Illumination | 01/20/2014 
- Kidnapping happened while in transit GasTech HQ: “Sanjorge, that was been assistiendo to a corporative meeting, was in 
+ Kidnapping happened while in transit GasTech HQ: 
+“Sanjorge, that was been assistiendo to a corporative meeting, was in 
 transit of the GAStech that constructed capitol when it happened the 
 kidnapping. Sanjorge and the four executives of the headquarters of 
 GAStech of the premises were to take care of a reception that was received 
@@ -86,4 +89,8 @@
  Possible inside job: 
 “The number of the missing has been revised from fourteen to ten.” 
 The Abila Post | 01/21/2014 
-    
\ No newline at end of file
+    1
+1
+2
+3
+4
\ No newline at end of file
