--- VAST32P_revision_61_July_18_10:50_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_62_July_18_10:50_AM	2019-07-19 13:59:36.000000000 -0400
@@ -57,7 +57,8 @@
 The General Post | 01/20/2014 
  “the GAStech executives left the building after their meeting this 
 morning”Kronos Star | 01/20/2014 
-  “The Protectors of Kronos (POK) are suspected of kidnapping up to 
+[ 
+ “The Protectors of Kronos (POK) are suspected of kidnapping up to 
 fourteen GAStech employees from their headquarters building earlier today. 
 Abila police quickly secured the building and interviewed the remaining 
 GAStech employees.”International News | 01/20/2014 
