--- VAST32P_revision_31_July_18_8:24_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_32_July_18_8:34_AM	2019-07-19 13:59:36.000000000 -0400
@@ -1,6 +1,7 @@
 Vast32P GasTech Scenario Notes 
  Who are POK? 
- Suspects: 
+Organization: 
+   Suspects: 
  E. Vann, Security (Org Chart) 
 (There is also an I. Vann in Security) 
  “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
