--- VAST32P_revision_69_July_18_11:05_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_70_July_18_11:05_AM	2019-07-19 13:59:36.000000000 -0400
@@ -53,7 +53,7 @@
 “The 20 of January were annual corporative meeting of GAStech with a 
 reception of the government of Kronos to follow. Shortly after tomorrow the meeting finished in the 10:00, Abila that the cars of the body of firemen 
 responded to the information of a fire in the office of GAStech.” 
- Fire Alarm, possible signal or distraction: 
+ Fire Alarm, possible signal or distraction, executives leave building: 
 “The information without confirming of a fire in the office could have served 
 like distraction kidnapping them or at least to have contributed to the 
 confusion during the event. The cars of the body of firemen of Avila were 
