--- VASTRDI_revision_64_June_19_2:21_PM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_65_June_20_8:29_AM	2019-06-27 13:08:01.000000000 -0400
@@ -1,6 +1,7 @@
 ID 3 individuals associated with the Protectors of Kronos 
  Describe their involvement, and provide evidence.  
   Eventually construct POK org chart, figure out why people disappeared 
+ Elian Karel- Leadership 
  Isia Vann -  
 -Edvard Vann detained, article mentions he has same name as  POK member.  
 -Works in Security 
