--- VASTRDI_revision_5_June_19_11:04_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_6_June_19_11:13_AM	2019-06-27 13:08:01.000000000 -0400
@@ -1,4 +1,4 @@
 ID 3 individuals associated with the Protectors of Kronos 
  Describe their involvement, and provide evidence.  
   Eventually construct POK org chart, figure out why people disappeared 
-   14 Kidnapped, Ransom of 20M, (sent by who?)protectors of kronos claimed responsibilty 
\ No newline at end of file
+   140 Kidnapped, Ransom of 20M, (protectors of kronos claimed responsibilty 
\ No newline at end of file
