--- VASTRDI_revision_30_June_19_11:46_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_31_June_19_11:51_AM	2019-06-27 13:08:01.000000000 -0400
@@ -13,6 +13,7 @@
 Interrogated and released after 6 hrs, indicated POK  is suspected. 
 Name is similar to POK Member - GASTECH EMPLOYEE INTERROGATED, RELEASED 
 Central Bulletin | 01/20/2014 
+Armed Forces of Kronos 
  Ravi Danisliau- fueler at airport 
 -Noted 2 departing planes 
 -1st departed 1230 with 7-8 business type passengers 
@@ -29,7 +30,6 @@
   1/20    1230  Departure of first plane with 7-8 People 
 1/20    1430 Departure of second plane to Rome with 7-8 people 
 1/20    1700  Edvard Vann Released - News Online Today | 01/20/2014 
- 1/20     UNKEmployees noticed missing -  
- 1/21 0934 estimate revised from 14 to 10. - ON THE SCENE BLOG By Haneson Ngohebo 
+   1/21 0934 estimate revised from 14 to 10. - ON THE SCENE BLOG By Haneson Ngohebo 
 1/21     1000News conference to take place  -ON THE SCENE BLOG By Haneson Ngohebo 
  
\ No newline at end of file
