--- VASTRDI_revision_34_June_19_11:53_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_35_June_19_12:59_PM	2019-06-27 13:08:01.000000000 -0400
@@ -31,6 +31,6 @@
   1/20    1230  Departure of first plane with 7-8 People 
 1/20    1430 Departure of second plane to Rome with 7-8 people 
 1/20    1700  Edvard Vann Released - News Online Today | 01/20/2014 
-   1/21 0934 estimate revised from 14 to 10. - ON THE SCENE BLOG By Haneson Ngohebo 
+   1/21  0934 estimate revised from 14 to 10. - ON THE SCENE BLOG By Haneson Ngohebo 
 1/21     1000News conference to take place  -ON THE SCENE BLOG By Haneson Ngohebo 
  
\ No newline at end of file
