--- VASTRDI_revision_20_June_19_11:30_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_21_June_19_11:34_AM	2019-06-27 13:08:01.000000000 -0400
@@ -10,7 +10,8 @@
  Kidnapping Expert - American John Rathburn  
 Lives in Tethys 
  Security Guard Edvard Vann 
-Interrogated and released after 6 hrs, indicated POK  is suspected 
+Interrogated and released after 6 hrs, indicated POK  is suspected. 
+Name is similar to POK Member 
    Timeline: 
  1/20    UNKKidnapping takes place  
 1/20    1000Fire trucks dispached to GASTech HQ on Kronos in response to fire alarm - 
