--- VASTRDI_revision_39_June_19_1:24_PM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_40_June_19_1:25_PM	2019-06-27 13:08:01.000000000 -0400
@@ -39,4 +39,7 @@
 1/21     1000News conference to take place  -ON THE SCENE BLOG By Haneson 
 Ngohebo 
   E-mail “Subject” Files 
-  
\ No newline at end of file
+ Sanjorge/Mies 
+Arpa/Onda 
+Osvaldo/Pantanal 
+Dedos/Orilla 
\ No newline at end of file
