--- VASTRDI_revision_7_June_19_11:13_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_8_June_19_11:15_AM	2019-06-27 13:08:01.000000000 -0400
@@ -1,5 +1,9 @@
 ID 3 individuals associated with the Protectors of Kronos 
  Describe their involvement, and provide evidence.  
   Eventually construct POK org chart, figure out why people disappeared 
-   10Either 10 or 14 Kidnapped, Ransom of 20M, (protectors of kronos claimed 
-responsibiltyresponsibility 
\ No newline at end of file
+   Either 10 or 14 Kidnapped, Ransom of 20M, (protectors of kronos claimed responsibility 
+ President and CEO Sten Sanjorge Jr,  
+CFO Ingrid Barranco,  
+CIO Ada campo-Corrente,  
+COO Orhan Strum, and environment civil servant  
+environment civil servant Willem Vasco-Pais  
\ No newline at end of file
