--- VASTRDI_revision_47_June_19_1:41_PM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_48_June_19_1:41_PM	2019-06-27 13:08:01.000000000 -0400
@@ -29,8 +29,8 @@
 1/20    1100 Edvard Vann taken into custody? 
 1/201155Fire alarmed cleared. False alarm 
 1/20    1157  noted as not having arrived at Government Reception - VOICES - a blog 
-approximately that what is important to people 
-  Centrum Sentinel | 01/20/2014 
+approximately that what is important to people  Centrum Sentinel | 01/20/2014 
+1/20   UNK  Miriam Avila confirms that leadership missing 
   1/20    1230  Departure of first plane with 7-8 People 
 1/20    1430 Departure of second plane to Rome with 7-8 people 
 1/20    1700  Edvard Vann Released - News Online Today | 01/20/2014 
