--- VASTRDI_revision_83_June_20_9:32_AM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_84_June_20_9:35_AM	2019-06-27 13:08:01.000000000 -0400
@@ -7,6 +7,7 @@
 -Jeroen Karel, Elian’s Father - Organized recruitment and personnel 
 -Died in prison Successor? 
 Silvia Marek 
+-Possible POK Leader 
 Mandor Vann (enlisted lingusit)  
 -Uncle to Isia Vann & Juliana Vann 
 -Political Strategist 
