--- VASTRDI_revision_60_June_19_2:10_PM	2019-06-27 13:08:01.000000000 -0400
+++ VASTRDI_revision_61_June_19_2:12_PM	2019-06-27 13:08:01.000000000 -0400
@@ -47,6 +47,11 @@
 administrative assistant in Tethys headquarters 
 1/20    1430 Departure of second plane to Rome with 7-8 people 
 1/20    1700  Edvard Vann Released - News Online Today | 01/20/2014 
+1/201853 UPDATE - Speculation that the 2:00 PM arrival of Kronos Government 
+official was in fact the arrival of a senior security executive and other officials from 
+GAStech's Tethys office including President and CEO Sten Sanjorge, Jr. This 
+speculation is based on information concerning the arrival, refueling and immediate 
+departure of Sanjorge Jr.'s private jet from the Tethys airport. 
 1/202021 UPDATE - A Kronos Government official who asked not to be identified but 
 was present at the government reception has gone on the record stating, 'Investigators 
 are working around the clock to ensure that the kidnapped GAStech employees are 
