--- VASTT93_revision_34_July_18_8:54_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_35_July_18_8:55_AM	2019-07-23 14:25:35.000000000 -0400
@@ -11,7 +11,7 @@
  Mestache Vardshvna (involved in protests, maybe a POK member) 
 https://vast.ncsu-las.net/?article=265 
   Mandor Vann (was listed as possible succesor to karel) has uncle to Isia Vann (gastech secuirty) 
-Isia Vann (works security at gastech, niecenephew of Mandor, known POK)  
+Isia Vann (works security at gastech, nephew of Mandor, known POK) older brother of  
  Henk Bodrogi (founder of POK) 
 https://vast.ncsu-las.net/?article=452 
  Jeroen Karel (martyred leaders father/founder of POK with henk) 
