--- VASTT93_revision_56_July_18_11:10_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_57_July_18_11:10_AM	2019-07-23 14:25:35.000000000 -0400
@@ -1,5 +1,5 @@
 VastT93 
- Missing on 1/210/2014 
+ Missing on 1/20/2014 
   3 people ASW POK  
  Elian Karel ( dead during imprisonment; POK leader)  
 Juliana Vann (dead by cancer, young girl source of anger for POK) 
@@ -34,7 +34,8 @@
 Same number in second, but less well dressed’ 
 Plane 1 depart 1230  
 Second plane depart 1430 destination Rome  
-  Perhaps someone should locate the caterers from the executive breakfast?   Shopping at alberts fine clothing  
+  Perhaps someone should locate the caterers from the executive breakfast?  
+ Shopping at alberts fine clothing  
  1/6/2014 20:26 
 Albert's Fine 
 Clothing 
@@ -154,7 +155,8 @@
 Clothing 
 283.38 
 Varro 
-Awelon 1/16/2014 
+Awelon 
+1/16/2014 
 20:25 
 Albert's Fine 
 Clothing 
@@ -208,4 +210,8 @@
 Benito hawelon 8 
 Claudio hawelonm 4 
 Dylan scozzese 11 
- 
\ No newline at end of file
+ 1
+1
+2
+3
+4
\ No newline at end of file
