--- VASTT93_revision_46_July_18_9:33_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_47_July_18_9:33_AM	2019-07-23 14:25:35.000000000 -0400
@@ -27,7 +27,8 @@
     Money made from ipo 
 $96.25 million for CFO Ingrid Barranco, and cio Campo-Corrente 
 adviser of safety Willem Vasco-Feed also gained $38.5  
-  2 private jets flew from abila  https://vast.ncsu-las.net/?article=123  
+ 1/20  
+2 private jets flew from abila  https://vast.ncsu-las.net/?article=123  
 7 or 8 passengers in first plane dressed business 
 Same number in second, but less well dressed’ 
 Plane 1 depart 1230  
