--- VASTT93_revision_54_July_18_9:45_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_55_July_18_10:58_AM	2019-07-23 14:25:35.000000000 -0400
@@ -91,9 +91,176 @@
 254.29 
 Isande 
 Borrasca 
+ 1/6/2014 20:26 
+Albert's Fine 
+Clothing 
+276.9 
+Lucas 
+Alcazar 
+1/6/2014 21:03 
+Albert's Fine 
+Clothing 
+121.76 
+Minke 
+Mies 
+1/7/2014 20:04 
+Albert's Fine 
+Clothing 
+308.36 
+Vira 
+Frente 
+1/7/2014 20:10 
+Albert's Fine 
+Clothing 
+78.91 
+Varro 
+Awelon 
+1/8/2014 20:45 
+Albert's Fine 
+Clothing 
+136.01 
+Felix 
+Balas 
+1/9/2014 20:21 
+Albert's Fine 
+Clothing 
+224.52 
+Varro 
+Awelon 
+1/9/2014 21:27 
+Albert's Fine 
+Clothing 
+229.98 
+Ada 
+Campo-Corrent
+e 
+1/10/2014 
+20:11 
+Albert's Fine 
+Clothing 
+126.13 
+Vira 
+Frente 
+1/11/2014 
+16:01 
+Albert's Fine 
+Clothing 
+211.47 
+Minke 
+Mies 1/11/2014 
+19:27 
+Albert's Fine 
+Clothing 
+105.33 
+Orhan 
+Strum 
+1/12/2014 
+15:49 
+Albert's Fine 
+Clothing 
+195.81 
+Adra 
+Nubarron 
+1/12/2014 
+19:58 
+Albert's Fine 
+Clothing 
+63.36 
+Axel 
+Calzas 
+1/13/2014 
+20:35 
+Albert's Fine 
+Clothing 
+288 
+Ingrid 
+Barranco 
+1/13/2014 
+20:42 
+Albert's Fine 
+Clothing 
+49.48 
+Gustav 
+Cazar 
+1/14/2014 
+20:05 
+Albert's Fine 
+Clothing 
+50.77 
+Varja 
+Lagos 
+1/14/2014 
+20:50 
+Albert's Fine 
+Clothing 
+228.72 
+Felix 
+Balas 
+1/15/2014 
+20:46 
+Albert's Fine 
+Clothing 
+228.15 
+Varro 
+Awelon 
+1/16/2014 
+20:08 
+Albert's Fine 
+Clothing 
+283.38 
+Varro 
+Awelon 
+1/16/2014 
+20:25 
+Albert's Fine 
+Clothing 
+91.71 
+Inga 
+Ferro 
+1/17/2014 
+19:44 
+Albert's Fine 
+Clothing 
+39.41 
+Gustav 
+Cazar 
+1/18/2014 
+15:20 
+Albert's Fine 
+Clothing 
+35.66 
+Vira 
+Frente 
+1/18/2014 
+15:57 
+Albert's Fine 
+Clothing 
+189.52 
+Anda 
+Ribera 
+1/18/2014 
+18:59 
+Albert's Fine 
+Clothing 
+226.49 
+Orhan 
+Strum 
+1/19/2014 
+15:39 
+Albert's Fine 
+Clothing 
+274.66 
+Adra 
+Nubarron 
+1/19/2014 
+20:02 
+Albert's Fine 
+Clothing 
+254.29 
+Isande 
+Borrasca 
   Henk Mies (truck driver) spent around 35k at airport… 
 Albena hafon 9k 
 Benito hawelon 8 
-Claudio hawelonm 4 
-Dylan scozzese 11 
+Claudio hawelonm 4 Dylan scozzese 11 
  
\ No newline at end of file
