--- VASTT93_revision_47_July_18_9:33_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_48_July_18_9:34_AM	2019-07-23 14:25:35.000000000 -0400
@@ -25,7 +25,8 @@
 Forluniau, Carla 
 Lais, Cornelia (security) 
     Money made from ipo 
-$96.25 million for CFO Ingrid Barranco, and cio Campo-Corrente 
+$96.25 million for CFO Ingrid Barranco, and cio Campo-Corrente and COO 
+Orhan Strum 
 adviser of safety Willem Vasco-Feed also gained $38.5  
  1/20  
 2 private jets flew from abila  https://vast.ncsu-las.net/?article=123  
