--- VASTT93_revision_32_July_18_8:53_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_33_July_18_8:54_AM	2019-07-23 14:25:35.000000000 -0400
@@ -10,8 +10,9 @@
 https://vast.ncsu-las.net/?article=398 
  Mestache Vardshvna (involved in protests, maybe a POK member) 
 https://vast.ncsu-las.net/?article=265 
-  Mandor Vann (was listed as possible succesor to karel) has ties to Isia Vann (gastech secuirty) 
-Isia Vann (works security at gastech, ties to Mandor, known POK)  
+  Mandor Vann (was listed as possible succesor to karel) has tiesuncle to Isia Vann (gastech 
+secuirty) 
+Isia Vann (works security at gastech, ties toniece of Mandor, known POK)  
  Henk Bodrogi (founder of POK) 
 https://vast.ncsu-las.net/?article=452 
  Jeroen Karel (martyred leaders father/founder of POK with henk) 
