--- VASTT93_revision_10_July_17_10:29_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_11_July_17_10:31_AM	2019-07-23 14:25:35.000000000 -0400
@@ -1,7 +1,7 @@
 VastT93 
  3 people ASW POK  
  Elian Karel ( dead during imprisonment)  
-Julian Vann (dead by cancer) 
+Julian Vann (dead by cancer, young girl source of anger for POK) 
  Mestache Vardshvna involved in protests 
 https://vast.ncsu-las.net/?article=265 
  Edvard Vann (suspected security for POK)  
