--- VASTUBX-1_revision_43_July_18_10:22_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_44_July_18_10:53_AM	2019-07-19 13:57:14.000000000 -0400
@@ -5,7 +5,9 @@
  Kronos Military is mandatory 
  Juliana Vann - 10 yr old killed by cancer, environmental 
 Petrus Gerhard - Homeland Illumination 
-Michale Kraft: ELlian Karel’s lawyer 
+Michale Kraft: Elian Karel’s lawyer 
+ Why was Sanjorge and a bunch of the low-ranking security and truck drivers (numerous PoK 
+poss. affiliates) all at the Gyro place at the same time the night before the kidnapping 
  PoK 
 Leader: Henk Bodrogi (1997 - 2001) with Bodrogi’s departure, original 7 members quit, 
 concerns about environment, and health from GasTech drilling 
