--- VASTUBX-1_revision_15_July_18_8:25_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_16_July_18_8:26_AM	2019-07-19 13:57:14.000000000 -0400
@@ -8,6 +8,6 @@
 Ilian Karel - killed in prison, assassination?, pre. PoK leader 
 Petrus Gerhard - reporter 
  PoK 
-Leader: Henk Bodrogi 1997 - 2001 with Bodrogi’s departure, original 7 members quit 
-Leader: Elian Karel 2001 -  
+Leader: Henk Bodrogi (1997 - 2001) with Bodrogi’s departure, original 7 members quit 
+Leader: Elian Karel (2001 - ) peaceful protests, recruiting, merging with small  
  Member: Carmine Osvaldo 
\ No newline at end of file
