--- VASTUBX-1_revision_6_July_17_11:18_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_7_July_17_11:19_AM	2019-07-19 13:57:14.000000000 -0400
@@ -3,4 +3,4 @@
 Info given by other: 14 kidnapped, 5 executives 
  Articles start in 1982 
  Kronos Military is mandatory 
- Silvia Marek -  
\ No newline at end of file
+ Silvia Marek - Protectors of Kronos protest leader 
\ No newline at end of file
