--- VASTUBX-1_revision_23_July_18_8:34_AM	2019-07-19 13:57:14.000000000 -0400
+++ VASTUBX-1_revision_24_July_18_8:34_AM	2019-07-19 13:57:14.000000000 -0400
@@ -13,5 +13,5 @@
 Leader: Elian Karel (2001 - 19 June 2009) peaceful protests, recruiting, merging with small 
 activist groups, including govt corruption in concerns. 2005-2008, organization becomes 
 increasingly radical and violent 
-Leader: Silvia Marek 
+Leader: Silvia Marek - peaceful protest 
  Member: Carmine Osvaldo 
\ No newline at end of file
