--- VASTBR1_revision_32_July_18_8:55_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_33_July_18_8:56_AM	2019-07-30 15:52:54.000000000 -0400
@@ -10,7 +10,8 @@
  Kidnapping happened in transit from GAStech building to capital. 
  Petrus Gerhard is contacted by POK to relay message. 
  Edvard Vann, a GAStech was interrogated. 
- 2 private jets with 14 people take off to Rome, Italy. 
+ 2 private jets with 14 people take off to Rome, Italy.  Second group 
+celebrating something. 
     Question to answer:  3 insiders of POK and supporting document/hypothesis. 
   Disappeared executives on 1/24/14: 
 Disappeared paid include:  
@@ -18,8 +19,7 @@
 CFO Ingrid Barranco,  
 Campo-Corrente of ADA of CIO,  
 ROUCOULEMENT Orhan de GAStech pianotent, and  
-environmental leader Willem Vasco-Feed.  
- he Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
+environmental leader Willem Vasco-Feed.   he Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
 outside of the capital Abila. They have been protesting for clean water, 
 human rights, and against the corruption of the Government of Kronos and 
 GAStech International. 
@@ -42,7 +42,8 @@
 Position: Site Control (Security) 
 Date of Hire: 01/11/2013 
 Email: Inga.Ferro@gastech.com.kronos 
-   RE: Security procedures for January 20 VIP visit From: Inga.Ferro@gastech.com.kronos 
+   RE: Security procedures for January 20 VIP visit 
+From: Inga.Ferro@gastech.com.kronos 
 01/17/2014, 18:23:00 
 To: Hennie.Osvaldo@gastech.com.kronos, Loreto.Bodrogi@gastech.com.kronos, 
 Varja.Lagos@gastech.com.kronos, Inga.Ferro@gastech.com.kronos, 
