--- VASTBR1_revision_31_July_18_8:52_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_32_July_18_8:55_AM	2019-07-30 15:52:54.000000000 -0400
@@ -10,7 +10,8 @@
  Kidnapping happened in transit from GAStech building to capital. 
  Petrus Gerhard is contacted by POK to relay message. 
  Edvard Vann, a GAStech was interrogated. 
-     Question to answer:  3 insiders of POK and supporting document/hypothesis. 
+ 2 private jets with 14 people take off to Rome, Italy. 
+    Question to answer:  3 insiders of POK and supporting document/hypothesis. 
   Disappeared executives on 1/24/14: 
 Disappeared paid include:  
 President and PRESIDENT Sten Sanjorge Jr,  
