--- VASTBR1_revision_3_July_17_10:34_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_4_July_17_10:35_AM	2019-07-30 15:52:54.000000000 -0400
@@ -1,7 +1,11 @@
 VastBR1@ncsu-last.net notes 
  Question to answer:  3 insiders of POK and supporting document/hypothesis. 
   Disappeared executives on 1/24/14: 
-Disappeared paid include: President and PRESIDENT Sten Sanjorge Jr, CFO 
-Ingrid Barranco, Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de 
-GAStech pianotent, and environmental leader Willem Vasco-Feed.  
+Disappeared paid include:  
+President and PRESIDENT Sten Sanjorge Jr,  
+CFO Ingrid Barranco,  
+Campo-Corrente of ADA of CIO,  
+R 
+OUCOULEMENT Orhan de GAStech pianotent, and  
+environmental leader Willem Vasco-Feed.  
      
\ No newline at end of file
