--- VASTBR1_revision_34_July_18_8:58_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_35_July_18_8:59_AM	2019-07-30 15:52:54.000000000 -0400
@@ -13,12 +13,13 @@
  2 private jets with 14 people take off to Rome, Italy.  Second group 
 celebrating something. 
  1/20/14 at 1035am fire trucks arrive at GAStech building. 
-    Question to answer:  3 insiders of POK and supporting document/hypothesis. 
+2021:  Government of Kronos confirms kidnapping. 
+     Question to answer:  3 insiders of POK and supporting document/hypothesis. 
   Disappeared executives on 1/24/14: 
 Disappeared paid include:  
-President and PRESIDENT Sten Sanjorge Jr,  
-CFO Ingrid Barranco,  
-Campo-Corrente of ADA of CIO,  ROUCOULEMENT Orhan de GAStech pianotent, and  
+President and PRESIDENT Sten Sanjorge Jr,  CFO Ingrid Barranco,  
+Campo-Corrente of ADA of CIO,  
+ROUCOULEMENT Orhan de GAStech pianotent, and  
 environmental leader Willem Vasco-Feed.  
  he Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
 outside of the capital Abila. They have been protesting for clean water, 
@@ -41,8 +42,7 @@
 Email: Loreto.Bodrogi@gastech.com.kronos 
  Ferro, Inga 
 Position: Site Control (Security) 
-Date of Hire: 01/11/2013 
-Email: Inga.Ferro@gastech.com.kronos 
+Date of Hire: 01/11/2013 Email: Inga.Ferro@gastech.com.kronos 
    RE: Security procedures for January 20 VIP visit 
 From: Inga.Ferro@gastech.com.kronos 
 01/17/2014, 18:23:00 
@@ -60,9 +60,9 @@
 Date of Hire: 03/18/2013 
 Email: Emile.Arpa@gastech.com.kronos 
  Awelon, Varro 
-Position: Janitor (Facilities) 
-Date of Hire: 10/22/2012 
-Email: Varro.Awelon@gastech.com.kronos  Mies, Henk 
+Position: Janitor (Facilities) Date of Hire: 10/22/2012 
+Email: Varro.Awelon@gastech.com.kronos 
+ Mies, Henk 
 Position: Truck Driver (Facilities) 
 Date of Hire: 07/12/2013 
 Email: Henk.Mies@gastech.com.kronos 
