--- VASTBR1_revision_17_July_17_11:27_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_18_July_17_11:28_AM	2019-07-30 15:52:54.000000000 -0400
@@ -37,5 +37,7 @@
 Kanon.Herrero@gastech.com.kronos, Hideki.Cocinaro@gastech.com.kronos, 
 Minke.Mies@gastech.com.kronos 
    Recent hire after 2010 and citizen of Kronos: 
-Lais, Cornelia 
- 
\ No newline at end of file
+Lais, CorneliaLais, Cornelia 
+Position: Assistant to Security Group Manager (Administration) 
+Date of Hire: 12/22/2011 
+Email: Cornelia.Lais@gastech.com.kronos   
\ No newline at end of file
