--- VASTBR1_revision_59_July_18_10:10_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_60_July_18_10:40_AM	2019-07-30 15:52:54.000000000 -0400
@@ -26,7 +26,8 @@
 some hours ago with 6 other employees. 
 1840 experts from GASTech arrive in Abilia thought security experts 
 2021:  Government of Kronos confirms kidnapping. 
-   12/16/13 POK attacks in broad day light 2 police officers 
+  2000:  Minister of Health Nespola concerned about cancer. 
+  12/16/13 POK attacks in broad day light 2 police officers 
 12/17/13 GASTech IPO. 
 6/22/13 POK demonstration in front of Abila capitol 
 2/8/13 Designer drug problem. 
@@ -46,8 +47,7 @@
  he Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
 outside of the capital Abila. They have been protesting for clean water, 
 human rights, and against the corruption of the Government of Kronos and 
-GAStech International. 
-  Suspects: 
+GAStech International.   Suspects: 
  Adan Morlun, truck driver, military experience. 
  Carla Forluniau Administrative Assistant | GASTech – Kronos | Abila, Kronos. Resume overlaps 
 with Onda, Inc. -2010.  But also works at GASTech starting 2005. 
