--- VASTBR1_revision_66_July_18_10:48_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_67_July_18_10:48_AM	2019-07-30 15:52:54.000000000 -0400
@@ -22,7 +22,8 @@
 12:45 police cordon off GASTech building 
 1345 confirm missing GASTech employees. 
 1430 private jet leaves Abila. Celebrating.  Heading to Rome. 
-1745 2 private jets leave airport.  First to Rome.  Second unknown. 18:10 jet arrives in arrived in Tethys Jr. of Sten Sanjorge of the bearing of 
+1745 2 private jets leave airport.  First to Rome.  Second unknown. 
+18:10 jet arrives in arrived in Tethys Jr. of Sten Sanjorge of the bearing of 
 some hours ago with 6 other employees. 
 1840 experts from GASTech arrive in Abilia thought security experts 
 2021:  Government of Kronos confirms kidnapping. 
@@ -48,7 +49,8 @@
 President and PRESIDENT Sten Sanjorge Jr,  
 CFO Ingrid Barranco,  
 Campo-Corrente of ADA of CIO,  
-ROUCOULEMENT Orhan de GAStech pianotent, and  environmental leader Willem Vasco-Feed.  
+ROUCOULEMENT Orhan de GAStech pianotent, and  
+environmental leader Willem Vasco-Feed.  
  he Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
 outside of the capital Abila. They have been protesting for clean water, 
 human rights, and against the corruption of the Government of Kronos and 
@@ -96,4 +98,9 @@
 Position: Truck Driver (Facilities) 
 Date of Hire: 07/12/2013 
 Email: Henk.Mies@gastech.com.kronos 
-   
\ No newline at end of file
+    Send to: VastBR1@ncsu-last.net
+   1
+1
+2
+3
+4
\ No newline at end of file
