--- VASTBR1_revision_48_July_18_9:34_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_49_July_18_9:35_AM	2019-07-30 15:52:54.000000000 -0400
@@ -18,10 +18,10 @@
 1145 all clear by fire department. 
 12:25 sees Sanjorge but not these others: have not seen Ingrid Barranca, 
 live-Current of the Ada, the Strum di Orhan, or Willem Vasco-Pais. 
-1230 21 planes take off from Abila. Business types 
+1230 1 planes take off from Abila. Business types 
 12:45 police cordon off GASTech building 
 1345 confirm missing GASTech employees. 
-1430 private jet leaves Abila. celebrating. 
+1430 private jet leaves Abila. cCelebrating.  Heading to Rome. 
 1745 2 private jets leave airport.  First to Rome.  Second unknown. 18:10 jet arrives in arrived in Tethys Jr. of Sten Sanjorge of the bearing of 
 some hours ago with 6 other employees. 
 1840 experts from GASTech arrive in Abilia thought security experts 
