--- VASTBR1_revision_42_July_18_9:11_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_43_July_18_9:12_AM	2019-07-30 15:52:54.000000000 -0400
@@ -19,10 +19,9 @@
  12:45 police cordon off GASTech building 
 1345 confirm missing GASTech employees. 
 1745 2 private jets leave airport.  First to Rome.  Second unknown. 
-18:10 jet arrives in Trst to Rome.  Second unknown. 
-18:10 jet arrives in hetysarrived in Tethys Jr. of Sten Sanjorge of the bearing 
-of some hours ago 2021:  Government of Kronos confirms kidnapping. 
-     Question to answer:  3 insiders of POK and supporting document/hypothesis. 
+18:10 jet arrives in arrived in Tethys Jr. of Sten Sanjorge of the bearing of 
+some hours ago with 6 other employees. 
+2021:  Government of Kronos confirms kidnapping.      Question to answer:  3 insiders of POK and supporting document/hypothesis. 
   Disappeared executives on 1/24/14: 
 Disappeared paid include:  
 President and PRESIDENT Sten Sanjorge Jr,  
@@ -44,7 +43,8 @@
  Vann, Edvard 
 Position: Perimeter Control (Security) 
 Date of Hire: 08/15/2013 
-Email: Edvard.Vann@gastech.com.kronos  Bodrogi, Loreto 
+Email: Edvard.Vann@gastech.com.kronos 
+ Bodrogi, Loreto 
 Position: Site Control (Security) 
 Date of Hire: 08/17/2013 
 Email: Loreto.Bodrogi@gastech.com.kronos 
