--- VASTBR1_revision_9_July_17_10:53_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_10_July_17_10:58_AM	2019-07-30 15:52:54.000000000 -0400
@@ -15,4 +15,6 @@
  Adan Morlun, truck driver, military experience. 
  Carla Forluniau Administrative Assistant | GASTech – Kronos | Abila, Kronos. Resume overlaps 
 with Onda, Inc. -2010.  But also works at GASTech starting 2005. 
-    
\ No newline at end of file
+ Minke Mies Security Officer | GASTech – Kronos | Abila, Kronos, recent employment at 
+GASTEC 2013. 
+  
\ No newline at end of file
