--- VASTBR1_revision_5_July_17_10:36_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_6_July_17_10:39_AM	2019-07-30 15:52:54.000000000 -0400
@@ -5,7 +5,10 @@
 President and PRESIDENT Sten Sanjorge Jr,  
 CFO Ingrid Barranco,  
 Campo-Corrente of ADA of CIO,  
-R 
-OUCOULEMENT Orhan de GAStech pianotent, and  
+ROUCOULEMENT Orhan de GAStech pianotent, and  
 environmental leader Willem Vasco-Feed.  
-     
\ No newline at end of file
+ he Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
+outside of the capital Abila. They have been protesting for clean water, 
+human rights, and against the corruption of the Government of Kronos and 
+GAStech International. 
+   
\ No newline at end of file
