--- VASTBR1_revision_40_July_18_9:08_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_41_July_18_9:11_AM	2019-07-30 15:52:54.000000000 -0400
@@ -19,6 +19,7 @@
  12:45 police cordon off GASTech building 
 1345 confirm missing GASTech employees. 
 1745 2 private jets leave airport.  First to Rome.  Second unknown. 
+18:10 jet arrives in Thetys 
 2021:  Government of Kronos confirms kidnapping. 
      Question to answer:  3 insiders of POK and supporting document/hypothesis. 
   Disappeared executives on 1/24/14: 
