--- VASTBR1_revision_21_July_17_11:29_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_22_July_18_8:32_AM	2019-07-30 15:52:54.000000000 -0400
@@ -1,5 +1,7 @@
 VastBR1@ncsu-last.net notes 
- Question to answer:  3 insiders of POK and supporting document/hypothesis. 
+  From news articles: 
+1/21/14:  revised kidnapping from 14 down to 10. 
+   Question to answer:  3 insiders of POK and supporting document/hypothesis. 
   Disappeared executives on 1/24/14: 
 Disappeared paid include:  
 President and PRESIDENT Sten Sanjorge Jr,  
@@ -20,9 +22,9 @@
  Loreto Bodrogi Security Officer, Abila, Kronos, 2013 started at GASTech. 
  Vann, Edvard 
 Position: Perimeter Control (Security) 
-Date of Hire: 08/15/2013 
-Email: Edvard.Vann@gastech.com.kronos 
- Bodrogi, Loreto Position: Site Control (Security) 
+Date of Hire: 08/15/2013 Email: Edvard.Vann@gastech.com.kronos 
+ Bodrogi, Loreto 
+Position: Site Control (Security) 
 Date of Hire: 08/17/2013 
 Email: Loreto.Bodrogi@gastech.com.kronos 
  Ferro, Inga 
@@ -39,8 +41,8 @@
    Recent hire after 2010 and citizen of Kronos: 
 Lais, Cornelia 
 Position: Assistant to Security Group Manager (Administration) 
-Date of Hire: 12/22/2011 
-Email: Cornelia.Lais@gastech.com.kronos Arpa, Emile 
+Date of Hire: 12/22/2011 Email: Cornelia.Lais@gastech.com.kronos 
+Arpa, Emile 
 Position: Janitor (Facilities) 
 Date of Hire: 03/18/2013 
 Email: Emile.Arpa@gastech.com.kronos 
