--- VASTBR1_revision_65_July_18_10:46_AM	2019-07-30 15:52:54.000000000 -0400
+++ VASTBR1_revision_66_July_18_10:48_AM	2019-07-30 15:52:54.000000000 -0400
@@ -27,6 +27,8 @@
 1840 experts from GASTech arrive in Abilia thought security experts 
 2021:  Government of Kronos confirms kidnapping. 
   2000:  Minister of Health Nespola concerned about cancer near Eldoris. 
+6/2001 Minister of health Nespola suddenly dies of heart attack.  Vincent 
+Kapelou takes post due to president Kapelou. 
 2001: Protectors of Kronos leader Henk Bodrogi 
 2001:  Nespola wants tax increase to help health issues.  Defeated. 
 1998:  Nespola investigates health issues near Eldoris. 
@@ -46,8 +48,7 @@
 President and PRESIDENT Sten Sanjorge Jr,  
 CFO Ingrid Barranco,  
 Campo-Corrente of ADA of CIO,  
-ROUCOULEMENT Orhan de GAStech pianotent, and  
-environmental leader Willem Vasco-Feed.  
+ROUCOULEMENT Orhan de GAStech pianotent, and  environmental leader Willem Vasco-Feed.  
  he Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
 outside of the capital Abila. They have been protesting for clean water, 
 human rights, and against the corruption of the Government of Kronos and 
@@ -71,7 +72,8 @@
 Position: Site Control (Security) 
 Date of Hire: 01/11/2013 
 Email: Inga.Ferro@gastech.com.kronos 
-   RE: Security procedures for January 20 VIP visit From: Inga.Ferro@gastech.com.kronos 
+   RE: Security procedures for January 20 VIP visit 
+From: Inga.Ferro@gastech.com.kronos 
 01/17/2014, 18:23:00 
 To: Hennie.Osvaldo@gastech.com.kronos, Loreto.Bodrogi@gastech.com.kronos, 
 Varja.Lagos@gastech.com.kronos, Inga.Ferro@gastech.com.kronos, 
