--- VASTFOB_revision_70_July_18_10:33_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_71_July_18_10:34_AM	2019-07-22 16:07:08.000000000 -0400
@@ -38,8 +38,8 @@
 ●Isia Vann- Security Officer, Armed Forces of Kronos ** 
 ●Edvard Vann- interrogated employee** 
       POK members 
-● 
-●Elian Karel- current2nd gen. leader of POK died 2009 
+●Silvia Marek- probable current leader of POK 
+●Elian Karel- 2nd gen. leader of POK died 2009 
 -Henk Bodrogi- previous leader (wife died from contaminated water) 
 ●Silvia Marek-founder of Save Our Wildlands (which later joined with 
 the POK)  
