--- VASTFOB_revision_6_July_17_10:07_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_7_July_17_10:08_AM	2019-07-22 16:07:08.000000000 -0400
@@ -4,7 +4,6 @@
 ●CIO Ada campo-Corrente  
 ●COO Orhan Strum  
 ●environment civil servant Willem Vasco-Pais van 
-● 
  - Kidnapped after a council meeting at hoofdkwartier GASTech. 
- - 03/26/2014 
+ - 03/26/2014 attack on  
    
\ No newline at end of file
