--- VASTFOB_revision_32_July_17_11:19_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_33_July_17_11:20_AM	2019-07-22 16:07:08.000000000 -0400
@@ -19,5 +19,5 @@
 ●Cornelia Lais- Executive Assistant, Armed Forces of Kronos 
 ●Isia Vann- Security Officer, Armed Forces of Kronos ** 
   POK members 
-●Mestache Vardshvna 
+●Mestache Vardshvna- spoke out during protest 
   
\ No newline at end of file
