--- VASTFOB_revision_22_July_17_10:37_AM	2019-07-22 16:07:08.000000000 -0400
+++ VASTFOB_revision_23_July_17_10:38_AM	2019-07-22 16:07:08.000000000 -0400
@@ -12,7 +12,8 @@
   People of interest 
 ●Adan Morlun- truck driver, Armed Forces of Kronos  
 ●Albina Hafon- truck driver, Armed Forces of Kronos  
-●Bertrand Ovan-Facilities Group Manager, retired major Armed Forces of Kronos 
-●Carla Forluniau-Armed Forces of Kronos  
-●Claudio Hawelon - 
+●Bertrand Ovan- Facilities Group Manager, retired major Armed Forces of Kronos 
+●Carla Forluniau- Armed Forces of Kronos  
+●Claudio Hawelon - truck driver, Armed Forces of Kronos 
+● 
    
\ No newline at end of file
