--- VASTJL1_revision_19_July_18_10:57_AM	2019-07-23 14:23:11.000000000 -0400
+++ VASTJL1_revision_20_July_18_10:57_AM	2019-07-23 14:23:11.000000000 -0400
@@ -25,10 +25,16 @@
 that a classified jet has arrived more soon today to the airport of Abila and that the jet has been 
 come to contact of in attending limousine 
 1915- Edvard Vann, interrogated and released from police, possibly due to similarities of name 
-with members of the POK. 1744 - Media reports are coming in that a number of GAStech employees were kidnapped. The 
+with members of the POK. 
+1744 - Media reports are coming in that a number of GAStech employees were kidnapped. The 
 identity of the passengers on the mystery jet that departed earlier today from the Abila Airport 
 remains unknown. 
 2232 - There are relationships that the civil employees of the airport have confirmed the arrival 
 of a classified jet from Tethys. Several passengers have left in an attending limousine. The 
 reporter they have followed the limousine, that he has guided directly to the centers of the police 
-of Abila. 
\ No newline at end of file
+of Abila. 
+1
+1
+2
+3
+4
\ No newline at end of file
