--- VASTJL1_revision_15_July_18_10:29_AM	2019-07-23 14:23:11.000000000 -0400
+++ VASTJL1_revision_16_July_18_10:32_AM	2019-07-23 14:23:11.000000000 -0400
@@ -19,8 +19,11 @@
 1430- The second plane also carried seven or eight 'of the businesses type', but seemed 'more 
 slackened much than the first group of passengers'. Danisliau indicated the second group of 
 passengers 'seemed to celebrate something.' The second plane left towards 14:30 around two 
-hours after the first plane. Danisliau confirmed that the second destination of the plane was 
+hours after the first plane. Danisliau confirmed that the second destination of the plane was  
 Rome, Italy.  
 1915- Edvard Vann, interrogated and released from police, possibly due to similarities of name 
 with members of the POK. 
- 
\ No newline at end of file
+2232 - There are relationships that the civil employees of the airport have confirmed the 
+arrival of a classified jet from Tethys. Several passengers have left in an attending 
+limousine. The reporter they have followed the limousine, that he has guided directly to 
+the centers of the police of Abila. 
\ No newline at end of file
