--- VASTJL1_revision_8_July_18_10:07_AM	2019-07-23 14:23:11.000000000 -0400
+++ VASTJL1_revision_9_July_18_10:10_AM	2019-07-23 14:23:11.000000000 -0400
@@ -12,4 +12,5 @@
 -Silvia Marek 
 -Mandor Vann 
  Reconstruct the Timeline of events on Jan. 20-21, 2014: 
-  
\ No newline at end of file
+ 1915- Edvard Vann, interrogated and released from police, possibly due to similarities of name 
+with members of the POK. 
\ No newline at end of file
