--- VASTJL1_revision_10_July_18_10:21_AM	2019-07-23 14:23:11.000000000 -0400
+++ VASTJL1_revision_11_July_18_10:22_AM	2019-07-23 14:23:11.000000000 -0400
@@ -14,12 +14,11 @@
  Reconstruct the Timeline of events on Jan. 20-21, 2014: 
  1915- Edvard Vann, interrogated and released from police, possibly due to similarities of name 
 with members of the POK. 
- According to Danisliau, the seven or eight passengers of the first plane 
-were 'businesses types' and seemed to be in a hurry. It was not exactly 
-clear where they were directed. The plane took off about local time of 
-Kronos 12:30. The second plane also carried seven or eight 'of the 
-businesses type', but seemed 'more slackened much than the first group of 
-passengers'. Danisliau indicated the second group of passengers 'seemed 
-to celebrate something.' The second plane left towards 14:30 around two 
-hours after the first plane. Danisliau confirmed that the second destination 
-of the plane was Rome, Italy.  
\ No newline at end of file
+ According to Danisliau, the seven or eight passengers of the first plane were 
+'businesses types' and seemed to be in a hurry. It was not exactly clear where they were 
+directed. The plane took off about local time of Kronos 12:30. The second plane also 
+carried seven or eight 'of the businesses type', but seemed 'more slackened much than 
+the first group of passengers'. Danisliau indicated the second group of passengers 
+'seemed to celebrate something.' The second plane left towards 14:30 around two 
+hours after the first plane. Danisliau confirmed that the second destination of the plane 
+was Rome, Italy.  
\ No newline at end of file
