--- VASTJL1_revision_16_July_18_10:32_AM	2019-07-23 14:23:11.000000000 -0400
+++ VASTJL1_revision_17_July_18_10:32_AM	2019-07-23 14:23:11.000000000 -0400
@@ -23,7 +23,7 @@
 Rome, Italy.  
 1915- Edvard Vann, interrogated and released from police, possibly due to similarities of name 
 with members of the POK. 
-2232 - There are relationships that the civil employees of the airport have confirmed the 
-arrival of a classified jet from Tethys. Several passengers have left in an attending 
-limousine. The reporter they have followed the limousine, that he has guided directly to 
-the centers of the police of Abila. 
\ No newline at end of file
+2232 - There are relationships that the civil employees of the airport have confirmed the arrival 
+of a classified jet from Tethys. Several passengers have left in an attending limousine. The 
+reporter they have followed the limousine, that he has guided directly to the centers of the police 
+of Abila. 
\ No newline at end of file
