--- VASTJL1_revision_17_July_18_10:32_AM	2019-07-23 14:23:11.000000000 -0400
+++ VASTJL1_revision_18_July_18_10:34_AM	2019-07-23 14:23:11.000000000 -0400
@@ -21,9 +21,11 @@
 passengers 'seemed to celebrate something.' The second plane left towards 14:30 around two 
 hours after the first plane. Danisliau confirmed that the second destination of the plane was  
 Rome, Italy.  
+1840 - There are relationships that the civil employees of the airport have freed the information 
+that a classified jet has arrived more soon today to the airport of Abila and that the jet has been 
+come to contact of in attending limousine 
 1915- Edvard Vann, interrogated and released from police, possibly due to similarities of name 
 with members of the POK. 
 2232 - There are relationships that the civil employees of the airport have confirmed the arrival 
-of a classified jet from Tethys. Several passengers have left in an attending limousine. The 
-reporter they have followed the limousine, that he has guided directly to the centers of the police 
+of a classified jet from Tethys. Several passengers have left in an attending limousine. The reporter they have followed the limousine, that he has guided directly to the centers of the police 
 of Abila. 
\ No newline at end of file
