--- VASTVD5_revision_20_July_17_11:05_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_21_July_17_11:22_AM	2019-07-30 16:05:39.000000000 -0400
@@ -9,6 +9,7 @@
 a.“Readers, the next post you read will be one of the most important you have ever 
 read, or indeed, will ever read. 21 Jan 2014” - prior to POK’s announcement of 
 their responsibility in the kidnapping 
+ Edvard vann, isia vann,  
  Employees of GASTech removed in Kronos / World Journal | 01/21/2014:  
 14 kidnapped (updated to 10 from Abila PD) 
 5 of 14 were execs 
