--- VASTVD5_revision_17_July_17_10:34_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_18_July_17_10:44_AM	2019-07-30 16:05:39.000000000 -0400
@@ -8,11 +8,11 @@
 environmental issues; active since 1998 
 a.“Readers, the next post you read will be one of the most important you have ever 
 read, or indeed, will ever read. 21 Jan 2014” - prior to POK’s announcement of 
-ent of their responsibility in the kidnapping 
+their responsibility in the kidnapping 
  Employees of GASTech removed in Kronos / World Journal | 01/21/2014:  
 14 kidnapped 
 5 of 14 were execs 
- PRESIDENT Sten Sanjorge Jr,  
+ PRESIDENT Sten Sanjorge Jr,  - not kidnapped 
 CFO Ingrid Barranco 
 Campo-Corrente of ADA of CIO,  
 ROUCOULEMENT Orhan de GAStech pianotent,  
