--- VASTVD5_revision_9_July_17_10:15_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_10_July_17_10:17_AM	2019-07-30 16:05:39.000000000 -0400
@@ -1,5 +1,7 @@
 Identify 3 potential associates of “Protectors of Kronos” (POK) and describe their involvement 
-  Employees of GASTech removed in Kronos / World Journal | 01/21/2014:  
+ 1.Silvia Marek - leader 
+2. 
+ Employees of GASTech removed in Kronos / World Journal | 01/21/2014:  
 14 kidnapped 
 5 of 14 were execs 
  PRESIDENT Sten Sanjorge Jr,  
