--- VASTVD5_revision_40_July_18_8:54_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_41_July_18_8:56_AM	2019-07-30 16:05:39.000000000 -0400
@@ -46,4 +46,10 @@
 a.Isia Vann 
 b.Loreto Bodrogi 
   Timelime of 01/20 - 01/21 
- 
\ No newline at end of file
+ 1853 UPDATE - Speculation that the 2:00 PM arrival of Kronos Government 
+official was in fact the arrival of a senior security executive and other 
+officials from GAStech's Tethys office including President and CEO Sten 
+Sanjorge, Jr. This speculation is based on information concerning the 
+arrival, refueling and immediate departure of Sanjorge Jr.'s private jet from 
+the Tethys airport. 
+  
\ No newline at end of file
