--- VASTVD5_revision_56_July_18_9:02_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_57_July_18_9:03_AM	2019-07-30 16:05:39.000000000 -0400
@@ -46,7 +46,7 @@
 b.Loreto Bodrogi 
                     Timelime of 01/20 - 01/21 
 01/20 
-1853 UPDATE - Speculation that the 2:00 PM arrival of Kronos Government official was in fact 
+ 1853 UPDATE - Speculation that the 2:00 PM arrival of Kronos Government official was in fact 
 the arrival of a senior security executive and other officials from GAStech's Tethys office 
 including President and CEO Sten Sanjorge, Jr. This speculation is based on information 
 concerning the arrival, refueling and immediate departure of Sanjorge Jr.'s private jet from the 
@@ -90,23 +90,33 @@
 scene though the crime has not be revealed. The police response seems out of proportion for a 
 false fire alarm. Police officials have declined to comment at this time. 
  1255 UPDATE - Calls to Abila's hospitals and the City Morgue have netted no new information. 
- 1035 UPDATE - The Abila Fire Department arrived on the scene roughly 10 minutes ago and 
-began searching the building. No smoke is visible, raising concerns of an actual bomb threat. 
- 1345 UPDATE - Confirmed report of missing GAStech employees from GAStech administrative 
+  1345 UPDATE - Confirmed report of missing GAStech employees from GAStech administrative 
 assistant in Tethys headquarters; GAStech - Kronos still not responding. The mystery of whom 
 and how many are missing remains. 
  1748 UPDATE - Two private jets left Abila Airport today with fourteen to sixteen passengers 
 between the two. The destination of one of the planes has been confirmed as Rome, Italy. The 
 other plane's destination has yet to be confirmed. 
+ 1853 UPDATE - Speculation that the 2:00 PM arrival of Kronos Government official was in fact 
+the arrival of a senior security executive and other officials from GAStech's Tethys office 
+including President and CEO Sten Sanjorge, Jr. This speculation is based on information 
+concerning the arrival, refueling and immediate departure of Sanjorge Jr.'s private jet from the 
+Tethys airport. 
  1915 UPDATE - GAStech employee Edvard Vann was 'allowed to leave' after nearly six hours of 
 interrogation by police, government, and GAStech officials. Vann was questioned and released 
 about fifteen minutes ago and instructed not to leave town. Vann was visibly angry about the 
 confusion, which apparently stemmed from the similarity of his name and known members of 
-the Protectors of Kronos. Vann said he was 'instructed not to communicate with the press, but I 
-feel I have to stress my innocence. I have nothing to do whatsoever with any members of the 
+the Protectors of Kronos. Vann said he was 'instructed not to communicate with the press, but I  feel I have to stress my innocence. I have nothing to do whatsoever with any members of the 
 terrorist group POK.' 
+ 1947 UPDATE - From the Abila Police news conference: 'There are approximately fourteen 
+individuals unaccounted for among the GAStech staff. It has not yet been determined if all of 
+these individuals are in fact missing.'Though it was confirmed that over a dozen individuals were 
+unaccounted for, the Abila Police said that that does not mean they are necessarily missing. Nor 
+does it mean that they have been abducted. At this point the case is under investigation and more 
+information will be released as facts are learned. The Abila Police stressed that anyone seeing 
+any unusual activity should immediately report it. 
  2021 UPDATE - A Kronos Government official who asked not to be identified but was present at 
-the government reception has gone on the record stating, 'Investigators are working around the  clock to ensure that the kidnapped GAStech employees are found and that the perpetrators are 
+the government reception has gone on the record stating, 'Investigators are working around the 
+clock to ensure that the kidnapped GAStech employees are found and that the perpetrators are 
 brought to justice.'This is the first confirmation that today's events surrounding GAStech - 
 Kronos are in fact a kidnapping. With fourteen individuals still unaccounted for, at this point, the 
 assumption is that all are the victims of a kidnapping. No group has come forward yet to claim 
