--- VASTVD5_revision_44_July_18_8:57_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_45_July_18_8:57_AM	2019-07-30 16:05:39.000000000 -0400
@@ -76,4 +76,9 @@
 everyone was being interviewed about the events of the day, about the 
 meeting, and the recent IPO. In addition to the GAStech employees, the 
 caterers for the breakfast meeting, who were dressed all in black, have 
-been released. 
\ No newline at end of file
+been released. 
+ 1545 UPDATE - These 'shadowy' figures are at the center of what appears 
+to be a kidnapping incident with the victims being the missing GAStech 
+leadership. At this time we can only speculate on the number of people 
+involved. In the confusion from the building evacuation an accurate count 
+of GAStech employees is nearly impossible. 
\ No newline at end of file
