--- VASTVD5_revision_64_July_18_9:06_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_65_July_18_9:06_AM	2019-07-30 16:05:39.000000000 -0400
@@ -45,8 +45,8 @@
 a.Isia Vann 
 b.Loreto Bodrogi 
                     Timelime of 01/20 - 01/21 
-01/20 
-1035 UPDATE - The Abila Fire Department arrived on the scene roughly 10 minutes ago and 
+ 01/20 
+ 1035 UPDATE - The Abila Fire Department arrived on the scene roughly 10 minutes ago and 
 began searching the building. No smoke is visible, raising concerns of an actual bomb threat. 
  1100 UPDATE - Still no indication from the AFD about what if anything has happened; 
 however, there are reports that the GAStech - Kronos employees gathered outside the building 
@@ -72,10 +72,9 @@
 assistant in Tethys headquarters; GAStech - Kronos still not responding. The mystery of whom 
 and how many are missing remains. 
  1545 UPDATE - These 'shadowy' figures are at the center of what appears to be a kidnapping 
-incident with the victims being the missing GAStech leadership. At this time we can only 
-speculate on the number of people involved. In the confusion from the building evacuation an 
+incident with the victims being the missing GAStech leadership. At this time we can only  speculate on the number of people involved. In the confusion from the building evacuation an 
 accurate count of GAStech employees is nearly impossible. 
-  1721 UPDATE - GAStech employees have continued to trickle out over the past two hours. A 
+ 1721 UPDATE - GAStech employees have continued to trickle out over the past two hours. A 
 few have agreed to communicate with members of the press indicating that they were instructed 
 not to say anything. They also declined to be named. Generally, the GAStech employees said that 
 everyone was being interviewed about the events of the day, about the meeting, and the recent 
@@ -103,10 +102,10 @@
 does it mean that they have been abducted. At this point the case is under investigation and more 
 information will be released as facts are learned. The Abila Police stressed that anyone seeing 
 any unusual activity should immediately report it. 
-  2021 UPDATE - A Kronos Government official who asked not to be identified but was present at 
+ 2021 UPDATE - A Kronos Government official who asked not to be identified but was present at 
 the government reception has gone on the record stating, 'Investigators are working around the 
-clock to ensure that the kidnapped GAStech employees are found and that the perpetrators are 
-brought to justice.'This is the first confirmation that today's events surrounding GAStech - 
-Kronos are in fact a kidnapping. With fourteen individuals still unaccounted for, at this point, the  assumption is that all are the victims of a kidnapping. No group has come forward yet to claim 
+clock to ensure that the kidnapped GAStech employees are found and that the perpetrators are  brought to justice.'This is the first confirmation that today's events surrounding GAStech - 
+Kronos are in fact a kidnapping. With fourteen individuals still unaccounted for, at this point, the 
+assumption is that all are the victims of a kidnapping. No group has come forward yet to claim 
 responsibility nor have any demands been made. 
  
\ No newline at end of file
