--- VASTN5Z_revision_11_July_18_10:05_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_12_July_18_10:06_AM	2019-07-19 13:39:19.000000000 -0400
@@ -11,6 +11,7 @@
 Carmine Osvaldo. Osvaldo, who had served in the military with Bodrogi, was 
 a member of the board for the Elodis Chamber of Commerce and who used his business 
 experience to form relationships with the Kronosian government and outside agencies. 
+Jeroen Karel, father of current POK leader Elian Karel, to organize recruitment and personnel. 
 Today - POK has grown under the charismatic leadership of Elian Karel to loosely organized 
 group with an estimated membership of 200-300 people. 
   Timeline on jan 20-21 
