--- VASTN5Z_revision_9_July_18_10:03_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_10_July_18_10:04_AM	2019-07-19 13:39:19.000000000 -0400
@@ -5,7 +5,10 @@
 Org over time 
 (1997ish) - The grassroots organization coalesced under the leadership of Henk Bodrogi, a 
 floodplain farmer 
-Today the - POK has grown under the charismatic leadership of Elian Karel to loosely organized 
+Carmine Osvaldo. Osvaldo, who had served in the military with Bodrogi, was 
+a member of the board for the Elodis Chamber of Commerce and who used his business 
+experience to form relationships with the Kronosian government and outside agencies. 
+Today - POK has grown under the charismatic leadership of Elian Karel to loosely organized 
 group with an estimated membership of 200-300 people. 
   Timeline on jan 20-21 
  explanation 
