--- VASTN5Z_revision_3_July_18_9:59_AM	2019-07-19 13:39:19.000000000 -0400
+++ VASTN5Z_revision_4_July_18_9:59_AM	2019-07-19 13:39:19.000000000 -0400
@@ -1,4 +1,8 @@
-   POK 5 year report notes: 
+Leaders 
+Network 
+Connections to gastech 
+Org over time 
+  POK 5 year report notes: 
 Today the POK has grown under the charismatic leadership of Elian Karel to loosely organized 
 group with an estimated membership of 200-300 people. 
  
\ No newline at end of file
