--- VASTWK2_revision_26_July_18_10:21_AM	2019-07-23 15:58:43.000000000 -0400
+++ VASTWK2_revision_27_July_18_10:21_AM	2019-07-23 15:58:43.000000000 -0400
@@ -26,4 +26,9 @@
 d.Security and environmental personnel  
 2.F 
 3.My accessesment is that the missing members will be returned with 
-the payment of a ransom  
\ No newline at end of file
+the payment of a ransom  
+1
+1
+2
+3
+4
\ No newline at end of file
