--- VASTL6X_revision_56_June_20_8:42_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_57_June_20_8:44_AM	2019-07-08 08:44:40.000000000 -0400
@@ -52,6 +52,7 @@
 Isia Vann - Security - Employee since 2007 - Former military (general discharge) 
 Edvard Vann - Security - Employee since 2013 - Former military (honorable discharge) 
  Henk Mies and Sanjorge emails re: files? 
+ Company officers discussing vacation spots shortly before the event 
                                       People from Elodis, which is a town affected by GASTech’s operations 
 Pondo Reese, editor of Elodis Weekly Journal 
 Henk Bodrogi, farmer - founder of POK 
