--- VASTL6X_revision_27_June_19_10:42_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_28_June_19_10:44_AM	2019-07-08 08:44:40.000000000 -0400
@@ -23,6 +23,7 @@
 COO Orhan Strum 
 environment civil servant Willem Vasco-Pais van GAStech 
  20 million (unknown currency) ransom demanded 
+ Jocelyn Reynolds - Wellness for All, water org that tested the water in Elodis 
         POK 
 founded in 1998 in Elodis, a rural township outside of the capital Abila. They have been 
 protesting for clean water, human rights, and against the corruption of the Government of 
@@ -41,5 +42,6 @@
 CHILDREN AND RUINED OUR WAY OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE 
 FORM OF $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE 
 PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 
-                      People from Elodis, which is a town affected by GASTech’s operations Pondo Reese, editor of Elodis Weekly Journal 
+                      People from Elodis, which is a town affected by GASTech’s operations 
+Pondo Reese, editor of Elodis Weekly Journal 
 Henk Pondrogi, farmer 
\ No newline at end of file
