--- VASTL6X_revision_43_June_19_11:02_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_44_June_19_11:03_AM	2019-07-08 08:44:40.000000000 -0400
@@ -49,6 +49,7 @@
 PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 
              GASTech Isia Vann - Security - Employee since 2007 - Former military (general discharge) 
 Edvard Vann - Security - Employee since 2013 - Former military (honorable discharge) 
+Loreto Bodrogi - Security - Employee since 2013 - Former military (honorable discharge) 
                       People from Elodis, which is a town affected by GASTech’s operations 
 Pondo Reese, editor of Elodis Weekly Journal 
 Henk Pondrogi, farmer - founder of POK 
