--- VASTL6X_revision_25_June_19_10:41_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_26_June_19_10:41_AM	2019-07-08 08:44:40.000000000 -0400
@@ -41,4 +41,5 @@
 CHILDREN AND RUINED OUR WAY OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE 
 FORM OF $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE 
 PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 
-                      People from Elodis, which is a town affected by GASTech’s operations Pondo Reese, editor of local paper 
\ No newline at end of file
+                      People from Elodis, which is a town affected by GASTech’s operations Pondo Reese, editor of local paperElodis Weekly Journal 
+ 
\ No newline at end of file
