--- VASTL6X_revision_29_June_19_10:48_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_30_June_19_10:49_AM	2019-07-08 08:44:40.000000000 -0400
@@ -45,4 +45,5 @@
                       People from Elodis, which is a town affected by GASTech’s operations 
 Pondo Reese, editor of Elodis Weekly Journal 
 Henk Pondrogi, farmer 
-Juliana Vann, girl who dies of leukemia due to benzene poisoning 
\ No newline at end of file
+Juliana Vann, girl who dies of leukemia due to benzene poisoningl 
+  
\ No newline at end of file
