--- VASTL6X_revision_39_June_19_10:57_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_40_June_19_11:00_AM	2019-07-08 08:44:40.000000000 -0400
@@ -25,7 +25,7 @@
  20 million (unknown currency) ransom demanded 
  Jocelyn Reynolds - Wellness for All, water org that tested the water in Elodis 
  There are people named Vann and Bodrogi who work in security at GASTech 
-      POK 
+       POK 
 founded in 1998 in Elodis, a rural township outside of the capital Abila. They have been 
 protesting for clean water, human rights, and against the corruption of the Government of 
 Kronos and GAStech International. 
@@ -47,7 +47,7 @@
 CHILDREN AND RUINED OUR WAY OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE 
 FORM OF $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE 
 PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 
-                      People from Elodis, which is a town affected by GASTech’s operations 
+                                    People from Elodis, which is a town affected by GASTech’s operations 
 Pondo Reese, editor of Elodis Weekly Journal 
 Henk Pondrogi, farmer - founder of POK 
 Juliana Vann, girl who dies of leukemia due to benzene poisoningl 
