--- VASTL6X_revision_9_June_19_10:03_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_10_June_19_10:04_AM	2019-07-08 08:44:40.000000000 -0400
@@ -19,4 +19,5 @@
 CFO Ingrid Barranco 
 CIO Ada campo-Corrente 
 COO Orhan Strum 
-environment civil servant Willem Vasco-Pais van GAStech 
\ No newline at end of file
+environment civil servant Willem Vasco-Pais van GAStech 
+ 20 million  
\ No newline at end of file
