--- VASTL6X_revision_45_June_19_11:07_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_46_June_19_11:12_AM	2019-07-08 08:44:40.000000000 -0400
@@ -29,7 +29,9 @@
 founded in 1998 in Elodis, a rural township outside of the capital Abila. They have been 
 protesting for clean water, human rights, and against the corruption of the Government of 
 Kronos and GAStech International. 
- Elian Karel - leader of POK in 2009 - dies in jail 19JUN09 -  
+ Leadership 
+Henk Bodrogi (founder) - Elian Karel - Silvia Marek 
+  Elian Karel - leader of POK in 2009 - dies in jail 19JUN09 -  
 Michale Kraft - Karel’s lawyer 
  Kriste Brontes - supporter? 
  Silvia Marek - leader of POK in 2012 
@@ -47,11 +49,12 @@
 CHILDREN AND RUINED OUR WAY OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE 
 FORM OF $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE 
 PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 
-             GASTech Isia Vann - Security - Employee since 2007 - Former military (general discharge) 
+             GASTech 
+Isia Vann - Security - Employee since 2007 - Former military (general discharge) 
 Edvard Vann - Security - Employee since 2013 - Former military (honorable discharge) 
 Loreto Bodrogi - Security - Employee since 2013 - Former military (honorable discharge) 
                       People from Elodis, which is a town affected by GASTech’s operations 
 Pondo Reese, editor of Elodis Weekly Journal 
-Henk PonBodrogi, farmer - founder of POK 
+Henk Bodrogi, farmer - founder of POK 
 Juliana Vann, girl who dies of leukemia due to benzene poisoningl 
   
\ No newline at end of file
