--- VASTL6X_revision_55_June_20_8:20_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_56_June_20_8:42_AM	2019-07-08 08:44:40.000000000 -0400
@@ -24,8 +24,7 @@
 environment civil servant Willem Vasco-Pais van GAStech 
  20 million (unknown currency) ransom demanded 
  Jocelyn Reynolds - Wellness for All, water org that tested the water in Elodis 
- There are people named Vann and Bodrogi who work in security at GASTechCo-founder Fluss 
-sympathetic to pollution issue 
+ Co-founder Fluss sympathetic to pollution issue 
     POK 
 founded in 1998 in Elodis, a rural township outside of the capital Abila. They have been 
 protesting for clean water, human rights, and against the corruption of the Government of 
@@ -52,7 +51,8 @@
 PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 GASTech 
 Isia Vann - Security - Employee since 2007 - Former military (general discharge) 
 Edvard Vann - Security - Employee since 2013 - Former military (honorable discharge) 
-                                        People from Elodis, which is a town affected by GASTech’s operations 
+ Henk Mies and Sanjorge emails re: files? 
+                                      People from Elodis, which is a town affected by GASTech’s operations 
 Pondo Reese, editor of Elodis Weekly Journal 
 Henk Bodrogi, farmer - founder of POK 
 Juliana Vann, girl who dies of leukemia due to benzene poisoning 
