--- VASTL6X_revision_36_June_19_10:55_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_37_June_19_10:56_AM	2019-07-08 08:44:40.000000000 -0400
@@ -32,6 +32,7 @@
  Elian Karel - leader of POK in 2009 - dies in jail 19JUN09 -  
 Michale Kraft - Karel’s lawyer 
  Kriste Brontes - supporter? 
+ Silvia Marek - leader of POK in  
   From VOICES blog 
 1200 - TO THE PEOPLE OF KRONOS: NO LONGER WILL WE ACQUIESCE TO THE 
 UNFETTERED KLEPTOCRACY THAT HAS BECOME OUR GOVERNMENT AND TO THE 
