--- VASTL6X_revision_0_June_19_9:38_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_1_June_19_9:41_AM	2019-07-08 08:44:40.000000000 -0400
@@ -1 +1,13 @@
- 
\ No newline at end of file
+Kronos (Abila/Elodis) 
+ 3 major industries 
+Agriculture 
+logging/forestry 
+Natural Gas 
+ Lots of poverty 
+ Gastech in island of Tethys to NE 
+Has strong relationship with Kronos govt 
+GAStech has bad environmental rap on Kronos 
+ Jan 2014 (20 yrs after gastech started on kronos) gastech has an IPO 
+Having a big party 
+Bunch of employees go missing Protectors of Kronos (POK) are implicated 
+POK hasn’t claimed responsibilty 
\ No newline at end of file
