--- VASTL6X_revision_53_June_19_11:27_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_54_June_19_11:48_AM	2019-07-08 08:44:40.000000000 -0400
@@ -48,11 +48,11 @@
 APPREHENDED THE ELITISTS WHO HAVE EXPLOITED OUR RESOURCES, KILLED OUR 
 CHILDREN AND RUINED OUR WAY OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE 
 FORM OF $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE 
-PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014  GASTech 
+PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 20144  GASTech 
 Isia Vann - Security - Employee since 2007 - Former military (general discharge) 
 Edvard Vann - Security - Employee since 2013 - Former military (honorable discharge) 
-                                       People from Elodis, which is a town affected by GASTech’s operations 
+                                        People from Elodis, which is a town affected by GASTech’s operations 
 Pondo Reese, editor of Elodis Weekly Journal 
 Henk Bodrogi, farmer - founder of POK 
-Juliana Vann, girl who dies of leukemia due to benzene poisoningl 
+Juliana Vann, girl who dies of leukemia due to benzene poisoning 
   
\ No newline at end of file
