--- VASTL6X_revision_22_June_19_10:33_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_23_June_19_10:40_AM	2019-07-08 08:44:40.000000000 -0400
@@ -14,7 +14,7 @@
  Ex 1 
 Identify at least three people who are members of the POK 
 Provide information about them 
- Kidnapping occurs 21 0JAN 14 
+ Kidnapping occurs 20JAN14 
  14 people missing (revised to 10) including: 
   Initially believed missing but weren’t due to being en route to 20 year partnership celebration 
 President and CEO Sten Sanjorge Jr 
@@ -40,4 +40,5 @@
 APPREHENDED THE ELITISTS WHO HAVE EXPLOITED OUR RESOURCES, KILLED OUR 
 CHILDREN AND RUINED OUR WAY OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE 
 FORM OF $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE 
-PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 
\ No newline at end of file
+PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 
+                      People from 
\ No newline at end of file
