--- VASTKD0_revision_5_June_19_9:44_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_6_June_19_9:58_AM	2019-06-27 13:14:16.000000000 -0400
@@ -1,5 +1,6 @@
--Kronos - Found natural gas 
+VastKD0@ncsu-las.net 
+ -Kronos - Found natural gas 
  -Gas Tech - company from Tethys 
   -Jan 2014 - Leaders of GAS Tech went public (Employees went missing during party) 
  - POK is the suspect of the disappearing people 
- TASK: locate , figure out what happened,  to missing employees  
\ No newline at end of file
+ TASK: locate , figure out what happened to missing employees  
\ No newline at end of file
