--- VASTKD0_revision_108_June_20_9:25_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_109_June_20_9:26_AM	2019-06-27 13:14:16.000000000 -0400
@@ -200,8 +200,9 @@
 Armed Forces of Kronos 
 Last Name Associated with previous POK Leader 
 -Vann, Isia (Current employee) - shares name of Julian Vann(Julian Vann died in 1998 of 
-the cancer caused by environmental toxins in its potable water) 
-- Osvaldo, Hennie (Current employee) - shares name of Carmine Osvaldo 
+the cancer caused by environmental toxins in its potable water) Daily Pegasus | 
+03/26/2014 
+ - Osvaldo, Hennie (Current employee) - shares name of Carmine Osvaldo 
 History  Document:  
 Bodrogi lacked experience in dealing with the Kronosian government and for which he turned to 
 his lifelong friend, Carmine Osvaldo. Osvaldo, who had served in the military with Bodrogi, was a 
