--- VASTKD0_revision_34_June_19_11:09_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_35_June_19_11:10_AM	2019-06-27 13:14:16.000000000 -0400
@@ -30,9 +30,10 @@
 Ingrid BARRANCO (CFO) 
 Orhan STRUM (COO) 
 Ada CAMPO-CORRENTE (CIO) 
-  Event: 
+ Event: 
 Happened during a transition to reception 
- POK Information: 
+Ingrid Brranco/ Ada Campo/  
+POK Information: 
 -The Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
 outside of the capital Abila. They have been protesting for clean water, 
 human rights, and against the corruption of the Government of Kronos and 
