--- VASTKD0_revision_7_June_19_9:59_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_8_June_19_10:03_AM	2019-06-27 13:14:16.000000000 -0400
@@ -3,4 +3,6 @@
  -Gas Tech - company from Tethys 
   -Jan 2014 - Leaders of GAS Tech went public (Employees went missing during party) 
  - Protectors Of Kronos is the suspect of the disappearing people 
- TASK: locate , figure out what happened to missing employees  
\ No newline at end of file
+ TASK: locate , figure out what happened to missing employees  
+  What we know from Articles: 
+ 
\ No newline at end of file
