--- VASTKD0_revision_58_June_19_11:42_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_59_June_19_11:42_AM	2019-06-27 13:14:16.000000000 -0400
@@ -162,4 +162,15 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2006 
 Discharge Type: General Discharge 
+ Arpa, Emile 
+Position: Janitor (Facilities) 
+Date of Hire: 03/18/2013 
+Email: Emile.Arpa@gastech.com.kronos 
+ Sex: Male DOB: 10/01/1992 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2012 
+Discharge Type: General Discharge 
  
\ No newline at end of file
