--- VASTKD0_revision_14_June_19_10:09_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_15_June_19_10:10_AM	2019-06-27 13:14:16.000000000 -0400
@@ -14,7 +14,8 @@
 -Sanjorge and others disappeared after a Council meeting at 
 hoofdkwartier GASTech. A fire alarm descended, and execs were 
 discovered missing shortly after that.  
-  Police informed reporters that four of the GAStech employees whose 
+ International News | 01/21/2014 
+Police informed reporters that four of the GAStech employees whose 
 whereabouts were unconfirmed yesterday were found overnight, leaving ten 
 people who are presumed missing. Police spokesman Adrien Carman said 
 Tethyn federal law enforcement agencies are assisting in the investigation. 
