--- VASTKD0_revision_44_June_19_11:24_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_45_June_19_11:29_AM	2019-06-27 13:14:16.000000000 -0400
@@ -42,4 +42,5 @@
 outside of the capital Abila. They have been protesting for clean water, 
 human rights, and against the corruption of the Government of Kronos and 
 GAStech International. 
-  - Edvard Vann- shares name of POK member 
\ No newline at end of file
+  - Edvard Vann- shares name of POK member 
+-Stenig Fusil 
\ No newline at end of file
