--- VASTKD0_revision_37_June_19_11:11_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_38_June_19_11:12_AM	2019-06-27 13:14:16.000000000 -0400
@@ -34,7 +34,7 @@
 Happened during a transition to reception 
 Ingrid Brranco/ Ada Campo/ Sten Sanjorge - has email contact 1/10/2014 
 about reception  
-POK Information: 
+ POK Information: 
 -The Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
 outside of the capital Abila. They have been protesting for clean water, 
 human rights, and against the corruption of the Government of Kronos and 
