--- VASTKD0_revision_116_June_20_10:15_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_117_June_20_10:15_AM	2019-06-27 13:14:16.000000000 -0400
@@ -212,9 +212,10 @@
 experience to form relationships with the Kronosian government and outside agencies. 
  ----------------------------------------------------------------------- 
 OUZERI & ELIAN- Pub or drinking establishment across from GasTech 
-(corner of Ermou Street and Taxiarchon Avenue)  
+(corner of Ermou Street and Taxiarchon Avenue)  Shares name of Elian 
+Karel 
+Elian Karel 
   You would need inside knowledge  
 Transportation 
-Area Knowledge 
-Military training  
+Area Knowledge Military training  
  
\ No newline at end of file
