--- VASTKD0_revision_38_June_19_11:12_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_39_June_19_11:12_AM	2019-06-27 13:14:16.000000000 -0400
@@ -32,7 +32,7 @@
 Ada CAMPO-CORRENTE (CIO) 
  Event: 
 Happened during a transition to reception 
-Ingrid Brranco/ Ada Campo/ Sten Sanjorge - has email contact 1/10/2014 
+ Ingrid Brranco/ Ada Campo/ Sten Sanjorge - has email contact 1/10/2014 
 about reception  
  POK Information: 
 -The Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
