--- VASTKD0_revision_61_June_19_11:43_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_62_June_19_11:43_AM	2019-06-27 13:14:16.000000000 -0400
@@ -197,4 +197,16 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2011 
 Discharge Type: General Discharge 
+ Forluniau, Carla 
+Position: Assistant to IT Group Manager (Administration) 
+Date of Hire: 03/07/2005 
+Email: Carla.Forluniau@gastech.com.kronos 
+ Sex: Female 
+DOB: 06/02/1981 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2001 
+Discharge Type: General Discharge 
  
\ No newline at end of file
