--- VASTKD0_revision_56_June_19_11:42_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_57_June_19_11:42_AM	2019-06-27 13:14:16.000000000 -0400
@@ -139,4 +139,15 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2007 
 Discharge Type: General Discharge 
+ Coginian, Dante 
+Position: Lead Janitor (Facilities) 
+Date of Hire: 01/07/1995 
+Email: Dante.Coginian@gastech.com.kronos  Sex: Male 
+DOB: 12/01/1971 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/1992 
+Discharge Type: General Discharge 
  
\ No newline at end of file
