--- VASTKD0_revision_35_June_19_11:10_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_36_June_19_11:10_AM	2019-06-27 13:14:16.000000000 -0400
@@ -32,7 +32,7 @@
 Ada CAMPO-CORRENTE (CIO) 
  Event: 
 Happened during a transition to reception 
-Ingrid Brranco/ Ada Campo/  
+Ingrid Brranco/ Ada Campo/ Sten Sanjore 
 POK Information: 
 -The Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
 outside of the capital Abila. They have been protesting for clean water, 
