--- VASTKD0_revision_2_June_19_9:42_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_3_June_19_9:43_AM	2019-06-27 13:14:16.000000000 -0400
@@ -1,5 +1,6 @@
 Kronos - Found natural gas 
-Gas Tech - company from Tethys  
-Jan 2014 - Leaders of GAS Tech went public (Employees went missing durring party) 
-POK is the susspect of the dissapearing people 
-locate , figureout what happen,  
\ No newline at end of file
+ Gas Tech - company from Tethys  
+-Jan 2014 - Leaders of GAS Tech went public (Employees went missing durring party) 
+ - POK is the susspectsuspect of the dissapearingdisappearing people 
+l 
+ TASK: locate , figureoutfigure out what happenhappened,  
\ No newline at end of file
