--- VASTKD0_revision_112_June_20_9:38_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_113_June_20_9:40_AM	2019-06-27 13:14:16.000000000 -0400
@@ -189,7 +189,7 @@
 Email: Carla.Forluniau@gastech.com.kronos 
  Sex: Female 
 DOB: 06/02/1981 
- Citizenship: Kronosvann 
+ Citizenship: Kronos 
 Passport Issued: 01/01/2000 
 Passport Expires: 01/01/2000  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2001 
@@ -199,9 +199,9 @@
 Passport Issue Dates All The Same 
 Armed Forces of Kronos 
 Last Name Associated with previous POK Leader 
--Vann, Isia (Current employee) - shares name of Julian Vann(Julian Vann died in 1998 of 
-the cancer caused by environmental toxins in its potable water) Daily Pegasus | 
-03/26/2014 
+-Vann, Isia (Current employee) - shares name of Julian Vann(Julian Vann died in 1998 
+of the cancer caused by environmental toxins in its potable water) Daily 
+Pegasus | 03/26/2014 
  - Osvaldo, Hennie (Current employee) - shares name of Carmine Osvaldo 
 History  Document (A Psycorps Analysis Brief By Fredrick N. Wagner and Westley B. Andrews / 
 January 2009): 
