--- VASTKD0_revision_30_June_19_10:52_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_31_June_19_10:52_AM	2019-06-27 13:14:16.000000000 -0400
@@ -33,4 +33,4 @@
  POK Information: 
 -The Protectors of Kronos, was founded in 1998 in Elodis, a rural township 
 outside of the capital Abila.  
-Edvard Vann- shares name of POK member 
\ No newline at end of file
+- Edvard Vann- shares name of POK member 
\ No newline at end of file
