--- VASTKD0_revision_113_June_20_9:40_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_114_June_20_10:11_AM	2019-06-27 13:14:16.000000000 -0400
@@ -209,7 +209,8 @@
 his lifelong friend, Carmine Osvaldo. Osvaldo, who had served in the military with Bodrogi, was a 
 member of the board for the Elodis Chamber of Commerce and who used his business 
 experience to form relationships with the Kronosian government and outside agencies. 
-   You would need inside knowledge  
+OUZERI & ELIAN- Pub or drinking establishment across from GasTech 
+  You would need inside knowledge  
 Transportation 
 Area Knowledge 
 Military training  
