--- VASTKD0_revision_10_June_19_10:04_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_11_June_19_10:06_AM	2019-06-27 13:14:16.000000000 -0400
@@ -5,8 +5,11 @@
  - Protectors Of Kronos is the suspect of the disappearing people 
  TASK: locate , figure out what happened to missing employees  
   What we know from Articles: 
-Fourteen employees, among which executive civil servants perhaps five, 
-feared have been kidnapped yesterday by the guards of Kronos. 
-Communicated missing includes: President and CEO Sten Sanjorge Jr, CFO 
-Ingrid Barranco, CIO Ada campo-Corrente, COO Orhan Strum, and 
-environment civil servant Willem Vasco-Pais van GAStech.  
\ No newline at end of file
+-Fourteen employees, among which executive civil servants perhaps 
+five, feared have been kidnapped yesterday by the guards of Kronos. 
+Communicated missing includes: President and CEO Sten Sanjorge 
+Jr, CFO Ingrid Barranco, CIO Ada campo-Corrente, COO Orhan Strum, 
+and environment civil servant Willem Vasco-Pais van GAStech.  
+-Sanjorge and others disappeared after a Council meeting at 
+hoofdkwartier GASTech. A fire alarm descended, and execs were 
+discovered missing shortly after that.  
\ No newline at end of file
