--- VASTKD0_revision_20_June_19_10:41_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_21_June_19_10:41_AM	2019-06-27 13:14:16.000000000 -0400
@@ -26,5 +26,6 @@
 kidnapping of GAStech employees demanding a $20 million ransom. 
  List of missing Employees: 
 SANJORGE, JR.-CEO 
- POK Information: 
+VASCO-PAIS-(ESA) 
+  POK Information: 
 Edvard Vann- shares name of POK member 
\ No newline at end of file
