--- VASTKD0_revision_48_June_19_11:38_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_49_June_19_11:39_AM	2019-06-27 13:14:16.000000000 -0400
@@ -43,5 +43,17 @@
 human rights, and against the corruption of the Government of Kronos and 
 GAStech International. 
  - Edvard Vann- shares name of POK member 
- EployeesEmployees with the EXACT SAME “General Discharge” 
- 
\ No newline at end of file
+      Employees with the EXACT SAME “General Discharge”: 
+Ferro, Inga 
+Position: Site Control (Security) 
+Date of Hire: 01/11/2013 
+Email: Inga.Ferro@gastech.com.kronos 
+ Sex: Female 
+DOB: 06/17/1989 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2012 
+Discharge Type: General Discharge 
+  
\ No newline at end of file
