--- VASTKD0_revision_50_June_19_11:40_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_51_June_19_11:40_AM	2019-06-27 13:14:16.000000000 -0400
@@ -68,4 +68,16 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2007 
 Discharge Type: General Discharge 
+ Osvaldo, Hennie 
+Position: Perimeter Control (Security) 
+Date of Hire: 06/07/2011 
+Email: Hennie.Osvaldo@gastech.com.kronos 
+ Sex: Male 
+DOB: 05/31/1988 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2010 
+Discharge Type: General Discharge 
   
\ No newline at end of file
