--- VASTKD0_revision_126_June_20_10:50_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_127_June_20_10:50_AM	2019-06-27 13:14:16.000000000 -0400
@@ -78,7 +78,7 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2010 
 Discharge Type: General Discharge 
-  Mies, Minke (Defensive Driving and Possible Relative to  
+  Mies, Minke (Defensive Driving and Possible Relative to Henk Mies) 
 Position: Perimeter Control (Security) 
 Date of Hire: 05/22/2013 
 Email: Minke.Mies@gastech.com.kronos 
