--- VASTA6W_revision_23_July_17_10:43_AM	2019-07-22 16:00:57.000000000 -0400
+++ VASTA6W_revision_24_July_17_10:46_AM	2019-07-22 16:00:57.000000000 -0400
@@ -20,6 +20,8 @@
 interrogated and released after 6 hours, indicated the Protectors of Kronos 
 were suspected as being the perpetrators behind today's events. 
 1/20/2014 - fueler Ravi Danisliau of the airport of Abila confirmed that two 
-jets private removed today after being supplied of fuel.  
+jets private removed today after being supplied of fuel. 
 1/20/2014 - 14-16 people between two private jets. One confirmed 
-destination of Rome, Italy. 
\ No newline at end of file
+destination of Rome, Italy. 
+1/20/2014 - Captain Federigo Accardi of the fire of Abila announces that it 
+was a false alarm 
\ No newline at end of file
