--- VASTA6W_revision_13_July_17_10:13_AM	2019-07-22 16:00:57.000000000 -0400
+++ VASTA6W_revision_14_July_17_10:15_AM	2019-07-22 16:00:57.000000000 -0400
@@ -1,5 +1,6 @@
 VastA6W@ncsu-las.net 
- 3/26/2014 - President of Kronos family boat sank. Captain of the boat Petra Hohimae 
+ 3/26/2014 - President of Kronos (Karel Kapelou II) family boat sank. Captain of the boat Petra 
+Hohimae 
 3/26/2014 - Mestache Vardshvna - Demonstrator (Protests in Abila). Elian Karel and Julian 
 Vann. JV died in 1998 of cancer from environmental toxins. 
 1/21/2014 - PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
