--- VASTA6W_revision_17_July_17_10:17_AM	2019-07-22 16:00:57.000000000 -0400
+++ VASTA6W_revision_18_July_17_10:18_AM	2019-07-22 16:00:57.000000000 -0400
@@ -7,7 +7,7 @@
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan Strum de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. (kidnapped 
 executives) John Rathburn is American expert in kidnapping located in 
-Tethys. ~14 employees total kidnapped. 
+Tethys. ~14 employees total kidnapped. Revised to 10. 
 1/21/2014 - Protectors of Kronos (POK) formed in 1998 in Elodis. 
 1/21/2014 - President and CEO Sten Sanjorge Jr, CFO Ingrid Barranco, CIO 
 Ada campo-Corrente, COO Orhan Strum, and environment civil servant 
