--- VASTA6W_revision_8_July_17_10:08_AM	2019-07-22 16:00:57.000000000 -0400
+++ VASTA6W_revision_9_July_17_10:11_AM	2019-07-22 16:00:57.000000000 -0400
@@ -1,4 +1,8 @@
 VastA6W@ncsu-las.net 
  3/26/2014 - President of Kronos family boat sank. Captain of the boat Petra Hohimae 
-3/26/2014 - Mestache Vardshvna - Demonstrator. (Protests in Abila). Elian Karel and Julian 
-Vann. JV died in 1998 of cancer from environmental toxins. 
\ No newline at end of file
+3/26/2014 - Mestache Vardshvna - Demonstrator (Protests in Abila). Elian Karel and Julian 
+Vann. JV died in 1998 of cancer from environmental toxins. 
+1/21/2014 - PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
+Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
+pianotent, and environmental leader Willem Vasco-Feed. (kidnapped 
+executives) 
\ No newline at end of file
