--- VASTA6W_revision_11_July_17_10:12_AM	2019-07-22 16:00:57.000000000 -0400
+++ VASTA6W_revision_12_July_17_10:12_AM	2019-07-22 16:00:57.000000000 -0400
@@ -6,4 +6,5 @@
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. (kidnapped 
 executives) John Rathburn is American expert in kidnapping located in 
-Tethys. 
\ No newline at end of file
+Tethys. 
+1/21/2014 - Protectors of Kronos (POK) formed in 1998 in Elodis 
\ No newline at end of file
