--- VASTA6W_revision_9_July_17_10:11_AM	2019-07-22 16:00:57.000000000 -0400
+++ VASTA6W_revision_10_July_17_10:11_AM	2019-07-22 16:00:57.000000000 -0400
@@ -5,4 +5,4 @@
 1/21/2014 - PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. (kidnapped 
-executives) 
\ No newline at end of file
+executives) John Rathburn is American expert in kidnapping 
\ No newline at end of file
