--- VASTWC0_revision_88_July_18_10:41_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_89_July_18_10:44_AM	2019-07-30 15:54:47.000000000 -0400
@@ -138,39 +138,51 @@
 f.1146 - Fire Captain Federigo Accardi has apparently given the 
 all-clear because several officials are entering the building, the 
 perimeter is being dismantled and the fire department is 
-packing their gear. g.1155 - The fire department spokesman announced the building 
+packing their gear. g.1153 - BREAKING NEWS Readers, this is correspondent Petrus 
+Gerhard writing from an unknown location. I was contacted a 
+short time ago by members of the Protectors of Kronos social 
+action movement to convey a very important message to the 
+people of Kronos. The POK was forced to go to great lengths to 
+in order to communicate with you without hindrance from the 
+government. A representative contacted me and asked me to 
+carry their message to you. To protect my safety they picked me 
+up from a public location, blindfolded me and drove me to a 
+location that even I do not recognize. Readers, the next post you 
+read will be one of the most important you have ever read, or 
+indeed, will ever read. 
+h.1155 - The fire department spokesman announced the building 
 is clear and it was a false alarm. Employees return to the 
 building. 
-h.1512 - Two police vehicles each with two policemen in inside 
+i.1512 - Two police vehicles each with two policemen in inside 
 have just left GAStech headquarters with no lights or sirens. 20 
 Jan 2014 
-i.1515 - BREAKING NEWS - Homeland Illumination 
+j.1515 - BREAKING NEWS - Homeland Illumination 
 correspondent Petrus Gerhard has just spoken with an airport 
 official who confirms that a private jet left Abila airport at 1430, 
 with a number of persons on board. Reports that the GAStech 
 executives were passengers on this jet are unconfirmed. 
-j.1744 - Media reports are coming in that a number of GAStech 
+k.1744 - Media reports are coming in that a number of GAStech 
 employees were kidnapped. The identity of the passengers on 
 the mystery jet that departed earlier today from the Abila Airport 
 remains unknown. 
-k.1837 - There are unconfirmed reports that the GAStech 
+l.1837 - There are unconfirmed reports that the GAStech 
 employees were kidnapped. Media outlets are speculating that 
 their newfound wealth might provide a motive. Perhaps 
 someone should locate the caterers from the executive 
-breakfast? 
-l.1942 - BREAKING NEWS - Kronos police spokesman has 
+breakfast? m.1942 - BREAKING NEWS - Kronos police spokesman has 
 released a statement. 'There are approximately fourteen 
 individuals unaccounted for among the GAStech staff. It has 
 not yet been determined if all of these individuals are in fact 
 missing.' They said the case is under investigation and asked 
 that anyone who might have information about the missing 
 employees please contact Abila police immediately. 
-m.2020 - Media outlets are reporting that a government 
+n.2020 - Media outlets are reporting that a government 
 spokesman who asked not to be named because he was not 
 authorized to speak to the press stated that the GAStech 
 employees had been kidnapped and the Kronos government 
 was working hard to identify the lawbreakers. One wonders if 
-they are also working hard to recover the GAStech people as well, or if this is an opportunity to profit from the misfortunate 
+they are also working hard to recover the GAStech people as 
+well, or if this is an opportunity to profit from the misfortunate 
 of others.  
  3.Plausible explanation of why employees are missing and recommend 
 next steps. 
\ No newline at end of file
