--- VASTWC0_revision_92_July_18_10:46_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_93_July_18_10:46_AM	2019-07-30 15:54:47.000000000 -0400
@@ -121,7 +121,8 @@
 could be provided with only half the funds used to orchestrate 
 this spectacle.  
 b.1000ish- VIP Security team kidnaps the officials and the three 
-other security force members. Isia  
+other security force members. Isia probable hand in the 
+execution of the kidnapping.  
 c.1018 - A fire alarm has gone off at GAStech headquarters. 
 People are evacuating the building, and so far no sign of 
 Sanjorge or his executive staff. Officials have asked reporters to 
