--- VASTWC0_revision_13_July_17_10:11_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_14_July_17_10:11_AM	2019-07-30 15:54:47.000000000 -0400
@@ -11,7 +11,7 @@
  4.Arpa, Emile- Janitor (Facilities) 
 a.Kronos citizenship 
 b.Kronos military service 
- 5.Arpa, Emile- Janitor (Facilities) 
+ 5.Arpa, EmileAwelon, Varro- Janitor (Facilities) 
 a.Kronos citizenship 
 b.Kronos military service 
 6. 
