--- VASTWC0_revision_44_July_18_8:42_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_45_July_18_8:54_AM	2019-07-30 15:54:47.000000000 -0400
@@ -76,4 +76,8 @@
 advocate for a more forceful approach to gaining attention to the plight of people living 
 downstream of Tiskele Bend. He has been arrested several times for civil disobedience and 
 trespassing during POK rallies. 
- 
\ No newline at end of file
+ Five Execs who were kidnapped 
+President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
+Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
+pianotent, and environmental leader Willem Vasco-Feed. 
+   
\ No newline at end of file
