--- VASTWC0_revision_85_July_18_10:38_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_86_July_18_10:39_AM	2019-07-30 15:54:47.000000000 -0400
@@ -108,42 +108,54 @@
 ii.Isia Vann, most of the janitors/truck drivers/security fall 
 into this category 
  2.Reconstruct timeline of events on Jan. 20-21, 2014 
-a.1018 - A fire alarm has gone off at GAStech headquarters. 
+a.0926 - Welcome to our coverage of today's events. I'm standing 
+outside the GAStech headquarters waiting for the GAStech 
+executives who are expected to appear within the hour. 
+GAStech CEO Sten Sanjorge is celebrating the successful initial 
+public offering of his company, which took place last month. 
+When they leave here the GAStech executives will travel to the 
+capital building where they will meet with President Kapelou 
+and several of his Ministers and Council members. A private 
+celebration of their successful alliance is planned to take place 
+later today. One can't help but wonder how much healthcare 
+could be provided with only half the funds used to orchestrate 
+this spectacle.  
+b.1018 - A fire alarm has gone off at GAStech headquarters. 
 People are evacuating the building, and so far no sign of 
 Sanjorge or his executive staff. Officials have asked reporters to 
 move away from the building to allow the fire department 
 access. This has the flavor of yet another media ploy to avoid 
 confrontation from the people of Kronos.  
-b.1025 - The building is being searched for signs of trouble by the 
+c.1025 - The building is being searched for signs of trouble by the 
 fire department. There has been no confirmation of a bomb 
 threat, but I haven't seen a bomb squad appear. 
-c.1052 - I just received word from Illumination correspondent 
+d.1052 - I just received word from Illumination correspondent 
 Petrus Gerhard that CEO Sanjorge arrived at the capitol building 
 a few minutes ago. 20 Jan 2014 
-d.1130 - Some of the fire officials have exited the building and are 
+e.1130 - Some of the fire officials have exited the building and are 
 talking to the officials, but no announcements have been made 
 about the events at GAStech headquarters.  
-e.1146 - Fire Captain Federigo Accardi has apparently given the 
+f.1146 - Fire Captain Federigo Accardi has apparently given the 
 all-clear because several officials are entering the building, the 
 perimeter is being dismantled and the fire department is 
-packing their gear. 
-f.1512 - Two police vehicles each with two policemen in inside 
+packing their gear. g.1512 - Two police vehicles each with two policemen in inside 
 have just left GAStech headquarters with no lights or sirens. 20 
 Jan 2014 
-g.1515 - BREAKING NEWS - Homeland Illumination 
+h.1515 - BREAKING NEWS - Homeland Illumination 
 correspondent Petrus Gerhard has just spoken with an airport 
 official who confirms that a private jet left Abila airport at 1430, 
 with a number of persons on board. Reports that the GAStech 
 executives were passengers on this jet are unconfirmed. 
-h.1744 - Media reports are coming in that a number of GAStech 
+i.1744 - Media reports are coming in that a number of GAStech 
 employees were kidnapped. The identity of the passengers on 
 the mystery jet that departed earlier today from the Abila Airport 
-remains unknown. i.1837 - There are unconfirmed reports that the GAStech 
+remains unknown. 
+j.1837 - There are unconfirmed reports that the GAStech 
 employees were kidnapped. Media outlets are speculating that 
 their newfound wealth might provide a motive. Perhaps 
 someone should locate the caterers from the executive 
 breakfast? 
-j.2020 - Media outlets are reporting that a government 
+k.2020 - Media outlets are reporting that a government 
 spokesman who asked not to be named because he was not 
 authorized to speak to the press stated that the GAStech 
 employees had been kidnapped and the Kronos government 
