--- VASTWC0_revision_89_July_18_10:44_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_90_July_18_10:44_AM	2019-07-30 15:54:47.000000000 -0400
@@ -153,36 +153,54 @@
 h.1155 - The fire department spokesman announced the building 
 is clear and it was a false alarm. Employees return to the 
 building. 
-i.1512 - Two police vehicles each with two policemen in inside 
+i.1200 - TO THE PEOPLE OF KRONOS: NO LONGER WILL WE 
+ACQUIESCE TO THE UNFETTERED KLEPTOCRACY THAT HAS 
+BECOME OUR GOVERNMENT AND TO THE IMPERIALIST 
+ACTIVITIES OF THE FOREIGN CORPORATIONS ON OUR 
+HOMELAND. WE HAVE TAKEN STEPS TO ENSURE OUR VOICE - 
+YOUR VOICES - ARE HEARD WITHOUT SUPPRESSION OR 
+DISTORTION. THE INACTIONS AND IGNORANCE OF OUR 
+ELECTED OFFICIALS HAVE FORCED US INTO THESE 
+MEASURES. YESTERDAY, WE THWARTED A CELEBRATION OF 
+GREED AND CORRUPTION BETWEEN FOREIGN BUSINESSMEN 
+WHO, IN COLLUSION WITH OUR GOVERNMENT OFFICIALS, 
+WORKED TO IMPOVERISH 99% OF THIS GREAT NATION TO 
+THEIR OWN BENEFIT. IN AN ACT OF DEFIANCE WE 
+APPREHENDED THE ELITISTS WHO HAVE EXPLOITED OUR 
+RESOURCES, KILLED OUR CHILDREN AND RUINED OUR WAY 
+OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE FORM OF 
+$20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE PEOPLE. THIS IS THE FIRST STEP 
+TOWARD TAKING OUR COUNTRY BACK. 
+j.1512 - Two police vehicles each with two policemen in inside 
 have just left GAStech headquarters with no lights or sirens. 20 
 Jan 2014 
-j.1515 - BREAKING NEWS - Homeland Illumination 
+k.1515 - BREAKING NEWS - Homeland Illumination 
 correspondent Petrus Gerhard has just spoken with an airport 
 official who confirms that a private jet left Abila airport at 1430, 
 with a number of persons on board. Reports that the GAStech 
 executives were passengers on this jet are unconfirmed. 
-k.1744 - Media reports are coming in that a number of GAStech 
+l.1744 - Media reports are coming in that a number of GAStech 
 employees were kidnapped. The identity of the passengers on 
 the mystery jet that departed earlier today from the Abila Airport 
 remains unknown. 
-l.1837 - There are unconfirmed reports that the GAStech 
+m.1837 - There are unconfirmed reports that the GAStech 
 employees were kidnapped. Media outlets are speculating that 
 their newfound wealth might provide a motive. Perhaps 
 someone should locate the caterers from the executive 
-breakfast? m.1942 - BREAKING NEWS - Kronos police spokesman has 
+breakfast? 
+n.1942 - BREAKING NEWS - Kronos police spokesman has 
 released a statement. 'There are approximately fourteen 
 individuals unaccounted for among the GAStech staff. It has 
 not yet been determined if all of these individuals are in fact 
 missing.' They said the case is under investigation and asked 
 that anyone who might have information about the missing 
 employees please contact Abila police immediately. 
-n.2020 - Media outlets are reporting that a government 
+o.2020 - Media outlets are reporting that a government 
 spokesman who asked not to be named because he was not 
 authorized to speak to the press stated that the GAStech 
 employees had been kidnapped and the Kronos government 
 was working hard to identify the lawbreakers. One wonders if 
-they are also working hard to recover the GAStech people as 
-well, or if this is an opportunity to profit from the misfortunate 
+they are also working hard to recover the GAStech people as well, or if this is an opportunity to profit from the misfortunate 
 of others.  
  3.Plausible explanation of why employees are missing and recommend 
 next steps. 
\ No newline at end of file
