--- VASTWC0_revision_77_July_18_10:34_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_78_July_18_10:35_AM	2019-07-30 15:54:47.000000000 -0400
@@ -108,16 +108,20 @@
 ii.Isia Vann, most of the janitors/truck drivers/security fall 
 into this category 
  2.Reconstruct timeline of events on Jan. 20-21, 2014 
-a.1018- fire alarm pulled at GAStech1018 - A fire alarm has gone 
-off at GAStech headquarters. People are evacuating the 
-building, and so far no sign of Sanjorge or his executive staff. 
-Officials have asked reporters to move away from the building 
-to allow the fire department access. This has the flavor of yet 
-another media ploy to avoid confrontation from the people of 
-Kronos.  
+a.1018 - A fire alarm has gone off at GAStech headquarters. 
+People are evacuating the building, and so far no sign of 
+Sanjorge or his executive staff. Officials have asked reporters to 
+move away from the building to allow the fire department 
+access. This has the flavor of yet another media ploy to avoid 
+confrontation from the people of Kronos.  
 b.1512 - Two police vehicles each with two policemen in inside 
 have just left GAStech headquarters with no lights or sirens. 20 
 Jan 2014 
-c. 
+c.1515 - BREAKING NEWS - Homeland Illumination 
+correspondent Petrus Gerhard has just spoken with an airport 
+official who confirms that a private jet left Abila airport at 1430, 
+with a number of persons on board. Reports that the GAStech 
+executives were passengers on this jet are unconfirmed. 
+d. 
  3.Plausible explanation of why employees are missing and recommend 
 next steps. 
\ No newline at end of file
