--- VASTWC0_revision_41_July_18_8:37_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_42_July_18_8:39_AM	2019-07-30 15:54:47.000000000 -0400
@@ -67,4 +67,7 @@
 Dante.Coginian@gastech.com.kronos, Henk.Mies@gastech.com.kronos, 
 Varro.Awelon@gastech.com.kronos, Dylan.Scozzese@gastech.com.kronos, 
 Albina.Hafon@gastech.com.kronos, Irene.Nant@gastech.com.kronos 
- Mandor Vann is closely related to Isia Vann 
\ No newline at end of file
+ Mandor Vann is closely related to Isia Vann 
+ A large number of the POK members recruited since 2001 are under-employed and under- 
+educated young males 
+ 
\ No newline at end of file
