--- VASTWC0_revision_68_July_18_9:37_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_69_July_18_9:38_AM	2019-07-30 15:54:47.000000000 -0400
@@ -81,7 +81,7 @@
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. 
  Poss security suspects - VIP visit 20JAN 
-Hennie.Osvaldo@gastech.com.kronos,  SUS  25yo 
+Hennie.Osvaldo@gastech.com.kronos,  SUS  25yo male 
 Loreto.Bodrogi@gastech.com.kronos,  SUS  24yo male Varja.Lagos@gastech.com.kronos,  
 Inga.Ferro@gastech.com.kronos, SUS 24 yo female 
 Kanon.Herrero@gastech.com.kronos,  
