--- VASTWC0_revision_95_July_18_10:48_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_96_July_18_10:49_AM	2019-07-30 15:54:47.000000000 -0400
@@ -210,4 +210,5 @@
 a.I believe that the GAStech employees went missing because the 
 POK has a more radical leadership and they are taking action 
 against the injustice that is happening within their community 
-at the hands of the GAStech executives.  
\ No newline at end of file
+at the hands of the GAStech executives. I recommend that the 
+company should compensate them  
\ No newline at end of file
