--- VASTWC0_revision_22_July_17_10:18_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_23_July_17_10:18_AM	2019-07-30 15:54:47.000000000 -0400
@@ -31,4 +31,7 @@
  11.Morlun, Valeria- Truck Driver (Facilities) 
 a.Kronos citizenship 
 b.Kronos military service 
-12. 
\ No newline at end of file
+ 12.Morlun, Adan- Truck Driver (Facilities) 
+a.Kronos citizenship 
+b.Kronos military service 
+13. 
\ No newline at end of file
