--- VASTWC0_revision_62_July_18_9:31_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_63_July_18_9:32_AM	2019-07-30 15:54:47.000000000 -0400
@@ -80,7 +80,7 @@
 President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. 
- Poss security suspects 
+ Poss security suspects - VIP visit 20JAN 
 Hennie.Osvaldo@gastech.com.kronos,  SUS 
 Loreto.Bodrogi@gastech.com.kronos,  SUS Varja.Lagos@gastech.com.kronos,  
 Inga.Ferro@gastech.com.kronos, SUS 
