--- VASTWC0_revision_54_July_18_9:11_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_55_July_18_9:12_AM	2019-07-30 15:54:47.000000000 -0400
@@ -90,8 +90,9 @@
 Stefano 
 ii.Save our Wildlands members 
  c.How has the organization’s structure changed over time- 
-i.Went from peaceful environmental protests to more 
-violent protests rallies have also been marked bywith an 
-increase in civil disobedience and militant, violent action. 
+i.Went from peaceful environmental protests to rallies with 
+an increase in civil disobedience and militant, violent 
+action. 
  d.Potential Connections- POK and GAStech- 
-i. 
\ No newline at end of file
+i.A large number of the POK members recruited since 2001 are 
+under-employed and under-educated young males 
\ No newline at end of file
