--- VASTWC0_revision_48_July_18_9:04_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_49_July_18_9:06_AM	2019-07-30 15:54:47.000000000 -0400
@@ -84,4 +84,6 @@
  1. 
 a.Leaders- 
 i.Elian Karel and if something should happen to him the 
-group would fall to Silvia Marek or Manderor Vann  
\ No newline at end of file
+group would fall to Silvia Marek or Mandor Vann  
+ii.Dominant personalities- Isia Vann, Lucio Jakb, Lorenzo Di 
+Stefano 
\ No newline at end of file
