--- VASTWC0_revision_96_July_18_10:49_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_97_July_18_10:50_AM	2019-07-30 15:54:47.000000000 -0400
@@ -203,8 +203,8 @@
 authorized to speak to the press stated that the GAStech 
 employees had been kidnapped and the Kronos government 
 was working hard to identify the lawbreakers. One wonders if 
-they are also working hard to recover the GAStech people as well, or if this is an opportunity to profit from the misfortunate 
-of others.  
+they are also working hard to recover the GAStech people as well, or if this is an opportunity to profit from the 
+misfortunatemisfortune of others.  
  3.Plausible explanation of why employees are missing and recommend 
 next steps. 
 a.I believe that the GAStech employees went missing because the 
