--- VASTWC0_revision_78_July_18_10:35_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_79_July_18_10:35_AM	2019-07-30 15:54:47.000000000 -0400
@@ -114,14 +114,17 @@
 move away from the building to allow the fire department 
 access. This has the flavor of yet another media ploy to avoid 
 confrontation from the people of Kronos.  
-b.1512 - Two police vehicles each with two policemen in inside 
+b.1052 - I just received word from Illumination correspondent 
+Petrus Gerhard that CEO Sanjorge arrived at the capitol building 
+a few minutes ago. 20 Jan 2014 
+c.1512 - Two police vehicles each with two policemen in inside 
 have just left GAStech headquarters with no lights or sirens. 20 
 Jan 2014 
-c.1515 - BREAKING NEWS - Homeland Illumination 
+d.1515 - BREAKING NEWS - Homeland Illumination 
 correspondent Petrus Gerhard has just spoken with an airport 
 official who confirms that a private jet left Abila airport at 1430, 
 with a number of persons on board. Reports that the GAStech 
 executives were passengers on this jet are unconfirmed. 
-d. 
+e. 
  3.Plausible explanation of why employees are missing and recommend 
 next steps. 
\ No newline at end of file
