--- VASTWC0_revision_63_July_18_9:32_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_64_July_18_9:35_AM	2019-07-30 15:54:47.000000000 -0400
@@ -81,8 +81,8 @@
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed. 
  Poss security suspects - VIP visit 20JAN 
-Hennie.Osvaldo@gastech.com.kronos,  SUS 
-Loreto.Bodrogi@gastech.com.kronos,  SUS Varja.Lagos@gastech.com.kronos,  
+Hennie.Osvaldo@gastech.com.kronos,  SUS  30yo 
+Loreto.Bodrogi@gastech.com.kronos,  SUS  Varja.Lagos@gastech.com.kronos,  
 Inga.Ferro@gastech.com.kronos, SUS 
 Kanon.Herrero@gastech.com.kronos,  
 Hideki.Cocinaro@gastech.com.kronos, 
