--- VASTWC0_revision_91_July_18_10:45_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_92_July_18_10:46_AM	2019-07-30 15:54:47.000000000 -0400
@@ -120,7 +120,8 @@
 later today. One can't help but wonder how much healthcare 
 could be provided with only half the funds used to orchestrate 
 this spectacle.  
-b. 
+b.1000ish- VIP Security team kidnaps the officials and the three 
+other security force members. Isia  
 c.1018 - A fire alarm has gone off at GAStech headquarters. 
 People are evacuating the building, and so far no sign of 
 Sanjorge or his executive staff. Officials have asked reporters to 
@@ -137,9 +138,9 @@
 talking to the officials, but no announcements have been made 
 about the events at GAStech headquarters.  
 g.1146 - Fire Captain Federigo Accardi has apparently given the 
-all-clear because several officials are entering the building, the 
-perimeter is being dismantled and the fire department is 
-packing their gear. h.1153 - BREAKING NEWS Readers, this is correspondent Petrus 
+all-clear because several officials are entering the building, the perimeter is being dismantled and the fire department is 
+packing their gear. 
+h.1153 - BREAKING NEWS Readers, this is correspondent Petrus 
 Gerhard writing from an unknown location. I was contacted a 
 short time ago by members of the Protectors of Kronos social 
 action movement to convey a very important message to the 
@@ -169,8 +170,8 @@
 THEIR OWN BENEFIT. IN AN ACT OF DEFIANCE WE 
 APPREHENDED THE ELITISTS WHO HAVE EXPLOITED OUR 
 RESOURCES, KILLED OUR CHILDREN AND RUINED OUR WAY 
-OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE FORM OF 
-$20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE PEOPLE. THIS IS THE FIRST STEP 
+OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE FORM OF $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE 
+SUFFERING OF THE PEOPLE. THIS IS THE FIRST STEP 
 TOWARD TAKING OUR COUNTRY BACK. 
 k.1512 - Two police vehicles each with two policemen in inside 
 have just left GAStech headquarters with no lights or sirens. 20 
