--- VAST32P_revision_5_July_17_10:44_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_6_July_17_10:45_AM	2019-07-19 13:59:36.000000000 -0400
@@ -2,4 +2,6 @@
  Who are KOS? 
  E.Vann, Security 
 Julian Vann died in 1998 of the cancer caused by environmental toxins in 
-its potable water.  
\ No newline at end of file
+its potable water.  
+Daily Pegasus | 03/26/2014 
+  
\ No newline at end of file
