--- VAST32P_revision_55_July_18_10:36_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_56_July_18_10:37_AM	2019-07-19 13:59:36.000000000 -0400
@@ -50,7 +50,7 @@
 family name with several known POK members.”News Online Today | 01/20/2014 
    What happened on 1/20-21? Timeline of Events 
  “1744 - Media reports are coming in that a number of GAStech employees 
-were kidnapped.” 
+were kidnapped...” 
 Homeland Illumination | 01/20/2014 
  “1837 - There are unconfirmed reports that the GAStech employees were 
 kidnapped.”Homeland Illumination | 01/20/2014 
