--- VAST32P_revision_10_July_17_10:58_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_11_July_17_11:02_AM	2019-07-19 13:59:36.000000000 -0400
@@ -12,6 +12,7 @@
 The Abila Post | 01/21/2014 
   Suspects: 
  E.Vann, Security 
+Relative died from GasTech pollution 
 “Julian Vann died in 1998 of the cancer caused by environmental toxins in 
 its potable water.”  
 Daily Pegasus | 03/26/2014 
