--- VAST32P_revision_15_July_17_11:13_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_16_July_17_11:14_AM	2019-07-19 13:59:36.000000000 -0400
@@ -22,12 +22,11 @@
 Relative died from GasTech pollution 
 “Julian Vann died in 1998 of the cancer caused by environmental toxins in 
 its potable water.”  
-Daily Pegasus | 03/26/2014 
+ Daily Pegasus | 03/26/2014 
 Possible case of mistaken identity: 
-“The evident confusion comes the similarity between its name and the one 
-from a protector of the member of Kronos. Vann, protector of security for GAStech, was furious, indicating, 'I do not have anything to do nobodies 
-with no members of group POK of the terrorist. I know nobody of Elodis. I 
-was born here in Abila and support to my government and my president.  
+“The evident confusion comes the similarity between its name and the one from a protector of the 
+member of Kronos. Vann, protector of security for GAStech, was furious, indicating, 'I do not have anything to do nobodies with no members of group POK of the terrorist. I know nobody of Elodis. I was 
+born here in Abila and support to my government and my president.  
 “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
 released after 6 hours, indicated the Protectors of Kronos were suspected 
 as being the perpetrators behind today's events.” 
