--- VAST32P_revision_38_July_18_9:10_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_39_July_18_9:13_AM	2019-07-19 13:59:36.000000000 -0400
@@ -9,7 +9,9 @@
 Kronos homage paid to their cases LEADER, Elian Karel, who on 19 June 
 2009 whereas resistance worker is held in a prison Kronos assassination. 
 We collect us today to remind Elian Karel, Silvia Marek told rap assembly, 
-much of which supported white candles in the honor of Karel.” 
+much of which supported white candles in the honor of Karel.”..He had on 
+its father, Jeroen and the founder of POK, the pace of Henk Bodrogi actual 
+gone paid attention which he would enter.” 
 The Wrap | 06/21/2012 
   Suspects: 
  E. Vann, Security (Org Chart) 
@@ -24,8 +26,7 @@
 Daily Pegasus | 03/26/2014 
  Name is actually Juliana Vann 
 “sweet-faced ten year dead old girl - Juliana Vann who died in 1998 of 
-cancer caused by the benzene in her drinking water.” 
-All News Today | 06/21/2013 
+cancer caused by the benzene in her drinking water.” All News Today | 06/21/2013 
   Vann Possible mistaken identity: 
 “The evident confusion comes the similarity between its name and the one 
 from a protector of the member of Kronos. Vann, protector of security for 
@@ -51,6 +52,5 @@
 like distraction kidnapping them or at least to have contributed to the 
 confusion during the event. The cars of the body of firemen of Avila were 
 sent to the headquarters of GAStech - of Kronos around 10:00 this local 
-time of Kronos in the morning to respond to a fire alarm.” 
-The General Post | 01/20/2014 
+time of Kronos in the morning to respond to a fire alarm.” The General Post | 01/20/2014 
   
\ No newline at end of file
