--- VAST32P_revision_46_July_18_9:26_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_47_July_18_9:26_AM	2019-07-19 13:59:36.000000000 -0400
@@ -19,12 +19,14 @@
 Homeland Illumination | 06/20/2012 
  “Karel took over control POK of Henk Bodrogi, of which bad health forced 
 him of the position.” 
-   Suspects: 
+The Wrap | 03/09/2009 
+    Suspects: 
  E. Vann, Security (Org Chart) 
 (There is also an I. Vann in Security) 
  “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
 released after 6 hours, indicated the Protectors of Kronos were suspected 
-as being the perpetrators behind today's events.” Who What News | 01/20/2014 
+as being the perpetrators behind today's events.” 
+Who What News | 01/20/2014 
  Possible relative died from GasTech pollution 
 “Julian Vann died in 1998 of the cancer caused by environmental toxins in 
 its potable water.”  
@@ -49,10 +51,10 @@
 kidnapping. Sanjorge and the four executives of the headquarters of 
 GAStech of the premises were to take care of a reception that was received 
 by president Kapelou in the building of the Capitol.” 
-The Guide | 01/21/2014 
- Possible inside job: 
+The Guide | 01/21/2014  Possible inside job: 
 “The number of the missing has been revised from fourteen to ten.” 
-The Abila Post | 01/21/2014  Fire Alarm, possible signal or distraction: 
+The Abila Post | 01/21/2014 
+ Fire Alarm, possible signal or distraction: 
 “The information without confirming of a fire in the office could have served 
 like distraction kidnapping them or at least to have contributed to the 
 confusion during the event. The cars of the body of firemen of Avila were 
