--- VAST32P_revision_19_July_17_11:15_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_20_July_17_11:25_AM	2019-07-19 13:59:36.000000000 -0400
@@ -19,13 +19,17 @@
 The General Post | 01/20/2014 
  Suspects: 
  Edvard Vann, Security 
-Relative died from GasTech pollution 
+RPossible relative elative died from GasTech pollution 
 “Julian Vann died in 1998 of the cancer caused by environmental toxins in 
 its potable water.”  
 Daily Pegasus | 03/26/2014 
- “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
+Name is actually Juliana Vann 
+“sweet-faced ten year dead old girl - Juliana Vann who died in 1998 of 
+cancer caused by the benzene in her drinking water.” 
+  “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
 released after 6 hours, indicated the Protectors of Kronos were suspected 
-as being the perpetrators behind today's events.” Who What News | 01/20/2014 
+as being the perpetrators behind today's events.” 
+Who What News | 01/20/2014 
 Vann Possible mistaken identity: 
 “The evident confusion comes the similarity between its name and the one 
 from a protector of the member of Kronos. Vann, protector of security for 
