--- VAST32P_revision_35_July_18_9:01_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_36_July_18_9:08_AM	2019-07-19 13:59:36.000000000 -0400
@@ -4,9 +4,8 @@
 “The guards of Kronos or POK were founded in 1997 by residents of rural 
 Elodis in accordance with the opinion of now old health for all (WFA), an 
 international group of safety of water.” 
-” 
 Athena Speaks | 08/24/2012 
-  Suspects: 
+   Suspects: 
  E. Vann, Security (Org Chart) 
 (There is also an I. Vann in Security) 
  “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
