--- VAST32P_revision_49_July_18_9:35_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_50_July_18_10:28_AM	2019-07-19 13:59:36.000000000 -0400
@@ -48,11 +48,11 @@
 The Explainer | 01/20/2014 
  “Edvard Vann of Abila, was questioned for hours because he shares a 
 family name with several known POK members.”News Online Today | 01/20/2014 
-   What happened on 6/20-21? 
- Kidnapping happened while in transit GasTech HQ: 
+   What happened on 6/20-21? Timeline of Events 
+  Kidnapping happened while in transit GasTech HQ: 
 “Sanjorge, that was been assistiendo to a corporative meeting, was in 
-transit of the GAStech that constructed capitol when it happened the 
-kidnapping. Sanjorge and the four executives of the headquarters of GAStech of the premises were to take care of a reception that was received 
+transit of the GAStech that constructed capitol when it happened the kidnapping. Sanjorge and the four executives of the headquarters of 
+GAStech of the premises were to take care of a reception that was received 
 by president Kapelou in the building of the Capitol.” 
 The Guide | 01/21/2014 
  Possible inside job: 
