--- VAST32P_revision_32_July_18_8:34_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_33_July_18_8:42_AM	2019-07-19 13:59:36.000000000 -0400
@@ -23,7 +23,9 @@
 with no members of group POK of the terrorist. I know nobody of Elodis. I 
 was born here in Abila and support to my government and my president.” 
 The Explainer | 01/20/2014 
-    What happened on 1/20-21? 
+ “Edvard Vann of Abila, was questioned for hours because he shares a 
+family name with several known POK members. 
+”News Online Today | 01/20/2014      What happened on 1/20-21? 
  Kidnapping happened while in transit GasTech HQ: 
 “Sanjorge, that was been assistiendo to a corporative meeting, was in 
 transit of the GAStech that constructed capitol when it happened the 
