--- VAST32P_revision_12_July_17_11:06_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_13_July_17_11:09_AM	2019-07-19 13:59:36.000000000 -0400
@@ -10,8 +10,14 @@
 Possible inside job: 
 “The number of the missing has been revised from fourteen to ten.” 
 The Abila Post | 01/21/2014 
+Fire Alarm, possible signal or distraction: 
+“The information without confirming of a fire in the office could have served 
+like distraction kidnapping them or at least to have contributed to the 
+confusion during the event. The cars of the body of firemen of Avila were 
+sent to the headquarters of GAStech - of Kronos around 10:00 this local 
+time of Kronos in the morning to respond to a fire alarm.” 
  Suspects: 
- E.Vdvard Vann, Security 
+ Edvard Vann, Security 
 Relative died from GasTech pollution 
 “Julian Vann died in 1998 of the cancer caused by environmental toxins in 
 its potable water.”  
@@ -19,5 +25,4 @@
 “Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
 released after 6 hours, indicated the Protectors of Kronos were suspected 
 as being the perpetrators behind today's events.” 
-Who What News | 01/20/2014 
-   
\ No newline at end of file
+Who What News | 01/20/2014    
\ No newline at end of file
