--- VAST32P_revision_27_July_17_11:43_AM	2019-07-19 13:59:36.000000000 -0400
+++ VAST32P_revision_28_July_18_8:22_AM	2019-07-19 13:59:36.000000000 -0400
@@ -1,5 +1,6 @@
 Vast32P GasTech Scenario Notes 
- Who are KOSPOK? 
+ Who are POK? 
+What happened 1/20-21? 
  Kidnapping happened while in transit GasTech HQ: 
 “Sanjorge, that was been assistiendo to a corporative meeting, was in 
 transit of the GAStech that constructed capitol when it happened the 
