--- VASTBTZ_revision_51_June_20_9:14_AM	2019-06-27 13:08:54.000000000 -0400
+++ VASTBTZ_revision_52_June_20_9:18_AM	2019-06-27 13:08:54.000000000 -0400
@@ -68,4 +68,7 @@
  Kidnapping happened in transit. EA Lais arranges travel 
  Three Mies that work at GAStech all ex Kronos military. One is EA, one truck driver, last is 
 security 
-HASR banned in Tethys but GAStech still does in Kronos 
\ No newline at end of file
+HASR banned in Tethys but GAStech still does in Kronos 
+ Inga.Ferro@gastech.com.kronos, Loreto.Bodrogi@gastech.com.kronos, 
+Isia.Vann@gastech.com.kronos, Hennie.Osvaldo@gastech.com.kronos, 
+Minke.Mies@gastech.com.kronos, Ruscella.Mies.Haber@gastech.com.kronos 
\ No newline at end of file
