--- VASTBTZ_revision_33_June_19_11:29_AM	2019-06-27 13:08:54.000000000 -0400
+++ VASTBTZ_revision_34_June_19_11:50_AM	2019-06-27 13:08:54.000000000 -0400
@@ -50,4 +50,6 @@
  President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech 
 pianotent, and environmental leader Willem Vasco-Feed.  Missing leads 
+ Alibi CC Isia Vann 
+Adan Morlun 
  
\ No newline at end of file
