--- VASTBTZ_revision_22_June_19_10:49_AM	2019-06-27 13:08:54.000000000 -0400
+++ VASTBTZ_revision_23_June_19_10:51_AM	2019-06-27 13:08:54.000000000 -0400
@@ -30,4 +30,5 @@
  Art 447 Our front trajectory is clear,' POK Co-leader that the carmine 
 Osvaldo said. ' 
  Art 277 We have a meeting who is planned with the minister, the guards of 
-Kronos to LEADER Henk Bodrogi said. 
\ No newline at end of file
+Kronos to LEADER Henk Bodrogi said. 
+ Art 86 Petrus Gerhard talks directly to POK 
\ No newline at end of file
