--- VASTCER_revision_19_June_19_11:19_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_20_June_19_11:21_AM	2019-06-27 13:13:16.000000000 -0400
@@ -25,4 +25,6 @@
 first-responder with the medical 
  Mestache Vardshvna, 23, of the Western coast of Kronos. 'We try to obtain 
 the attention of our government,' it said. 'It is important, thus I took leave of 
-work to be part of this important event 
\ No newline at end of file
+work to be part of this important event 
+ Elon Carol had been implied POK since its beginning since his/her father, 
+Jeroen Carol, were an original member 
\ No newline at end of file
