--- VASTCER_revision_40_June_19_11:47_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_41_June_19_11:52_AM	2019-06-27 13:13:16.000000000 -0400
@@ -1,7 +1,7 @@
 Goal: Find 3 people associated with POK. Annotate Evidence and reasons for decision 
 1 Most recent leader of POK(2012): Silvia Marek 
 2  
-  1 Edvard Vann? 
+  1 Edvard Vann? and Isia Vann (same name as Juliana Vann both work in same  
 2 Elian Karel (Dead) 
 3 Juliana Vann (Dead) 
 4 Isia Vann?? 
@@ -34,6 +34,6 @@
 connect himself appealing on the citizens of Kronos at her for a day of 
 peaceful protest tomorrow.  
 Silvia Marek, leader of the Protectors of Kronos social action organization  
- Blotter-Abila arrested for “MISS MAINTAINING THE ORDER” day after POK 
-riots in Siopia 
+ Blotter arrested for “MISS MAINTAINING THE ORDER” day after POK riots in 
+Siopia 
  2/23/2009 
\ No newline at end of file
