--- VASTCER_revision_39_June_19_11:45_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_40_June_19_11:47_AM	2019-06-27 13:13:16.000000000 -0400
@@ -28,8 +28,8 @@
 Mestache Vardshvna, 23, of the Western coast of Kronos. 'We try to obtain 
 the attention of our government,' it said. 'It is important, thus I took leave of 
 work to be part of this important event 
- Elian (Elon/Elan) Carol had been implied POK since its beginning since 
-his/her father, Jeroen Carol, were an original member 
+ Elian (Elan) Carol had been implied POK since its beginning since his/her 
+father, Jeroen Carol, were an original member 
  Silvia Marek, LEADER of the guards of Kronos social action organization is 
 connect himself appealing on the citizens of Kronos at her for a day of 
 peaceful protest tomorrow.  
