--- VASTCER_revision_6_June_19_10:17_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_7_June_19_10:19_AM	2019-06-27 13:13:16.000000000 -0400
@@ -1,11 +1,12 @@
 Goal: Find 3 people associated with POK. Annotate Evidence and reasons for decision 
   POK protest clean water, human rights, and against the corruption of the 
 Government of Kronos and GAStech International. 
- The posters with coloured images of Elian KarelElian Karel and Juliana 
-Vann were recalls of the heavy toll already paid by chiefs and the innocent 
-ones. Karel was the former chief of the guards of Kronos before his 
-untimely death in 2009. Its death, thought with being a murder by, 
-encouraged much cancel gatherings to bring the message of the guards of 
-Kronos to the public. Juliana Vann died in 1998 of the cancer caused by 
-environmental toxins in its drinking water. 
- Mestache Vardshvna is a protestor 
\ No newline at end of file
+ The posters with coloured images of Elian Karel and Juliana Vann were 
+recalls of the heavy toll already paid by chiefs and the innocent ones. Karel 
+was the former chief of the guards of Kronos before his untimely death in 
+2009. Its death, thought with being a murder by, encouraged much cancel 
+gatherings to bring the message of the guards of Kronos to the public. 
+Juliana Vann died in 1998 of the cancer caused by environmental toxins in 
+its drinking water. 
+ Protectors of Kronos or POK were founded in 1997 by residents of rural 
+Elodis on the advice of the now defunct Wellness for All (WFA) 
\ No newline at end of file
