--- VASTCER_revision_52_June_20_8:29_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_53_June_20_8:29_AM	2019-06-27 13:13:16.000000000 -0400
@@ -2,8 +2,8 @@
  Goal: Find 3 people associated with POK. Annotate Evidence and reasons for decision 
 1 Most recent leader of POK(2012): Silvia Marek 
 2  
-  1 Edvard Vann and Isia Vann (same name as Juliana Vann both are perimeter control, Edvard 
-was interrogated for 6 hours and considered a possible suspect) 
+  1 Edvard Vann and Isia Vann (sSame last name as Juliana Vann both are perimeter control, 
+Edvard was interrogated for 6 hours and considered a possible suspect) 
 2 Elian Karel (Dead) 
 3 Juliana Vann (Dead) 
 4 Jeroen Karol (Father of Elian and original member of POK, in charge of recruitment and 
