--- VASTCER_revision_36_June_19_11:43_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_37_June_19_11:43_AM	2019-06-27 13:13:16.000000000 -0400
@@ -6,7 +6,7 @@
 3 Juliana Vann (Dead) 
 4 Isia Vann?? 
 5 Jeroen Carol (father of Elian and original member) 
-6 Mestache Vardshvna (protestor  
+6 Mestache Vardshvna (protestor , had pictures of deceased POK leaders) 
   POK protest clean water, human rights, and against the corruption of the 
 Government of Kronos and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were 
