--- VASTCER_revision_25_June_19_11:27_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_26_June_19_11:27_AM	2019-06-27 13:13:16.000000000 -0400
@@ -1,10 +1,10 @@
 Goal: Find 3 people associated with POK. Annotate Evidence and reasons for decision 
-1 Leader of POK : Silvia Marek 
+1 Leader of POK: Silvia Marek 
    1 Edvard Vann? 
 2 Elian Karel (Dead) 
 3 Juliana Vann (Dead) 
 4 Isia Vann?? 
-  POK protest clean water, human rights, and against the corruption of the 
+   POK protest clean water, human rights, and against the corruption of the 
 Government of Kronos and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were 
 recalls of the heavy toll already paid by chiefs and the innocent ones. Karel 
