--- VASTCER_revision_17_June_19_11:07_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_18_June_19_11:17_AM	2019-06-27 13:13:16.000000000 -0400
@@ -21,4 +21,5 @@
 from a protector of the member of Kronos. Vann, 
  Edvard Vann of Abila, was questioned for hours because he shares a family 
 name with several known POK members. 
- 
\ No newline at end of file
+ Bianca known as. Sgt Antonio Arresta of the department of fire of Abila, a 
+first-responder with the medical 
\ No newline at end of file
