--- VASTCER_revision_72_June_20_10:38_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_73_June_20_10:38_AM	2019-06-27 13:13:16.000000000 -0400
@@ -21,8 +21,8 @@
 unspecified emergency 
 12:40pm Additional police have arrived at GAStech headquarters. They are setting up crowd 
 control barricades around the building 
-1:50pm Attempts are made to call GAStech  
-     POK protest clean water, human rights, and against the corruption of the Government of Kronos 
+1:50pm Attempts are made to call GAStech phones but no one answers 
+      POK protest clean water, human rights, and against the corruption of the Government of Kronos 
 and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were recalls of the heavy toll 
 already paid by chiefs and the innocent ones. Karel was the former chief of the guards of 
