--- VASTCER_revision_4_June_19_10:16_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_5_June_19_10:16_AM	2019-06-27 13:13:16.000000000 -0400
@@ -7,4 +7,5 @@
 2009. Its death, thought with being a murder by, encouraged much cancel 
 gatherings to bring the message of the guards of Kronos to the public. 
 Juliana Vann died in 1998 of the cancer caused by environmental toxins in 
-its drinking water. 
\ No newline at end of file
+its drinking water. 
+ Mestache Vardshvna is a protestor 
\ No newline at end of file
