--- VASTCER_revision_57_June_20_8:40_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_58_June_20_8:43_AM	2019-06-27 13:13:16.000000000 -0400
@@ -6,7 +6,7 @@
 control, Edvard was interrogated for 6 hours and considered a possible suspect) 
 Mandor Vann is the uncle of Isia and Juliana Vann 
 2 Elian Karel (Dead former leader of POK)  
-3 Juliana Vann (Dead) 
+3 Juliana Vann (Deadied at age 10 from water contamination) 
 4 Jeroen Karol (Father of Elian and original member of POK, in charge of recruitment and 
 personel) 
 5 Mestache Vardshvna (Protestor, had pictures of deceased POK leaders) 
