--- VASTCER_revision_42_June_19_11:54_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_43_June_19_11:56_AM	2019-06-27 13:13:16.000000000 -0400
@@ -1,40 +1,33 @@
 Goal: Find 3 people associated with POK. Annotate Evidence and reasons for decision 
 1 Most recent leader of POK(2012): Silvia Marek 
 2  
-  1 Edvard Vann and Isia Vann (same name as Juliana Vann both work in same are perimeter 
-control, Edvard was interrogated for 6 hours and considered a possible suspect) 
+  1 Edvard Vann and Isia Vann (same name as Juliana Vann both are perimeter control, Edvard 
+was interrogated for 6 hours and considered a possible suspect) 
 2 Elian Karel (Dead) 
 3 Juliana Vann (Dead) 
-4 Isia Vann?? 
-54 Jeroen Carol (father of Elian and original member of POK) 
-65 Mestache Vardshvna (protestor, had pictures of deceased POK leaders) 
-  POK protest clean water, human rights, and against the corruption of the 
-Government of Kronos and GAStech International. 
- The posters with coloured images of Elian Karel and Juliana Vann were 
-recalls of the heavy toll already paid by chiefs and the innocent ones. Karel 
-was the former chief of the guards of Kronos before his untimely death in 
-2009. Its death, thought with being a murder by, encouraged much cancel 
-gatherings to bring the message of the guards of Kronos to the public. 
-Juliana Vann died in 1998 of the cancer caused by environmental toxins in 
-its drinking water. 
- Protectors of Kronos or POK were founded in 1997 by residents of rural 
-Elodis on the advice of the now defunct Wellness for All (WFA) 
- Abila Fire Department trucks were dispatched to GAStech - Kronos 
-headquarters around 10:00 this morning local Kronos time to respond to a 
-fire alarm. 
- The evident confusion comes the similarity between its name and the one 
-from a protector of the member of Kronos. Vann, 
- Edvard Vann of Abila, was questioned for hours because he shares a family 
-name with several known POK members. 
-Mestache Vardshvna, 23, of the Western coast of Kronos. 'We try to obtain 
-the attention of our government,' it said. 'It is important, thus I took leave of 
-work to be part of this important event 
- Elian (Elan) Carol had been implied POK since its beginning since his/her 
-father, Jeroen Carol, were an original member 
- Silvia Marek, LEADER of the guards of Kronos social action organization is 
-connect himself appealing on the citizens of Kronos at her for a day of 
-peaceful protest tomorrow.  
+4 Jeroen Carol (father of Elian and original member of POK) 
+5 Mestache Vardshvna (protestor, had pictures of deceased POK leaders) 
+  POK protest clean water, human rights, and against the corruption of the Government of Kronos 
+and GAStech International. 
+ The posters with coloured images of Elian Karel and Juliana Vann were recalls of the heavy toll 
+already paid by chiefs and the innocent ones. Karel was the former chief of the guards of 
+Kronos before his untimely death in 2009. Its death, thought with being a murder by, encouraged 
+much cancel gatherings to bring the message of the guards of Kronos to the public. Juliana 
+Vann died in 1998 of the cancer caused by environmental toxins in its drinking water. 
+ Protectors of Kronos or POK were founded in 1997 by residents of rural Elodis on the advice of 
+the now defunct Wellness for All (WFA) 
+ Abila Fire Department trucks were dispatched to GAStech - Kronos headquarters around 10:00 
+this morning local Kronos time to respond to a fire alarm. 
+ The evident confusion comes the similarity between its name and the one from a protector of 
+the member of Kronos. Vann, 
+ Edvard Vann of Abila, was questioned for hours because he shares a family name with several 
+known POK members. 
+Mestache Vardshvna, 23, of the Western coast of Kronos. 'We try to obtain the attention of our 
+government,' it said. 'It is important, thus I took leave of work to be part of this important event 
+ Elian (Elan) Carol had been implied POK since its beginning since his/her father, Jeroen Carol, 
+were an original member 
+ Silvia Marek, LEADER of the guards of Kronos social action organization is connect himself 
+appealing on the citizens of Kronos at her for a day of peaceful protest tomorrow.  
 Silvia Marek, leader of the Protectors of Kronos social action organization  
- Blotter arrested for “MISS MAINTAINING THE ORDER” day after POK riots in 
-Siopia 
+ Blotter arrested for “MISS MAINTAINING THE ORDER” day after POK riots in Siopia 
  2/23/2009 
\ No newline at end of file
