--- VASTCER_revision_26_June_19_11:27_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_27_June_19_11:28_AM	2019-06-27 13:13:16.000000000 -0400
@@ -4,6 +4,7 @@
 2 Elian Karel (Dead) 
 3 Juliana Vann (Dead) 
 4 Isia Vann?? 
+5 Jeroen Carol (father of Elian and original member) 
    POK protest clean water, human rights, and against the corruption of the 
 Government of Kronos and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were 
