--- VASTCER_revision_47_June_20_8:24_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_48_June_20_8:26_AM	2019-06-27 13:13:16.000000000 -0400
@@ -8,8 +8,8 @@
 3 Juliana Vann (Dead) 
 4 Jeroen Karol (father of Elian and original member of POK) 
 5 Mestache Vardshvna (protestor, had pictures of deceased POK leaders) 
-6 Carmine Osvaldo  
-7 Henk Bodrogi 
+6 Carmine Osvaldo (Began leadership of POK) 
+7 Henk Bodrogi (Recruited by Osvaldo for assistance) 
   POK protest clean water, human rights, and against the corruption of the Government of Kronos 
 and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were recalls of the heavy toll 
