--- VASTCER_revision_62_June_20_8:49_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_63_June_20_8:51_AM	2019-06-27 13:13:16.000000000 -0400
@@ -3,7 +3,7 @@
 1 Most recent leader of POK(2012): Silvia Marek 
 2  
   1 Edvard Vann and Isia Vann - Isia Vann is Juliana Vann’s older brother,  both are perimeter 
-control, Edvard was interrogated for 6 hours and considered a possible suspect 
+control for GAStech, Edvard was interrogated for 6 hours and considered a possible suspect 
 Mandor Vann - The uncle of Isia and Juliana Vann 
 2 Elian Karel - Dead former leader of POK 
 3 Juliana Vann - Died at age 10 from water contamination 
