--- VAST5KP_revision_68_June_20_9:17_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_69_June_20_9:18_AM	2019-06-27 13:10:44.000000000 -0400
@@ -43,7 +43,7 @@
 between the main office and a Kronos Government reception. 
  Recommendations/What to do next: 
 At this point, based on supplemental data I have identified Isia Vann as a prominent member of 
-the POK who is a proponent of using violence. My next step would be to notify the police of the 
-GAStech - POK connection of Isia Vann. Also, I would start an in depth persona development on 
-Isia Vann. 
+the POK who is a proponent of using violence. Additionally I believe I may have identified a 
+possible love interest in Rachel My next step would be to notify the police of the GAStech - POK 
+connection of Isia Vann. Also, I would start an in depth persona development on Isia Vann. 
           
\ No newline at end of file
