--- VAST5KP_revision_61_June_20_9:05_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_62_June_20_9:06_AM	2019-06-27 13:10:44.000000000 -0400
@@ -34,7 +34,8 @@
 Tiskele Bend. (POK Notes) 
 ■Same last name as Juliana Vann, a POK martyr (The annual ends of the 
 collection POK in babble, again) 
-   General knowledge: 
+Other people of interest 
+  General knowledge: 
 ●Based on reporting from “The Truth” it appears a fire was set at the main office of 
 Gastech in order to cause a distraction with the kidnapping taking place somewhere 
 between the main office and a Kronos Government reception. 
