--- VAST5KP_revision_77_June_20_9:43_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_78_June_20_9:44_AM	2019-06-27 13:10:44.000000000 -0400
@@ -41,8 +41,10 @@
 ○Would be a target to a terrorist organization trying to gain information  
 ●Henk Mies 
 ○Truck driver for GAStech 
-○I found it interesting that there is direct email communication between  
- General knowledge: 
+○I found it interesting that there is direct email communication between Henk 
+Mies and Sten Sanjorge Jr 
+ ○ the CEO of GAStech referring to files 
+General knowledge: 
 ●Based on reporting from “The Truth” it appears a fire was set at the main office of 
 Gastech in order to cause a distraction with the kidnapping taking place somewhere 
 between the main office and a Kronos Government reception. 
