--- VAST5KP_revision_67_June_20_9:16_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_68_June_20_9:17_AM	2019-06-27 13:10:44.000000000 -0400
@@ -36,7 +36,7 @@
 collection POK in babble, again) 
  Other people of interest: 
 ●Rachel.Pantanal@gastech.com.kronos 
-○ 
+○Possible girlfriend of Isia Vann 
    General knowledge: 
 ●Based on reporting from “The Truth” it appears a fire was set at the main office of 
 Gastech in order to cause a distraction with the kidnapping taking place somewhere 
