--- VASTGZ2_revision_14_June_19_10:52_AM	2019-06-27 13:14:02.000000000 -0400
+++ VASTGZ2_revision_15_June_19_10:57_AM	2019-06-27 13:14:02.000000000 -0400
@@ -9,11 +9,14 @@
 -Says they are from Kronos and in the Kronovian Armed forces 
  Mies Harber Ruscella is not listed in resumes 
 It is shown as Minke Mies 
- Minke.Mies@gastech.com.kronos 
-Henk.Mies@gastech.com.kronos,  
   POK - 1998 in Elodis, a rural township outside of the capital Abila. They have 
 been protesting for clean water, human rights, and against the corruption of 
 the Government of Kronos and GAStech International. 
  Security , Truck Driver, and Assistant to Engineering group manager all 
 same last name. 
+ To: Nils.Calixto@gastech.com.kronos, Sven.Flecha@gastech.com.kronos 
+They always appear in emails together with only one member included 
+Sometimes none 
+ (other than the re-organization emails) 
+Could be trying to recruit via email. 
  
\ No newline at end of file
