--- VASTGZ2_revision_16_June_19_10:59_AM	2019-06-27 13:14:02.000000000 -0400
+++ VASTGZ2_revision_17_June_19_11:05_AM	2019-06-27 13:14:02.000000000 -0400
@@ -19,5 +19,5 @@
 Sometimes none 
  (other than the re-organization emails) 
 Could be trying to recruit via email. 
- Says from tethys but types like Kronovians 
+ Sven: Says from tethys but types like Kronovians 
  
\ No newline at end of file
