--- VASTOIC_revision_95_July_18_11:03_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_96_July_18_11:18_AM	2019-07-30 15:57:21.000000000 -0400
@@ -64,4 +64,5 @@
 Compare list to security personnel  
 Get flight manifests from Abila airport  
 Request security footage from GAStech HQ in Abila 
-  
\ No newline at end of file
+ Hideki.Cocinaro@gastech.com.kronos, Varja.Lagos@gastech.com.kronos, 
+Isia.Vann@gastech.com.kronos 
\ No newline at end of file
