--- VASTOIC_revision_48_July_18_9:04_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_49_July_18_9:04_AM	2019-07-30 15:57:21.000000000 -0400
@@ -16,8 +16,8 @@
 Silvia Marek - founded the group Save Our Wildlands and was recruited by Karel in ‘05 
 (encourages peaceful engagement) 
 Mandor Vann - retired military, linguist, political strategist, related to Isia and Juliana Vann 
-Isia Vann - brother of Juliana Vann (deceased) -  
-  The gourp’s first actions included contacting international agencies and speaking to GAStech 
+Isia Vann - brother of Juliana Vann (deceased) - history of civil disobedience and trespassing 
+   The gourp’s first actions included contacting international agencies and speaking to GAStech 
 CEO Sten Sanjorge Jr.  After the meeting and test results from the WFA on the river water the 
 group increased it’s lobbying in the capital of Abila, conducted peaceful protests, and met with 
 low-level government personnel.  Cesare Nespola, Minister of Health, attempted to introduce 
