--- VASTOIC_revision_75_July_18_10:05_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_76_July_18_10:13_AM	2019-07-30 15:57:21.000000000 -0400
@@ -1,13 +1,7 @@
 VastOIC 
  Fire occurred around 1/20/14 1000 
  GAStech Edvard Vann interrogated - said POK was responsible 
- KOP  
-Julian Vann (deceased) 
-Elian Karel 
-Henk Bodrogi - leader 
- Vann, Isia (securtiy) 
-Vann, Edvard (security) 
-Bodrogi, Loreto (security)GAStech Security: 
+ GAStech Security: 
 Felix.Resumir@gastech.com.kronos 
 Varja.Lagos@gastech.com.kronos 
 Edvard.Vann@gastech.com.kronos 
@@ -18,7 +12,8 @@
 Kanon.Herrero@gastech.com.kronos 
 Hennie.Osvaldo@gastech.com.kronos 
 Minke.Mies@gastech.com.kronos 
-  GAStech Employees of Interest:  
+Cornelia.Lais@gastech.com.kronos 
+   GAStech Employees of Interest:  
 Isia Vann (security) 
 Edvard Vann (security) 
 Hennie Osvaldo (security) 
@@ -38,8 +33,7 @@
   The gourp’s first actions included contacting international agencies and speaking to GAStech 
 CEO Sten Sanjorge Jr.  After the meeting and test results from the WFA on the river water the 
 group increased it’s lobbying in the capital of Abila, conducted peaceful protests, and met with 
-low-level government personnel.  Cesare Nespola, Minister of Health, attempted to introduce 
-legislation to tax the oil and gas production to support healthcare and clean water, which was 
+low-level government personnel.  Cesare Nespola, Minister of Health, attempted to introduce legislation to tax the oil and gas production to support healthcare and clean water, which was 
 defeated.  After which the group began protesting at the Tiskele Bend fields.  
  Once Kerel succeeded Bodrogi the remaining founding members all left the POK.  Karel began 
 recruiting other small environmental organizations into the POK and the peaceful protests 
