--- VASTOIC_revision_31_July_18_8:32_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_32_July_18_8:36_AM	2019-07-30 15:57:21.000000000 -0400
@@ -11,6 +11,7 @@
   Origional Members: 
 Henk Bodrogi - farmer, military  
 Carmine Osvaldo - Elodis chamber of commerce, military with Bodrogi 
+Cesare Nespola - Minister of Health 
 Elian Karel 
    The gourp’s first actions included contacting international agencies and speaking to GAStech 
 CEO Sten Sanjorge Jr.  After the meeting and test results from the WFA on the river water the 
