--- VASTOIC_revision_82_July_18_10:33_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_83_July_18_10:39_AM	2019-07-30 15:57:21.000000000 -0400
@@ -38,7 +38,7 @@
 recruiting other small environmental organizations into the POK and the peaceful protests 
 began to become more violent and militant.  
 Time Line: 
-1/20/2014 - GAStech annual corporate mettingmeeting 
+1/20/2014 - GAStech annual corporate meeting 
 1/20/2014 - Fire reported at GAStech HQ at 1000 
 1/20/2014 - 15 GAStech employees reported missing 
 1/20/2014 - GAStech Edvard Vann interrogated - said POK was responsible 
@@ -47,10 +47,11 @@
 1/20/2014 - GAStech Senior Security Officer at 1400 
 1/20/2014 - Immediate arrival/refueling and departure of GAStech CEO’s private jet 
 ●Departed Tethys, returning to Abila w/ 6 GAStech employees onboard 
+1/20/2014 - Private jet departed Abila with relaxed business types at 1430 
 1/21/2014 - 5 Gastech execs located, 10 employees still missing at 0900 
 1/21/2014 - Pertrus Gerhard released public statement from POK at 1153 
  Possible Explanation: 
-Several members of GAStech security who have possible familiral family connections to POK 
-leadership may have abducted the missing employees and concealed their 
-movement/abduction from the building using a fire alarm.  
+Several members of GAStech security who have possible family connections to POK leadership 
+may have abducted the missing employees and concealed their movement/abduction from the 
+building using a fire alarm.  
  
\ No newline at end of file
