--- VASTOIC_revision_88_July_18_10:44_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_89_July_18_10:45_AM	2019-07-30 15:57:21.000000000 -0400
@@ -46,9 +46,11 @@
 1/20/2014 - $20M ransom demand made by group claiming to be the POK (for CEO) 
 1/20/2014 - Private jet departed Abila with 7-8 business types (in a hurry) at 1230 
 1/20/2014 - GAStech Senior Security Officer at arrived in Abila at 1400 
+●Departed Abila to UNK LOC 
 1/20/2014 - Immediate arrival/refueling and departure of GAStech CEO’s private jet 
 ●Departed Tethys, returning to Abila w/ 6 GAStech employees onboard 
 1/20/2014 - Private jet departed Abila with 7-8 relaxed business types at 1430 
+●Departed Abilia to Italy 
 1/21/2014 - 5 Gastech execs located, 10 employees still missing at 0900 
 1/21/2014 - Pertrus Gerhard released public statement from POK at 1153 
  Possible Explanation: 
