--- VASTOIC_revision_57_July_18_9:20_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_58_July_18_9:20_AM	2019-07-30 15:57:21.000000000 -0400
@@ -1,5 +1,5 @@
 VastOIC 
- Fire occurred around 1/20/14 101500 
+ Fire occurred around 1/20/14 1000 
  GAStech Edvard Vann interrogated - said POK was responsible 
  KOP  
 Julian Vann (deceased) 
@@ -32,4 +32,4 @@
 defeated.  After which the group began protesting at the Tiskele Bend fields.  
  Once Kerel succeeded Bodrogi the remaining founding members all left the POK.  Karel begain 
 recruiting other small environmental organizations into the POK and the peaceful protests 
-began to become more violent and militant.   
\ No newline at end of file
+began to become more violent and militant.  Fire occurred around 1/20/14 1000 
\ No newline at end of file
