--- VASTOIC_revision_93_July_18_11:02_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_94_July_18_11:03_AM	2019-07-30 15:57:21.000000000 -0400
@@ -1,5 +1,5 @@
 VastOIC 
-    GAStech Security emails: 
+ GAStech Security emails: 
 Felix.Resumir@gastech.com.kronos 
 Varja.Lagos@gastech.com.kronos 
 Edvard.Vann@gastech.com.kronos 
@@ -11,7 +11,7 @@
 Hennie.Osvaldo@gastech.com.kronos 
 Minke.Mies@gastech.com.kronos 
 Cornelia.Lais@gastech.com.kronos (manager) 
-   GAStech Employees of Interest:  
+  GAStech Employees of Interest:  
 Isia Vann (security) 
 Edvard Vann (security) 
 Hennie Osvaldo (security) 
@@ -20,6 +20,7 @@
 Henk Bodrogi - farmer, military  
 Carmine Osvaldo - Elodis chamber of commerce, military with Bodrogi 
 Jeroen Karel 
+l - Founder, father of Elian 
 Elian Karel - Bodrogi’s successor, leader and major recruiter for the POK, could turn the group 
 into a political organization 
 Silvia Marek - founded the group Save Our Wildlands and was recruited by Karel in ‘05 
@@ -29,11 +30,12 @@
 ●Security Officer at GAStech 
 Lucio Jakab - co-founder of SOW, leads communications for the group 
 Lorenjzo Di Stefano - college professor at UofAbila 
-  The gourp’s first actions included contacting international agencies and speaking to GAStech 
-CEO Sten Sanjorge Jr.  After the meeting and test results from the WFA on the river water the 
-group increased it’s lobbying in the capital of Abila, conducted peaceful protests, and met with 
-low-level government personnel.  Cesare Nespola, Minister of Health, attempted to introduce legislation to tax the oil and gas production to support healthcare and clean water, which was 
-defeated.  After which the group began protesting at the Tiskele Bend fields.  
+  The gourp’sgroup's first actions included contacting international agencies and speaking to 
+GAStech CEO Sten Sanjorge Jr.  After the meeting and test results from the WFA on the river 
+water the group increased it’s lobbying in the capital of Abila, conducted peaceful protests, and 
+met with low-level government personnel.  Cesare Nespola, Minister of Health, attempted to 
+introduce legislation to tax the oil and gas production to support healthcare and clean water, 
+which was defeated.  After which the group began protesting at the Tiskele Bend fields.  
  Once Kerel succeeded Bodrogi the remaining founding members all left the POK.  Karel began 
 recruiting other small environmental organizations into the POK and the peaceful protests 
 began to become more violent and militant.  
