--- VASTOIC_revision_12_July_17_11:13_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_13_July_17_11:15_AM	2019-07-30 15:57:21.000000000 -0400
@@ -1,10 +1,7 @@
 VastOIC 
  Fire occurred around 1/20/14 1015 
  GAStech Edvard Vann interrogated - said POK was responsible 
- Kronos personnel 
-●Mies Haber, Ruscella 
-●Forluniau, Carla 
-●Lais, Cornelia 
-KOP  
+ KOP  
 Julian Vann (deceased) 
-   
\ No newline at end of file
+Karel 
+  
\ No newline at end of file
