--- VASTOIC_revision_59_July_18_9:21_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_60_July_18_9:29_AM	2019-07-30 15:57:21.000000000 -0400
@@ -33,4 +33,6 @@
  Once Kerel succeeded Bodrogi the remaining founding members all left the POK.  Karel begain 
 recruiting other small environmental organizations into the POK and the peaceful protests 
 began to become more violent and militant.  Time Line: 
- Fire occurred around 1/20/14 1000 
\ No newline at end of file
+ 1/20/2014 - Fire reported at GAStech HQ 
+ Fire occurred around 1/20/14 10001/20/2014 - 15 GAStech employees reported missing 
+ 
\ No newline at end of file
