--- VASTOIC_revision_61_July_18_9:29_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_62_July_18_9:31_AM	2019-07-30 15:57:21.000000000 -0400
@@ -35,5 +35,6 @@
 began to become more violent and militant.  Time Line: 
  1/20/2014 - Fire reported at GAStech HQ 
 1/20/2014 - 15 GAStech employees reported missing 
+1/20/2014 - GAStech CEO found 
 1/21/2014 - 5 Gastech execs located, 10 employees still missing 
  
\ No newline at end of file
