--- VASTOIC_revision_35_July_18_8:41_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_36_July_18_8:41_AM	2019-07-30 15:57:21.000000000 -0400
@@ -18,5 +18,5 @@
 low-level government personnel.  Cesare Nespola, Minister of Health, attempted to introduce 
 legislation to tax the oil and gas production to support healthcare and clean water, which was 
 defeated.  After which the group began protesting at the Tiskele Bend fields.  
- Once Kerel succeded Bodrogi the remaining founding members all left the POK.  
+ Once Kerel succededsucceeded Bodrogi the remaining founding members all left the POK.  
  
\ No newline at end of file
