--- VASTOIC_revision_98_July_18_11:29_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_99_July_18_11:30_AM	2019-07-30 15:57:21.000000000 -0400
@@ -21,7 +21,7 @@
 Carmine Osvaldo - Elodis chamber of commerce, military with Bodrogi 
 Jeroen Karel - Founder, father of Elian 
 Elian Karel - Bodrogi’s successor, leader and major recruiter for the POK, could turn the group 
-into a political organization 
+into a political organization (deceased) 
 Silvia Marek - founded the group Save Our Wildlands and was recruited by Karel in ‘05 
 (encourages peaceful engagement) 
 Mandor Vann - retired military, linguist, political strategist, related to Isia and Juliana Vann 
