--- VASTOIC_revision_94_July_18_11:03_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_95_July_18_11:03_AM	2019-07-30 15:57:21.000000000 -0400
@@ -19,8 +19,7 @@
   Leadership: 
 Henk Bodrogi - farmer, military  
 Carmine Osvaldo - Elodis chamber of commerce, military with Bodrogi 
-Jeroen Karel 
-l - Founder, father of Elian 
+Jeroen Karel - Founder, father of Elian 
 Elian Karel - Bodrogi’s successor, leader and major recruiter for the POK, could turn the group 
 into a political organization 
 Silvia Marek - founded the group Save Our Wildlands and was recruited by Karel in ‘05 
@@ -30,16 +29,16 @@
 ●Security Officer at GAStech 
 Lucio Jakab - co-founder of SOW, leads communications for the group 
 Lorenjzo Di Stefano - college professor at UofAbila 
-  The gourp’sgroup's first actions included contacting international agencies and speaking to 
-GAStech CEO Sten Sanjorge Jr.  After the meeting and test results from the WFA on the river 
-water the group increased it’s lobbying in the capital of Abila, conducted peaceful protests, and 
-met with low-level government personnel.  Cesare Nespola, Minister of Health, attempted to 
-introduce legislation to tax the oil and gas production to support healthcare and clean water, 
-which was defeated.  After which the group began protesting at the Tiskele Bend fields.  
+  The group's first actions included contacting international agencies and speaking to GAStech 
+CEO Sten Sanjorge Jr.  After the meeting and test results from the WFA on the river water the 
+group increased it’s lobbying in the capital of Abila, conducted peaceful protests, and met with 
+low-level government personnel.  Cesare Nespola, Minister of Health, attempted to introduce 
+legislation to tax the oil and gas production to support healthcare and clean water, which was 
+defeated.  After which the group began protesting at the Tiskele Bend fields.  
  Once Kerel succeeded Bodrogi the remaining founding members all left the POK.  Karel began 
 recruiting other small environmental organizations into the POK and the peaceful protests 
 began to become more violent and militant.  
-Time Line: 
+  Time Line: 
 1/20/2014 - GAStech annual corporate meeting 
 1/20/2014 - Fire reported at GAStech HQ at 1000 
 1/20/2014 - 15 GAStech employees reported missing 
