--- VASTOIC_revision_60_July_18_9:29_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_61_July_18_9:29_AM	2019-07-30 15:57:21.000000000 -0400
@@ -34,5 +34,6 @@
 recruiting other small environmental organizations into the POK and the peaceful protests 
 began to become more violent and militant.  Time Line: 
  1/20/2014 - Fire reported at GAStech HQ 
- Fire occurred around 1/20/14 10001/20/2014 - 15 GAStech employees reported missing 
+1/20/2014 - 15 GAStech employees reported missing 
+1/21/2014 - 5 Gastech execs located, 10 employees still missing 
  
\ No newline at end of file
