--- VASTBNJ_revision_70_June_20_10:16_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_71_June_20_10:16_AM	2019-06-27 13:11:32.000000000 -0400
@@ -23,9 +23,9 @@
 ○Passport Issued: 01/01/2000 / Passport Expires: 01/01/2000 
 ○Military: Armed Forces of Kronos / Discharge: 10/01/1984 / Honorable 
  ●Forluniau, Carla 
-●Position: Assistant to IT Group Manager (Administration) 
-●Date of Hire: 03/07/2005 
-●Email: Carla.Forluniau@gastech.com.kronos 
+○Position: Assistant to IT Group Manager (Administration) 
+○Date of Hire: 03/07/2005 
+○Email: Carla.Forluniau@gastech.com.kronos 
 ● 
 ●Sex: Female 
 ●DOB: 06/02/1981 
