--- VASTBNJ_revision_12_June_19_10:15_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_13_June_19_10:17_AM	2019-06-27 13:11:32.000000000 -0400
@@ -9,4 +9,8 @@
 succeed today. 
 ○Competition among Kronosian and foreign companies is hard. It is difficult to gain the 
 hearts and the spirits of the population.  
-○ 
\ No newline at end of file
+○people think between the culture of Kronosian and, culture of Tethian of word. The 
+personnel of Kronos function just like hard, they are respectful of their work and have the 
+same desire for success. They can also appreciate the effective communication and a 
+gifted chief. Which foreign chief must make is to learn how to communicate the 'manner 
+of Kronosian' and to take time to include/understand their personnel. 
\ No newline at end of file
