--- VASTBNJ_revision_53_June_20_9:33_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_54_June_20_9:33_AM	2019-06-27 13:11:32.000000000 -0400
@@ -9,11 +9,10 @@
 Hennie.Osvaldo@gastech.com.kronos, Felix.Resumir@gastech.com.kronos, 
 Minke.Mies@gastech.com.kronos 
  ●KIDNAPPING AT GASTECH KRONOS|Who What News | 01/20/2014 
-○Edvard Vann, a GAStech - Kronos security guard who was interrogated and released 
-after 6 hours, indicated the Protectors of Kronos were suspected as being the 
-perpetrators behind today's events. Speculation about the motivation behind the 
-kidnapping has also been rampant due to the recent GAStech IPO. At this time no one has 
-claimed responsibility for the kidnapping. 
+●Edvard Vann, a GAStech - Kronos security guard who was interrogated and released after 6 
+hours, indicated the Protectors of Kronos were suspected as being the perpetrators behind 
+today's events. Speculation about the motivation behind the kidnapping has also been rampant 
+due to the recent GAStech IPO. At this time no one has claimed responsibility for the kidnapping. 
 ●Maha Salo |  
 ●-The perspicacity of an expert: prosperity as foreign PRESIDENT in Kronos|World Journal | 
 03/26/2014: 
