--- VASTBNJ_revision_69_June_20_10:15_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_70_June_20_10:16_AM	2019-06-27 13:11:32.000000000 -0400
@@ -22,8 +22,23 @@
 ○Sex: Female / DOB: 04/26/1964 / Citizenship: Kronos 
 ○Passport Issued: 01/01/2000 / Passport Expires: 01/01/2000 
 ○Military: Armed Forces of Kronos / Discharge: 10/01/1984 / Honorable 
- ●Discharge Type:  Discharge 
- ●-The perspicacity of an expert: prosperity as foreign PRESIDENT in Kronos|World Journal | 
+ ●Forluniau, Carla 
+●Position: Assistant to IT Group Manager (Administration) 
+●Date of Hire: 03/07/2005 
+●Email: Carla.Forluniau@gastech.com.kronos 
+● 
+●Sex: Female 
+●DOB: 06/02/1981 
+● 
+●Citizenship: Kronos 
+●Passport Issued: 01/01/2000 
+●Passport Expires: 01/01/2000 
+● 
+●Military Branch: Armed Forces of Kronos 
+●Discharge Date: 10/01/2001 
+●Discharge Type: General Discharge 
+● 
+  ●-The perspicacity of an expert: prosperity as foreign PRESIDENT in Kronos|World Journal | 
 03/26/2014: 
 ○GAStech, the principal industry of Krono during more than 15 years, after its discovery of 
 its immense gas layers close to the town of Elodis. 
