--- VASTBNJ_revision_71_June_20_10:16_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_72_June_20_10:21_AM	2019-06-27 13:11:32.000000000 -0400
@@ -26,10 +26,11 @@
 ○Position: Assistant to IT Group Manager (Administration) 
 ○Date of Hire: 03/07/2005 
 ○Email: Carla.Forluniau@gastech.com.kronos 
-● 
-●Sex: Female 
+○ 
+●Sex: Female / DOB: 06/02/1981/ Citizenship: Kronos 
 ●DOB: 06/02/1981 
-● 
+●
+Passport Issued: 01/01/2000 / Expires: 01/01/2000 
 ●Citizenship: Kronos 
 ●Passport Issued: 01/01/2000 
 ●Passport Expires: 01/01/2000 
