--- VAST6D8_revision_23_June_20_10:16_AM	2019-06-27 13:12:58.000000000 -0400
+++ VAST6D8_revision_24_June_20_10:18_AM	2019-06-27 13:12:58.000000000 -0400
@@ -1,4 +1,4 @@
-Edvard Vann, protector of security for GAStech, (part of  
+Edvard Vann, protector of security for GAStech, (part of sec 
  Elan Carol or Elian Karel died 2009 former member 
 Juliana Vann died 1998 
  Jeroen Carol member since 1997 (Elan Carol father) 
