--- VASTPSZ_revision_18_July_18_8:59_AM	2019-07-22 15:24:53.000000000 -0400
+++ VASTPSZ_revision_19_July_18_8:59_AM	2019-07-22 15:24:53.000000000 -0400
@@ -10,7 +10,7 @@
 Disappeared after council meeting at hoofdkwartier  
 Distraction - Fire alarm 
 Jeroen Carol, were an original member of POK 
-two founders - Sten Sanjorge Sr and Hank Fluss 
+GAStech two founders - Sten Sanjorge Sr and Hank Fluss 
     Emile Arpa 
 Irene Nant 
 Mies Haber, Ruscella 
