--- VASTXYZ-2_revision_2_July_18_8:54_AM	2019-07-30 16:01:41.000000000 -0400
+++ VASTXYZ-2_revision_3_July_18_8:55_AM	2019-07-30 16:01:41.000000000 -0400
@@ -4,13 +4,13 @@
 International, has confirmed that members of the GAStech leadership are in 
 fact missing. The group was expected to arrive at the Government 
 reception following this morning's meeting at GAStech - Kronos 
-headquarters, but never showed up. Avila denied that any GAStech 
-employee has been abducted. The GAStech - Kronos leadership was also 
-handsomely rewarded. The IPO filings show that the executive group 
-earned a collective $327.25 million from the newly vested shares including 
-$96.25 million each for CFO Ingrid Barranco, CIO Ada Campo-Corrente, and 
-COO Orhan Strum. maintains an office in Centrum, the capital of Tethys, 
-most of GAStech International's operations take place on Kronos including 
-production of about 5 million cubic meters of gas daily. Avila works in 
-GAStech's Centrum headquarters. 
+headquarters, but never showed up. (indication a driver was involved). Avila 
+denied that any GAStech employee has been abducted. The GAStech - 
+Kronos leadership was also handsomely rewarded. The IPO filings show 
+that the executive group earned a collective $327.25 million from the newly 
+vested shares including $96.25 million each for CFO Ingrid Barranco, CIO 
+Ada Campo-Corrente, and COO Orhan Strum. maintains an office in 
+Centrum, the capital of Tethys, most of GAStech International's operations 
+take place on Kronos including production of about 5 million cubic meters 
+of gas daily. Avila works in GAStech's Centrum headquarters. 
  
\ No newline at end of file
