--- VASTXYZ-2_revision_5_July_18_9:03_AM	2019-07-30 16:01:41.000000000 -0400
+++ VASTXYZ-2_revision_6_July_18_9:03_AM	2019-07-30 16:01:41.000000000 -0400
@@ -13,21 +13,8 @@
 Centrum, the capital of Tethys, most of GAStech International's operations 
 take place on Kronos including production of about 5 million cubic meters 
 of gas daily. Avila works in GAStech's Centrum headquarters. 
-        KOPERACHTIGE THE ATTACK OF TIME OF 
-THE DAY POK 
-News Desk | 12/18/2013 
-ABILA, Kronos - koperachtige the attack of the day time on two peace civil 
-servants POK are only another violent, terroristic operation in a long tie of 
-such operations. The two civil servants were left bloodily and were beaten, 
-and were linked in the tree-tame of their vandalized vehicle. The 
-spokesman Adrien Carman of the police force aforesaid of the attack, 
-attacking a police force civil servant and destroying government 
-possession is serious crime. I want it public insure that these people to 
-justice will shortly be caught and will be brought. Furthermore the 
-worsening situation, schendden the two criminals the car of the civil 
-servants by painting the emblem POK nevel. If they two can attack police 
-force civil servants in light day, which is following?THE POK MEETING DRAWS THOUSANDS OF THOSE IN 
-FAVOR 
+        THE POK MEETING DRAWS THOUSANDS OF 
+THOSE IN FAVOR 
 Daily Pegasus | 03/09/2009 
 ABILA, Kronos - the thousands of demonstrators who take the flags that 
 read 'remember Julian' and singing 'we wished truth' reunited in front of the 
@@ -52,8 +39,8 @@
 'What we needed it is the transparency in our government. We need to know 
 what happens to all the money that comes inside from these foreign 
 companies. The leader Elian Karel of the protest dealed with the 
-demonstrators in the park the city of Abila. Karel assumed the control the 
-direction of POK of Henk Bodrogi, that to bad forced it health under the position. Karel spoke eloquent on the principles of the campaign of POK 
+demonstrators in the park the city of Abila. Karel assumed the control the direction of POK of Henk Bodrogi, that to bad forced it health under the 
+position. Karel spoke eloquent on the principles of the campaign of POK 
 like effort of the towns for the clean water, to the dynamic movement it has 
 the improved transparency demanding turned and responsibility of the 
 government.'We are not requesting any desrazonable thing,' Karel speaks 
