--- VASTKW0_revision_11_June_20_9:21_AM	2019-06-27 13:09:31.000000000 -0400
+++ VASTKW0_revision_12_June_20_9:31_AM	2019-06-27 13:09:31.000000000 -0400
@@ -4,6 +4,6 @@
 1400 2nd Plane departs (8 people; more relaxed) 
  Possible Members of the POK 
 Isia Vann (Security officer) 
-Osvaldo 
-Loreto Borgidodrogi (Security Officer) 
+Hennie Osvaldo 
+Loreto Bodrogi (Security Officer) 
  
\ No newline at end of file
