--- VASTKW0_revision_7_June_20_8:52_AM	2019-06-27 13:09:31.000000000 -0400
+++ VASTKW0_revision_8_June_20_9:19_AM	2019-06-27 13:09:31.000000000 -0400
@@ -2,4 +2,8 @@
 1000 Fire Trucks respond to fire alarm at GasTech 
 12:30 Plane Departs (8 business like people in a hurry) 
 1400 2nd Plane departs (8 people; more relaxed) 
+ Possible Members of the POK 
+Isia Vann 
+Osvaldo 
+Borgido 
  
\ No newline at end of file
