--- VASTVD5_revision_14_July_17_10:32_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_15_July_17_10:33_AM	2019-07-30 16:05:39.000000000 -0400
@@ -6,6 +6,7 @@
 metadata 
 3.Petrus Gerhard - News correspondent who seemingly exclusively writes about POK and 
 environmental issues; active since 1998 
+a.“The next news  
  Employees of GASTech removed in Kronos / World Journal | 01/21/2014:  
 14 kidnapped 
 5 of 14 were execs 
