--- VASTVD5_revision_51_July_18_9:00_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_52_July_18_9:00_AM	2019-07-30 16:05:39.000000000 -0400
@@ -1,16 +1,17 @@
- Identify 3 potential associates of “Protectors of Kronos” (POK) and describe their involvement 
+ Identify 3 potential associates of “Protectors of Kronos” (POK) and describe their 
+involvement 
  1.Silvia Marek - leader, calls for public assembly and protest 
 2.Loreto Bodrogi - Possible relative of Henk Bodrogi (former POK leader) 
 a.Works in security for GASTech 
-b.Involved in security planning and procedures for VIP events on 01/20 via email 
-metadata 
-3.Petrus Gerhard - News correspondent who seemingly exclusively writes about POK and 
-environmental issues; active since 1998 
-a.“Readers, the next post you read will be one of the most important you have ever 
-read, or indeed, will ever read. 21 Jan 2014” - prior to POK’s announcement of 
-their responsibility in the kidnapping 
- Edvard vann, isia vann - possible relation to Juliana Vann, both briefed on 20 JAN security 
-protocols / on the security staff 
+b.Involved in security planning and procedures for VIP events on 01/20 via 
+email metadata 
+3.Petrus Gerhard - News correspondent who seemingly exclusively writes about 
+POK and environmental issues; active since 1998 
+a.“Readers, the next post you read will be one of the most important you 
+have ever read, or indeed, will ever read. 21 Jan 2014” - prior to POK’s 
+announcement of their responsibility in the kidnapping 
+ Edvard vann, isia vann - possible relation to Juliana Vann, both briefed on 20 JAN 
+security protocols / on the security staff 
  *Isia Vann - according to Psycorps report, older brother of Juliana Vann 
  Employees of GASTech removed in Kronos / World Journal | 01/21/2014:  
 14 kidnapped (updated to 10 from Abila PD) 
@@ -24,12 +25,11 @@
  COLLECTION WHICH IS PLANNED TO SUPPORT OF THE RAISED DUTY OF 
 RESPONSIBILITY OF THE GOVERNMENT / The Wrap | 04/11/2012 
  Claims Silvia Marek as a/the leader of POK 
-  List of people with access to 01/20 Security Procedures 
-Hennie.Osvaldo@gastech.com.kronos, Loreto.Bodrogi@gastech.com.kronos, 
+  List of people with access to 01/20 Security Procedures  Hennie.Osvaldo@gastech.com.kronos, Loreto.Bodrogi@gastech.com.kronos, 
 Varja.Lagos@gastech.com.kronos, Inga.Ferro@gastech.com.kronos, 
 Kanon.Herrero@gastech.com.kronos, Hideki.Cocinaro@gastech.com.kronos, 
 Minke.Mies@gastech.com.kronos 
-   1.POK membership, power structure 
+  1.POK membership, power structure 
 a.Silvia Marek, leader 
 b.Petrus Gerhard, spokesman/media outlet, perhaps not direct member 
 c.High probability members 
@@ -39,88 +39,79 @@
 a.Henk Bodrogi and Carmine Osvaldo formed POK; peaceful protests 
 b.Elian Karel in charge after Bodrogi until his death in prison, 2009; peaceful 
 protests, seeking political involvement for POK 
-c.Silvia Marek current leader; initially involved in peaceful protests; radicalized and 
-engaged in terroristic acts more recently -- probably motivated by Elian Karel’s 
-mysterious death in prison 
+c.Silvia Marek current leader; initially involved in peaceful protests; 
+radicalized and engaged in terroristic acts more recently -- probably 
+motivated by Elian Karel’s mysterious death in prison 
 3.POK connection between members and GASTech employees 
 a.Isia Vann 
 b.Loreto Bodrogi 
   Timelime of 01/20 - 01/21 
- 1853 UPDATE - Speculation that the 2:00 PM arrival of Kronos Government 
-official was in fact the arrival of a senior security executive and other 
-officials from GAStech's Tethys office including President and CEO Sten 
-Sanjorge, Jr. This speculation is based on information concerning the 
-arrival, refueling and immediate departure of Sanjorge Jr.'s private jet from 
-the Tethys airport. 
- 1947 UPDATE - From the Abila Police news conference: 'There are 
-approximately fourteen individuals unaccounted for among the GAStech 
-staff. It has not yet been determined if all of these individuals are in fact 
-missing.'Though it was confirmed that over a dozen individuals were 
-unaccounted for, the Abila Police said that that does not mean they are 
-necessarily missing. Nor does it mean that they have been abducted. At 
-this point the case is under investigation and more information will be 
-released as facts are learned. The Abila Police stressed that anyone seeing 
-any unusual activity should immediately report it. 
-  1330 UPDATE - At 1315 we placed a call to GAStech headquarters. A 
-hesitant administrative assistant to Orhan Strum, Chief Operating Officer of 
-GAStech claimed Mr. Strum was unavailable. Linda Lagos the assistant to 
-Mr. Strum declined to answer any further questions about the incident, 
-whether or not there was a fire, a false alarm, a bomb threat, or whether or 
-not anyone is missing. Uneasy throughout the call, Ms. Lagos repeated that 
-Orhan Strum was unavailable at this time, said good bye and hung up. 
-Additional calls were placed with no answer on the GAStech end of the line. 
- 1721 UPDATE - GAStech employees have continued to trickle out over the 
-past two hours. A few have agreed to communicate with members of the 
-press indicating that they were instructed not to say anything. They also 
-declined to be named. Generally, the GAStech employees said that 
-everyone was being interviewed about the events of the day, about the 
-meeting, and the recent IPO. In addition to the GAStech employees, the 
-caterers for the breakfast meeting, who were dressed all in black, have 
-been released. 
- 1545 UPDATE - These 'shadowy' figures are at the center of what appears 
-to be a kidnapping incident with the victims being the missing GAStech 
-leadership. At this time we can only speculate on the number of people 
-involved. In the confusion from the building evacuation an accurate count 
-of GAStech employees is nearly impossible. 
- 1158 UPDATE - The Abila Fire Department finished a thorough search of the 
-GAStech building finding evidence of fire. At 11:45 AM they gave the all 
-clear for the GAStech employees to re-enter the building. An AFD 
-spokesman stated that the fire alarm appears to be a false alarm and that it 
-was not a part of a planned fire drill. The spokesman declined to answer 
-whether or not the fire alarm was intentionally pulled. 
- 2021 UPDATE - A Kronos Government official who asked not to be identified 
-but was present at the government reception has gone on the record  stating, 'Investigators are working around the clock to ensure that the 
-kidnapped GAStech employees are found and that the perpetrators are 
-brought to justice.'This is the first confirmation that today's events 
-surrounding GAStech - Kronos are in fact a kidnapping. With fourteen 
-individuals still unaccounted for, at this point, the assumption is that all are 
-the victims of a kidnapping. No group has come forward yet to claim 
-responsibility nor have any demands been made. 
- 1245 UPDATE - Abila Police have cordoned off the GAStech headquarters 
-with additional officers arriving to secure the perimeter and to investigate 
-what can only be described as a crime scene though the crime has not be 
-revealed. The police response seems out of proportion for a false fire 
-alarm. Police officials have declined to comment at this time. 
- 1255 UPDATE - Calls to Abila's hospitals and the City Morgue have netted 
-no new information. 
- 1035 UPDATE - The Abila Fire Department arrived on the scene roughly 10 
-minutes ago and began searching the building. No smoke is visible, raising 
-concerns of an actual bomb threat. 
- 1345 UPDATE - Confirmed report of missing GAStech employees from 
-GAStech administrative assistant in Tethys headquarters; GAStech - Kronos 
-still not responding. The mystery of whom and how many are missing 
-remains. 
- 1748 UPDATE - Two private jets left Abila Airport today with fourteen to 
-sixteen passengers between the two. The destination of one of the planes 
-has been confirmed as Rome, Italy. The other plane's destination has yet to 
-be confirmed. 
- 1915 UPDATE - GAStech employee Edvard Vann was 'allowed to leave' after 
-nearly six hours of interrogation by police, government, and GAStech  officials. Vann was questioned and released about fifteen minutes ago and 
-instructed not to leave town. Vann was visibly angry about the confusion, 
-which apparently stemmed from the similarity of his name and known 
-members of the Protectors of Kronos. Vann said he was 'instructed not to 
-communicate with the press, but I feel I have to stress my innocence. I have 
-nothing to do whatsoever with any members of the terrorist group POK.' 
- 1100 UPDATE - Still no indication from the AFD about what if anything has 
-happened; however, there are reports that the GAStech - Kronos employees 
-gathered outside the building heard the fire alarm. 
\ No newline at end of file
+ 1853 UPDATE - Speculation that the 2:00 PM arrival of Kronos Government official was 
+in fact the arrival of a senior security executive and other officials from GAStech's 
+Tethys office including President and CEO Sten Sanjorge, Jr. This speculation is based 
+on information concerning the arrival, refueling and immediate departure of Sanjorge 
+Jr.'s private jet from the Tethys airport. 
+ 1947 UPDATE - From the Abila Police news conference: 'There are approximately 
+fourteen individuals unaccounted for among the GAStech staff. It has not yet been 
+determined if all of these individuals are in fact missing.'Though it was confirmed that 
+over a dozen individuals were unaccounted for, the Abila Police said that that does not 
+mean they are necessarily missing. Nor does it mean that they have been abducted. At 
+this point the case is under investigation and more information will be released as facts  are learned. The Abila Police stressed that anyone seeing any unusual activity should 
+immediately report it. 
+ 1330 UPDATE - At 1315 we placed a call to GAStech headquarters. A hesitant 
+administrative assistant to Orhan Strum, Chief Operating Officer of GAStech claimed Mr. 
+Strum was unavailable. Linda Lagos the assistant to Mr. Strum declined to answer any 
+further questions about the incident, whether or not there was a fire, a false alarm, a 
+bomb threat, or whether or not anyone is missing. Uneasy throughout the call, Ms. 
+Lagos repeated that Orhan Strum was unavailable at this time, said good bye and hung 
+up. Additional calls were placed with no answer on the GAStech end of the line. 
+ 1721 UPDATE - GAStech employees have continued to trickle out over the past two 
+hours. A few have agreed to communicate with members of the press indicating that 
+they were instructed not to say anything. They also declined to be named. Generally, the 
+GAStech employees said that everyone was being interviewed about the events of the 
+day, about the meeting, and the recent IPO. In addition to the GAStech employees, the 
+caterers for the breakfast meeting, who were dressed all in black, have been released. 
+ 1545 UPDATE - These 'shadowy' figures are at the center of what appears to be a 
+kidnapping incident with the victims being the missing GAStech leadership. At this time 
+we can only speculate on the number of people involved. In the confusion from the 
+building evacuation an accurate count of GAStech employees is nearly impossible. 
+ 1158 UPDATE - The Abila Fire Department finished a thorough search of the GAStech 
+building finding evidence of fire. At 11:45 AM they gave the all clear for the GAStech 
+employees to re-enter the building. An AFD spokesman stated that the fire alarm 
+appears to be a false alarm and that it was not a part of a planned fire drill. The 
+spokesman declined to answer whether or not the fire alarm was intentionally pulled. 
+ 2021 UPDATE - A Kronos Government official who asked not to be identified but was 
+present at the government reception has gone on the record stating, 'Investigators are 
+working around the clock to ensure that the kidnapped GAStech employees are found 
+and that the perpetrators are brought to justice.'This is the first confirmation that today's 
+events surrounding GAStech - Kronos are in fact a kidnapping. With fourteen individuals 
+still unaccounted for, at this point, the assumption is that all are the victims of a 
+kidnapping. No group has come forward yet to claim responsibility nor have any 
+demands been made. 
+  1245 UPDATE - Abila Police have cordoned off the GAStech headquarters with 
+additional officers arriving to secure the perimeter and to investigate what can only be 
+described as a crime scene though the crime has not be revealed. The police response 
+seems out of proportion for a false fire alarm. Police officials have declined to comment 
+at this time. 
+ 1255 UPDATE - Calls to Abila's hospitals and the City Morgue have netted no new 
+information. 
+ 1035 UPDATE - The Abila Fire Department arrived on the scene roughly 10 minutes ago 
+and began searching the building. No smoke is visible, raising concerns of an actual 
+bomb threat. 
+ 1345 UPDATE - Confirmed report of missing GAStech employees from GAStech 
+administrative assistant in Tethys headquarters; GAStech - Kronos still not responding. 
+The mystery of whom and how many are missing remains. 
+ 1748 UPDATE - Two private jets left Abila Airport today with fourteen to sixteen 
+passengers between the two. The destination of one of the planes has been confirmed 
+as Rome, Italy. The other plane's destination has yet to be confirmed. 
+ 1915 UPDATE - GAStech employee Edvard Vann was 'allowed to leave' after nearly six 
+hours of interrogation by police, government, and GAStech officials. Vann was 
+questioned and released about fifteen minutes ago and instructed not to leave town. 
+Vann was visibly angry about the confusion, which apparently stemmed from the 
+similarity of his name and known members of the Protectors of Kronos. Vann said he 
+was 'instructed not to communicate with the press, but I feel I have to stress my 
+innocence. I have nothing to do whatsoever with any members of the terrorist group 
+POK.' 
+ 1100 UPDATE - Still no indication from the AFD about what if anything has happened; 
+however, there are reports that the GAStech - Kronos employees gathered outside the 
+building heard the fire alarm. 
\ No newline at end of file
