--- VASTVD5_revision_72_July_18_10:25_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_73_July_18_10:29_AM	2019-07-30 16:05:39.000000000 -0400
@@ -44,7 +44,7 @@
 3.POK connection between members and GASTech employees 
 a.Isia Vann 
 b.Loreto Bodrogi 
-                    Timelime of 01/20 - 01/21 The Abila Post 
+ Timelime of 01/20 - 01/21 The Abila Post 
  01/20 
  ●1035 UPDATE - The Abila Fire Department arrived on the scene roughly 10 minutes ago 
 and began searching the building. No smoke is visible, raising concerns of an actual 
@@ -59,8 +59,7 @@
 declined to answer whether or not the fire alarm was intentionally pulled. 
 ●1245 UPDATE - Abila Police have cordoned off the GAStech headquarters with 
 additional officers arriving to secure the perimeter and to investigate what can only be 
-described as a crime scene though the crime has not be revealed. The police response 
-seems out of proportion for a false fire alarm. Police officials have declined to comment 
+described as a crime scene though the crime has not be revealed. The police response  seems out of proportion for a false fire alarm. Police officials have declined to comment 
 at this time. 
 ●1255 UPDATE - Calls to Abila's hospitals and the City Morgue have netted no new 
 information. 
@@ -80,7 +79,8 @@
 building evacuation an accurate count of GAStech employees is nearly impossible. 
 ●1721 UPDATE - GAStech employees have continued to trickle out over the past two 
 hours. A few have agreed to communicate with members of the press indicating that they 
-were instructed not to say anything. They also declined to be named. Generally, the  GAStech employees said that everyone was being interviewed about the events of the 
+were instructed not to say anything. They also declined to be named. Generally, the 
+GAStech employees said that everyone was being interviewed about the events of the 
 day, about the meeting, and the recent IPO. In addition to the GAStech employees, the 
 caterers for the breakfast meeting, who were dressed all in black, have been released. 
 ●1748 UPDATE - Two private jets left Abila Airport today with fourteen to sixteen 
@@ -97,8 +97,7 @@
 was visibly angry about the confusion, which apparently stemmed from the similarity of 
 his name and known members of the Protectors of Kronos. Vann said he was 'instructed 
 not to communicate with the press, but I feel I have to stress my innocence. I have 
-nothing to do whatsoever with any members of the terrorist group POK.' 
-●1947 UPDATE - From the Abila Police news conference: 'There are approximately 
+nothing to do whatsoever with any members of the terrorist group POK.'  ●1947 UPDATE - From the Abila Police news conference: 'There are approximately 
 fourteen individuals unaccounted for among the GAStech staff. It has not yet been 
 determined if all of these individuals are in fact missing.'Though it was confirmed that 
 over a dozen individuals were unaccounted for, the Abila Police said that that does not 
@@ -116,7 +115,8 @@
 demands been made. 
  01/21 
  ●934 UPDATE - Abila Police indicated that they are working with law enforcement and 
-government officials from Tethys. The number of the missing has been revised from  fourteen to ten. The names of the missing - and presumably kidnapped - have yet to be 
+government officials from Tethys. The number of the missing has been revised from 
+fourteen to ten. The names of the missing - and presumably kidnapped - have yet to be 
 released. Police stated that given the evolving nature of the incident announcing the 
 names of the missing employees would simply add to the confusion, speculation, and 
 potentially hinder the ongoing investigation. A GAStech International news conference 
@@ -126,4 +126,8 @@
 building heard the fire alarm. 
 ●1245 UPDATE - RANSOM DEMANDS FROM POK - The Protectors of Kronos have 
 released a statement claiming responsibility for the kidnapping of GAStech employees 
-demanding a $20 million ransom. 
\ No newline at end of file
+demanding a $20 million ransom. 
+ Course of Action 
+1.Need complete list of unaccounted for personnel 
+2.Need full access to e-mails between the security staff, especially Bodrigo and Vann 
+3. 
\ No newline at end of file
