--- VASTVD5_revision_73_July_18_10:29_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_74_July_18_10:30_AM	2019-07-30 16:05:39.000000000 -0400
@@ -130,4 +130,5 @@
  Course of Action 
 1.Need complete list of unaccounted for personnel 
 2.Need full access to e-mails between the security staff, especially Bodrigo and Vann 
-3. 
\ No newline at end of file
+3.Need list of personnel of landed in Rome on the second private jet 
+ 
\ No newline at end of file
