--- VASTVD5_revision_15_July_17_10:33_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_16_July_17_10:34_AM	2019-07-30 16:05:39.000000000 -0400
@@ -6,7 +6,9 @@
 metadata 
 3.Petrus Gerhard - News correspondent who seemingly exclusively writes about POK and 
 environmental issues; active since 1998 
-a.“The next news  
+a.“The next news Readers, the next post you read will be one of the most important 
+you have ever read, or indeed, will ever read. 21 Jan 2014” - prior to POK’s 
+announcement of  
  Employees of GASTech removed in Kronos / World Journal | 01/21/2014:  
 14 kidnapped 
 5 of 14 were execs 
