--- VASTVD5_revision_22_July_17_12:41_PM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_23_July_18_8:07_AM	2019-07-30 16:05:39.000000000 -0400
@@ -9,7 +9,8 @@
 a.“Readers, the next post you read will be one of the most important you have ever 
 read, or indeed, will ever read. 21 Jan 2014” - prior to POK’s announcement of 
 their responsibility in the kidnapping 
- Edvard vann, isia vann,  
+ Edvard vann, isia vann,  - possible relation to Juliana Vann, both briefed on 20 JAN security 
+protocols / on the security staff 
  Employees of GASTech removed in Kronos / World Journal | 01/21/2014:  
 14 kidnapped (updated to 10 from Abila PD) 
 5 of 14 were execs 
