--- VASTVD5_revision_74_July_18_10:30_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_75_July_18_11:04_AM	2019-07-30 16:05:39.000000000 -0400
@@ -131,4 +131,5 @@
 1.Need complete list of unaccounted for personnel 
 2.Need full access to e-mails between the security staff, especially Bodrigo and Vann 
 3.Need list of personnel of landed in Rome on the second private jet 
- 
\ No newline at end of file
+4.Retrieve surveillance tapes from Guy’s Gyros on 01/19 - Sanjorge Jr present at the 
+restaurant at the same time as a significant portion of the security staff   
\ No newline at end of file
