--- VASTVD5_revision_35_July_18_8:46_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_36_July_18_8:47_AM	2019-07-30 16:05:39.000000000 -0400
@@ -32,7 +32,10 @@
    1.POK membership, power structure 
 a.Silvia Marek, leader 
 b.Petrus Gerhard, spokesman/media outlet, perhaps not direct member 
-c. 
+c.High probability members 
+i.Isia Vann 
+ii.Loreto Bodrogi 
+iii. 
 2.POK change over time 
 a.Henk Bodrogi and Carmine Osvaldo formed POK; peaceful protests 
 b.Elian Karel in charge after Bodrogi until his death in prison, 2009; peaceful 
