--- VASTVD5_revision_59_July_18_9:03_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_60_July_18_9:04_AM	2019-07-30 16:05:39.000000000 -0400
@@ -55,13 +55,7 @@
 anyone is missing. Uneasy throughout the call, Ms. Lagos repeated that Orhan Strum was 
 unavailable at this time, said good bye and hung up. Additional calls were placed with no answer 
 on the GAStech end of the line. 
- 1721 UPDATE - GAStech employees have continued to trickle out over the past two hours. A 
-few have agreed to communicate with members of the press indicating that they were instructed 
-not to say anything. They also declined to be named. Generally, the GAStech employees said that 
-everyone was being interviewed about the events of the day, about the meeting, and the recent 
-IPO. In addition to the GAStech employees, the caterers for the breakfast meeting, who were 
-dressed all in black, have been released. 
- 1545 UPDATE - These 'shadowy' figures are at the center of what appears to be a kidnapping 
+   1545 UPDATE - These 'shadowy' figures are at the center of what appears to be a kidnapping 
 incident with the victims being the missing GAStech leadership. At this time we can only 
 speculate on the number of people involved. In the confusion from the building evacuation an 
 accurate count of GAStech employees is nearly impossible. 
@@ -73,14 +67,17 @@
 re-enter the building. An AFD spokesman stated that the fire alarm appears to be a false alarm 
 and that it was not a part of a planned fire drill. The spokesman declined to answer whether or 
 not the fire alarm was intentionally pulled. 
-  1245 UPDATE - Abila Police have cordoned off the GAStech headquarters with additional 
+ 1245 UPDATE - Abila Police have cordoned off the GAStech headquarters with additional 
 officers arriving to secure the perimeter and to investigate what can only be described as a crime 
 scene though the crime has not be revealed. The police response seems out of proportion for a 
 false fire alarm. Police officials have declined to comment at this time. 
- 1255 UPDATE - Calls to Abila's hospitals and the City Morgue have netted no new information. 
-  1345 UPDATE - Confirmed report of missing GAStech employees from GAStech administrative 
+ 1255 UPDATE - Calls to Abila's hospitals and the City Morgue have netted no new information.   1345 UPDATE - Confirmed report of missing GAStech employees from GAStech administrative 
 assistant in Tethys headquarters; GAStech - Kronos still not responding. The mystery of whom 
 and how many are missing remains. 
+ 1545 UPDATE - These 'shadowy' figures are at the center of what appears to be a kidnapping 
+incident with the victims being the missing GAStech leadership. At this time we can only 
+speculate on the number of people involved. In the confusion from the building evacuation an 
+accurate count of GAStech employees is nearly impossible. 
  1721 UPDATE - GAStech employees have continued to trickle out over the past two hours. A 
 few have agreed to communicate with members of the press indicating that they were instructed 
 not to say anything. They also declined to be named. Generally, the GAStech employees said that 
@@ -103,9 +100,9 @@
 feel I have to stress my innocence. I have nothing to do whatsoever with any members of the 
 terrorist group POK.' 
  1947 UPDATE - From the Abila Police news conference: 'There are approximately fourteen 
-individuals unaccounted for among the GAStech staff. It has not yet been determined if all of  these individuals are in fact missing.'Though it was confirmed that over a dozen individuals were 
-unaccounted for, the Abila Police said that that does not mean they are necessarily missing. Nor 
-does it mean that they have been abducted. At this point the case is under investigation and more 
+individuals unaccounted for among the GAStech staff. It has not yet been determined if all of 
+these individuals are in fact missing.'Though it was confirmed that over a dozen individuals were 
+unaccounted for, the Abila Police said that that does not mean they are necessarily missing. Nor  does it mean that they have been abducted. At this point the case is under investigation and more 
 information will be released as facts are learned. The Abila Police stressed that anyone seeing 
 any unusual activity should immediately report it. 
  2021 UPDATE - A Kronos Government official who asked not to be identified but was present at 
