--- VASTVD5_revision_48_July_18_8:58_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_49_July_18_8:59_AM	2019-07-30 16:05:39.000000000 -0400
@@ -105,4 +105,12 @@
 no new information. 
  1035 UPDATE - The Abila Fire Department arrived on the scene roughly 10 
 minutes ago and began searching the building. No smoke is visible, raising 
-concerns of an actual bomb threat. 
\ No newline at end of file
+concerns of an actual bomb threat. 
+ 1345 UPDATE - Confirmed report of missing GAStech employees from 
+GAStech administrative assistant in Tethys headquarters; GAStech - Kronos 
+still not responding. The mystery of whom and how many are missing 
+remains. 
+ 1748 UPDATE - Two private jets left Abila Airport today with fourteen to 
+sixteen passengers between the two. The destination of one of the planes 
+has been confirmed as Rome, Italy. The other plane's destination has yet to 
+be confirmed. 
\ No newline at end of file
