--- VASTVD5_revision_6_July_17_10:01_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_7_July_17_10:02_AM	2019-07-30 16:05:39.000000000 -0400
@@ -2,8 +2,9 @@
   Employees of GASTech removed in Kronos /World Journal | 01/21/2014:  
 14 kidnapped 
 5 of 14 were execs 
- President Sanjorge; PRESIDENT Sten Sanjorge Jr,  
+ PRESIDENT Sten Sanjorge Jr,  
 CFO Ingrid Barranco 
 Campo-Corrente of ADA of CIO,  
 ROUCOULEMENT Orhan de GAStech pianotent,  
-Environmental leader Willem Vasco-Feed 
\ No newline at end of file
+Environmental leader Willem Vasco-Feed 
+ False fire alarm occurred prior to the kidnappings at the GASTech offices 
\ No newline at end of file
