--- VASTVD5_revision_47_July_18_8:58_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_48_July_18_8:58_AM	2019-07-30 16:05:39.000000000 -0400
@@ -100,4 +100,9 @@
 with additional officers arriving to secure the perimeter and to investigate 
 what can only be described as a crime scene though the crime has not be 
 revealed. The police response seems out of proportion for a false fire 
-alarm. Police officials have declined to comment at this time. 
\ No newline at end of file
+alarm. Police officials have declined to comment at this time. 
+ 1255 UPDATE - Calls to Abila's hospitals and the City Morgue have netted 
+no new information. 
+ 1035 UPDATE - The Abila Fire Department arrived on the scene roughly 10 
+minutes ago and began searching the building. No smoke is visible, raising 
+concerns of an actual bomb threat. 
\ No newline at end of file
