--- VASTVD5_revision_33_July_18_8:38_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_34_July_18_8:39_AM	2019-07-30 16:05:39.000000000 -0400
@@ -37,4 +37,7 @@
 c.Silvia Marek current leader; peaceful protests until more robust demonstrations 
 and this kidnapping -- probably motivated by Elian Karel’s mysterious death in 
 prison 
-3.POK connection between members and GASTech employees 
\ No newline at end of file
+3.POK connection between members and GASTech employees 
+a.Isia Vann 
+b.Loreto Bodrogi 
+c. 
\ No newline at end of file
