--- VASTVD5_revision_29_July_18_8:36_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_30_July_18_8:36_AM	2019-07-30 16:05:39.000000000 -0400
@@ -32,5 +32,6 @@
    1.POK membership, power structure 
 2.POK change over time 
 a.Henk Bodrogi and Carmine Osvaldo formed POK 
-b.Elian Karel in charge of  
+b.Elian Karel in charge of  after Bodrogi until his death in prison, 2009 
+c. 
 3.POK connection between members and GASTech employees 
\ No newline at end of file
