--- VASTVD5_revision_69_July_18_9:10_AM	2019-07-30 16:05:39.000000000 -0400
+++ VASTVD5_revision_70_July_18_9:33_AM	2019-07-30 16:05:39.000000000 -0400
@@ -31,9 +31,10 @@
   1.POK membership, power structure 
 a.Silvia Marek, leader 
 b.Petrus Gerhard, spokesman/media outlet, perhaps not direct member 
-c.High probability members 
-i.Isia Vann 
-ii.Loreto Bodrogi 
+c.Isia Vann 
+d.High probability members 
+e.Isia Vann 
+i.Loreto Bodrogi 
 2.POK change over time - Leaders and Disposition 
 a.Henk Bodrogi and Carmine Osvaldo formed POK; peaceful protests 
 b.Elian Karel in charge after Bodrogi until his death in prison, 2009; peaceful 
@@ -49,27 +50,21 @@
  ●1035 UPDATE - The Abila Fire Department arrived on the scene roughly 10 minutes ago 
 and began searching the building. No smoke is visible, raising concerns of an actual 
 bomb threat. 
-● 
 ●1100 UPDATE - Still no indication from the AFD about what if anything has happened; 
 however, there are reports that the GAStech - Kronos employees gathered outside the 
 building heard the fire alarm. 
-● 
 ●1158 UPDATE - The Abila Fire Department finished a thorough search of the GAStech 
 building finding evidence of fire. At 11:45 AM they gave the all clear for the GAStech 
 employees to re-enter the building. An AFD spokesman stated that the fire alarm appears 
 to be a false alarm and that it was not a part of a planned fire drill. The spokesman 
 declined to answer whether or not the fire alarm was intentionally pulled. 
-● 
 ●1245 UPDATE - Abila Police have cordoned off the GAStech headquarters with 
 additional officers arriving to secure the perimeter and to investigate what can only be 
 described as a crime scene though the crime has not be revealed. The police response 
 seems out of proportion for a false fire alarm. Police officials have declined to comment 
 at this time. 
-● 
-● 
 ●1255 UPDATE - Calls to Abila's hospitals and the City Morgue have netted no new 
 information. 
-● 
 ●1330 UPDATE - At 1315 we placed a call to GAStech headquarters. A hesitant 
 administrative assistant to Orhan Strum, Chief Operating Officer of GAStech claimed Mr. 
 Strum was unavailable. Linda Lagos the assistant to Mr. Strum declined to answer any 
@@ -77,32 +72,26 @@
 threat, or whether or not anyone is missing. Uneasy throughout the call, Ms. Lagos 
 repeated that Orhan Strum was unavailable at this time, said good bye and hung up. 
 Additional calls were placed with no answer on the GAStech end of the line. 
-● 
 ●1345 UPDATE - Confirmed report of missing GAStech employees from GAStech 
 administrative assistant in Tethys headquarters; GAStech - Kronos still not responding. 
-The mystery of whom and how many are missing remains.  ● 
+The mystery of whom and how many are missing remains. 
 ●1545 UPDATE - These 'shadowy' figures are at the center of what appears to be a 
 kidnapping incident with the victims being the missing GAStech leadership. At this time 
 we can only speculate on the number of people involved. In the confusion from the 
 building evacuation an accurate count of GAStech employees is nearly impossible. 
-● 
 ●1721 UPDATE - GAStech employees have continued to trickle out over the past two 
-hours. A few have agreed to communicate with members of the press indicating that they 
-were instructed not to say anything. They also declined to be named. Generally, the 
+hours. A few have agreed to communicate with members of the press indicating that they  were instructed not to say anything. They also declined to be named. Generally, the 
 GAStech employees said that everyone was being interviewed about the events of the 
 day, about the meeting, and the recent IPO. In addition to the GAStech employees, the 
 caterers for the breakfast meeting, who were dressed all in black, have been released. 
-● 
 ●1748 UPDATE - Two private jets left Abila Airport today with fourteen to sixteen 
 passengers between the two. The destination of one of the planes has been confirmed as 
 Rome, Italy. The other plane's destination has yet to be confirmed. 
-● 
 ●1853 UPDATE - Speculation that the 2:00 PM arrival of Kronos Government official was 
 in fact the arrival of a senior security executive and other officials from GAStech's Tethys 
 office including President and CEO Sten Sanjorge, Jr. This speculation is based on 
 information concerning the arrival, refueling and immediate departure of Sanjorge Jr.'s 
 private jet from the Tethys airport. 
-● 
 ●1915 UPDATE - GAStech employee Edvard Vann was 'allowed to leave' after nearly six 
 hours of interrogation by police, government, and GAStech officials. Vann was 
 questioned and released about fifteen minutes ago and instructed not to leave town. Vann 
@@ -110,7 +99,6 @@
 his name and known members of the Protectors of Kronos. Vann said he was 'instructed 
 not to communicate with the press, but I feel I have to stress my innocence. I have 
 nothing to do whatsoever with any members of the terrorist group POK.' 
-● 
 ●1947 UPDATE - From the Abila Police news conference: 'There are approximately 
 fourteen individuals unaccounted for among the GAStech staff. It has not yet been 
 determined if all of these individuals are in fact missing.'Though it was confirmed that 
@@ -118,7 +106,7 @@
 mean they are necessarily missing. Nor does it mean that they have been abducted. At 
 this point the case is under investigation and more information will be released as facts 
 are learned. The Abila Police stressed that anyone seeing any unusual activity should 
-immediately report it.  ● 
+immediately report it. 
 ●2021 UPDATE - A Kronos Government official who asked not to be identified but was 
 present at the government reception has gone on the record stating, 'Investigators are 
 working around the clock to ensure that the kidnapped GAStech employees are found and 
@@ -128,18 +116,16 @@
 kidnapping. No group has come forward yet to claim responsibility nor have any 
 demands been made. 
  01/21 
- ●934 UPDATE - Abila Police indicated that they are working with law enforcement and 
+  ●934 UPDATE - Abila Police indicated that they are working with law enforcement and 
 government officials from Tethys. The number of the missing has been revised from 
 fourteen to ten. The names of the missing - and presumably kidnapped - have yet to be 
 released. Police stated that given the evolving nature of the incident announcing the 
 names of the missing employees would simply add to the confusion, speculation, and 
 potentially hinder the ongoing investigation. A GAStech International news conference 
 has been announced for 10:00 this morning Kronos time. 
-● 
 ●1100 UPDATE - Still no indication from the AFD about what if anything has happened; 
 however, there are reports that the GAStech - Kronos employees gathered outside the 
 building heard the fire alarm. 
-● 
 ●1245 UPDATE - RANSOM DEMANDS FROM POK - The Protectors of Kronos have 
 released a statement claiming responsibility for the kidnapping of GAStech employees 
 demanding a $20 million ransom. 
\ No newline at end of file
