--- VASTL6X_revision_4_June_19_9:44_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_5_June_19_9:44_AM	2019-07-08 08:44:40.000000000 -0400
@@ -4,7 +4,7 @@
 logging/forestry 
 Natural Gas 
  Lots of poverty despite natural resources 
- Gastech in island of Tethys to NE 
+ Gastech incompany from island of Tethys to NE 
 Has strong relationship with Kronos govt 
 GAStech has bad environmental rap on Kronos 
  Jan 2014 (20 yrs after gastech started on kronos) gastech has an IPO 
