--- VASTL6X_revision_48_June_19_11:13_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_49_June_19_11:14_AM	2019-07-08 08:44:40.000000000 -0400
@@ -49,7 +49,8 @@
 CHILDREN AND RUINED OUR WAY OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE 
 FORM OF $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE 
 PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 
-           GASTech 
+ 4 
+       GASTech 
 Isia Vann - Security - Employee since 2007 - Former military (general discharge) 
 Edvard Vann - Security - Employee since 2013 - Former military (honorable discharge) 
 Loreto Bodrogi - Security - Employee since 2013 - Former military (honorable discharge) 
