--- VASTL6X_revision_49_June_19_11:14_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_50_June_19_11:15_AM	2019-07-08 08:44:40.000000000 -0400
@@ -30,7 +30,7 @@
 protesting for clean water, human rights, and against the corruption of the Government of 
 Kronos and GAStech International. 
  Leadership 
-Henk Bodrogi (founder) - Elian Karel - Silvia Marek 
+Henk Bodrogi (founder) -> Elian Karel -> Silvia Marek 
   Elian Karel - leader of POK in 2009 - dies in jail 19JUN09 -  
 Michale Kraft - Karel’s lawyer 
  Kriste Brontes - supporter? 
@@ -48,9 +48,7 @@
 APPREHENDED THE ELITISTS WHO HAVE EXPLOITED OUR RESOURCES, KILLED OUR 
 CHILDREN AND RUINED OUR WAY OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE 
 FORM OF $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE 
-PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 
- 4 
-       GASTech 
+PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 20144  GASTech 
 Isia Vann - Security - Employee since 2007 - Former military (general discharge) 
 Edvard Vann - Security - Employee since 2013 - Former military (honorable discharge) 
 Loreto Bodrogi - Security - Employee since 2013 - Former military (honorable discharge) 
