--- VASTL6X_revision_34_June_19_10:54_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_35_June_19_10:54_AM	2019-07-08 08:44:40.000000000 -0400
@@ -29,7 +29,7 @@
 founded in 1998 in Elodis, a rural township outside of the capital Abila. They have been 
 protesting for clean water, human rights, and against the corruption of the Government of 
 Kronos and GAStech International. 
- Elian Karel - leader of POK in 2009 
+ Elian Karel - leader of POK in 2009 - dies in jail 19JUN09 
 Michale Kraft - Karel’s lawyer 
  Kriste Brontes - supporter? 
   From VOICES blog 
