--- VASTL6X_revision_6_June_19_9:59_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_7_June_19_10:00_AM	2019-07-08 08:44:40.000000000 -0400
@@ -12,5 +12,5 @@
 Bunch of employees go missing / Protectors of Kronos (POK) are implicated 
 POK hasn’t claimed responsibility 
  Ex 1 
-Find three people who are members of the POK 
+Find Identify at least three people who are members of the POK 
 Provide information about them 
\ No newline at end of file
