--- VASTL6X_revision_58_June_20_8:47_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_59_June_20_8:47_AM	2019-07-08 08:44:40.000000000 -0400
@@ -52,7 +52,8 @@
 Isia Vann - Security - Employee since 2007 - Former military (general discharge) 
 Edvard Vann - Security - Employee since 2013 - Former military (honorable discharge) 
  Henk Mies and Sanjorge emails re: files? 
- Company officers discussing vacation spots shortly before the event 
+ Company officers discussing vacation spots and hiring financial advisors shortly before the 
+event  
  Emails re: how to prepare for VIP visit 
                                       People from Elodis, which is a town affected by GASTech’s operations 
 Pondo Reese, editor of Elodis Weekly Journal 
