--- VASTL6X_revision_44_June_19_11:03_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_45_June_19_11:07_AM	2019-07-08 08:44:40.000000000 -0400
@@ -52,6 +52,6 @@
 Loreto Bodrogi - Security - Employee since 2013 - Former military (honorable discharge) 
                       People from Elodis, which is a town affected by GASTech’s operations 
 Pondo Reese, editor of Elodis Weekly Journal 
-Henk Pondrogi, farmer - founder of POK 
+Henk PonBodrogi, farmer - founder of POK 
 Juliana Vann, girl who dies of leukemia due to benzene poisoningl 
   
\ No newline at end of file
