--- VASTL6X_revision_38_June_19_10:56_AM	2019-07-08 08:44:40.000000000 -0400
+++ VASTL6X_revision_39_June_19_10:57_AM	2019-07-08 08:44:40.000000000 -0400
@@ -49,6 +49,6 @@
 PEOPLE. THIS IS THE FIRST STEP TOWARD TAKING OUR COUNTRY BACK. 21 Jan 2014 
                       People from Elodis, which is a town affected by GASTech’s operations 
 Pondo Reese, editor of Elodis Weekly Journal 
-Henk Pondrogi, farmer 
+Henk Pondrogi, farmer - founder of POK 
 Juliana Vann, girl who dies of leukemia due to benzene poisoningl 
   
\ No newline at end of file
