--- VASTT93_revision_43_July_18_9:30_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_44_July_18_9:31_AM	2019-07-23 14:25:35.000000000 -0400
@@ -27,4 +27,5 @@
     Money made from ipo 
 $96.25 million for CFO Ingrid Barranco, and cio Campo-Corrente 
 adviser of safety Willem Vasco-Feed also gained $38.5  
- 
\ No newline at end of file
+  2 private jets flew from abila  
+7 or 8 passengers in first plane dressed business 
\ No newline at end of file
