--- VASTT93_revision_16_July_17_10:39_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_17_July_17_10:39_AM	2019-07-23 14:25:35.000000000 -0400
@@ -5,7 +5,8 @@
  Mestache Vardshvna involved in protests 
 https://vast.ncsu-las.net/?article=265 
  Edvard Vann (security guard @ gastech, same last name as the young girl, 
-perhaps coincidence; claims to have nothing to do with POK)  
+perhaps coincidence; claims to have nothing to do with POK questioned 
+and released)  
 https://vast.ncsu-las.net/?article=63 
  Kriste Brontes (at a meeting with Karel, the leader of POK) 
 https://vast.ncsu-las.net/?article=398 
