--- VASTT93_revision_17_July_17_10:39_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_18_July_17_10:41_AM	2019-07-23 14:25:35.000000000 -0400
@@ -1,7 +1,7 @@
 VastT93 
  3 people ASW POK  
  Elian Karel ( dead during imprisonment; POK leader)  
-Julian Vann (dead by cancer, young girl source of anger for POK) 
+Juliana Vann (dead by cancer, young girl source of anger for POK) 
  Mestache Vardshvna involved in protests 
 https://vast.ncsu-las.net/?article=265 
  Edvard Vann (security guard @ gastech, same last name as the young girl, 
