--- VASTT93_revision_48_July_18_9:34_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_49_July_18_9:37_AM	2019-07-23 14:25:35.000000000 -0400
@@ -33,4 +33,5 @@
 7 or 8 passengers in first plane dressed business 
 Same number in second, but less well dressed’ 
 Plane 1 depart 1230  
-Second plane depart 1430 destination Rome  
\ No newline at end of file
+Second plane depart 1430 destination Rome  
+  Perhaps someone should locate the caterers from the executive breakfast?  
\ No newline at end of file
