--- VASTT93_revision_22_July_17_10:44_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_23_July_17_10:45_AM	2019-07-23 14:25:35.000000000 -0400
@@ -10,8 +10,9 @@
 https://vast.ncsu-las.net/?article=63 
  Kriste Brontes (at a meeting with Karel, the leader of POK) 
 https://vast.ncsu-las.net/?article=398 
- Silvia Marek (spoke at gathering remembering dead POK leader)’ 
-https://vast.ncsu-las.net/?article=452 
+ Silvia Marek (current leader POKspoke at gathering remembering dead POK 
+leader)’ 
+   https://vast.ncsu-las.net/?article=452 
  Henk Bodrogi (founder of POK) 
 https://vast.ncsu-las.net/?article=452 
  Jeroen Karel (martyred leaders father/founder of POK with henk) 
