--- VASTT93_revision_55_July_18_10:58_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_56_July_18_11:10_AM	2019-07-23 14:25:35.000000000 -0400
@@ -1,5 +1,5 @@
 VastT93 
- Missing on 1/21/2014 
+ Missing on 1/210/2014 
   3 people ASW POK  
  Elian Karel ( dead during imprisonment; POK leader)  
 Juliana Vann (dead by cancer, young girl source of anger for POK) 
@@ -35,62 +35,6 @@
 Plane 1 depart 1230  
 Second plane depart 1430 destination Rome  
   Perhaps someone should locate the caterers from the executive breakfast?   Shopping at alberts fine clothing  
- 1/16/2014 
-20:08 
-Albert's Fine 
-Clothing 
-283.38 
-Varro 
-Awelon 
-1/16/2014 
-20:25 
-Albert's Fine 
-Clothing 
-91.71 
-Inga 
-Ferro 
-1/17/2014 
-19:44 
-Albert's Fine 
-Clothing 
-39.41 
-Gustav 
-Cazar 
-1/18/2014 
-15:20 
-Albert's Fine 
-Clothing 
-35.66 
-Vira 
-Frente 
-1/18/2014 
-15:57 
-Albert's Fine 
-Clothing 
-189.52 
-Anda 
-Ribera 
-1/18/2014 
-18:59 
-Albert's Fine 
-Clothing 
-226.49 
-Orhan 
-Strum 
-1/19/2014 
-15:39 
-Albert's Fine 
-Clothing 
-274.66 
-Adra 
-Nubarron 
-1/19/2014 
-20:02 
-Albert's Fine 
-Clothing 
-254.29 
-Isande 
-Borrasca 
  1/6/2014 20:26 
 Albert's Fine 
 Clothing 
@@ -147,7 +91,8 @@
 Clothing 
 211.47 
 Minke 
-Mies 1/11/2014 
+Mies 
+1/11/2014 
 19:27 
 Albert's Fine 
 Clothing 
@@ -209,8 +154,7 @@
 Clothing 
 283.38 
 Varro 
-Awelon 
-1/16/2014 
+Awelon 1/16/2014 
 20:25 
 Albert's Fine 
 Clothing 
@@ -262,5 +206,6 @@
   Henk Mies (truck driver) spent around 35k at airport… 
 Albena hafon 9k 
 Benito hawelon 8 
-Claudio hawelonm 4 Dylan scozzese 11 
+Claudio hawelonm 4 
+Dylan scozzese 11 
  
\ No newline at end of file
