--- VASTT93_revision_25_July_17_10:49_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_26_July_17_10:56_AM	2019-07-23 14:25:35.000000000 -0400
@@ -2,15 +2,7 @@
  3 people ASW POK  
  Elian Karel ( dead during imprisonment; POK leader)  
 Juliana Vann (dead by cancer, young girl source of anger for POK) 
- Mestache Vardshvna involved in protests 
-https://vast.ncsu-las.net/?article=265 
- Edvard Vann (security guard @ gastech, same last name as the young girl, 
-perhaps coincidence; claims to have nothing to do with POK questioned 
-and released)  
-https://vast.ncsu-las.net/?article=63 
- Kriste Brontes (at a meeting with Karel, the leader of POK) 
-https://vast.ncsu-las.net/?article=398 
- Silvia Marek (current leader POKspoke at gathering remembering dead 
+   Silvia Marek (current leader POKspoke at gathering remembering dead 
 POK leader)  
 https://vast.ncsu-las.net/?article=163  
 https://vast.ncsu-las.net/?article=452 
@@ -21,8 +13,11 @@
   Edvard Vann (security guard @ gastech, same last name as the young girl, 
 perhaps coincidence; claims to have nothing to do with POK questioned 
 and released)  
-https://vast.ncsu-las.net/?article=63   Henk Bodrogi (founder of POK) 
+https://vast.ncsu-las.net/?article=63 
+  Henk Bodrogi (founder of POK) 
 https://vast.ncsu-las.net/?article=452 
  Jeroen Karel (martyred leaders father/founder of POK with henk) 
 https://vast.ncsu-las.net/?article=452 
+  Kronos citizens who work for gastech 
+Mies Haber, Ruscella 
    
\ No newline at end of file
