--- VASTT93_revision_49_July_18_9:37_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_50_July_18_9:40_AM	2019-07-23 14:25:35.000000000 -0400
@@ -34,4 +34,61 @@
 Same number in second, but less well dressed’ 
 Plane 1 depart 1230  
 Second plane depart 1430 destination Rome  
-  Perhaps someone should locate the caterers from the executive breakfast?  
\ No newline at end of file
+  Perhaps someone should locate the caterers from the executive breakfast?   Shopping at alberts fine clothing  
+ 1/16/2014 
+20:08 
+Albert's Fine 
+Clothing 
+283.38 
+Varro 
+Awelon 
+1/16/2014 
+20:25 
+Albert's Fine 
+Clothing 
+91.71 
+Inga 
+Ferro 
+1/17/2014 
+19:44 
+Albert's Fine 
+Clothing 
+39.41 
+Gustav 
+Cazar 
+1/18/2014 
+15:20 
+Albert's Fine 
+Clothing 
+35.66 
+Vira 
+Frente 
+1/18/2014 
+15:57 
+Albert's Fine 
+Clothing 
+189.52 
+Anda 
+Ribera 
+1/18/2014 
+18:59 
+Albert's Fine 
+Clothing 
+226.49 
+Orhan 
+Strum 
+1/19/2014 
+15:39 
+Albert's Fine 
+Clothing 
+274.66 
+Adra 
+Nubarron 
+1/19/2014 
+20:02 
+Albert's Fine 
+Clothing 
+254.29 
+Isande 
+Borrasca 
+ 
\ No newline at end of file
