--- VASTT93_revision_24_July_17_10:45_AM	2019-07-23 14:25:35.000000000 -0400
+++ VASTT93_revision_25_July_17_10:49_AM	2019-07-23 14:25:35.000000000 -0400
@@ -10,11 +10,18 @@
 https://vast.ncsu-las.net/?article=63 
  Kriste Brontes (at a meeting with Karel, the leader of POK) 
 https://vast.ncsu-las.net/?article=398 
- Silvia Marek (current leader POKspoke at gathering remembering dead POK 
-leader)  
+ Silvia Marek (current leader POKspoke at gathering remembering dead 
+POK leader)  
 https://vast.ncsu-las.net/?article=163  
 https://vast.ncsu-las.net/?article=452 
- Henk Bodrogi (founder of POK) 
+ Kriste Brontes (at a meeting with Karel, the leader of POK) 
+https://vast.ncsu-las.net/?article=398 
+ Mestache Vardshvna (involved in protests, maybe a POK member) 
+https://vast.ncsu-las.net/?article=265 
+  Edvard Vann (security guard @ gastech, same last name as the young girl, 
+perhaps coincidence; claims to have nothing to do with POK questioned 
+and released)  
+https://vast.ncsu-las.net/?article=63   Henk Bodrogi (founder of POK) 
 https://vast.ncsu-las.net/?article=452 
  Jeroen Karel (martyred leaders father/founder of POK with henk) 
 https://vast.ncsu-las.net/?article=452 
