--- VAST9L7_revision_18_June_19_10:16_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_19_June_19_10:16_AM	2019-06-27 13:09:46.000000000 -0400
@@ -43,4 +43,8 @@
 Position: Truck Driver (Facilities) 
 Date of Hire: 12/02/2003 
 Email: Valeria.Morlun@gastech.com.kronos 
+ Morlun, Adan 
+Position: Truck Driver (Facilities) 
+Date of Hire: 01/14/2006 
+Email: Adan.Morlun@gastech.com.kronos 
         
\ No newline at end of file
