--- VAST9L7_revision_39_June_19_10:31_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_40_June_19_10:34_AM	2019-06-27 13:09:46.000000000 -0400
@@ -12,19 +12,7 @@
 Position: Assistant to Security Group Manager (Administration) 
 Date of Hire: 12/22/2011 
 Email: Cornelia.Lais@gastech.com.kronos 
- Azada, Lars 
-Position: Engineer (Engineering) 
-Date of Hire: 12/01/1995 
-Email: Lars.Azada@gastech.com.kronos 
- Sex: Male 
-DOB: 04/30/1973 
- Citizenship: Tethys 
-Passport Issued: 06/01/2011 
-Passport Expires: 05/31/2021 
- Military Branch: Tethan Defense Force: Air 
-Discharge Date: 10/01/1995 
-Discharge Type: General Discharge 
-  Arpa, Emile (K) 
+Arpa, Emile (K) 
 Position: Janitor (Facilities) 
 Date of Hire: 03/18/2013 
 Email: Emile.Arpa@gastech.com.kronos 
