--- VAST9L7_revision_6_June_19_10:05_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_7_June_19_10:05_AM	2019-06-27 13:09:46.000000000 -0400
@@ -1,6 +1,7 @@
-Military Service & Citizenship 
- Mies Haber, Ruscella (A.F.O.K) -  
+Military Service  
+ Mies Haber, Ruscella (A.F.O.K)  
 Position: Assistant to Engineering Group Manager (Administration) 
 Date of Hire: 04/02/2003 
 Email: Ruscella.Mies.Haber@gastech.com.kronos 
-  
\ No newline at end of file
+ Forluniau, Carla (A.F.O.K) 
+   
\ No newline at end of file
