--- VAST9L7_revision_75_June_19_11:18_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_76_June_19_11:19_AM	2019-06-27 13:09:46.000000000 -0400
@@ -8,9 +8,6 @@
 Position: Assistant to IT Group Manager (Administration) 
 Date of Hire: 03/07/2005 
 Email: Carla.Forluniau@gastech.com.kronos 
-Lais, Cornelia G.D. Manager (Administration) 
-Date of Hire: 03/07/2005 
-Email: Carla.Forluniau@gastech.com.kronos 
 Lais, Cornelia G.D. (arranges travel) 
 Position: Assistant to Security Group Manager (Administration) 
 Date of Hire: 12/22/2011 
@@ -23,7 +20,7 @@
 Position: Janitor (Facilities) 
 Date of Hire: 10/22/2012 
 Email: Varro.Awelon@gastech.com.kronos 
- Coginian, Dante G.D. 
+ Coginian, Dante G.D. (OUTSTANDING LEADER) 
 Position: Lead Janitor (Facilities) 
 Date of Hire: 01/07/1995 
 Email: Dante.Coginian@gastech.com.kronos 
