--- VAST9L7_revision_26_June_19_10:19_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_27_June_19_10:19_AM	2019-06-27 13:09:46.000000000 -0400
@@ -75,4 +75,8 @@
 Position: Perimeter Control (Security) 
 Date of Hire: 08/15/2013 
 Email: Edvard.Vann@gastech.com.kronos 
-          
\ No newline at end of file
+ Bodrogi, Loreto 
+Position: Site Control (Security) 
+Date of Hire: 08/17/2013 
+Email: Loreto.Bodrogi@gastech.com.kronos 
+         
\ No newline at end of file
