--- VAST9L7_revision_79_June_19_11:28_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_80_June_19_11:29_AM	2019-06-27 13:09:46.000000000 -0400
@@ -60,7 +60,7 @@
 Position: Truck Driver (Facilities) 
 Date of Hire: 03/01/2007 
 Email: Dylan.Scozzese@gastech.com.kronos 
- Mies, Minke G.D. 
+ Mies, Minke G.D. (takes necessary action when needed) 
 Position: Perimeter Control (Security) 
 Date of Hire: 05/22/2013 
 Email: Minke.Mies@gastech.com.kronos 
