--- VAST9L7_revision_19_June_19_10:16_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_20_June_19_10:16_AM	2019-06-27 13:09:46.000000000 -0400
@@ -47,4 +47,8 @@
 Position: Truck Driver (Facilities) 
 Date of Hire: 01/14/2006 
 Email: Adan.Morlun@gastech.com.kronos 
-        
\ No newline at end of file
+ Morluniau, Cecilia 
+Position: Truck Driver (Facilities) 
+Date of Hire: 04/30/2009 
+Email: Cecilia.Morluniau@gastech.com.kronos 
+         
\ No newline at end of file
