--- VAST9L7_revision_27_June_19_10:19_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_28_June_19_10:20_AM	2019-06-27 13:09:46.000000000 -0400
@@ -79,4 +79,8 @@
 Position: Site Control (Security) 
 Date of Hire: 08/17/2013 
 Email: Loreto.Bodrogi@gastech.com.kronos 
+ Ferro, Inga 
+Position: Site Control (Security) 
+Date of Hire: 01/11/2013 
+Email: Inga.Ferro@gastech.com.kronos 
          
\ No newline at end of file
