--- VAST9L7_revision_97_June_20_10:14_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_98_June_20_10:51_AM	2019-06-27 13:09:46.000000000 -0400
@@ -121,5 +121,6 @@
  LEADER: Elian Karel 
 Associations through Military and University of Abila 
 Additional data on second day provided further insight to the necessary connections to 
-develop an organizational structrurestructure. 
- Personnel  
\ No newline at end of file
+develop an organizational structure. 
+ Personnel were able to extract the “missing” during the confusion they created with a fire 
+alarm,  
\ No newline at end of file
