--- VAST9L7_revision_14_June_19_10:12_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_15_June_19_10:15_AM	2019-06-27 13:09:46.000000000 -0400
@@ -27,4 +27,8 @@
 Position: Truck Driver (Facilities) 
 Date of Hire: 09/12/2009 
 Email: Albina.Hafon@gastech.com.kronos 
-          
\ No newline at end of file
+ Hawelon, Benito 
+Position: Truck Driver (Facilities) 
+Date of Hire: 02/21/2000 
+Email: Benito.Hawelon@gastech.com.kronos 
+         
\ No newline at end of file
