--- VAST9L7_revision_100_June_20_10:56_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_101_June_20_10:56_AM	2019-06-27 13:09:46.000000000 -0400
@@ -118,9 +118,14 @@
 Position: Janitor (Facilities) 
 Date of Hire: 03/18/2013 
 Email: Emile.Arpa@gastech.com.kronos 
- Associations through Military and , University of Abila, and the children of the founders of 
-POK 
+ Associations through Military, University of Abila, and the children of the founders of POK 
 Additional data on second day provided further insight to the necessary connections to 
 develop an organizational structure. 
  Personnel were able to extract the “missing” during the confusion they created with a fire 
-alarm,  
\ No newline at end of file
+alarm,  
+1
+1
+2
+3
+4
+5
\ No newline at end of file
