--- VAST9L7_revision_84_June_19_11:56_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_85_June_19_11:57_AM	2019-06-27 13:09:46.000000000 -0400
@@ -114,4 +114,7 @@
 Position: Truck Driver (Facilities) 
 Date of Hire: 07/12/2013 
 Email: Henk.Mies@gastech.com.kronos 
-  
\ No newline at end of file
+ Arpa, Emile G.D. (Little Oversight) 
+Position: Janitor (Facilities) 
+Date of Hire: 03/18/2013 
+Email: Emile.Arpa@gastech.com.kronos 
\ No newline at end of file
