--- VAST9L7_revision_48_June_19_10:39_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_49_June_19_10:39_AM	2019-06-27 13:09:46.000000000 -0400
@@ -36,7 +36,7 @@
 Position: Truck Driver (Facilities) 
 Date of Hire: 03/13/2007 
 Email: Claudio.Hawelon@gastech.com.kronos 
- Mies, Henk 
+ Mies, Henk H.D. 
 Position: Truck Driver (Facilities) 
 Date of Hire: 07/12/2013 
 Email: Henk.Mies@gastech.com.kronos 
