--- VAST9L7_revision_44_June_19_10:37_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_45_June_19_10:37_AM	2019-06-27 13:09:46.000000000 -0400
@@ -16,11 +16,12 @@
 Position: Janitor (Facilities) 
 Date of Hire: 03/18/2013 
 Email: Emile.Arpa@gastech.com.kronos 
- Awelon, Varro (K)G.D. 
+ Awelon, Varro G.D. 
 Position: Janitor (Facilities) 
 Date of Hire: 10/22/2012 
 Email: Varro.Awelon@gastech.com.kronos 
  Coginian, Dante (K) 
+G.D. 
 Position: Lead Janitor (Facilities) 
 Date of Hire: 01/07/1995 
 Email: Dante.Coginian@gastech.com.kronos 
