--- VAST9L7_revision_61_June_19_10:54_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_62_June_19_10:55_AM	2019-06-27 13:09:46.000000000 -0400
@@ -85,6 +85,7 @@
 Date of Hire: 01/11/2013 
 Email: Inga.Ferro@gastech.com.kronos 
   Articles w/ associated Date of interest 
-INFLAMMATION OF THE PROTESTS IN ABILA IN ANSWER TO CALLS TO THE ACTION 
-Daily Pegasus | 03/26/2014 
-      
\ No newline at end of file
+-INFLAMMATION OF THE PROTESTS IN ABILA IN ANSWER TO CALLS TO THE ACTION 
+Daily Pegasus | 03/26/2014 (7 SEP 2012) 
+ - 
+     
\ No newline at end of file
