--- VAST9L7_revision_33_June_19_10:23_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_34_June_19_10:27_AM	2019-06-27 13:09:46.000000000 -0400
@@ -1,4 +1,5 @@
-Employees with Kronos Military Service 
+Employees with Kronos Military Service  
+Mandatory for both genders (i.e. Israel) (review discharges) 
  Mies Haber, Ruscella (K) 
 Position: Assistant to Engineering Group Manager (Administration) 
 Date of Hire: 04/02/2003 
@@ -76,7 +77,7 @@
 Date of Hire: 08/15/2013 
 Email: Edvard.Vann@gastech.com.kronos 
  Bodrogi, Loreto 
- Position: Site Control (Security) 
+Position: Site Control (Security) 
 Date of Hire: 08/17/2013 
 Email: Loreto.Bodrogi@gastech.com.kronos 
  Ferro, Inga 
