--- VAST9L7_revision_20_June_19_10:16_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_21_June_19_10:16_AM	2019-06-27 13:09:46.000000000 -0400
@@ -51,4 +51,8 @@
 Position: Truck Driver (Facilities) 
 Date of Hire: 04/30/2009 
 Email: Cecilia.Morluniau@gastech.com.kronos 
+ Nant, Irene 
+Position: Truck Driver (Facilities) 
+Date of Hire: 11/30/2008 
+Email: Irene.Nant@gastech.com.kronos 
          
\ No newline at end of file
