--- VAST9L7_revision_81_June_19_11:55_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_82_June_19_11:55_AM	2019-06-27 13:09:46.000000000 -0400
@@ -102,4 +102,7 @@
 June 2011 
  Loretto Bodrogi (Kronos citizen, directs personnel during emergencies) 
     P.O.I.’s 
-  
\ No newline at end of file
+ Ferro, Inga G.D. 
+Position: Site Control (Security) 
+Date of Hire: 01/11/2013 
+Email: Inga.Ferro@gastech.com.kronos 
\ No newline at end of file
