--- VAST9L7_revision_12_June_19_10:11_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_13_June_19_10:11_AM	2019-06-27 13:09:46.000000000 -0400
@@ -19,4 +19,8 @@
 Position: Janitor (Facilities) 
 Date of Hire: 10/22/2012 
 Email: Varro.Awelon@gastech.com.kronos 
-      
\ No newline at end of file
+ Coginian, Dante (K) 
+Position: Lead Janitor (Facilities) 
+Date of Hire: 01/07/1995 
+Email: Dante.Coginian@gastech.com.kronos 
+       
\ No newline at end of file
