--- VASTA6W_revision_19_July_17_10:20_AM	2019-07-22 16:00:57.000000000 -0400
+++ VASTA6W_revision_20_July_17_10:24_AM	2019-07-22 16:00:57.000000000 -0400
@@ -12,4 +12,7 @@
 1/21/2014 - President and CEO Sten Sanjorge Jr, CFO Ingrid Barranco, CIO 
 Ada campo-Corrente, COO Orhan Strum, and environment civil servant 
 Willem Vasco-Pais van GAStech. 
-1/21/2014 - President/CEO avoided (or escaped?) kidnapping. 
\ No newline at end of file
+1/21/2014 - President/CEO avoided (or escaped?) kidnapping. 
+1/21/2014 - President and CEO Sten Sanjorge Jr, CFO Ingrid Precipice, 
+Field-Corrente of Ada of I BACK WATER, rasgueo of Orhan de GAStech of 
+the COO, and environmental official Willem Basoue-Country. 
\ No newline at end of file
