--- VASTA6W_revision_26_July_17_11:13_AM	2019-07-22 16:00:57.000000000 -0400
+++ VASTA6W_revision_27_July_17_11:13_AM	2019-07-22 16:00:57.000000000 -0400
@@ -30,5 +30,12 @@
 million each for CFO Ingrid Barranco, CIO Ada Campo-Corrente, and COO 
 Orhan Strum. 
 1/20/2014 - Sten St. George, Jr. by $1.96 eye-jumping billion for its part 
-control of international company of drilling and exploration of gas. The environmental adviser of safety Willem Vasco-Feed also gained $38.5 
-million.  
\ No newline at end of file
+control of international company of drilling and exploration of gas. The 
+environmental adviser of safety Willem Vasco-Feed also gained $38.5 
+million.  
+ Send to: VastA6W@ncsu-las.net
+   1
+1
+2
+3
+4
\ No newline at end of file
