--- VASTOMS_revision_32_June_19_10:51_AM	2019-06-27 13:15:28.000000000 -0400
+++ VASTOMS_revision_33_June_19_10:52_AM	2019-06-27 13:15:28.000000000 -0400
@@ -105,6 +105,7 @@
  WHO ARE the PROTESTERS
  Athena Speaks | 08/24/2012
  popular leader of the POK who died in prison
- 10 year old girl who dies from illnesses
+ WHO ARE the PROTESTERS
+ Athena Speaks | 08/24/2012
  Mestache Vardshvna BUMP OF PROTESTS IN ABILA IN RESPONSE TO THE CALLS TO THE ACTION
  The Orb | 09/08/2012
