--- VASTOMS_revision_133_June_20_10:44_AM	2019-06-27 13:15:28.000000000 -0400
+++ VASTOMS_revision_134_June_20_10:44_AM	2019-06-27 13:15:28.000000000 -0400
@@ -157,5 +157,6 @@
  Update, 3:05 PM: Some police have left the scene, but others remain to outside to secure the building while others are still in the building. There is much speculation about their activities, but police have yet to respond to our queries for information.
  Update, 3:20 PM: GAStech employees are being allowed to leave the building.
  Update, 6:40 PM: There is speculation that the GAStech personnel who have arrived from Tethys are security experts who are assisting police with the investigation.
+ Update, 7:15 PM: Abila police have announced there will be a short statement made to the press at 7:30 pm.
  Update, 7:40 PM: Carman informed the press that police do not believe the missing people have left the island of Kronos.
  Update, 8:15 PM: A government official, who declined to be identified, says that police and government investigators are working around the clock to ensure the GAStech employees are found and that the kidnappers are brought to justice.
