--- VASTOMS_revision_137_June_20_10:45_AM	2019-06-27 13:15:28.000000000 -0400
+++ VASTOMS_revision_138_June_20_10:45_AM	2019-06-27 13:15:28.000000000 -0400
@@ -151,6 +151,7 @@
 ---Reported Timeline---
  Breaking: Emergency at GAStech Headquarters Building [Updates] Cato Rossini, Marcella Trapani
  Kronos Star | 01/20/2014
+ Update, 10:30 AM: Abila fire department responds to the alarm. Firemen are seen entering the building.
  Update, 11:45 AM: Fire department officials allow GAStech employees to re-enter the building.
  Update, 12:20 PM: The Abila police have been called to respond to GAStech headquarters for an unspecified emergency. (editor's note - The call was overheard on a police scanner. The Abila police know this channel is monitored by the media and often do not give the reason for the response)
  Update, 12:40 PM: Additional police have arrived at GAStech headquarters. They are setting up crowd control barricades around the building. Police have not commented on their activities.
@@ -162,6 +163,7 @@
  Update, 3:20 PM: GAStech employees are being allowed to leave the building.
  Update, 3:40 PM: One GAStech employee, who declined to be identified, reports having seen several people dressed in black in the building before the fire alarm. These people were not GAStech employees are 'were acting odd and lurking about'.
  Update, 5:50 PM: According to a GAStech employee, the GAStech executives left the building after their meeting this morning and were headed to the reception at the capitol building. When asked about why we were unable to reach them by phone, the employee replied that it was standard corporate security procedure for the executives to be rushed to a secure location in the event of a threat like the kidnapping.
+ Update, 6:30 PM: The GAStech executive jet arrived at the airport. The occupants were taken by limousine to the GAStech building.
  Update, 6:40 PM: There is speculation that the GAStech personnel who have arrived from Tethys are security experts who are assisting police with the investigation.
  Update, 7:15 PM: Abila police have announced there will be a short statement made to the press at 7:30 pm.
  Update, 7:40 PM: Carman informed the press that police do not believe the missing people have left the island of Kronos.
