--- VASTOMS_revision_138_June_20_10:45_AM	2019-06-27 13:15:28.000000000 -0400
+++ VASTOMS_revision_139_June_20_10:46_AM	2019-06-27 13:15:28.000000000 -0400
@@ -162,6 +162,7 @@
  Update, 3:05 PM: Some police have left the scene, but others remain to outside to secure the building while others are still in the building. There is much speculation about their activities, but police have yet to respond to our queries for information.
  Update, 3:20 PM: GAStech employees are being allowed to leave the building.
  Update, 3:40 PM: One GAStech employee, who declined to be identified, reports having seen several people dressed in black in the building before the fire alarm. These people were not GAStech employees are 'were acting odd and lurking about'.
+ Update, 5:30 PM: We have several reports from GAStech employees that confirm a kidnapping took place, but at this time there is no information on who was involved. Unofficial reports from the police indicate that they believe the Protectors of Kronos (POK) are behind the kidnapping and that they requested government assistance with the investigation. We have received no information which could confirm the POK's role in this alleged kidnapping, but note that they have grown increasingly violent over the past 5 years.
  Update, 5:50 PM: According to a GAStech employee, the GAStech executives left the building after their meeting this morning and were headed to the reception at the capitol building. When asked about why we were unable to reach them by phone, the employee replied that it was standard corporate security procedure for the executives to be rushed to a secure location in the event of a threat like the kidnapping.
  Update, 6:30 PM: The GAStech executive jet arrived at the airport. The occupants were taken by limousine to the GAStech building.
  Update, 6:40 PM: There is speculation that the GAStech personnel who have arrived from Tethys are security experts who are assisting police with the investigation.
