--- VASTOMS_revision_20_June_19_10:22_AM	2019-06-27 13:15:28.000000000 -0400
+++ VASTOMS_revision_21_June_19_10:23_AM	2019-06-27 13:15:28.000000000 -0400
@@ -18,7 +18,7 @@
                           
  2 jets leave seven or eight passengers of the first plane were 'business type' and appeared to be in a hurry. also took seven or eight 'business types', but similar 'relaxed much that the first group of passengers'. Danisliau said to second group of similar passengers 'to celebrate something.' THE ABILA AIRPORT TODAY CONFIRMS TWO JETS DEPRIVED TO THE LEFT                      
   The Explainer | 01/20/2014                      
-                          
+                         
                           
                           
                           
