--- VASTWC0_revision_43_July_18_8:41_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_44_July_18_8:42_AM	2019-07-30 15:54:47.000000000 -0400
@@ -70,4 +70,10 @@
  Mandor Vann is closely related to Isia Vann 
  A large number of the POK members recruited since 2001 are under-employed and under- 
 educated young males 
- Isia Vann 
\ No newline at end of file
+ Isia Vann- Isia Vann, an Elodis native, is the older brother of Juliana Vann, the 10-year-old 
+girl who died in 1998 after an illness related to drinking contaminated water from the 
+Tiskele River, and nephew to Mandor Vann. Isia is one of several POK members who 
+advocate for a more forceful approach to gaining attention to the plight of people living 
+downstream of Tiskele Bend. He has been arrested several times for civil disobedience and 
+trespassing during POK rallies. 
+ 
\ No newline at end of file
