--- VASTWC0_revision_90_July_18_10:44_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_91_July_18_10:45_AM	2019-07-30 15:54:47.000000000 -0400
@@ -120,25 +120,26 @@
 later today. One can't help but wonder how much healthcare 
 could be provided with only half the funds used to orchestrate 
 this spectacle.  
-b.1018 - A fire alarm has gone off at GAStech headquarters. 
+b. 
+c.1018 - A fire alarm has gone off at GAStech headquarters. 
 People are evacuating the building, and so far no sign of 
 Sanjorge or his executive staff. Officials have asked reporters to 
 move away from the building to allow the fire department 
 access. This has the flavor of yet another media ploy to avoid 
 confrontation from the people of Kronos.  
-c.1025 - The building is being searched for signs of trouble by the 
+d.1025 - The building is being searched for signs of trouble by the 
 fire department. There has been no confirmation of a bomb 
 threat, but I haven't seen a bomb squad appear. 
-d.1052 - I just received word from Illumination correspondent 
+e.1052 - I just received word from Illumination correspondent 
 Petrus Gerhard that CEO Sanjorge arrived at the capitol building 
 a few minutes ago. 20 Jan 2014 
-e.1130 - Some of the fire officials have exited the building and are 
+f.1130 - Some of the fire officials have exited the building and are 
 talking to the officials, but no announcements have been made 
 about the events at GAStech headquarters.  
-f.1146 - Fire Captain Federigo Accardi has apparently given the 
+g.1146 - Fire Captain Federigo Accardi has apparently given the 
 all-clear because several officials are entering the building, the 
 perimeter is being dismantled and the fire department is 
-packing their gear. g.1153 - BREAKING NEWS Readers, this is correspondent Petrus 
+packing their gear. h.1153 - BREAKING NEWS Readers, this is correspondent Petrus 
 Gerhard writing from an unknown location. I was contacted a 
 short time ago by members of the Protectors of Kronos social 
 action movement to convey a very important message to the 
@@ -150,10 +151,10 @@
 location that even I do not recognize. Readers, the next post you 
 read will be one of the most important you have ever read, or 
 indeed, will ever read. 
-h.1155 - The fire department spokesman announced the building 
+i.1155 - The fire department spokesman announced the building 
 is clear and it was a false alarm. Employees return to the 
 building. 
-i.1200 - TO THE PEOPLE OF KRONOS: NO LONGER WILL WE 
+j.1200 - TO THE PEOPLE OF KRONOS: NO LONGER WILL WE 
 ACQUIESCE TO THE UNFETTERED KLEPTOCRACY THAT HAS 
 BECOME OUR GOVERNMENT AND TO THE IMPERIALIST 
 ACTIVITIES OF THE FOREIGN CORPORATIONS ON OUR 
@@ -171,31 +172,31 @@
 OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE FORM OF 
 $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE SUFFERING OF THE PEOPLE. THIS IS THE FIRST STEP 
 TOWARD TAKING OUR COUNTRY BACK. 
-j.1512 - Two police vehicles each with two policemen in inside 
+k.1512 - Two police vehicles each with two policemen in inside 
 have just left GAStech headquarters with no lights or sirens. 20 
 Jan 2014 
-k.1515 - BREAKING NEWS - Homeland Illumination 
+l.1515 - BREAKING NEWS - Homeland Illumination 
 correspondent Petrus Gerhard has just spoken with an airport 
 official who confirms that a private jet left Abila airport at 1430, 
 with a number of persons on board. Reports that the GAStech 
 executives were passengers on this jet are unconfirmed. 
-l.1744 - Media reports are coming in that a number of GAStech 
+m.1744 - Media reports are coming in that a number of GAStech 
 employees were kidnapped. The identity of the passengers on 
 the mystery jet that departed earlier today from the Abila Airport 
 remains unknown. 
-m.1837 - There are unconfirmed reports that the GAStech 
+n.1837 - There are unconfirmed reports that the GAStech 
 employees were kidnapped. Media outlets are speculating that 
 their newfound wealth might provide a motive. Perhaps 
 someone should locate the caterers from the executive 
 breakfast? 
-n.1942 - BREAKING NEWS - Kronos police spokesman has 
+o.1942 - BREAKING NEWS - Kronos police spokesman has 
 released a statement. 'There are approximately fourteen 
 individuals unaccounted for among the GAStech staff. It has 
 not yet been determined if all of these individuals are in fact 
 missing.' They said the case is under investigation and asked 
 that anyone who might have information about the missing 
 employees please contact Abila police immediately. 
-o.2020 - Media outlets are reporting that a government 
+p.2020 - Media outlets are reporting that a government 
 spokesman who asked not to be named because he was not 
 authorized to speak to the press stated that the GAStech 
 employees had been kidnapped and the Kronos government 
