--- VASTWC0_revision_38_July_17_10:33_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_39_July_17_11:13_AM	2019-07-30 15:54:47.000000000 -0400
@@ -60,4 +60,10 @@
 b.Kronos military service 
  21.Ferro, Inga- Site Control (Security) 
 a.Kronos citizenship 
-b.Kronos military service 
\ No newline at end of file
+b.Kronos military service 
+   Claudio.Hawelon@gastech.com.kronos, Cecilia.Morluniau@gastech.com.kronos, 
+Benito.Hawelon@gastech.com.kronos, Emile.Arpa@gastech.com.kronos, 
+Adan.Morlun@gastech.com.kronos, Valeria.Morlun@gastech.com.kronos, 
+Dante.Coginian@gastech.com.kronos, Henk.Mies@gastech.com.kronos, 
+Varro.Awelon@gastech.com.kronos, Dylan.Scozzese@gastech.com.kronos, 
+Albina.Hafon@gastech.com.kronos, Irene.Nant@gastech.com.kronos 
\ No newline at end of file
