--- VASTWC0_revision_30_July_17_10:21_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_31_July_17_10:23_AM	2019-07-30 15:54:47.000000000 -0400
@@ -43,4 +43,7 @@
  15.Scozzese, Dylan- Truck Driver (Facilities) 
 a.Kronos citizenship 
 b.Kronos military service 
-16. 
\ No newline at end of file
+ 16.Mies, Minke- Perimeter Control (Security) 
+a.Kronos citizenship 
+b.Kronos military service 
+17. 
\ No newline at end of file
