--- VASTWC0_revision_36_July_17_10:26_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_37_July_17_10:26_AM	2019-07-30 15:54:47.000000000 -0400
@@ -54,7 +54,9 @@
 b.Kronos military service 
  19.Vann, Edvard- Perimeter Control (Security) 
 a.Kronos citizenship 
-b.Kronos military service 20.Vann, IsiaBodrogi, Loreto- Site Control (Security) 
+b.Kronos military service 20.Bodrogi, Loreto- Site Control (Security) 
 a.Kronos citizenship 
 b.Kronos military service 
-21. 
\ No newline at end of file
+ 21.Ferro, Inga- Site Control (Security) 
+a.Kronos citizenship 
+b.Kronos military service 
\ No newline at end of file
