--- VASTWC0_revision_98_July_18_11:35_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_99_July_18_11:35_AM	2019-07-30 15:54:47.000000000 -0400
@@ -24,7 +24,8 @@
  8.Hawelon, Benito- Truck Driver (Facilities) 
 a.Kronos citizenship 
 b.Kronos military service 
- 9.Hawelon, Claudio- Truck Driver (Facilities) a.Kronos citizenship 
+ 9.Hawelon, Claudio- Truck Driver (Facilities) 
+a.Kronos citizenship 
 b.Kronos military service 
 10.Mies, Henk- Truck Driver (Facilities) 
 a.Kronos citizenship 
@@ -53,7 +54,8 @@
  18.Vann, Isia- Perimeter Control (Security) 
 a.Kronos citizenship 
 b.Kronos military service 
- 19.Vann, Edvard- Perimeter Control (Security) a.Kronos citizenship 
+ 19.Vann, Edvard- Perimeter Control (Security) 
+a.Kronos citizenship 
 b.Kronos military service 
  20.Bodrogi, Loreto- Site Control (Security) 
 a.Kronos citizenship 
@@ -82,7 +84,8 @@
 pianotent, and environmental leader Willem Vasco-Feed. 
  Poss security suspects - VIP visit 20JAN 
 Hideki.Cocinaro@gastech.com.kronos,  
-Varja.Lagos@gastech.com.kronos,  Isia.Vann@gastech.com.kronos SUS  27yo male 
+Varja.Lagos@gastech.com.kronos,  
+Isia.Vann@gastech.com.kronos SUS  27yo male 
   Hennie.Osvaldo@gastech.com.kronos,  SUS  25yo male 
 Loreto.Bodrogi@gastech.com.kronos,  SUS  24yo male POSS Son of Henk Bodrogi 
 Varja.Lagos@gastech.com.kronos,  
@@ -139,7 +142,8 @@
 talking to the officials, but no announcements have been made 
 about the events at GAStech headquarters.  
 g.1146 - Fire Captain Federigo Accardi has apparently given the 
-all-clear because several officials are entering the building, the perimeter is being dismantled and the fire department is 
+all-clear because several officials are entering the building, the 
+perimeter is being dismantled and the fire department is 
 packing their gear. 
 h.1153 - BREAKING NEWS Readers, this is correspondent Petrus 
 Gerhard writing from an unknown location. I was contacted a 
@@ -171,7 +175,8 @@
 THEIR OWN BENEFIT. IN AN ACT OF DEFIANCE WE 
 APPREHENDED THE ELITISTS WHO HAVE EXPLOITED OUR 
 RESOURCES, KILLED OUR CHILDREN AND RUINED OUR WAY 
-OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE FORM OF $20 MILLION WHICH WILL BE USED TO ALLEVIATE THE 
+OF LIFE. WE ARE DEMANDING RECOMPENSE IN THE FORM OF 
+$20 MILLION WHICH WILL BE USED TO ALLEVIATE THE 
 SUFFERING OF THE PEOPLE. THIS IS THE FIRST STEP 
 TOWARD TAKING OUR COUNTRY BACK. 
 k.1512 - Two police vehicles each with two policemen in inside 
@@ -203,7 +208,8 @@
 authorized to speak to the press stated that the GAStech 
 employees had been kidnapped and the Kronos government 
 was working hard to identify the lawbreakers. One wonders if 
-they are also working hard to recover the GAStech people as well, or if this is an opportunity to profit from the misfortune of 
+they are also working hard to recover the GAStech people as 
+well, or if this is an opportunity to profit from the misfortune of 
 others.  
  3.Plausible explanation of why employees are missing and recommend 
 next steps. 
@@ -211,4 +217,12 @@
 POK has a more radical leadership and they are taking action 
 against the injustice that is happening within their community 
 at the hands of the GAStech executives. I recommend that the 
-company should compensate them  
\ No newline at end of file
+company should compensate them  
+1
+1
+2
+3
+4
+5
+6
+7
\ No newline at end of file
