--- VASTWC0_revision_16_July_17_10:12_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_17_July_17_10:12_AM	2019-07-30 15:54:47.000000000 -0400
@@ -14,7 +14,7 @@
  5.Awelon, Varro- Janitor (Facilities) 
 a.Kronos citizenship 
 b.Kronos military service 
- 6.Awelon, Varro- Lead Janitor (Facilities) 
+ 6.Awelon, VarroCoginian, Dante- Lead Janitor (Facilities) 
 a.Kronos citizenship 
 b.Kronos military service 
 7. 
\ No newline at end of file
