--- VASTWC0_revision_71_July_18_9:41_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_72_July_18_9:43_AM	2019-07-30 15:54:47.000000000 -0400
@@ -82,7 +82,7 @@
 pianotent, and environmental leader Willem Vasco-Feed. 
  Poss security suspects - VIP visit 20JAN 
 Hideki.Cocinaro@gastech.com.kronos,  
-Varja.Lagos@gastech.com.kronos,  Isia.Vann@gastech.com.kronos 
+Varja.Lagos@gastech.com.kronos,  Isia.Vann@gastech.com.kronos SUS  27yo male 
   Hennie.Osvaldo@gastech.com.kronos,  SUS  25yo male 
 Loreto.Bodrogi@gastech.com.kronos,  SUS  24yo male 
 Varja.Lagos@gastech.com.kronos,  
