--- VASTWC0_revision_42_July_18_8:39_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_43_July_18_8:41_AM	2019-07-30 15:54:47.000000000 -0400
@@ -70,4 +70,4 @@
  Mandor Vann is closely related to Isia Vann 
  A large number of the POK members recruited since 2001 are under-employed and under- 
 educated young males 
- 
\ No newline at end of file
+ Isia Vann 
\ No newline at end of file
