--- VASTWC0_revision_49_July_18_9:06_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_50_July_18_9:06_AM	2019-07-30 15:54:47.000000000 -0400
@@ -85,5 +85,8 @@
 a.Leaders- 
 i.Elian Karel and if something should happen to him the 
 group would fall to Silvia Marek or Mandor Vann  
-ii.Dominant personalities- Isia Vann, Lucio Jakb, Lorenzo Di 
-Stefano 
\ No newline at end of file
+Dominant personalities- Isia Vann, Lucio Jakb, Lorenzo Di Stefano 
+b.Part of the extended Network- 
+i.Dominant personalities- Isia Vann, Lucio Jakb, Lorenzo Di 
+Stefano 
+ii. 
\ No newline at end of file
