--- VASTWC0_revision_40_July_17_11:23_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_41_July_18_8:37_AM	2019-07-30 15:54:47.000000000 -0400
@@ -66,4 +66,5 @@
 Adan.Morlun@gastech.com.kronos, Valeria.Morlun@gastech.com.kronos, 
 Dante.Coginian@gastech.com.kronos, Henk.Mies@gastech.com.kronos, 
 Varro.Awelon@gastech.com.kronos, Dylan.Scozzese@gastech.com.kronos, 
-Albina.Hafon@gastech.com.kronos, Irene.Nant@gastech.com.kronos 
\ No newline at end of file
+Albina.Hafon@gastech.com.kronos, Irene.Nant@gastech.com.kronos 
+ Mandor Vann is closely related to Isia Vann 
\ No newline at end of file
