--- VASTWC0_revision_18_July_17_10:13_AM	2019-07-30 15:54:47.000000000 -0400
+++ VASTWC0_revision_19_July_17_10:14_AM	2019-07-30 15:54:47.000000000 -0400
@@ -20,4 +20,6 @@
  7.Hafon, Albina- Truck Driver (Facilities) 
 a.Kronos citizenship 
 b.Kronos military service 
-8. 
\ No newline at end of file
+ 8.Hawelon, Benito- Truck Driver (Facilities) 
+a.Kronos citizenship 
+b.Kronos military service 
\ No newline at end of file
