--- VASTOIC_revision_51_July_18_9:06_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_52_July_18_9:07_AM	2019-07-30 15:57:21.000000000 -0400
@@ -18,7 +18,7 @@
 Mandor Vann - retired military, linguist, political strategist, related to Isia and Juliana Vann 
 Isia Vann - brother of Juliana Vann (deceased) - history of civil disobedience and trespassing 
 Lucio Jakab - co-founder of SOW, leads communications for the group 
-Lorenjzo Di Stefano -  
+Lorenjzo Di Stefano - college professor at UofAbila 
   The gourp’s first actions included contacting international agencies and speaking to GAStech 
 CEO Sten Sanjorge Jr.  After the meeting and test results from the WFA on the river water the 
 group increased it’s lobbying in the capital of Abila, conducted peaceful protests, and met with 
