--- VASTOIC_revision_30_July_18_8:30_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_31_July_18_8:32_AM	2019-07-30 15:57:21.000000000 -0400
@@ -13,4 +13,6 @@
 Carmine Osvaldo - Elodis chamber of commerce, military with Bodrogi 
 Elian Karel 
    The gourp’s first actions included contacting international agencies and speaking to GAStech 
-CEO sSten Sanjorge Jr.  
\ No newline at end of file
+CEO Sten Sanjorge Jr.  After the meeting and test results from the WFA on the river water the 
+group increased it’s lobbying in the capital of Abila, conducted peaceful protests, and met with 
+low-level government personnel.  
\ No newline at end of file
