--- VASTOIC_revision_73_July_18_9:59_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_74_July_18_9:59_AM	2019-07-30 15:57:21.000000000 -0400
@@ -12,7 +12,7 @@
 Isia Vann (security) 
 Edvard Vann (security) 
 Hennie Osvaldo (security) 
-Loreto Bodrogi (security) 
+Loreto Bodrogi (Site control security) 
   Leadership: 
 Henk Bodrogi - farmer, military  
 Carmine Osvaldo - Elodis chamber of commerce, military with Bodrogi 
