--- VASTOIC_revision_29_July_18_8:29_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_30_July_18_8:30_AM	2019-07-30 15:57:21.000000000 -0400
@@ -12,5 +12,5 @@
 Henk Bodrogi - farmer, military  
 Carmine Osvaldo - Elodis chamber of commerce, military with Bodrogi 
 Elian Karel 
-   The gourps first actions included contacting international agencies and speaking to GAStech 
-CEO sten Sanjorge Jr.  
\ No newline at end of file
+   The gourp’s first actions included contacting international agencies and speaking to GAStech 
+CEO sSten Sanjorge Jr.  
\ No newline at end of file
