--- VASTOIC_revision_27_July_18_8:24_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_28_July_18_8:24_AM	2019-07-30 15:57:21.000000000 -0400
@@ -10,6 +10,6 @@
 Bodrogi, Loreto (security) 
   Origional Members: 
 Henk Bodrogi - farmer, military  
-Carmine Osvaldo - elodis chamber of commerce, military with Bodrogi 
+Carmine Osvaldo - eElodis chamber of commerce, military with Bodrogi 
 Elian Karel 
     
\ No newline at end of file
