--- VASTOIC_revision_80_July_18_10:31_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_81_July_18_10:31_AM	2019-07-30 15:57:21.000000000 -0400
@@ -50,5 +50,5 @@
 1/21/2014 - 5 Gastech execs located, 10 employees still missing at 0900 
 1/21/2014 - Pertrus Gerhard released public statement from POK at 1153 
  Possible Explanation: 
-Several members of GAStech security who have  
+Several members of GAStech security who have possible familiral  
  
\ No newline at end of file
