--- VASTOIC_revision_65_July_18_9:35_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_66_July_18_9:36_AM	2019-07-30 15:57:21.000000000 -0400
@@ -30,13 +30,14 @@
 low-level government personnel.  Cesare Nespola, Minister of Health, attempted to introduce 
 legislation to tax the oil and gas production to support healthcare and clean water, which was 
 defeated.  After which the group began protesting at the Tiskele Bend fields.  
- Once Kerel succeeded Bodrogi the remaining founding members all left the POK.  Karel begain 
-recruiting other small environmental organizations into the POK and the peaceful protests 
-began to become more violent and militant.  Time Line: 
+ Once Kerel succeeded Bodrogi the remaining founding members all left the POK.  Karel 
+begainbegan recruiting other small environmental organizations into the POK and the peaceful 
+protests began to become more violent and militant.  Time Line: 
 1/20/2014 - GAStech annual corporate metting 
 1/20/2014 - Fire reported at GAStech HQ at 1000 
 1/20/2014 - 15 GAStech employees reported missing 
 1/20/2014 - GAStech CEO found 
-1/20/2014 - GAStech Senior Security Officer 
+1/20/2014 - GAStech Senior Security Officer at 1400 
+1/20/2014 - Immediate arrival refueling and departure of GAStech CEO’s private jet 
 1/21/2014 - 5 Gastech execs located, 10 employees still missing 
  
\ No newline at end of file
