--- VASTOIC_revision_84_July_18_10:39_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_85_July_18_10:41_AM	2019-07-30 15:57:21.000000000 -0400
@@ -53,5 +53,6 @@
  Possible Explanation: 
 Several members of GAStech security who have possible family connections to POK leadership 
 may have abducted the missing employees and concealed their movement/abduction from the 
-building using a fire alarm.  
+building using a fire alarm.  The missing employees may have been forced to board a private 
+plane which left Abila towards an unknown location 
  
\ No newline at end of file
