--- VASTOIC_revision_68_July_18_9:40_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_69_July_18_9:47_AM	2019-07-30 15:57:21.000000000 -0400
@@ -37,6 +37,7 @@
 1/20/2014 - Fire reported at GAStech HQ at 1000 
 1/20/2014 - 15 GAStech employees reported missing 
 1/20/2014 - GAStech CEO found 
+1/20/2014 - $20M ransom demand made by group claiming to be the POK (for CEO) 
 1/20/2014 - GAStech Senior Security Officer at 1400 
 1/20/2014 - Immediate arrival/refueling and departure of GAStech CEO’s private jet 
 ●Departed Tethys, returning to Abila w/ 6 GAStech employees onboard 
