--- VASTOIC_revision_42_July_18_8:59_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_43_July_18_9:00_AM	2019-07-30 15:57:21.000000000 -0400
@@ -11,7 +11,8 @@
   Leadership: 
 Henk Bodrogi - farmer, military  
 Carmine Osvaldo - Elodis chamber of commerce, military with Bodrogi 
-Elian Karel - Bodrogi’s successor , leader and major recruiter for the POK 
+Elian Karel - Bodrogi’s successor, leader and major recruiter for the POK, could turn the group 
+into a political organization 
 Silvia Marek (encourages peaceful engagement) 
 Mandor Vann - military, related to Isia and Juliana Vann 
    The gourp’s first actions included contacting international agencies and speaking to GAStech 
