--- VASTOIC_revision_34_July_18_8:39_AM	2019-07-30 15:57:21.000000000 -0400
+++ VASTOIC_revision_35_July_18_8:41_AM	2019-07-30 15:57:21.000000000 -0400
@@ -11,10 +11,12 @@
   Origional Members: 
 Henk Bodrogi - farmer, military  
 Carmine Osvaldo - Elodis chamber of commerce, military with Bodrogi 
- Elian Karel - Bodrogi’s successor  
+Elian Karel - Bodrogi’s successor  
    The gourp’s first actions included contacting international agencies and speaking to GAStech 
 CEO Sten Sanjorge Jr.  After the meeting and test results from the WFA on the river water the 
 group increased it’s lobbying in the capital of Abila, conducted peaceful protests, and met with 
 low-level government personnel.  Cesare Nespola, Minister of Health, attempted to introduce 
 legislation to tax the oil and gas production to support healthcare and clean water, which was 
-defeated. 
\ No newline at end of file
+defeated.  After which the group began protesting at the Tiskele Bend fields.  
+ Once Kerel succeded Bodrogi the remaining founding members all left the POK.  
+ 
\ No newline at end of file
