--- VAST9L7_revision_65_June_19_10:58_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_66_June_19_10:58_AM	2019-06-27 13:09:46.000000000 -0400
@@ -88,7 +88,7 @@
 -INFLAMMATION OF THE PROTESTS IN ABILA IN ANSWER TO CALLS TO THE ACTION 
 Daily Pegasus | 03/26/2014 (7 SEP 2012) 
  -Employees of GASTech removed in Kronos 
-World Journal | 01/21/2014 (President and PRESIDENT Sten Sanjorge Jr, CFO Ingrid Barranco, 
+World Journal | 01/21/2014 ( 5 of 14: President Sten Sanjorge Jr, CFO Ingrid Barranco, 
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech pianotent, and 
 environmental leader Willem Vasco-Feed.) 
       
\ No newline at end of file
