--- VAST9L7_revision_94_June_20_10:09_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_95_June_20_10:12_AM	2019-06-27 13:09:46.000000000 -0400
@@ -119,4 +119,7 @@
 Date of Hire: 03/18/2013 
 Email: Emile.Arpa@gastech.com.kronos 
  LEADER: Elian Karel 
-Associations through Military and University of Abila 
\ No newline at end of file
+Associations through Military and University of Abila 
+Additional data on second day provided further insight to the necessary connections to 
+develop an organizational strructrure. 
+ Personnel  
\ No newline at end of file
