--- VAST9L7_revision_67_June_19_11:00_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_68_June_19_11:00_AM	2019-06-27 13:09:46.000000000 -0400
@@ -92,5 +92,7 @@
 Campo-Corrente of ADA of CIO, ROUCOULEMENT Orhan de GAStech pianotent, and 
 environmental leader Willem Vasco-Feed.) 
 -RANSOM DEMANDS MADE IN GASTECH KIDNAPPING 
-Who What News | 01/21/2014 ( 
+Who What News | 01/21/2014 (The Protectors of Kronos, was founded in 1998 in 
+Elodis, a rural township outside of the capital Abila) 
+- 
      
\ No newline at end of file
