--- VAST9L7_revision_96_June_20_10:12_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_97_June_20_10:14_AM	2019-06-27 13:09:46.000000000 -0400
@@ -121,5 +121,5 @@
  LEADER: Elian Karel 
 Associations through Military and University of Abila 
 Additional data on second day provided further insight to the necessary connections to 
-develop an organizational strructrure. 
+develop an organizational structrurestructure. 
  Personnel  
\ No newline at end of file
