--- VAST9L7_revision_10_June_19_10:07_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_11_June_19_10:10_AM	2019-06-27 13:09:46.000000000 -0400
@@ -11,4 +11,8 @@
 Position: Assistant to Security Group Manager (Administration) 
 Date of Hire: 12/22/2011 
 Email: Cornelia.Lais@gastech.com.kronos 
-      
\ No newline at end of file
+ Arpa, Emile (K) 
+Position: Janitor (Facilities) 
+Date of Hire: 03/18/2013 
+Email: Emile.Arpa@gastech.com.kronos 
+     
\ No newline at end of file
