--- VAST9L7_revision_42_June_19_10:34_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_43_June_19_10:36_AM	2019-06-27 13:09:46.000000000 -0400
@@ -12,7 +12,7 @@
 Position: Assistant to Security Group Manager (Administration) 
 Date of Hire: 12/22/2011 
 Email: Cornelia.Lais@gastech.com.kronos 
-Arpa, Emile (K) 
+Arpa, Emile (K)G.D. 
 Position: Janitor (Facilities) 
 Date of Hire: 03/18/2013 
 Email: Emile.Arpa@gastech.com.kronos 
