--- VAST9L7_revision_76_June_19_11:19_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_77_June_19_11:20_AM	2019-06-27 13:09:46.000000000 -0400
@@ -12,7 +12,7 @@
 Position: Assistant to Security Group Manager (Administration) 
 Date of Hire: 12/22/2011 
 Email: Cornelia.Lais@gastech.com.kronos 
-Arpa, Emile G.D. 
+Arpa, Emile G.D. (Little Oversight) 
 Position: Janitor (Facilities) 
 Date of Hire: 03/18/2013 
 Email: Emile.Arpa@gastech.com.kronos 
