--- VAST9L7_revision_11_June_19_10:10_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_12_June_19_10:11_AM	2019-06-27 13:09:46.000000000 -0400
@@ -15,4 +15,8 @@
 Position: Janitor (Facilities) 
 Date of Hire: 03/18/2013 
 Email: Emile.Arpa@gastech.com.kronos 
-     
\ No newline at end of file
+ Awelon, Varro (K) 
+Position: Janitor (Facilities) 
+Date of Hire: 10/22/2012 
+Email: Varro.Awelon@gastech.com.kronos 
+      
\ No newline at end of file
