--- VAST9L7_revision_73_June_19_11:10_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_74_June_19_11:16_AM	2019-06-27 13:09:46.000000000 -0400
@@ -4,7 +4,7 @@
 Position: Assistant to Engineering Group Manager (Administration) 
 Date of Hire: 04/02/2003 
 Email: Ruscella.Mies.Haber@gastech.com.kronos 
-Forluniau, Carla G.D. 
+Forluniau, Carla G.D. (specialized coordination of large group events) 
 Position: Assistant to IT Group Manager (Administration) 
 Date of Hire: 03/07/2005 
 Email: Carla.Forluniau@gastech.com.kronos 
