--- VAST9L7_revision_17_June_19_10:15_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_18_June_19_10:16_AM	2019-06-27 13:09:46.000000000 -0400
@@ -39,4 +39,8 @@
 Position: Truck Driver (Facilities) 
 Date of Hire: 07/12/2013 
 Email: Henk.Mies@gastech.com.kronos 
+ Morlun, Valeria 
+Position: Truck Driver (Facilities) 
+Date of Hire: 12/02/2003 
+Email: Valeria.Morlun@gastech.com.kronos 
         
\ No newline at end of file
