--- VAST9L7_revision_82_June_19_11:55_AM	2019-06-27 13:09:46.000000000 -0400
+++ VAST9L7_revision_83_June_19_11:55_AM	2019-06-27 13:09:46.000000000 -0400
@@ -105,4 +105,9 @@
  Ferro, Inga G.D. 
 Position: Site Control (Security) 
 Date of Hire: 01/11/2013 
-Email: Inga.Ferro@gastech.com.kronos 
\ No newline at end of file
+Email: Inga.Ferro@gastech.com.kronos 
+ Mies, Minke G.D. (takes necessary action when needed) 
+Position: Perimeter Control (Security) 
+Date of Hire: 05/22/2013 
+Email: Minke.Mies@gastech.com.kronos 
+  
\ No newline at end of file
