--- VASTBNJ_revision_15_June_19_10:20_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_16_June_19_10:26_AM	2019-06-27 13:11:32.000000000 -0400
@@ -15,8 +15,9 @@
 gifted chief. Which foreign chief must make is to learn how to communicate the 'manner 
 of Kronosian' and to take time to include/understand their personnel. 
 ○The people of Kronos do not like to take the responsability. They are frightened to lose 
-their work and do not like to be forced in sayinginto saying something which they do not 
-want withwant. The foreign Presidents must take account of this and have to take time to 
-include/understand how to communicate. Patience is key. Transparency can be an 
-excellent manner of coming to end this. It is a European value which Kronosians are very 
-sensitive to.  
\ No newline at end of file
+their work and do not like to be forced into saying something which they do not want. The 
+foreign Presidents must take account of this and have to take time to include/understand 
+how to communicate. Patience is key. Transparency can be an excellent manner of 
+coming to end this. It is a European value which Kronosians are very sensitive to.  
+○The people of Kronos work of overtime and socializes with their colleagues. It is very 
+different from Tethys where there is a clear border between work and the family. 
\ No newline at end of file
