--- VASTBNJ_revision_60_June_20_9:37_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_61_June_20_10:10_AM	2019-06-27 13:11:32.000000000 -0400
@@ -13,8 +13,8 @@
 today's events (KIDNAPPING AT GASTECH KRONOS|Who What News | 01/20/2014) . 
 Speculation about the motivation behind the kidnapping has also been rampant due to the recent 
 GAStech IPO. At this time no one has claimed responsibility for the kidnapping. 
-●Maha Salo | Motivator | VOICES - a blog about what is important to the people. 
-Posted a blog 01/21/2014 : 1200 
+●Maha Salo | Motivator | VOICES - a blog about what is important to the people. Posted a 
+blog 01/21/2014 : 1200 
 ●-The perspicacity of an expert: prosperity as foreign PRESIDENT in Kronos|World Journal | 
 03/26/2014: 
 ○GAStech, the principal industry of Krono during more than 15 years, after its discovery of 
