--- VASTBNJ_revision_82_June_20_10:33_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_83_June_20_10:44_AM	2019-06-27 13:11:32.000000000 -0400
@@ -15,7 +15,8 @@
 GAStech IPO. At this time no one has claimed responsibility for the kidnapping. 
 ●Maha Salo | Motivator | VOICES - a blog about what is important to the people. Posted a 
 blog 01/21/2014 : 1200 
-●Mies Haber, Ruscella 
+●Mies Haber, Ruscella (Could be a leader  due to she is on leadership possition on 
+gastech and she could  
 ○Position: Assistant to Engineering Group Manager (Administration) 
 ○Date of Hire: 04/02/2003 
 ○Email: Ruscella.Mies.Haber@gastech.com.kronos 
