--- VASTBNJ_revision_42_June_19_11:26_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_43_June_19_11:38_AM	2019-06-27 13:11:32.000000000 -0400
@@ -43,6 +43,8 @@
 SUPPRESSION OR DISTORTION. THE INACTIONS AND IGNORANCE OF 
 OUR ELECTED OFFICIALS HAVE FORCED US INTO THESE MEASURES.  
 ●KIDNAPPING AT GASTECH KRONOS|Who What News | 01/20/2014 
-○Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
-released after 6 hours, indicated the Protectors of Kronos were suspected as 
-being the perpetrators behind today's events. 
\ No newline at end of file
+○Edvard Vann, a GAStech - Kronos security guard who was interrogated and released 
+after 6 hours, indicated the Protectors of Kronos were suspected as being the 
+perpetrators behind today's events. Speculation about the motivation behind the 
+kidnapping has also been rampant due to the recent GAStech IPO. At this time no one has 
+claimed responsibility for the kidnapping. 
\ No newline at end of file
