--- VASTBNJ_revision_45_June_19_11:48_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_46_June_19_11:48_AM	2019-06-27 13:11:32.000000000 -0400
@@ -2,8 +2,8 @@
 1.Id at least 3 individuals that you believe may be associated with the POK (protectors of 
 Kronos) 
 2. Describes their involvement and provide evidence to support your hypothesis  
-Emails;: Inspection Request for site 
-Hideki.Cocinaro@gastech.com.kronos, Varja.Lagos@gastech.com.kronos, 
+Emails: Inspection Request for site 
+ Hideki.Cocinaro@gastech.com.kronos, Varja.Lagos@gastech.com.kronos, 
 Loreto.Bodrogi@gastech.com.kronos, Edvard.Vann@gastech.com.kronos, 
 Stenig.Fusil@gastech.com.kronos, 
 Hennie.Osvaldo@gastech.com.kronos, Felix.Resumir@gastech.com.kronos, 
