--- VASTBNJ_revision_11_June_19_10:14_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_12_June_19_10:15_AM	2019-06-27 13:11:32.000000000 -0400
@@ -7,4 +7,6 @@
 its immense gas layers close to the town of Elodis. 
 ○the economic environment changed since the Eighties. It is much more difficult to 
 succeed today. 
+○Competition among Kronosian and foreign companies is hard. It is difficult to gain the 
+hearts and the spirits of the population.  
 ○ 
\ No newline at end of file
