--- VASTBNJ_revision_66_June_20_10:13_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_67_June_20_10:14_AM	2019-06-27 13:11:32.000000000 -0400
@@ -19,14 +19,12 @@
 ○Position: Assistant to Engineering Group Manager (Administration) 
 ○Date of Hire: 04/02/2003 
 ○Email: Ruscella.Mies.Haber@gastech.com.kronos 
-○ 
-○Sex: Female 
-○ / DOB: 04/26/1964 / Citizenship: Kronos 
- ●Citizenship: Kronos 
-●Passport Issued: 01/01/2000 
+○Sex: Female / DOB: 04/26/1964 / Citizenship: Kronos 
+ ○ 
+○Passport Issued: 01/01/2000 / Passport Expires: 01/01/2000 
 ●Passport Expires: 01/01/2000 
-● 
-●Military Branch: Armed Forces of Kronos 
+○ 
+○Military Branch: Armed Forces of Kronos / Discharge: 10/01/1984 
 ●Discharge Date: 10/01/1984 
 ●Discharge Type: Honorable Discharge 
 ● 
