--- VASTBNJ_revision_39_June_19_11:23_AM	2019-06-27 13:11:32.000000000 -0400
+++ VASTBNJ_revision_40_June_19_11:25_AM	2019-06-27 13:11:32.000000000 -0400
@@ -42,4 +42,6 @@
 SUPPRESSION OR DISTORTION. THE INACTIONS AND IGNORANCE OF OUR ELECTED 
 OFFICIALS HAVE FORCED US INTO THESE MEASURES.  
 ●KIDNAPPING AT GASTECH KRONOS|Who What News | 01/20/2014 
-● 
\ No newline at end of file
+○Edvard Vann, a GAStech - Kronos security guard who was interrogated and 
+released after 6 hours, indicated the Protectors of Kronos were suspected as 
+being the perpetrators behind today's events. 
\ No newline at end of file
