--- VASTKD0_revision_115_June_20_10:12_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_116_June_20_10:15_AM	2019-06-27 13:14:16.000000000 -0400
@@ -202,13 +202,15 @@
 -Vann, Isia (Current employee) - shares name of Julian Vann(Julian Vann died in 1998 
 of the cancer caused by environmental toxins in its potable water) Daily 
 Pegasus | 03/26/2014 
- - Osvaldo, Hennie (Current employee) - shares name of Carmine Osvaldo 
+------------------------------------------------------------- 
+- Osvaldo, Hennie (Current employee) - shares name of Carmine Osvaldo 
 History  Document (A Psycorps Analysis Brief By Fredrick N. Wagner and Westley B. Andrews / 
 January 2009): 
 Bodrogi lacked experience in dealing with the Kronosian government and for which he turned to 
 his lifelong friend, Carmine Osvaldo. Osvaldo, who had served in the military with Bodrogi, was a 
 member of the board for the Elodis Chamber of Commerce and who used his business 
 experience to form relationships with the Kronosian government and outside agencies. 
+ ----------------------------------------------------------------------- 
 OUZERI & ELIAN- Pub or drinking establishment across from GasTech 
 (corner of Ermou Street and Taxiarchon Avenue)  
   You would need inside knowledge  
