--- VASTKD0_revision_109_June_20_9:26_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_110_June_20_9:27_AM	2019-06-27 13:14:16.000000000 -0400
@@ -203,7 +203,8 @@
 the cancer caused by environmental toxins in its potable water) Daily Pegasus | 
 03/26/2014 
  - Osvaldo, Hennie (Current employee) - shares name of Carmine Osvaldo 
-History  Document:  
+History  Document (A Psycorps Analysis Brief By Fredrick N. Wagner and Westley B. Andrews / 
+January 200:9 ): 
 Bodrogi lacked experience in dealing with the Kronosian government and for which he turned to 
 his lifelong friend, Carmine Osvaldo. Osvaldo, who had served in the military with Bodrogi, was a 
 member of the board for the Elodis Chamber of Commerce and who used his business 
