--- VASTKD0_revision_129_June_20_11:28_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_130_June_20_11:28_AM	2019-06-27 13:14:16.000000000 -0400
@@ -21,7 +21,8 @@
 Tethyn federal law enforcement agencies are assisting in the investigation. 
 The POK rhetoric has become increasingly anarchist and their actions more 
 criminal and violent over the past several years. 
-The Abila Post | 01/21/2014 -1245 UPDATE - RANSOM DEMANDS FROM POK - The Protectors of 
+The Abila Post | 01/21/2014 
+-1245 UPDATE - RANSOM DEMANDS FROM POK - The Protectors of 
 Kronos have released a statement claiming responsibility for the 
 kidnapping of GAStech employees demanding a $20 million ransom. 
  List of missing Employees: 
@@ -43,7 +44,8 @@
 human rights, and against the corruption of the Government of Kronos and 
 GAStech International. 
  - Edvard Vann- shares name of POK member 
-     Employees with the EXACT SAME “General Discharge”: Ferro, Inga 
+     Employees with the EXACT SAME “General Discharge”: 
+Ferro, Inga 
 Position: Site Control (Security) 
 Date of Hire: 01/11/2013 
 Email: Inga.Ferro@gastech.com.kronos 
@@ -67,7 +69,8 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2007 
 Discharge Type: General Discharge 
- Osvaldo, Hennie Position: Perimeter Control (Security) 
+ Osvaldo, Hennie 
+Position: Perimeter Control (Security) 
 Date of Hire: 06/07/2011 
 Email: Hennie.Osvaldo@gastech.com.kronos 
  Sex: Male 
@@ -91,7 +94,8 @@
 Discharge Date: 10/01/2011 
 Discharge Type: General Discharge 
  Nant, Irene 
-Position: Truck Driver (Facilities) Date of Hire: 11/30/2008 
+Position: Truck Driver (Facilities) 
+Date of Hire: 11/30/2008 
 Email: Irene.Nant@gastech.com.kronos 
  Sex: Female 
 DOB: 03/13/1985 
@@ -116,7 +120,8 @@
  Hawelon, Claudio 
 Position: Truck Driver (Facilities) 
 Date of Hire: 03/13/2007 
-Email: Claudio.Hawelon@gastech.com.kronos  Sex: Male 
+Email: Claudio.Hawelon@gastech.com.kronos 
+ Sex: Male 
 DOB: 08/13/1983 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
@@ -140,7 +145,8 @@
 Position: Lead Janitor (Facilities) 
 Date of Hire: 01/07/1995 
 Email: Dante.Coginian@gastech.com.kronos 
- Sex: Male DOB: 12/01/1971 
+ Sex: Male 
+DOB: 12/01/1971 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
 Passport Expires: 01/01/2000 
@@ -191,7 +197,8 @@
 DOB: 06/02/1981 
  Citizenship: Kronos 
 Passport Issued: 01/01/2000 
-Passport Expires: 01/01/2000  Military Branch: Armed Forces of Kronos 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2001 
 Discharge Type: General Discharge 
    Recommendations to Law Enforcement: 
@@ -218,10 +225,15 @@
 Elian Karel- Current leader of the POK, Karel is a dynamic speaker whose charismatic 
 leadership abilities have won him a substantial group of loyal supporters. 
 --------------------------- 
-Mies, Minke (Defensive Driving and Possible Relative to Henk Mies) They Had : 
+Mies, Minke (Defensive Driving and Possible Relative to Henk Mies) 
+They Had : 
 Inside knowledge/ Surveillance  
 Transportation - Truck/Cars 
 Egress Routes: Airplane via Abila airport or Boat  
 Area Knowledge 
 Military training  
- 
\ No newline at end of file
+ 1
+1
+2
+3
+4
\ No newline at end of file
