--- VASTKD0_revision_121_June_20_10:22_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_122_June_20_10:23_AM	2019-06-27 13:14:16.000000000 -0400
@@ -217,7 +217,7 @@
 Karel 
 Elian Karel- Current leader of the POK, Karel is a dynamic speaker whose charismatic 
 leadership abilities have won him a substantial group of loyal supporters. 
-  You would need inside knowledge  Transportation - Truck 
+  You would need inside knowledge  Transportation - Truck/Cars 
 Egress Routes: Airplane via Abila airport or Boat  
 Area Knowledge 
 Military training  
