--- VASTKD0_revision_19_June_19_10:20_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_20_June_19_10:41_AM	2019-06-27 13:14:16.000000000 -0400
@@ -24,5 +24,7 @@
 The Abila Post | 01/21/2014 -1245 UPDATE - RANSOM DEMANDS FROM POK - The Protectors of 
 Kronos have released a statement claiming responsibility for the 
 kidnapping of GAStech employees demanding a $20 million ransom. 
-  POK Information: 
-Vann-Edvard Vann- potential member namwshares name of POK member 
\ No newline at end of file
+ List of missing Employees: 
+SANJORGE, JR.-CEO 
+ POK Information: 
+Edvard Vann- shares name of POK member 
\ No newline at end of file
