--- VASTKD0_revision_57_June_19_11:42_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_58_June_19_11:42_AM	2019-06-27 13:14:16.000000000 -0400
@@ -150,4 +150,16 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/1992 
 Discharge Type: General Discharge 
+ Awelon, Varro 
+Position: Janitor (Facilities) 
+Date of Hire: 10/22/2012 
+Email: Varro.Awelon@gastech.com.kronos 
+ Sex: Male 
+DOB: 05/05/1986 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2006 
+Discharge Type: General Discharge 
  
\ No newline at end of file
