--- VASTKD0_revision_110_June_20_9:27_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_111_June_20_9:38_AM	2019-06-27 13:14:16.000000000 -0400
@@ -189,7 +189,7 @@
 Email: Carla.Forluniau@gastech.com.kronos 
  Sex: Female 
 DOB: 06/02/1981 
- Citizenship: Kronos 
+ Citizenship: Kronosvann 
 Passport Issued: 01/01/2000 
 Passport Expires: 01/01/2000  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2001 
@@ -204,7 +204,7 @@
 03/26/2014 
  - Osvaldo, Hennie (Current employee) - shares name of Carmine Osvaldo 
 History  Document (A Psycorps Analysis Brief By Fredrick N. Wagner and Westley B. Andrews / 
-January 200:9 ): 
+January 2009): 
 Bodrogi lacked experience in dealing with the Kronosian government and for which he turned to 
 his lifelong friend, Carmine Osvaldo. Osvaldo, who had served in the military with Bodrogi, was a 
 member of the board for the Elodis Chamber of Commerce and who used his business 
