--- VASTKD0_revision_51_June_19_11:40_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_52_June_19_11:40_AM	2019-06-27 13:14:16.000000000 -0400
@@ -80,4 +80,16 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2010 
 Discharge Type: General Discharge 
+  Mies, Minke 
+Position: Perimeter Control (Security) 
+Date of Hire: 05/22/2013 
+Email: Minke.Mies@gastech.com.kronos 
+ Sex: Male 
+DOB: 11/19/1992 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2011 
+Discharge Type: General Discharge 
   
\ No newline at end of file
