--- VASTKD0_revision_13_June_19_10:08_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_14_June_19_10:09_AM	2019-06-27 13:14:16.000000000 -0400
@@ -14,7 +14,12 @@
 -Sanjorge and others disappeared after a Council meeting at 
 hoofdkwartier GASTech. A fire alarm descended, and execs were 
 discovered missing shortly after that.  
- The Abila Post | 01/21/2014 
--1245 UPDATE - RANSOM DEMANDS FROM POK - The Protectors of 
+  Police informed reporters that four of the GAStech employees whose 
+whereabouts were unconfirmed yesterday were found overnight, leaving ten 
+people who are presumed missing. Police spokesman Adrien Carman said 
+Tethyn federal law enforcement agencies are assisting in the investigation. 
+The POK rhetoric has become increasingly anarchist and their actions more 
+criminal and violent over the past several years. 
+The Abila Post | 01/21/2014 -1245 UPDATE - RANSOM DEMANDS FROM POK - The Protectors of 
 Kronos have released a statement claiming responsibility for the 
 kidnapping of GAStech employees demanding a $20 million ransom. 
\ No newline at end of file
