--- VASTKD0_revision_11_June_19_10:06_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_12_June_19_10:07_AM	2019-06-27 13:14:16.000000000 -0400
@@ -5,6 +5,7 @@
  - Protectors Of Kronos is the suspect of the disappearing people 
  TASK: locate , figure out what happened to missing employees  
   What we know from Articles: 
+The Continent | 01/21/2014 
 -Fourteen employees, among which executive civil servants perhaps 
 five, feared have been kidnapped yesterday by the guards of Kronos. 
 Communicated missing includes: President and CEO Sten Sanjorge 
