--- VASTKD0_revision_6_June_19_9:58_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_7_June_19_9:59_AM	2019-06-27 13:14:16.000000000 -0400
@@ -2,5 +2,5 @@
  -Kronos - Found natural gas 
  -Gas Tech - company from Tethys 
   -Jan 2014 - Leaders of GAS Tech went public (Employees went missing during party) 
- - POK is the suspect of the disappearing people 
+ - Protectors Of Kronos is the suspect of the disappearing people 
  TASK: locate , figure out what happened to missing employees  
\ No newline at end of file
