--- VASTKD0_revision_114_June_20_10:11_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_115_June_20_10:12_AM	2019-06-27 13:14:16.000000000 -0400
@@ -210,6 +210,7 @@
 member of the board for the Elodis Chamber of Commerce and who used his business 
 experience to form relationships with the Kronosian government and outside agencies. 
 OUZERI & ELIAN- Pub or drinking establishment across from GasTech 
+(corner of Ermou Street and Taxiarchon Avenue)  
   You would need inside knowledge  
 Transportation 
 Area Knowledge 
