--- VASTKD0_revision_55_June_19_11:41_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_56_June_19_11:42_AM	2019-06-27 13:14:16.000000000 -0400
@@ -127,4 +127,16 @@
  Military Branch: Armed Forces of Kronos 
 Discharge Date: 10/01/2003 
 Discharge Type: General Discharge 
+ Hafon, Albina 
+Position: Truck Driver (Facilities) 
+Date of Hire: 09/12/2009 
+Email: Albina.Hafon@gastech.com.kronos 
+ Sex: Female 
+DOB: 06/14/1987 
+ Citizenship: Kronos 
+Passport Issued: 01/01/2000 
+Passport Expires: 01/01/2000 
+ Military Branch: Armed Forces of Kronos 
+Discharge Date: 10/01/2007 
+Discharge Type: General Discharge 
  
\ No newline at end of file
