--- VASTKD0_revision_99_June_20_9:19_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_100_June_20_9:20_AM	2019-06-27 13:14:16.000000000 -0400
@@ -204,7 +204,8 @@
 Passport Issue Dates All The Same 
 Armed Forces of Kronos 
 Last Name Associated with previous POK Leader 
-  You would need inside knowledge  
+-Vann, Isia (Current employee) - shares name of Julian Vann 
+   You would need inside knowledge  
 Transportation 
 Area Knowledge 
 Military training  
