--- VASTKD0_revision_18_June_19_10:18_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_19_June_19_10:20_AM	2019-06-27 13:14:16.000000000 -0400
@@ -25,4 +25,4 @@
 Kronos have released a statement claiming responsibility for the 
 kidnapping of GAStech employees demanding a $20 million ransom. 
   POK Information: 
-Vann- potential member namw 
\ No newline at end of file
+Vann-Edvard Vann- potential member namwshares name of POK member 
\ No newline at end of file
