--- VASTKD0_revision_95_June_20_8:51_AM	2019-06-27 13:14:16.000000000 -0400
+++ VASTKD0_revision_96_June_20_9:09_AM	2019-06-27 13:14:16.000000000 -0400
@@ -202,4 +202,9 @@
 New Hire 
 Passport Issue Dates All The Same 
 Armed Forces of Kronos 
-Last Name Associated with previous POK Leader 
\ No newline at end of file
+Last Name Associated with previous POK Leader 
+  You would need inside knowledge  
+Transportation 
+Area Knowledge 
+Military training  
+ 
\ No newline at end of file
