--- VASTUBX-2_revision_179_July_18_11:24_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_180_July_18_11:26_AM	2019-07-30 15:49:45.000000000 -0400
@@ -1,7 +1,7 @@
 ---GasTech employees---
- Name/Position Tethys Citizen Kronos Citizen Unknown Time Employed w/ GasTech Ex-Kronos Mil. Ex-Tethys Mil. Notes
+ Name/Position Tethys Citizen Kronos Citizen Time Employed w/ GasTech Ex-Kronos Mil. Ex-Tethys Mil. Notes
  Ada Campo/ CIO Y 10
- Ingrid Barranco/ CFO Y 11
+ Ingrid Barranco/ CFO 11
  Orhan Strum/ COO Y ?
  Sten Sanjorge, Jr. / CEO Y NA
  William Vasco-Pais/ Env. Safety Off. Y 4
