--- VASTUBX-2_revision_75_July_17_10:35_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_76_July_17_10:35_AM	2019-07-30 15:49:45.000000000 -0400
@@ -3,12 +3,12 @@
  Ada Campo/ CIO Y 10
  Ingrid Barranco/ CFO Y 11
  Orhan Strum/ CIO Y y ?
- Sten Sanjorge, Jr. / CEO Y Y NA
+ Sten Sanjorge, Jr. / CEO Y NA
  William Vasco-Pais/ Env. Safety Off. Y 4
  Adan Morlun / Truck Driver Y 8 y only other employement is with GasTech
  Albina Hafon / Truck Driver Y 5 y (possible early discharge?), only other employement is with GasTech, Employment Gap of 2 years
  Anda Ribera / Exec. Asst. Y Y 5
- Axel Calzas / Snr. Drill Tech y y 17
+ Axel Calzas / Snr. Drill Tech y 17
  Bertrand Ovan / Facilities Manager Y 16 y GasTech only other employment
  Carla Forluniau / Admin. Asst. Y 9 y prior - Onda Inc employment overlap with Kronos employment?
  Claudio Hawelon /Truck Driver y 7 y prior- employment on-time delivery
@@ -21,4 +21,4 @@
  Hideki Cocinaro / Sec. Officer y 4 y
  Irene Nant / Truck Driver y 6 y (early discharge?), prior w/ Hernandez Retail
  Isak Baza / IT tech y 16 prior w/ Pharma Inc
- Isanda Borassca / Snr. Drill Tech.
+ Isanda Borassca / Snr. Drill Tech. y
