--- VASTUBX-2_revision_59_July_17_10:27_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_60_July_17_10:28_AM	2019-07-30 15:49:45.000000000 -0400
@@ -5,15 +5,15 @@
  Orhan Strum/ CIO Y y ?
  Sten Sanjorge, Jr. / CEO Y Y NA
  William Vasco-Pais/ Env. Safety Off. Y 4
- Adan Morlun / Truck Driver Y 8 Prior- Kronos Military, only other employement is with GasTech
- Albina Hafon / Truck Driver Y 5 Prior- Kronos Military (possible early discharge?), only other employement is with GasTech, Employment Gap of 2 years
+ Adan Morlun / Truck Driver Y 8 y only other employement is with GasTech
+ Albina Hafon / Truck Driver Y 5 y (possible early discharge?), only other employement is with GasTech, Employment Gap of 2 years
  Anda Ribera / Exec. Asst. Y Y 5
  Axel Calzas / Snr. Drill Tech y y 17
- Bertrand Ovan / Facilities Manager Y 16 Prior- Tethys Military Officer, GasTech only other employment
- Carla Forluniau / Admin. Asst. Y 9 Prior- Kronos Military, prior - Onda Inc employment overlap with Kronos employment?
- Claudio Hawelon /Truck Driver y 7 Prior - Kronos Military, prior- employment on-time delivery
- Cornelia Lais / Exec. Asst. y 3 Prior-Kronos military
- Dante Coginian / Cleaning TL y 19 Prior-Kronos military
+ Bertrand Ovan / Facilities Manager Y 16 y GasTech only other employment
+ Carla Forluniau / Admin. Asst. Y 9 y prior - Onda Inc employment overlap with Kronos employment?
+ Claudio Hawelon /Truck Driver y 7 y prior- employment on-time delivery
+ Cornelia Lais / Exec. Asst. y 3 y
+ Dante Coginian / Cleaning TL y 19 y
  Elsa Orilla / Drill Tech Y 10
- Emile Arpa / Cleaning y 1 Prior-Kronos military (early discharge?)
- Felix Balas y 10 Prior-Tethys military, prior employment Delekanium Engineering
+ Emile Arpa / Cleaning y 1 y (early discharge?)
+ Felix Balas y 10 y prior employment Delekanium Engineering
