--- VASTUBX-2_revision_43_July_17_10:19_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_44_July_17_10:19_AM	2019-07-30 15:49:45.000000000 -0400
@@ -12,3 +12,4 @@
  Bertrand Ovan / Facilities Manager Y 16 Prior- Tethys Military Officer, GasTech only other employment
  Carla Forluniau / Admin. Asst. Y 9 Prior- Kronos Military, prior - Onda Inc employment overlap with Kronos employment?
  Claudio Hawelon /Truck Driver y 7 Prior - Kronos Military, prior- employment on-time delivery
+ Cornelia Lais / Executive Asst.
