--- VASTUBX-2_revision_65_July_17_10:30_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_66_July_17_10:31_AM	2019-07-30 15:49:45.000000000 -0400
@@ -18,3 +18,4 @@
  Emile Arpa / Cleaning y 1 y (early discharge?)
  Felix Balas / Research y 10 y prior employment Delekanium Engineering
  Felix Resumir / Security Grp. Manager y 11 y prior employment Tarryn Security
+ Hideki Cocinaro / Sec. Officer y 4
