--- VASTUBX-2_revision_77_July_17_10:36_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_78_July_17_10:36_AM	2019-07-30 15:49:45.000000000 -0400
@@ -21,4 +21,4 @@
  Hideki Cocinaro / Sec. Officer y 4 y
  Irene Nant / Truck Driver y 6 y (early discharge?), prior w/ Hernandez Retail
  Isak Baza / IT tech y 16 prior w/ Pharma Inc
- Isanda Borassca / Snr. Drill Tech. y
+ Isanda Borassca / Snr. Drill Tech. y 10
