--- VASTUBX-2_revision_48_July_17_10:22_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_49_July_17_10:23_AM	2019-07-30 15:49:45.000000000 -0400
@@ -14,3 +14,4 @@
  Claudio Hawelon /Truck Driver y 7 Prior - Kronos Military, prior- employment on-time delivery
  Cornelia Lais / Exec. Asst. y Prior-Kronos military
  Dante Coginian / Cleaning TL y Prior-Kronos military
+ Elsa Orilla / Drill Tech
