--- VASTUBX-2_revision_89_July_17_10:42_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_90_July_17_10:43_AM	2019-07-30 15:49:45.000000000 -0400
@@ -26,3 +26,4 @@
  Lars Azada / Research Engineer 19 y
  Lidelse Dedos / Engineering Group Manager y 11 prior w/ G&A and w/ Sanli
  Linda Lagos / Exec. Asst. y 4 prior w/ Bank of Kronos, avoided military conscription?
+ Linnea Bergen / IT Group Manager y 10 prior w/ IT4U
