--- VASTUBX-2_revision_69_July_17_10:32_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_70_July_17_10:32_AM	2019-07-30 15:49:45.000000000 -0400
@@ -19,4 +19,4 @@
  Felix Balas / Research y 10 y prior employment Delekanium Engineering
  Felix Resumir / Security Grp. Manager y 11 y prior employment Tarryn Security
  Hideki Cocinaro / Sec. Officer y 4 y
- Irene Nant / Truck Driver y (early discharge?)
+ Irene Nant / Truck Driver 6 y (early discharge?)
