--- VASTUBX-2_revision_76_July_17_10:35_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_77_July_17_10:36_AM	2019-07-30 15:49:45.000000000 -0400
@@ -2,12 +2,12 @@
  Name/Position Tethys Association Kronos Association Neither/Unknown Time Employed w/ GasTech Prior Kronos Military Prior Tethys Military Notes Kidnapped
  Ada Campo/ CIO Y 10
  Ingrid Barranco/ CFO Y 11
- Orhan Strum/ CIO Y y ?
+ Orhan Strum/ CIO Y ?
  Sten Sanjorge, Jr. / CEO Y NA
  William Vasco-Pais/ Env. Safety Off. Y 4
  Adan Morlun / Truck Driver Y 8 y only other employement is with GasTech
  Albina Hafon / Truck Driver Y 5 y (possible early discharge?), only other employement is with GasTech, Employment Gap of 2 years
- Anda Ribera / Exec. Asst. Y Y 5
+ Anda Ribera / Exec. Asst. Y 5
  Axel Calzas / Snr. Drill Tech y 17
  Bertrand Ovan / Facilities Manager Y 16 y GasTech only other employment
  Carla Forluniau / Admin. Asst. Y 9 y prior - Onda Inc employment overlap with Kronos employment?
