--- VASTUBX-2_revision_23_July_17_10:06_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_24_July_17_10:08_AM	2019-07-30 15:49:45.000000000 -0400
@@ -9,3 +9,4 @@
  Albina Hafon / Truck Driver Y Prior-Military, only other employement is with GasTech, Employment Gap of 2 years
  Anda Ribera / Executive Assistant Y Y
  Axel Calzas / Snr. Drill Tech y y
+ Bertrand Ovan / Facilities Manager Y
