--- VASTUBX-2_revision_61_July_17_10:29_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_62_July_17_10:29_AM	2019-07-30 15:49:45.000000000 -0400
@@ -16,4 +16,5 @@
  Dante Coginian / Cleaning TL y 19 y
  Elsa Orilla / Drill Tech Y 10
  Emile Arpa / Cleaning y 1 y (early discharge?)
- Felix Balas y 10 y prior employment Delekanium Engineering
+ Felix Balas / Research y 10 y prior employment Delekanium Engineering
+ Felix Resumir /
