--- VASTUBX-2_revision_19_July_17_10:04_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_20_July_17_10:04_AM	2019-07-30 15:49:45.000000000 -0400
@@ -6,4 +6,4 @@
  Sten Sanjorge, Jr. / CEO Y Y
  William Vasco-Pais/ Env. Safety Off.
  Adan Morlun / Truck Driver Y Prior-Military, only other employement is with GasTech
- Albina Hafon / Truck Driver Y Prior-Military, only other employement is with GasTech
+ Albina Hafon / Truck Driver Y Prior-Military, only other employement is with GasTech, Employment Gap of 2 years
