--- VASTUBX-2_revision_171_July_18_10:35_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_172_July_18_10:37_AM	2019-07-30 15:49:45.000000000 -0400
@@ -39,7 +39,7 @@
  Kare Orilla / Drill Tech y 9
  Brand Tempestad / Drill Tech y 10
  Hank Fluss / GasTech COO dead, widow recieved his share of IPO
- Gustav Cazar / Hydraulic tech y
+ Gustav Cazar / Hydraulic tech y 5
  Birgitta Frente / Geologist y 15
  Vira Frente / Hydraulic Tech y 13
  Adra Nubarron / Geologist y 20
