--- VASTUBX-2_revision_52_July_17_10:24_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_53_July_17_10:25_AM	2019-07-30 15:49:45.000000000 -0400
@@ -15,4 +15,4 @@
  Cornelia Lais / Exec. Asst. y 3 Prior-Kronos military
  Dante Coginian / Cleaning TL y 19 Prior-Kronos military
  Elsa Orilla / Drill Tech Y 10
- Emile Arpa / Cleaning 1
+ Emile Arpa / Cleaning 1 Prior-Kronos military (early discharge?)
