--- VASTUBX-2_revision_98_July_17_10:48_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_99_July_17_10:48_AM	2019-07-30 15:49:45.000000000 -0400
@@ -30,4 +30,4 @@
  Loreto Bodrogi / Sec. Officer y 1 y prior w/ RQS security
  Minke Mies / Sec. Officer y 1 y prior w/ Bank of Kronos
  Nils Calixto / IT Tech y 10
- Rachel Pantanal / Exec. Asst. y 1
+ Rachel Pantanal / Exec. Asst. y 1 prior w/ XYZ Manufacturing
