--- VASTUBX-2_revision_111_July_17_10:56_AM	2019-07-30 15:49:45.000000000 -0400
+++ VASTUBX-2_revision_112_July_17_10:56_AM	2019-07-30 15:49:45.000000000 -0400
@@ -1,5 +1,5 @@
 ---Sheet1---
- Name/Position Tethys Citizen Kronos Citizen Unknown Time Employed w/ GasTech Prior Kronos Mil. Prior Tethys Military Notes
+ Name/Position Tethys Citizen Kronos Citizen Unknown Time Employed w/ GasTech Ex-Kronos Mil. Ex-Tethys Mil. Notes
  Ada Campo/ CIO Y 10
  Ingrid Barranco/ CFO Y 11
  Orhan Strum/ CIO Y ?
