--- VAST5KP_revision_78_June_20_9:44_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_79_June_20_9:44_AM	2019-06-27 13:10:44.000000000 -0400
@@ -1,5 +1,4 @@
- Send to: Vast5KP@ncsu-las.net
- Vast5KP@ncsu-las.net Computer 8 
+Vast5KP@ncsu-las.net Computer 8 
   Security Email: 
  ●Hennie.Osvaldo@gastech.com.kronos,  
 ○Military (General, Kronos) 
@@ -42,8 +41,7 @@
 ●Henk Mies 
 ○Truck driver for GAStech 
 ○I found it interesting that there is direct email communication between Henk 
-Mies and Sten Sanjorge Jr 
- ○ the CEO of GAStech referring to files 
+Mies and Sten Sanjorge Jr the CEO of GAStech referring to files 
 General knowledge: 
 ●Based on reporting from “The Truth” it appears a fire was set at the main office of 
 Gastech in order to cause a distraction with the kidnapping taking place somewhere 
@@ -55,4 +53,9 @@
 GAStech - POK connection of Isia Vann, and start an in depth persona development on Isia 
 Vann. I would also recommend the police interview Rachel Pantanal to see if she may have any 
 information on the attack. 
-          
\ No newline at end of file
+           Send to: Vast5KP@ncsu-las.net
+   1
+1
+2
+3
+4
\ No newline at end of file
