--- VAST5KP_revision_51_June_20_8:38_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_52_June_20_8:38_AM	2019-06-27 13:10:44.000000000 -0400
@@ -39,5 +39,5 @@
 Gastech in order to cause a distraction with the kidnapping taking place somewhere 
 between the main office and a Kronos Government reception. 
  Recommendations/What to do next: 
-At this point based on supplemental data I have identifies IIsia Vann as a  
+At this point, based on supplemental data I have identifies Isia Vann as a  
           
\ No newline at end of file
