--- VAST5KP_revision_25_June_19_11:16_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_26_June_19_11:17_AM	2019-06-27 13:10:44.000000000 -0400
@@ -19,4 +19,6 @@
 ●Edvard Vann (possible member) - questioned by police for many hours, denied all 
 involvement Atricle (TO BECOME INVOLVED FOR ABOVE OF EVENTS IN GASTECH) 
  Other Security Personnel of interest 
-       
\ No newline at end of file
+●Vann, Isia 
+● 
+        
\ No newline at end of file
