--- VAST5KP_revision_45_June_19_11:58_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_46_June_20_8:33_AM	2019-06-27 13:10:44.000000000 -0400
@@ -28,7 +28,7 @@
 ■Denied all involvement (INTERROGATED, SENT EMPLOYEE OF GASTECH) 
 ■Same last name as Juliana Vann, a POK martyr (The annual ends of the 
 collection POK in babble, again) 
-○Isia Vann (possible member) 
+○Isia Vann (possible member) INSIDER THREAT (PROMINENT POK MEMBER) 
 ■Same last name as Juliana Vann, a POK martyr (The annual ends of the 
 collection POK in babble, again) 
   General knowledge: 
