--- VAST5KP_revision_28_June_19_11:19_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_29_June_19_11:20_AM	2019-06-27 13:10:44.000000000 -0400
@@ -21,6 +21,6 @@
 ○questioned by police for many hours,  
 ○denied all involvement Atricle (TO BECOME INVOLVED FOR ABOVE OF EVENTS 
 IN GASTECH)  
-○Same last name as  
+○Same last name as Juliana Vann, a POK martyr  
  Other Security Personnel of interest 
          
\ No newline at end of file
