--- VAST5KP_revision_36_June_19_11:33_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_37_June_19_11:34_AM	2019-06-27 13:10:44.000000000 -0400
@@ -23,7 +23,7 @@
 ○Edvard Vann (possible member) -  
 ■questioned by police for many hours,  
 ■Atricle (TO BECOME INVOLVED FOR ABOVE OF EVENTS IN GASTECH)  
-■Denied all involvement ( 
+■Denied all involvement (INTERROGATED, SENT EMPLOYEE OF GASTECH) 
 ■Same last name as Juliana Vann, a POK martyr (The annual ends of the 
 collection POK in babble, again) 
   General knowledge: 
