--- VAST5KP_revision_13_June_19_10:58_AM	2019-06-27 13:10:44.000000000 -0400
+++ VAST5KP_revision_14_June_19_10:59_AM	2019-06-27 13:10:44.000000000 -0400
@@ -1,6 +1,7 @@
 Security Email: 
  ●Hennie.Osvaldo@gastech.com.kronos,  
 ●Loreto.Bodrogi@gastech.com.kronos,  
+○Military service 
 ●Varja.Lagos@gastech.com.kronos,  
 ○(Possible Head of Security - Sent out original security Email for 1/20 vip visit)  
 ●Inga.Ferro@gastech.com.kronos,  
