--- VASTCER_revision_69_June_20_10:35_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_70_June_20_10:36_AM	2019-06-27 13:13:16.000000000 -0400
@@ -17,6 +17,8 @@
  Timeline: 
 10:30am Fire alarm goes off and firemen enter the building 
 11:45am Employees are allowed to reenter the building 
+12:20pm The Abila police have been called to respond to GAStech headquarters for an 
+unspecified emergency 
        POK protest clean water, human rights, and against the corruption of the Government of Kronos 
 and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were recalls of the heavy toll 
