--- VASTCER_revision_56_June_20_8:40_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_57_June_20_8:40_AM	2019-06-27 13:13:16.000000000 -0400
@@ -2,8 +2,9 @@
  Goal: Find 3 people associated with POK. Annotate Evidence and reasons for decision 
 1 Most recent leader of POK(2012): Silvia Marek 
 2  
-  1 Edvard Vann and Isia Vann (Isia Vann is related to Juliana Vann’s older brother,  both are 
-perimeter control, Edvard was interrogated for 6 hours and considered a possible suspect) 
+  1 Edvard Vann and Isia Vann (Isia Vann is Juliana Vann’s older brother,  both are perimeter 
+control, Edvard was interrogated for 6 hours and considered a possible suspect) 
+Mandor Vann is the uncle of Isia and Juliana Vann 
 2 Elian Karel (Dead former leader of POK)  
 3 Juliana Vann (Dead) 
 4 Jeroen Karol (Father of Elian and original member of POK, in charge of recruitment and 
