--- VASTCER_revision_30_June_19_11:35_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_31_June_19_11:35_AM	2019-06-27 13:13:16.000000000 -0400
@@ -34,4 +34,5 @@
 connect himself appealing on the citizens of Kronos at her for a day of 
 peaceful protest tomorrow.  
 Silvia Marek, leader of the Protectors of Kronos social action organization  
- Blotter-Abila arrested shortly after POK riots in Siopia 
\ No newline at end of file
+ Blotter-Abila arrested for “MISS MAINTAINING THE ORDER” shortly day 
+after POK riots in Siopia 
\ No newline at end of file
