--- VASTCER_revision_66_June_20_8:56_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_67_June_20_9:00_AM	2019-06-27 13:13:16.000000000 -0400
@@ -14,7 +14,8 @@
 7 Henk Bodrogi - Recruited by Osvaldo for assistance 
 8 Lucio Jakab - Started up Save Our Wildlands and eventually joined the POK 
 9 Lorenzo Di Stefano - Professor of environmental science and POK activist 
-  POK protest clean water, human rights, and against the corruption of the Government of Kronos 
+ Timeline: 
+       POK protest clean water, human rights, and against the corruption of the Government of Kronos 
 and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were recalls of the heavy toll 
 already paid by chiefs and the innocent ones. Karel was the former chief of the guards of 
