--- VASTCER_revision_18_June_19_11:17_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_19_June_19_11:19_AM	2019-06-27 13:13:16.000000000 -0400
@@ -22,4 +22,7 @@
  Edvard Vann of Abila, was questioned for hours because he shares a family 
 name with several known POK members. 
  Bianca known as. Sgt Antonio Arresta of the department of fire of Abila, a 
-first-responder with the medical 
\ No newline at end of file
+first-responder with the medical 
+ Mestache Vardshvna, 23, of the Western coast of Kronos. 'We try to obtain 
+the attention of our government,' it said. 'It is important, thus I took leave of 
+work to be part of this important event 
\ No newline at end of file
