--- VASTCER_revision_65_June_20_8:52_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_66_June_20_8:56_AM	2019-06-27 13:13:16.000000000 -0400
@@ -10,6 +10,7 @@
 personel 
 5 Mestache Vardshvna - Protestor, had pictures of deceased POK leaders 
 6 Carmine Osvaldo - Began organizing what would soon become the POK around 1997 
+Hennie Osvaldo - Security for GAStech possible relation 
 7 Henk Bodrogi - Recruited by Osvaldo for assistance 
 8 Lucio Jakab - Started up Save Our Wildlands and eventually joined the POK 
 9 Lorenzo Di Stefano - Professor of environmental science and POK activist 
