--- VASTCER_revision_35_June_19_11:41_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_36_June_19_11:43_AM	2019-06-27 13:13:16.000000000 -0400
@@ -6,7 +6,8 @@
 3 Juliana Vann (Dead) 
 4 Isia Vann?? 
 5 Jeroen Carol (father of Elian and original member) 
-   POK protest clean water, human rights, and against the corruption of the 
+6 Mestache Vardshvna (protestor  
+  POK protest clean water, human rights, and against the corruption of the 
 Government of Kronos and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were 
 recalls of the heavy toll already paid by chiefs and the innocent ones. Karel 
@@ -24,9 +25,7 @@
 from a protector of the member of Kronos. Vann, 
  Edvard Vann of Abila, was questioned for hours because he shares a family 
 name with several known POK members. 
- Bianca known as. Sgt Antonio Arresta of the department of fire of Abila, a 
-first-responder with the medical 
- Mestache Vardshvna, 23, of the Western coast of Kronos. 'We try to obtain 
+Mestache Vardshvna, 23, of the Western coast of Kronos. 'We try to obtain 
 the attention of our government,' it said. 'It is important, thus I took leave of 
 work to be part of this important event 
  Elian (Elon) Carol had been implied POK since its beginning since his/her 
