--- VASTCER_revision_41_June_19_11:52_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_42_June_19_11:54_AM	2019-06-27 13:13:16.000000000 -0400
@@ -1,12 +1,13 @@
 Goal: Find 3 people associated with POK. Annotate Evidence and reasons for decision 
 1 Most recent leader of POK(2012): Silvia Marek 
 2  
-  1 Edvard Vann? and Isia Vann (same name as Juliana Vann both work in same  
+  1 Edvard Vann and Isia Vann (same name as Juliana Vann both work in same are perimeter 
+control, Edvard was interrogated for 6 hours and considered a possible suspect) 
 2 Elian Karel (Dead) 
 3 Juliana Vann (Dead) 
 4 Isia Vann?? 
-5 Jeroen Carol (father of Elian and original member) 
-6 Mestache Vardshvna (protestor, had pictures of deceased POK leaders) 
+54 Jeroen Carol (father of Elian and original member of POK) 
+65 Mestache Vardshvna (protestor, had pictures of deceased POK leaders) 
   POK protest clean water, human rights, and against the corruption of the 
 Government of Kronos and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were 
