--- VASTCER_revision_70_June_20_10:36_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_71_June_20_10:37_AM	2019-06-27 13:13:16.000000000 -0400
@@ -19,7 +19,9 @@
 11:45am Employees are allowed to reenter the building 
 12:20pm The Abila police have been called to respond to GAStech headquarters for an 
 unspecified emergency 
-       POK protest clean water, human rights, and against the corruption of the Government of Kronos 
+12:40pm Additional police have arrived at GAStech headquarters. They are setting up crowd 
+control barricades around the building 
+      POK protest clean water, human rights, and against the corruption of the Government of Kronos 
 and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were recalls of the heavy toll 
 already paid by chiefs and the innocent ones. Karel was the former chief of the guards of 
