--- VASTCER_revision_15_June_19_10:55_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_16_June_19_11:07_AM	2019-06-27 13:13:16.000000000 -0400
@@ -17,4 +17,7 @@
 headquarters around 10:00 this morning local Kronos time to respond to a 
 fire alarm. 
  The evident confusion comes the similarity between its name and the one 
-from a protector of the member of Kronos. Vann, 
\ No newline at end of file
+from a protector of the member of Kronos. Vann, 
+ Edvard Vann of Abila, was questioned for hours because he shares a family 
+name with several known POK members. 
+ 
\ No newline at end of file
