--- VASTCER_revision_61_June_20_8:49_AM	2019-06-27 13:13:16.000000000 -0400
+++ VASTCER_revision_62_June_20_8:49_AM	2019-06-27 13:13:16.000000000 -0400
@@ -13,9 +13,8 @@
 6 Carmine Osvaldo - Began organizing what would soon become the POK around 1997 
 7 Henk Bodrogi - Recruited by Osvaldo for assistance 
 8 Lucio Jakab - Started up Save Our Wildlands and eventually joined the POK 
-9 Lorenzo Di Stefano -  
-Professor of environmental science 
- POK protest clean water, human rights, and against the corruption of the Government of Kronos 
+9 Lorenzo Di Stefano - Professor of environmental science and POK activist 
+  POK protest clean water, human rights, and against the corruption of the Government of Kronos 
 and GAStech International. 
  The posters with coloured images of Elian Karel and Juliana Vann were recalls of the heavy toll 
 already paid by chiefs and the innocent ones. Karel was the former chief of the guards of 
